{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-05-25 17:07:49.259273\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import os\n",
    "import glob\n",
    "import datetime\n",
    "import subprocess\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import ReadMapping\n",
    "from pygentoolbox import ReadMapping_v2\n",
    "\n",
    "from pygentoolbox import SeqsPerFeatureAndFeatureTypeFixed\n",
    "from pygentoolbox import CalculateRelativeBaseCoverageOfFeatureType\n",
    "\n",
    "# need pygentoolbox.Tools also\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "# forward = ['Ptiwi09_RIP_Early25bp25bp.fastq.gz']\n",
    "forward = ['EV_E1_L12_R1.fastq.gz', 'EV_E2_L12_R1.fastq.gz']\n",
    "#for i in range(15, 42): # align each seq length 15-41 and then 50bps # this is EV_late_totalsRNA\n",
    "#    name = f'190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_{i}bp.fastq.gz'\n",
    "#    forward.append(name)\n",
    "#forward.append('190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_50bp.fastq.gz')\n",
    "reverse = ['EV_E1_L12_R2.fastq.gz', 'EV_E1_L12_R2.fastq.gz']\n",
    "#reverse = [''] * len(forward)\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=True\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p2'\n",
    "#hisat_args='-p2 --no-softclip'\n",
    "#hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-05-25 17:07:57.225532\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R1.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R2.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp  already exists\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R1.fastq.gz -I /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R2.fastq.gz -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz -O /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2  already exists\n",
      "Starting Hisat2 paired end: aligning\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz, /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -1 /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R1.trim.fastq.gz -2 /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2/EV_E1_L12_R1.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2/EV_E1_L12_R1.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E2_L12_R1.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R2.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp  already exists\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E2_L12_R1.fastq.gz -I /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/EV_E1_L12_R2.fastq.gz -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz -O /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2  already exists\n",
      "Starting Hisat2 paired end: aligning\n",
      "/media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz, /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -1 /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E2_L12_R1.trim.fastq.gz -2 /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/EV_E1_L12_R2.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2/EV_E2_L12_R1.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Fukai/Hisat2/Chundi_EV/Pt_51_Mac/fastp/hisat2/EV_E2_L12_R1.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2021-05-26 01:24:45.999320\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-02-12 12:26:36.276854\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import os\n",
    "import glob\n",
    "import datetime\n",
    "import subprocess\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import ReadMapping\n",
    "from pygentoolbox import ReadMapping_v2\n",
    "from pygentoolbox import SplitFastqFileBySeqLength\n",
    "\n",
    "from pygentoolbox import SeqsPerFeatureAndFeatureTypeFixed\n",
    "from pygentoolbox import CalculateRelativeBaseCoverageOfFeatureType\n",
    "\n",
    "# need pygentoolbox.Tools also\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "# forward = ['Ptiwi09_RIP_Early25bp25bp.fastq.gz']\n",
    "forward = ['190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.fastq.gz', '190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.fastq.gz']\n",
    "reverse = [''] * len(forward)\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p2 --no-softclip'\n",
    "#hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-02-12 12:28:51.058352\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz\n",
      "hisat2 -q -p2 --no-softclip -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And22bp.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz\n",
      "hisat2 -q -p2 --no-softclip -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/EV_Total_sRNA/Late/Pt_51_Mac/fastp/hisat2/190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_23And24bp.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2021-02-12 12:29:44.494259\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
