{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2019-09-06 11:30:24.026132\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "import pandas as pd\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved_human_proteins\n",
    "from pygentoolbox.Tools import reverse_complement_to_dna\n",
    "from pygentoolbox.Tools import translate_AA_from_dna_human\n",
    "from pygentoolbox.Tools import translate_AA_from_dna_human_terminate_at_stop\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output file: D:\\LinuxShare\\Humans\\Genome\\Annotations\\ref_GRCh38.p12_top_level.mRNA.wProtID.gff3\n",
      "Reading in annotation: D:\\LinuxShare\\Humans\\Genome\\Annotations\\ref_GRCh38.p12_top_level.mRNA.wProtID.ChromOnly.gff3\n",
      "Finished reading in annotation\n",
      "Reading in \"normal\" proteins: D:\\LinuxShare\\Humans\\Genome\\Seqs\\protein.fa\n",
      "Counting total lines\n",
      "Number of lines in genome file: 1251806\n",
      "Reading in interleaved fasta as noninterleaved fasta dictionary\n",
      "Current line: 1000000\n",
      "Finished reading in proteins\n",
      "Reading in genome: D:\\LinuxShare\\Humans\\Genome\\Seqs\\GRCh38_top_level.fa\n",
      "Counting total lines\n",
      "Number of lines in genome file: 45847621\n",
      "Reading in interleaved fasta as noninterleaved fasta dictionary\n",
      "Current line: 5000000\n",
      "Current line: 10000000\n",
      "Current line: 15000000\n",
      "Current line: 20000000\n",
      "Current line: 25000000\n",
      "Current line: 30000000\n",
      "Current line: 35000000\n",
      "Current line: 40000000\n",
      "Current line: 45000000\n",
      "Finished reading in genome\n",
      "2019-09-06 11:34:48.312323\n"
     ]
    }
   ],
   "source": [
    "humanannotation = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Annotations\\\\ref_GRCh38.p12_top_level.gff3'\n",
    "from pygentoolbox.TrimAnnotationByFeature import trim_to_mRNA_and_add_protein_id\n",
    "trim_to_mRNA_and_add_protein_id(humanannotation)  # Keep only mRNA features, but add the protein ID from the CDS features\n",
    "## i manually deleted the entries that were not assigned to chromosomes\n",
    "\n",
    "################################\n",
    "annotationfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Annotations\\\\ref_GRCh38.p12_top_level.mRNA.wProtID.ChromOnly.gff3'\n",
    "annotation = []\n",
    "print('Reading in annotation: %s' % annotationfile)\n",
    "with open(annotationfile, 'r') as FILE:\n",
    "    annotation = [line.strip().split('\\t') for line in FILE if line[0] != '#']  # skip header\n",
    "print('Finished reading in annotation')\n",
    "\n",
    "scaffolds = [s[0] for s in annotation]  # collect ordered scaffold names\n",
    "\n",
    "################################\n",
    "proteinfile = annotationfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\protein.fa'\n",
    "print('Reading in \"normal\" proteins: %s' % proteinfile)\n",
    "namesproteins, dproteins = read_interleaved_fasta_as_noninterleaved_human_proteins(proteinfile)\n",
    "print('Finished reading in proteins')\n",
    "\n",
    "################################\n",
    "genomefile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\GRCh38_top_level.fa'\n",
    "print('Reading in genome: %s' % genomefile)\n",
    "names, dgenome = read_interleaved_fasta_as_noninterleaved(genomefile)\n",
    "print('Finished reading in genome')\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "forward +\n",
      "start\n",
      "ATG\n",
      "end\n",
      "TAG\n",
      "+1 frameshift\n",
      "AGA\n",
      "AGATCTTATATAACTGTGAGATTAATCTCA\n",
      "Translate\n",
      "#####  Positive Control Forward Translation: should match AA sequence  #####\n",
      "MVTEFIFLGL\n",
      "MVTEFIFLGL\n",
      "#####  Positive Control Forward Translation  #####\n",
      "#####  +1 frameshift translation  #####\n",
      "RSYITVRLIS\n",
      "RSYITVRLIS\n",
      "reverse -\n",
      "end\n",
      "TTA\n",
      "start\n",
      "CAT\n",
      "+1 frameshift\n",
      "TTT\n",
      "Translate\n",
      "#####  Positive Control Reverse Translation: should match AA sequence  #####\n",
      "MDGENHSVVS\n",
      "MDGENHSVVS\n",
      "#####  Positive Control Reverse Translation  #####\n",
      "#####  +1 frameshift translation  #####\n",
      "GTTTAATGAGAAGGGCTTATTATATCATTT\n",
      "Reverse complement, the 'reverse -' sequence\n",
      "AAATGATATAATAAGCCCTTCTCATTAAAC\n",
      "K*YNKPFSLN\n",
      "K\n"
     ]
    }
   ],
   "source": [
    "# for forward (+)\n",
    "# gff3 entry\n",
    "# NC_000001.11\tBestRefSeq\tmRNA\t69091\t70008\t.\t+\t.\tID=rna9;Parent=gene8;Dbxref=GeneID:79501,Genbank:NM_001005484.1,HGNC:HGNC:14825;Name=NM_001005484.1;gbkey=mRNA;gene=OR4F5;product=olfactory receptor family 4 subfamily F member 5;transcript_id=NM_001005484.1\n",
    "# subtract one from the start position if it is forward (+) to get start codon\n",
    "# to get stop codon use ending coordinate and subtract 3 for start\n",
    "# start should match atggtgactg\n",
    "# end should match agttttag\n",
    "\n",
    "print('forward +')\n",
    "print('start')\n",
    "print(dgenome['NC_000001.11'][69090:69093])\n",
    "#print(dgenome['NC_000001.11'][70000:70008])\n",
    "print('end')\n",
    "print(dgenome['NC_000001.11'][70005:70008])\n",
    "print('+1 frameshift')\n",
    "print(dgenome['NC_000001.11'][70006:70009])\n",
    "peplength = 30\n",
    "start = 69091\n",
    "end = 70008\n",
    "# forward +1 frameshift\n",
    "print(dgenome['NC_000001.11'][end-2:end+peplength-2])  # end-2:end+1+peplength-3\n",
    "print('Translate')\n",
    "print('#####  Positive Control Forward Translation: should match AA sequence  #####')\n",
    "print('MVTEFIFLGL')\n",
    "print(translate_AA_from_dna_human(dgenome['NC_000001.11'][start-1:start+peplength-1]))\n",
    "print('#####  Positive Control Forward Translation  #####')\n",
    "\n",
    "print('#####  +1 frameshift translation  #####')\n",
    "PlusOneAAseq = translate_AA_from_dna_human(dgenome['NC_000001.11'][end-2:end+peplength-2])\n",
    "print(PlusOneAAseq)\n",
    "PlusOneAAseq = translate_AA_from_dna_human_terminate_at_stop(dgenome['NC_000001.11'][end-2:end+peplength-2])\n",
    "print(PlusOneAAseq)\n",
    "\n",
    "# for reverse (-)\n",
    "# gff3 entry\n",
    "# NC_000001.11\tBestRefSeq\tmRNA\t450740\t451678\t.\t-\t.\tID=rna41;Parent=gene23;Dbxref=GeneID:729759,Genbank:NM_001005221.2,HGNC:HGNC:31275;Name=NM_001005221.2;gbkey=mRNA;gene=OR4F29;product=olfactory receptor family 4 subfamily F member 29;transcript_id=NM_001005221.2\n",
    "# subtract one from the start position if it is forward (+)\n",
    "# to get stop codon use ending coordinate and subtract 3 for start\n",
    "# start should match atggatggag \n",
    "# end should match atctcataa\n",
    "\n",
    "print('reverse -')\n",
    "print('end')\n",
    "print(dgenome['NC_000001.11'][450739:450742])  # 'end of protein'\n",
    "print('start')\n",
    "print(dgenome['NC_000001.11'][451675:451678])  # 'start of protein'\n",
    "# need to reverse complement to get 5' -> 3' order desired\n",
    "print('+1 frameshift')\n",
    "print(dgenome['NC_000001.11'][450738:450741])  # 'end of protein'\n",
    "end = 450740\n",
    "start = 451678\n",
    "\n",
    "print('Translate')\n",
    "print('#####  Positive Control Reverse Translation: should match AA sequence  #####')\n",
    "print('MDGENHSVVS')\n",
    "revcomp, rev = reverse_complement_to_dna(dgenome['NC_000001.11'][start-peplength:start])\n",
    "print(translate_AA_from_dna_human(revcomp))\n",
    "print('#####  Positive Control Reverse Translation  #####')\n",
    "\n",
    "# reverse +1 frameshift\n",
    "print('#####  +1 frameshift translation  #####')\n",
    "print(dgenome['NC_000001.11'][end-peplength+1:end+1])  # long form # [end-2-peplength+3:end+1\n",
    "print('Reverse complement, the \\'reverse -\\' sequence')\n",
    "revcomp, rev = reverse_complement_to_dna(dgenome['NC_000001.11'][end-peplength+1:end+1])  # long form # [end-2-peplength+3:end+1]\n",
    "print(revcomp)\n",
    "PlusOneAAseq = translate_AA_from_dna_human(revcomp)\n",
    "print(PlusOneAAseq)\n",
    "PlusOneAAseq = translate_AA_from_dna_human_terminate_at_stop(revcomp)\n",
    "print(PlusOneAAseq)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Extracting DNA sequences and translating\n",
      "Output +1 frameshift peptides up to 300 in length to file: D:\\LinuxShare\\Humans\\Genome\\Annotations\\PlusOneFrameshift_SkipFirstStopCodonBase_AnnotAndPeptides.tsv\n",
      "Output +1 frameshift peptides up to 300 in length to file: D:\\LinuxShare\\Humans\\Genome\\Seqs\\PlusOneFrameshift_SkipFirstStopCodonBase_Peptides.fa\n",
      "Output +1 frameshift peptides up to 300 in length to file: D:\\LinuxShare\\Humans\\Genome\\Seqs\\PlusOneFrameshift_SkipFirstStopCodonBase_ProteinAndPeptides.fa\n",
      "Output +1 frameshift peptides up to 300 in length to file: D:\\LinuxShare\\Humans\\Genome\\Seqs\\PlusOneFrameshift_SkipFirstStopCodonBase_Peptides.Min10.fa\n",
      "Output +1 frameshift peptides up to 300 in length to file: D:\\LinuxShare\\Humans\\Genome\\Seqs\\PlusOneFrameshift_SkipFirstStopCodonBase_ProteinAndPeptides.Min10.fa\n",
      "2019-09-06 11:48:33.316973\n"
     ]
    }
   ],
   "source": [
    "print('Extracting DNA sequences and translating')\n",
    "dnaextension = 900  # number of nucleotides to consider when extending peptides after +1 frameshift\n",
    "minPeptideLength = 10\n",
    "output = []\n",
    "outputPlusOneAAOnly = []\n",
    "outputProteinAndPlusOneAA = []\n",
    "outputPlusOneAAOnlyCutoff = []\n",
    "outputProteinAndPlusOneAACutoff = []\n",
    "for a in annotation:\n",
    "    if a[6] == '+':\n",
    "        start, end = int(a[3]), int(a[4])\n",
    "        PlusOneAAseq = translate_AA_from_dna_human_terminate_at_stop(dgenome[a[0]][end-2:end+dnaextension-2])  # end-2:end+1+peplength-3\n",
    "    elif a[6] == '-':\n",
    "        end, start = int(a[3]), int(a[4])  # the end coordinate is column three for reverse reads\n",
    "        revcomp, rev = reverse_complement_to_dna(dgenome[a[0]][end-peplength+1:end+1])  # long form # [end-2-peplength+3:end+1\n",
    "        PlusOneAAseq = translate_AA_from_dna_human_terminate_at_stop(revcomp)\n",
    "    else:\n",
    "        print('no orientation information... a problem')\n",
    "    output.append('\\t'.join(a + [PlusOneAAseq]))\n",
    "    protID = a[-1].split(';')[-1]  # isolating the protein ID from the annotation\n",
    "    outputPlusOneAAOnly.append('>%s\\n%s' % (protID, PlusOneAAseq))\n",
    "    outputProteinAndPlusOneAA.append('>%s\\n%s' % (protID, dproteins[protID] + PlusOneAAseq))\n",
    "    if len(PlusOneAAseq) > minPeptideLength:\n",
    "        outputPlusOneAAOnlyCutoff.append('>%s\\n%s' % (protID, PlusOneAAseq))\n",
    "        outputProteinAndPlusOneAACutoff.append('>%s\\n%s' % (protID, dproteins[protID] + PlusOneAAseq))\n",
    "    \n",
    "    \n",
    "outfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Annotations\\\\PlusOneFrameshift_SkipFirstStopCodonBase_AnnotAndPeptides.tsv'\n",
    "with open(outfile, 'w') as OUT:\n",
    "    OUT.write('\\n'.join(output))\n",
    "print('Output +1 frameshift peptides up to %d in length to file: %s' % (dnaextension/3, outfile))\n",
    "\n",
    "outfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\PlusOneFrameshift_SkipFirstStopCodonBase_Peptides.fa'\n",
    "with open(outfile, 'w') as OUT:\n",
    "    OUT.write('\\n'.join(outputPlusOneAAOnly))\n",
    "print('Output +1 frameshift peptides up to %d in length to file: %s' % (dnaextension/3, outfile))\n",
    "\n",
    "outfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\PlusOneFrameshift_SkipFirstStopCodonBase_ProteinAndPeptides.fa'\n",
    "with open(outfile, 'w') as OUT:\n",
    "    OUT.write('\\n'.join(outputProteinAndPlusOneAA))\n",
    "print('Output +1 frameshift peptides up to %d in length to file: %s' % (dnaextension/3, outfile))\n",
    "\n",
    "outfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\PlusOneFrameshift_SkipFirstStopCodonBase_Peptides.Min%d.fa' % minPeptideLength\n",
    "with open(outfile, 'w') as OUT:\n",
    "    OUT.write('\\n'.join(outputPlusOneAAOnlyCutoff))\n",
    "print('Output +1 frameshift peptides up to %d in length to file: %s' % (dnaextension/3, outfile))\n",
    "\n",
    "outfile = 'D:\\\\LinuxShare\\\\Humans\\\\Genome\\\\Seqs\\\\PlusOneFrameshift_SkipFirstStopCodonBase_ProteinAndPeptides.Min%d.fa' % minPeptideLength\n",
    "with open(outfile, 'w') as OUT:\n",
    "    OUT.write('\\n'.join(outputProteinAndPlusOneAACutoff))\n",
    "print('Output +1 frameshift peptides up to %d in length to file: %s' % (dnaextension/3, outfile))\n",
    "    \n",
    "print(datetime.datetime.now())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "## add Plus one frameshift peptides to actaul protein sequences and output them to a fasta file\n",
    "## need to keep all possible protein transcripts\n",
    "## for all of these add the protein ID to the mRNA line\n",
    "## example: ID=cds4;Parent=rna43;Dbxref=GeneID:105378947,Genbank:XP_011540840.1;Name=XP_011540840.1;\n",
    "## extract the \"XP_011540840.1\"\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
