{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-05-25 13:57:55.837367\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp  Created \n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz -I /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz -o /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz -O /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastqc\n",
      "\n",
      "Starting PEAR:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear  Created \n",
      "pear -f /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R1.trim.fastq.gz -r /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/fastp/WT_E_L1_R2.trim.fastq.gz -o  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim\n",
      "Finished PEAR, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim\n",
      "\n",
      "Start merging files\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "altreadfile1='/path/to/altreads1.fastq'\n",
    "altreadfile2='/path/to/altreads2.fastq'\n",
    "#altreadfile = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz'\n",
    "steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=15\n",
    "dnalen=15\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-05-27 16:56:01.359887\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start Charseq Bridge Removal on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz\n",
      "python2 /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/char_bridge_trackall.py --FASTQGZ /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz --NAME /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20. --minRNA 20 --minDNA 20\n",
      "Finished Charseq Bridge Removal, output files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fa\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "from pygentoolbox import CharSeqPipeline\n",
    "# skipping BLAST\n",
    "\n",
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/NotUsed.fastq.gz'\n",
    "\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['charbridge', 'fastq2fasta', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-08 17:27:33.840741\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fa\n",
      "\n",
      "Start calculations of percent free-floating human RNA contamination\n",
      "Expected percent of human-spiked-in of total RNA is 1 percent\n",
      "Start BLASTn on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fa\n",
      "To Database:\n",
      "/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/blastn  already exists\n",
      "Pipe step 1.1\n",
      "blastn -query /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fa -db /media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa -outfmt 6\n",
      "Pipe step 1.2\n",
      "Pipe step 1.3\n",
      "Pipe step 2.1\n",
      "Pipe step 2.2\n",
      "Pipe step 2.3\n",
      "Pipe step 3.1\n",
      "Pipe step 3.2\n",
      "Pipe step 3.3\n",
      "Finished BLASTn, output results to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/blastn/best_bit_score_per_query.blastn.RNA.tsv\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/blastn/human.rna.freefloating.tsv\n",
      "\n",
      "Measured percent of free-floating human RNA results: 0.037941\n",
      "Expected percent of free-floating RNA from Paramecium tetraurelia \"contaminating\" results: 3.794134\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['fastq2fasta', 'blast']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-05-27 16:01:04.650781\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA15RNA15.rna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA15RNA15.dna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA15RNA15.dna.bridgePE.fastq'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA15RNA15.rna.bridgePE.fastq'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=15\n",
    "dnalen=15\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-09 16:56:12.535216\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-26 08:21:38.957471\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting to Record RNA:DNA contacts\n",
      "Required MQ = 0\n",
      "Reading in RNA file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.sam\n",
      "Reading in DNA file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.sam\n",
      "Recording RNA and DNA contacts if they RNA and DNA portions map from the same original parent sequence\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ0.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ0.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ0.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.sort.sam'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.sam'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=0\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-23 10:13:21.556635\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 400\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 400\n"
     ]
    },
    {
     "ename": "KeyError",
     "evalue": "'scaffold51_106:305419:305819'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-3e30cc0bd2b9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     23\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     24\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 25\u001b[0;31m \u001b[0mCharSeqPipeline\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mforwardreadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreversereadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mblastdatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maligndatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile1\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile2\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msteps\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mREsequence\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwindowsize\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcontactcutoff\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mMQ\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mmain\u001b[0;34m(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn)\u001b[0m\n\u001b[1;32m    572\u001b[0m         \u001b[0mcontactfile\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'RNA.DNA.Contacts.pt.wthickness.txt'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    573\u001b[0m         \u001b[0;31m### Divide support for each RNA DNA contact by number of DPNII sites surrounding each contact ###\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 574\u001b[0;31m         \u001b[0mnormalizedsupportfile\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnormalize_rna_dna_contact_support\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcontactfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdRECounts\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcontactcutoff\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwindowsize\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    575\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    576\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'##########\\n    Fin\\n##########'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mnormalize_rna_dna_contact_support\u001b[0;34m(contactfile, dRECounts, contactcutoff, windowsize)\u001b[0m\n\u001b[1;32m     43\u001b[0m                     \u001b[0mstart\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     44\u001b[0m                 \u001b[0mkey\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'scaffold51_%s:%d:%d'\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0ml\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m3\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstart\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 45\u001b[0;31m                 \u001b[0;32mif\u001b[0m \u001b[0mdRECounts\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     46\u001b[0m                     \u001b[0;31m# if there are no restriction sites in this window do not correct for them... but then how did it map here... hmmm... maybe window too small?\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     47\u001b[0m                     \u001b[0mnormsupport\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msupport\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyError\u001b[0m: 'scaffold51_106:305419:305819'"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/pear/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51_with_ies.fa'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn, genomefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-09 21:01:16.657276\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.SeqsPerFeatureAndFeatureTypeAndGOTerms import main\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading Gff3 file: /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gene.gff3\n",
      "Number of scaffolds: 539\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.sam\n",
      "Number of scaffolds: 697\n",
      "Total number of lines in sam file (excluding header): 7600243\n",
      "Number of GO term entries (lines in file) = 86461\n",
      "Counting reads for all feature and Count reads per feature type (ex: exon, gene, CDS, etc.)\n",
      "scaffold51_1\n",
      "scaffold51_10\n",
      "scaffold51_100\n",
      "scaffold51_101\n",
      "scaffold51_102\n",
      "scaffold51_103\n",
      "scaffold51_104\n",
      "scaffold51_105\n",
      "scaffold51_106\n",
      "scaffold51_107\n",
      "scaffold51_108\n",
      "scaffold51_109\n",
      "scaffold51_11\n",
      "scaffold51_110\n",
      "scaffold51_111\n",
      "scaffold51_112\n",
      "scaffold51_113\n",
      "scaffold51_114\n",
      "scaffold51_115\n",
      "scaffold51_116\n",
      "scaffold51_117\n",
      "scaffold51_118\n",
      "scaffold51_119\n",
      "scaffold51_12\n",
      "scaffold51_120\n",
      "scaffold51_121\n",
      "scaffold51_122\n",
      "scaffold51_123\n",
      "scaffold51_124\n",
      "scaffold51_125\n",
      "scaffold51_126\n",
      "scaffold51_127\n",
      "scaffold51_128\n",
      "scaffold51_129\n",
      "scaffold51_13\n",
      "scaffold51_130\n",
      "scaffold51_131\n",
      "scaffold51_132\n",
      "scaffold51_133\n",
      "scaffold51_134\n",
      "scaffold51_135\n",
      "scaffold51_136\n",
      "scaffold51_137\n",
      "scaffold51_138\n",
      "scaffold51_139\n",
      "scaffold51_14\n",
      "scaffold51_140\n",
      "scaffold51_141\n",
      "scaffold51_142\n",
      "scaffold51_143\n",
      "scaffold51_144\n",
      "scaffold51_145\n",
      "scaffold51_146\n",
      "scaffold51_147\n",
      "scaffold51_148\n",
      "scaffold51_149\n",
      "scaffold51_15\n",
      "scaffold51_150\n",
      "scaffold51_151\n",
      "scaffold51_152\n",
      "scaffold51_153\n",
      "scaffold51_154\n",
      "scaffold51_155\n",
      "scaffold51_156\n",
      "scaffold51_157\n",
      "scaffold51_158\n",
      "scaffold51_159\n",
      "scaffold51_16\n",
      "scaffold51_160\n",
      "scaffold51_161\n",
      "scaffold51_162\n",
      "scaffold51_163\n",
      "scaffold51_164\n",
      "scaffold51_165\n",
      "scaffold51_166\n",
      "scaffold51_167\n",
      "scaffold51_168\n",
      "scaffold51_169\n",
      "scaffold51_17\n",
      "scaffold51_170\n",
      "scaffold51_171\n",
      "scaffold51_172\n",
      "scaffold51_173\n",
      "scaffold51_174\n",
      "scaffold51_175\n",
      "scaffold51_176\n",
      "scaffold51_177\n",
      "scaffold51_178\n",
      "scaffold51_179\n",
      "scaffold51_18\n",
      "scaffold51_180\n",
      "scaffold51_181\n",
      "scaffold51_182\n",
      "scaffold51_183\n",
      "scaffold51_184\n",
      "scaffold51_185\n",
      "scaffold51_186\n",
      "scaffold51_187\n",
      "scaffold51_188\n",
      "scaffold51_189\n",
      "scaffold51_19\n",
      "scaffold51_190\n",
      "scaffold51_191\n",
      "scaffold51_192\n",
      "scaffold51_193\n",
      "scaffold51_194\n",
      "scaffold51_195\n",
      "scaffold51_196\n",
      "scaffold51_197\n",
      "scaffold51_198\n",
      "scaffold51_199\n",
      "scaffold51_2\n",
      "scaffold51_20\n",
      "scaffold51_200\n",
      "scaffold51_201\n",
      "scaffold51_202\n",
      "scaffold51_203\n",
      "scaffold51_204\n",
      "scaffold51_205\n",
      "scaffold51_206\n",
      "scaffold51_207\n",
      "scaffold51_208\n",
      "scaffold51_209\n",
      "scaffold51_21\n",
      "scaffold51_210\n",
      "scaffold51_211\n",
      "scaffold51_212\n",
      "scaffold51_213\n",
      "scaffold51_214\n",
      "scaffold51_215\n",
      "scaffold51_216\n",
      "scaffold51_217\n",
      "scaffold51_218\n",
      "scaffold51_219\n",
      "scaffold51_22\n",
      "scaffold51_220\n",
      "scaffold51_221\n",
      "scaffold51_222\n",
      "scaffold51_223\n",
      "scaffold51_224\n",
      "scaffold51_225\n",
      "scaffold51_226\n",
      "scaffold51_227\n",
      "scaffold51_228\n",
      "scaffold51_229\n",
      "scaffold51_23\n",
      "scaffold51_230\n",
      "scaffold51_231\n",
      "scaffold51_232\n",
      "scaffold51_233\n",
      "scaffold51_234\n",
      "scaffold51_235\n",
      "scaffold51_236\n",
      "scaffold51_237\n",
      "scaffold51_238\n",
      "scaffold51_239\n",
      "scaffold51_24\n",
      "scaffold51_240\n",
      "scaffold51_241\n",
      "scaffold51_242\n",
      "scaffold51_243\n",
      "scaffold51_244\n",
      "scaffold51_245\n",
      "scaffold51_246\n",
      "scaffold51_248\n",
      "scaffold51_249\n",
      "scaffold51_25\n",
      "scaffold51_250\n",
      "scaffold51_251\n",
      "scaffold51_252\n",
      "scaffold51_253\n",
      "scaffold51_254\n",
      "scaffold51_255\n",
      "scaffold51_256\n",
      "scaffold51_257\n",
      "scaffold51_258\n",
      "scaffold51_259\n",
      "scaffold51_26\n",
      "scaffold51_261\n",
      "scaffold51_262\n",
      "scaffold51_263\n",
      "scaffold51_264\n",
      "scaffold51_265\n",
      "scaffold51_266\n",
      "scaffold51_267\n",
      "scaffold51_268\n",
      "scaffold51_269\n",
      "scaffold51_27\n",
      "scaffold51_270\n",
      "scaffold51_271\n",
      "scaffold51_272\n",
      "scaffold51_273\n",
      "scaffold51_274\n",
      "scaffold51_275\n",
      "scaffold51_276\n",
      "scaffold51_278\n",
      "scaffold51_279\n",
      "scaffold51_28\n",
      "scaffold51_280\n",
      "scaffold51_281\n",
      "scaffold51_282\n",
      "scaffold51_283\n",
      "scaffold51_284\n",
      "scaffold51_285\n",
      "scaffold51_286\n",
      "scaffold51_287\n",
      "scaffold51_288\n",
      "scaffold51_289\n",
      "scaffold51_29\n",
      "scaffold51_290\n",
      "scaffold51_291\n",
      "scaffold51_293\n",
      "scaffold51_294\n",
      "scaffold51_295\n",
      "scaffold51_296\n",
      "scaffold51_297\n",
      "scaffold51_298\n",
      "scaffold51_299\n",
      "scaffold51_3\n",
      "scaffold51_30\n",
      "scaffold51_300\n",
      "scaffold51_301\n",
      "scaffold51_302\n",
      "scaffold51_303\n",
      "scaffold51_304\n",
      "scaffold51_305\n",
      "scaffold51_306\n",
      "scaffold51_307\n",
      "scaffold51_308\n",
      "scaffold51_309\n",
      "scaffold51_31\n",
      "scaffold51_310\n",
      "scaffold51_311\n",
      "scaffold51_312\n",
      "scaffold51_313\n",
      "scaffold51_314\n",
      "scaffold51_315\n",
      "scaffold51_316\n",
      "scaffold51_317\n",
      "scaffold51_318\n",
      "scaffold51_319\n",
      "scaffold51_32\n",
      "scaffold51_320\n",
      "scaffold51_321\n",
      "scaffold51_322\n",
      "scaffold51_323\n",
      "scaffold51_325\n",
      "scaffold51_326\n",
      "scaffold51_327\n",
      "scaffold51_328\n",
      "scaffold51_329\n",
      "scaffold51_33\n",
      "scaffold51_331\n",
      "scaffold51_332\n",
      "scaffold51_333\n",
      "scaffold51_334\n",
      "scaffold51_335\n",
      "scaffold51_336\n",
      "scaffold51_337\n",
      "scaffold51_338\n",
      "scaffold51_339\n",
      "scaffold51_34\n",
      "scaffold51_340\n",
      "scaffold51_341\n",
      "scaffold51_342\n",
      "scaffold51_343\n",
      "scaffold51_344\n",
      "scaffold51_345\n",
      "scaffold51_347\n",
      "scaffold51_348\n",
      "scaffold51_349\n",
      "scaffold51_35\n",
      "scaffold51_350\n",
      "scaffold51_351\n",
      "scaffold51_352\n",
      "scaffold51_353\n",
      "scaffold51_354\n",
      "scaffold51_355\n",
      "scaffold51_356\n",
      "scaffold51_357\n",
      "scaffold51_358\n",
      "scaffold51_359\n",
      "scaffold51_36\n",
      "scaffold51_361\n",
      "scaffold51_363\n",
      "scaffold51_364\n",
      "scaffold51_365\n",
      "scaffold51_366\n",
      "scaffold51_368\n",
      "scaffold51_37\n",
      "scaffold51_370\n",
      "scaffold51_371\n",
      "scaffold51_372\n",
      "scaffold51_373\n",
      "scaffold51_374\n",
      "scaffold51_375\n",
      "scaffold51_377\n",
      "scaffold51_378\n",
      "scaffold51_379\n",
      "scaffold51_38\n",
      "scaffold51_380\n",
      "scaffold51_381\n",
      "scaffold51_382\n",
      "scaffold51_383\n",
      "scaffold51_384\n",
      "scaffold51_385\n",
      "scaffold51_386\n",
      "scaffold51_388\n",
      "scaffold51_389\n",
      "scaffold51_39\n",
      "scaffold51_390\n",
      "scaffold51_391\n",
      "scaffold51_392\n",
      "scaffold51_393\n",
      "scaffold51_394\n",
      "scaffold51_395\n",
      "scaffold51_396\n",
      "scaffold51_397\n",
      "scaffold51_4\n",
      "scaffold51_40\n",
      "scaffold51_400\n",
      "scaffold51_401\n",
      "scaffold51_402\n",
      "scaffold51_404\n",
      "scaffold51_406\n",
      "scaffold51_409\n",
      "scaffold51_41\n",
      "scaffold51_410\n",
      "scaffold51_411\n",
      "scaffold51_414\n",
      "scaffold51_415\n",
      "scaffold51_417\n",
      "scaffold51_418\n",
      "scaffold51_42\n",
      "scaffold51_420\n",
      "scaffold51_421\n",
      "scaffold51_422\n",
      "scaffold51_423\n",
      "scaffold51_424\n",
      "scaffold51_425\n",
      "scaffold51_426\n",
      "scaffold51_427\n",
      "scaffold51_428\n",
      "scaffold51_429\n",
      "scaffold51_43\n",
      "scaffold51_431\n",
      "scaffold51_433\n",
      "scaffold51_434\n",
      "scaffold51_435\n",
      "scaffold51_436\n",
      "scaffold51_438\n",
      "scaffold51_44\n",
      "scaffold51_440\n",
      "scaffold51_442\n",
      "scaffold51_443\n",
      "scaffold51_444\n",
      "scaffold51_445\n",
      "scaffold51_446\n",
      "scaffold51_448\n",
      "scaffold51_45\n",
      "scaffold51_450\n",
      "scaffold51_451\n",
      "scaffold51_452\n",
      "scaffold51_453\n",
      "scaffold51_454\n",
      "scaffold51_458\n",
      "scaffold51_459\n",
      "scaffold51_46\n",
      "scaffold51_460\n",
      "scaffold51_461\n",
      "scaffold51_462\n",
      "scaffold51_463\n",
      "scaffold51_464\n",
      "scaffold51_465\n",
      "scaffold51_466\n",
      "scaffold51_467\n",
      "scaffold51_468\n",
      "scaffold51_469\n",
      "scaffold51_47\n",
      "scaffold51_472\n",
      "scaffold51_473\n",
      "scaffold51_474\n",
      "scaffold51_475\n",
      "scaffold51_476\n",
      "scaffold51_477\n",
      "scaffold51_478\n",
      "scaffold51_48\n",
      "scaffold51_480\n",
      "scaffold51_483\n",
      "scaffold51_485\n",
      "scaffold51_488\n",
      "scaffold51_489\n",
      "scaffold51_49\n",
      "scaffold51_490\n",
      "scaffold51_491\n",
      "scaffold51_492\n",
      "scaffold51_493\n",
      "scaffold51_495\n",
      "scaffold51_497\n",
      "scaffold51_498\n",
      "scaffold51_499\n",
      "scaffold51_5\n",
      "scaffold51_50\n",
      "scaffold51_501\n",
      "scaffold51_504\n",
      "scaffold51_505\n",
      "scaffold51_507\n",
      "scaffold51_51\n",
      "scaffold51_511\n",
      "scaffold51_514\n",
      "scaffold51_515\n",
      "scaffold51_516\n",
      "scaffold51_52\n",
      "scaffold51_522\n",
      "scaffold51_523\n",
      "scaffold51_526\n",
      "scaffold51_528\n",
      "scaffold51_53\n",
      "scaffold51_530\n",
      "scaffold51_533\n",
      "scaffold51_535\n",
      "scaffold51_536\n",
      "scaffold51_539\n",
      "scaffold51_54\n",
      "scaffold51_540\n",
      "scaffold51_541\n",
      "scaffold51_543\n",
      "scaffold51_548\n",
      "scaffold51_55\n",
      "scaffold51_550\n",
      "scaffold51_554\n",
      "scaffold51_556\n",
      "scaffold51_56\n",
      "scaffold51_562\n",
      "scaffold51_564\n",
      "scaffold51_565\n",
      "scaffold51_566\n",
      "scaffold51_567\n",
      "scaffold51_57\n",
      "scaffold51_572\n",
      "scaffold51_575\n",
      "scaffold51_579\n",
      "scaffold51_58\n",
      "scaffold51_580\n",
      "scaffold51_582\n",
      "scaffold51_584\n",
      "scaffold51_585\n",
      "scaffold51_589\n",
      "scaffold51_59\n",
      "scaffold51_591\n",
      "scaffold51_592\n",
      "scaffold51_593\n",
      "scaffold51_596\n",
      "scaffold51_597\n",
      "scaffold51_599\n",
      "scaffold51_6\n",
      "scaffold51_60\n",
      "scaffold51_600\n",
      "scaffold51_602\n",
      "scaffold51_609\n",
      "scaffold51_61\n",
      "scaffold51_610\n",
      "scaffold51_611\n",
      "scaffold51_612\n",
      "scaffold51_613\n",
      "scaffold51_614\n",
      "scaffold51_616\n",
      "scaffold51_617\n",
      "scaffold51_62\n",
      "scaffold51_625\n",
      "scaffold51_63\n",
      "scaffold51_633\n",
      "scaffold51_636\n",
      "scaffold51_637\n",
      "scaffold51_64\n",
      "scaffold51_641\n",
      "scaffold51_643\n",
      "scaffold51_648\n",
      "scaffold51_65\n",
      "scaffold51_651\n",
      "scaffold51_652\n",
      "scaffold51_655\n",
      "scaffold51_658\n",
      "scaffold51_659\n",
      "scaffold51_66\n",
      "scaffold51_661\n",
      "scaffold51_663\n",
      "scaffold51_665\n",
      "scaffold51_667\n",
      "scaffold51_668\n",
      "scaffold51_67\n",
      "scaffold51_671\n",
      "scaffold51_674\n",
      "scaffold51_675\n",
      "scaffold51_678\n",
      "scaffold51_68\n",
      "scaffold51_681\n",
      "scaffold51_685\n",
      "scaffold51_687\n",
      "scaffold51_688\n",
      "scaffold51_689\n",
      "scaffold51_69\n",
      "scaffold51_690\n",
      "scaffold51_693\n",
      "scaffold51_696\n",
      "scaffold51_697\n",
      "scaffold51_7\n",
      "scaffold51_70\n",
      "scaffold51_71\n",
      "scaffold51_72\n",
      "scaffold51_73\n",
      "scaffold51_74\n",
      "scaffold51_75\n",
      "scaffold51_76\n",
      "scaffold51_77\n",
      "scaffold51_78\n",
      "scaffold51_79\n",
      "scaffold51_8\n",
      "scaffold51_80\n",
      "scaffold51_81\n",
      "scaffold51_82\n",
      "scaffold51_83\n",
      "scaffold51_84\n",
      "scaffold51_85\n",
      "scaffold51_86\n",
      "scaffold51_87\n",
      "scaffold51_88\n",
      "scaffold51_89\n",
      "scaffold51_9\n",
      "scaffold51_90\n",
      "scaffold51_91\n",
      "scaffold51_92\n",
      "scaffold51_93\n",
      "scaffold51_94\n",
      "scaffold51_95\n",
      "scaffold51_96\n",
      "scaffold51_97\n",
      "scaffold51_98\n",
      "scaffold51_99\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.allfeatures.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.allfeatures.sorted.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.allfeatures.sorted.RPKM.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.featuretypes.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.GOtermFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.GOtermDefinitionFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.GOterm.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.GOtermDefinition.counts.tsv\n"
     ]
    }
   ],
   "source": [
    "#GOfile only has GO terms for genes and therefore we must limit the Gff3file to only genes\n",
    "\n",
    "Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3'\n",
    "samfilelist = ['/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.dna.bridgePE.sort.sam']\n",
    "GOfile = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/Ptetraurelia51_GOTerms.tsv'\n",
    "\n",
    "# count number of reads per feature\n",
    "# count number of reads per feature type (exon, mRNA, 5' UTR, etc.) and sort so counts are decreasing\n",
    "\n",
    "main(Gff3file, samfilelist, GOfile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-10 10:24:21.124165\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.SeqsPerFeatureAndFeatureTypeAndGOTerms import main\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading Gff3 file: /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3\n",
      "Number of scaffolds: 539\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.sam\n",
      "Number of scaffolds: 697\n",
      "Total number of lines in sam file (excluding header): 7206389\n",
      "Number of GO term entries (lines in file) = 86461\n",
      "Counting reads for all feature and Count reads per feature type (ex: exon, gene, CDS, etc.)\n",
      "scaffold51_1\n",
      "scaffold51_10\n",
      "scaffold51_100\n",
      "scaffold51_101\n",
      "scaffold51_102\n",
      "scaffold51_103\n",
      "scaffold51_104\n",
      "scaffold51_105\n",
      "scaffold51_106\n",
      "scaffold51_107\n",
      "scaffold51_108\n",
      "scaffold51_109\n",
      "scaffold51_11\n",
      "scaffold51_110\n",
      "scaffold51_111\n",
      "scaffold51_112\n",
      "scaffold51_113\n",
      "scaffold51_114\n",
      "scaffold51_115\n",
      "scaffold51_116\n",
      "scaffold51_117\n",
      "scaffold51_118\n",
      "scaffold51_119\n",
      "scaffold51_12\n",
      "scaffold51_120\n",
      "scaffold51_121\n",
      "scaffold51_122\n",
      "scaffold51_123\n",
      "scaffold51_124\n",
      "scaffold51_125\n",
      "scaffold51_126\n",
      "scaffold51_127\n",
      "scaffold51_128\n",
      "scaffold51_129\n",
      "scaffold51_13\n",
      "scaffold51_130\n",
      "scaffold51_131\n",
      "scaffold51_132\n",
      "scaffold51_133\n",
      "scaffold51_134\n",
      "scaffold51_135\n",
      "scaffold51_136\n",
      "scaffold51_137\n",
      "scaffold51_138\n",
      "scaffold51_139\n",
      "scaffold51_14\n",
      "scaffold51_140\n",
      "scaffold51_141\n",
      "scaffold51_142\n",
      "scaffold51_143\n",
      "scaffold51_144\n",
      "scaffold51_145\n",
      "scaffold51_146\n",
      "scaffold51_147\n",
      "scaffold51_148\n",
      "scaffold51_149\n",
      "scaffold51_15\n",
      "scaffold51_150\n",
      "scaffold51_151\n",
      "scaffold51_152\n",
      "scaffold51_153\n",
      "scaffold51_154\n",
      "scaffold51_155\n",
      "scaffold51_156\n",
      "scaffold51_157\n",
      "scaffold51_158\n",
      "scaffold51_159\n",
      "scaffold51_16\n",
      "scaffold51_160\n",
      "scaffold51_161\n",
      "scaffold51_162\n",
      "scaffold51_163\n",
      "scaffold51_164\n",
      "scaffold51_165\n",
      "scaffold51_166\n",
      "scaffold51_167\n",
      "scaffold51_168\n",
      "scaffold51_169\n",
      "scaffold51_17\n",
      "scaffold51_170\n",
      "scaffold51_171\n",
      "scaffold51_172\n",
      "scaffold51_173\n",
      "scaffold51_174\n",
      "scaffold51_175\n",
      "scaffold51_176\n",
      "scaffold51_177\n",
      "scaffold51_178\n",
      "scaffold51_179\n",
      "scaffold51_18\n",
      "scaffold51_180\n",
      "scaffold51_181\n",
      "scaffold51_182\n",
      "scaffold51_183\n",
      "scaffold51_184\n",
      "scaffold51_185\n",
      "scaffold51_186\n",
      "scaffold51_187\n",
      "scaffold51_188\n",
      "scaffold51_189\n",
      "scaffold51_19\n",
      "scaffold51_190\n",
      "scaffold51_191\n",
      "scaffold51_192\n",
      "scaffold51_193\n",
      "scaffold51_194\n",
      "scaffold51_195\n",
      "scaffold51_196\n",
      "scaffold51_197\n",
      "scaffold51_198\n",
      "scaffold51_199\n",
      "scaffold51_2\n",
      "scaffold51_20\n",
      "scaffold51_200\n",
      "scaffold51_201\n",
      "scaffold51_202\n",
      "scaffold51_203\n",
      "scaffold51_204\n",
      "scaffold51_205\n",
      "scaffold51_206\n",
      "scaffold51_207\n",
      "scaffold51_208\n",
      "scaffold51_209\n",
      "scaffold51_21\n",
      "scaffold51_210\n",
      "scaffold51_211\n",
      "scaffold51_212\n",
      "scaffold51_213\n",
      "scaffold51_214\n",
      "scaffold51_215\n",
      "scaffold51_216\n",
      "scaffold51_217\n",
      "scaffold51_218\n",
      "scaffold51_219\n",
      "scaffold51_22\n",
      "scaffold51_220\n",
      "scaffold51_221\n",
      "scaffold51_222\n",
      "scaffold51_223\n",
      "scaffold51_224\n",
      "scaffold51_225\n",
      "scaffold51_226\n",
      "scaffold51_227\n",
      "scaffold51_228\n",
      "scaffold51_229\n",
      "scaffold51_23\n",
      "scaffold51_230\n",
      "scaffold51_231\n",
      "scaffold51_232\n",
      "scaffold51_233\n",
      "scaffold51_234\n",
      "scaffold51_235\n",
      "scaffold51_236\n",
      "scaffold51_237\n",
      "scaffold51_238\n",
      "scaffold51_239\n",
      "scaffold51_24\n",
      "scaffold51_240\n",
      "scaffold51_241\n",
      "scaffold51_242\n",
      "scaffold51_243\n",
      "scaffold51_244\n",
      "scaffold51_245\n",
      "scaffold51_246\n",
      "scaffold51_248\n",
      "scaffold51_249\n",
      "scaffold51_25\n",
      "scaffold51_250\n",
      "scaffold51_251\n",
      "scaffold51_252\n",
      "scaffold51_253\n",
      "scaffold51_254\n",
      "scaffold51_255\n",
      "scaffold51_256\n",
      "scaffold51_257\n",
      "scaffold51_258\n",
      "scaffold51_259\n",
      "scaffold51_26\n",
      "scaffold51_261\n",
      "scaffold51_262\n",
      "scaffold51_263\n",
      "scaffold51_264\n",
      "scaffold51_265\n",
      "scaffold51_266\n",
      "scaffold51_267\n",
      "scaffold51_268\n",
      "scaffold51_269\n",
      "scaffold51_27\n",
      "scaffold51_270\n",
      "scaffold51_271\n",
      "scaffold51_272\n",
      "scaffold51_273\n",
      "scaffold51_274\n",
      "scaffold51_275\n",
      "scaffold51_276\n",
      "scaffold51_278\n",
      "scaffold51_279\n",
      "scaffold51_28\n",
      "scaffold51_280\n",
      "scaffold51_281\n",
      "scaffold51_282\n",
      "scaffold51_283\n",
      "scaffold51_284\n",
      "scaffold51_285\n",
      "scaffold51_286\n",
      "scaffold51_287\n",
      "scaffold51_288\n",
      "scaffold51_289\n",
      "scaffold51_29\n",
      "scaffold51_290\n",
      "scaffold51_291\n",
      "scaffold51_293\n",
      "scaffold51_294\n",
      "scaffold51_295\n",
      "scaffold51_296\n",
      "scaffold51_297\n",
      "scaffold51_298\n",
      "scaffold51_299\n",
      "scaffold51_3\n",
      "scaffold51_30\n",
      "scaffold51_300\n",
      "scaffold51_301\n",
      "scaffold51_302\n",
      "scaffold51_303\n",
      "scaffold51_304\n",
      "scaffold51_305\n",
      "scaffold51_306\n",
      "scaffold51_307\n",
      "scaffold51_308\n",
      "scaffold51_309\n",
      "scaffold51_31\n",
      "scaffold51_310\n",
      "scaffold51_311\n",
      "scaffold51_312\n",
      "scaffold51_313\n",
      "scaffold51_314\n",
      "scaffold51_315\n",
      "scaffold51_316\n",
      "scaffold51_317\n",
      "scaffold51_318\n",
      "scaffold51_319\n",
      "scaffold51_32\n",
      "scaffold51_320\n",
      "scaffold51_321\n",
      "scaffold51_322\n",
      "scaffold51_323\n",
      "scaffold51_325\n",
      "scaffold51_326\n",
      "scaffold51_327\n",
      "scaffold51_328\n",
      "scaffold51_329\n",
      "scaffold51_33\n",
      "scaffold51_331\n",
      "scaffold51_332\n",
      "scaffold51_333\n",
      "scaffold51_334\n",
      "scaffold51_335\n",
      "scaffold51_336\n",
      "scaffold51_337\n",
      "scaffold51_338\n",
      "scaffold51_339\n",
      "scaffold51_34\n",
      "scaffold51_340\n",
      "scaffold51_341\n",
      "scaffold51_342\n",
      "scaffold51_343\n",
      "scaffold51_344\n",
      "scaffold51_345\n",
      "scaffold51_347\n",
      "scaffold51_348\n",
      "scaffold51_349\n",
      "scaffold51_35\n",
      "scaffold51_350\n",
      "scaffold51_351\n",
      "scaffold51_352\n",
      "scaffold51_353\n",
      "scaffold51_354\n",
      "scaffold51_355\n",
      "scaffold51_356\n",
      "scaffold51_357\n",
      "scaffold51_358\n",
      "scaffold51_359\n",
      "scaffold51_36\n",
      "scaffold51_361\n",
      "scaffold51_363\n",
      "scaffold51_364\n",
      "scaffold51_365\n",
      "scaffold51_366\n",
      "scaffold51_368\n",
      "scaffold51_37\n",
      "scaffold51_370\n",
      "scaffold51_371\n",
      "scaffold51_372\n",
      "scaffold51_373\n",
      "scaffold51_374\n",
      "scaffold51_375\n",
      "scaffold51_377\n",
      "scaffold51_378\n",
      "scaffold51_379\n",
      "scaffold51_38\n",
      "scaffold51_380\n",
      "scaffold51_381\n",
      "scaffold51_382\n",
      "scaffold51_383\n",
      "scaffold51_384\n",
      "scaffold51_385\n",
      "scaffold51_386\n",
      "scaffold51_388\n",
      "scaffold51_389\n",
      "scaffold51_39\n",
      "scaffold51_390\n",
      "scaffold51_391\n",
      "scaffold51_392\n",
      "scaffold51_393\n",
      "scaffold51_394\n",
      "scaffold51_395\n",
      "scaffold51_396\n",
      "scaffold51_397\n",
      "scaffold51_4\n",
      "scaffold51_40\n",
      "scaffold51_400\n",
      "scaffold51_401\n",
      "scaffold51_402\n",
      "scaffold51_404\n",
      "scaffold51_406\n",
      "scaffold51_409\n",
      "scaffold51_41\n",
      "scaffold51_410\n",
      "scaffold51_411\n",
      "scaffold51_414\n",
      "scaffold51_415\n",
      "scaffold51_417\n",
      "scaffold51_418\n",
      "scaffold51_42\n",
      "scaffold51_420\n",
      "scaffold51_421\n",
      "scaffold51_422\n",
      "scaffold51_423\n",
      "scaffold51_424\n",
      "scaffold51_425\n",
      "scaffold51_426\n",
      "scaffold51_427\n",
      "scaffold51_428\n",
      "scaffold51_429\n",
      "scaffold51_43\n",
      "scaffold51_431\n",
      "scaffold51_433\n",
      "scaffold51_434\n",
      "scaffold51_435\n",
      "scaffold51_436\n",
      "scaffold51_438\n",
      "scaffold51_44\n",
      "scaffold51_440\n",
      "scaffold51_442\n",
      "scaffold51_443\n",
      "scaffold51_444\n",
      "scaffold51_445\n",
      "scaffold51_446\n",
      "scaffold51_448\n",
      "scaffold51_45\n",
      "scaffold51_450\n",
      "scaffold51_451\n",
      "scaffold51_452\n",
      "scaffold51_453\n",
      "scaffold51_454\n",
      "scaffold51_458\n",
      "scaffold51_459\n",
      "scaffold51_46\n",
      "scaffold51_460\n",
      "scaffold51_461\n",
      "scaffold51_462\n",
      "scaffold51_463\n",
      "scaffold51_464\n",
      "scaffold51_465\n",
      "scaffold51_466\n",
      "scaffold51_467\n",
      "scaffold51_468\n",
      "scaffold51_469\n",
      "scaffold51_47\n",
      "scaffold51_472\n",
      "scaffold51_473\n",
      "scaffold51_474\n",
      "scaffold51_475\n",
      "scaffold51_476\n",
      "scaffold51_477\n",
      "scaffold51_478\n",
      "scaffold51_48\n",
      "scaffold51_480\n",
      "scaffold51_483\n",
      "scaffold51_485\n",
      "scaffold51_488\n",
      "scaffold51_489\n",
      "scaffold51_49\n",
      "scaffold51_490\n",
      "scaffold51_491\n",
      "scaffold51_492\n",
      "scaffold51_493\n",
      "scaffold51_495\n",
      "scaffold51_497\n",
      "scaffold51_498\n",
      "scaffold51_499\n",
      "scaffold51_5\n",
      "scaffold51_50\n",
      "scaffold51_501\n",
      "scaffold51_504\n",
      "scaffold51_505\n",
      "scaffold51_507\n",
      "scaffold51_51\n",
      "scaffold51_511\n",
      "scaffold51_514\n",
      "scaffold51_515\n",
      "scaffold51_516\n",
      "scaffold51_52\n",
      "scaffold51_522\n",
      "scaffold51_523\n",
      "scaffold51_526\n",
      "scaffold51_528\n",
      "scaffold51_53\n",
      "scaffold51_530\n",
      "scaffold51_533\n",
      "scaffold51_535\n",
      "scaffold51_536\n",
      "scaffold51_539\n",
      "scaffold51_54\n",
      "scaffold51_540\n",
      "scaffold51_541\n",
      "scaffold51_543\n",
      "scaffold51_548\n",
      "scaffold51_55\n",
      "scaffold51_550\n",
      "scaffold51_554\n",
      "scaffold51_556\n",
      "scaffold51_56\n",
      "scaffold51_562\n",
      "scaffold51_564\n",
      "scaffold51_565\n",
      "scaffold51_566\n",
      "scaffold51_567\n",
      "scaffold51_57\n",
      "scaffold51_572\n",
      "scaffold51_575\n",
      "scaffold51_579\n",
      "scaffold51_58\n",
      "scaffold51_580\n",
      "scaffold51_582\n",
      "scaffold51_584\n",
      "scaffold51_585\n",
      "scaffold51_589\n",
      "scaffold51_59\n",
      "scaffold51_591\n",
      "scaffold51_592\n",
      "scaffold51_593\n",
      "scaffold51_596\n",
      "scaffold51_597\n",
      "scaffold51_599\n",
      "scaffold51_6\n",
      "scaffold51_60\n",
      "scaffold51_600\n",
      "scaffold51_602\n",
      "scaffold51_609\n",
      "scaffold51_61\n",
      "scaffold51_610\n",
      "scaffold51_611\n",
      "scaffold51_612\n",
      "scaffold51_613\n",
      "scaffold51_614\n",
      "scaffold51_616\n",
      "scaffold51_617\n",
      "scaffold51_62\n",
      "scaffold51_625\n",
      "scaffold51_63\n",
      "scaffold51_633\n",
      "scaffold51_636\n",
      "scaffold51_637\n",
      "scaffold51_64\n",
      "scaffold51_641\n",
      "scaffold51_643\n",
      "scaffold51_648\n",
      "scaffold51_65\n",
      "scaffold51_651\n",
      "scaffold51_652\n",
      "scaffold51_655\n",
      "scaffold51_658\n",
      "scaffold51_659\n",
      "scaffold51_66\n",
      "scaffold51_661\n",
      "scaffold51_663\n",
      "scaffold51_665\n",
      "scaffold51_667\n",
      "scaffold51_668\n",
      "scaffold51_67\n",
      "scaffold51_671\n",
      "scaffold51_674\n",
      "scaffold51_675\n",
      "scaffold51_678\n",
      "scaffold51_68\n",
      "scaffold51_681\n",
      "scaffold51_685\n",
      "scaffold51_687\n",
      "scaffold51_688\n",
      "scaffold51_689\n",
      "scaffold51_69\n",
      "scaffold51_690\n",
      "scaffold51_693\n",
      "scaffold51_696\n",
      "scaffold51_697\n",
      "scaffold51_7\n",
      "scaffold51_70\n",
      "scaffold51_71\n",
      "scaffold51_72\n",
      "scaffold51_73\n",
      "scaffold51_74\n",
      "scaffold51_75\n",
      "scaffold51_76\n",
      "scaffold51_77\n",
      "scaffold51_78\n",
      "scaffold51_79\n",
      "scaffold51_8\n",
      "scaffold51_80\n",
      "scaffold51_81\n",
      "scaffold51_82\n",
      "scaffold51_83\n",
      "scaffold51_84\n",
      "scaffold51_85\n",
      "scaffold51_86\n",
      "scaffold51_87\n",
      "scaffold51_88\n",
      "scaffold51_89\n",
      "scaffold51_9\n",
      "scaffold51_90\n",
      "scaffold51_91\n",
      "scaffold51_92\n",
      "scaffold51_93\n",
      "scaffold51_94\n",
      "scaffold51_95\n",
      "scaffold51_96\n",
      "scaffold51_97\n",
      "scaffold51_98\n",
      "scaffold51_99\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.allfeatures.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.allfeatures.sorted.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.allfeatures.sorted.RPKM.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.featuretypes.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.GOtermFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.GOtermDefinitionFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.GOterm.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.GOtermDefinition.counts.tsv\n"
     ]
    }
   ],
   "source": [
    "#GOfile only has GO terms for genes and therefore we must limit the Gff3file to only genes\n",
    "\n",
    "Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3'\n",
    "samfilelist = ['/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.sort.sam']\n",
    "GOfile = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/Ptetraurelia51_GOTerms.tsv'\n",
    "\n",
    "# count number of reads per feature\n",
    "# count number of reads per feature type (exon, mRNA, 5' UTR, etc.) and sort so counts are decreasing\n",
    "\n",
    "main(Gff3file, samfilelist, GOfile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-23 11:08:54.019758\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.SeqsPerFeatureAndFeatureTypeFixed import main\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading Gff3 file: /media/sf_LinuxShare/Ciliates/Genomes/Annotations/internal_eliminated_sequence_PGM_ParTIES.pt_51_with_ies.gff3\n",
      "Number of scaffolds: 511\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.sam\n",
      "Number of scaffolds (in header): 697\n",
      "Total number of lines in sam file (excluding header): 7141929\n",
      "Mapped reads, so calculating seqs per feature and feature type\n",
      "Counting reads for all feature and Count reads per feature type (ex: exon, gene, CDS, etc.)\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.allfeatures.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.allfeatures.sorted.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.featuretypes.fixed.tsv\n"
     ]
    }
   ],
   "source": [
    "#GOfile only has GO terms for genes and therefore we must limit the Gff3file to only genes\n",
    "\n",
    "#Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3'\n",
    "Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/internal_eliminated_sequence_PGM_ParTIES.pt_51_with_ies.gff3'\n",
    "### this sam file is mapped to mac+IES genome, connections normalized at every 400bps\n",
    "samfilelist = ['/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/hisat2/WT_E_L1_R1R2.trim.Ass50Million.DNA20RNA20.rna.bridgePE.fastq.sort.sam']\n",
    "\n",
    "# count number of reads per feature\n",
    "# count number of reads per feature type (exon, mRNA, 5' UTR, etc.) and sort so counts are decreasing\n",
    "\n",
    "main(Gff3file, samfilelist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
