{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-02-26 17:13:30.868658\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R1.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R2.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp  Created \n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R1.fastq.gz -I /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R2.fastq.gz -o /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz -O /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastqc\n",
      "\n",
      "Starting PEAR:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear  Created \n",
      "pear -f /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R1.trim.fastq.gz -r /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/fastp/500_LK_L1_R2.trim.fastq.gz -o  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trim\n",
      "Finished PEAR, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trim\n",
      "\n",
      "Start merging files\n",
      "Finished merging files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq\n",
      "\n",
      "Start packing files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq\n",
      "gzip -f -k /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq\n",
      "Finished packing:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq.gz\n",
      "\n",
      "Start Charseq Bridge Removal on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq.gz\n",
      "python2 /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/char_bridge_trackall.py --FASTQGZ /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.fastq.gz --NAME /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15. --minRNA 15 --minDNA 15\n",
      "Finished Charseq Bridge Removal, output files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.dna.bridgePE.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fa\n",
      "\n",
      "Start calculations of percent free-floating human RNA contamination\n",
      "Expected percent of human-spiked-in of total RNA is 1 percent\n",
      "Start BLASTn on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fa\n",
      "To Database:\n",
      "/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/blastn  Created \n",
      "Finished BLASTn, output results to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/blastn/best_bit_score_per_query.blastn.RNA.tsv\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/blastn/human.rna.freefloating.tsv\n",
      "\n",
      "Measured percent of free-floating human RNA results: 11.764706\n",
      "Expected percent of free-floating RNA from Paramecium tetraurelia \"contaminating\" results: 1176.470588\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.rna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/pear/500_LK_L1_R1R2.trimAssUnFUnR.DNA15RNA15.dna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/TestMyPipe/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/TestMyPipe/500_LK_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "rnalen=15\n",
    "dnalen=15\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "    "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
