{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-09-22 16:21:56.424940\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split0.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split0.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split1.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split1.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split2.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split2.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split3.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split3.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split4.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split4.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split5.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split5.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split6.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split6.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split7.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split7.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split8.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split8.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split9.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split9.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split10.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split10.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split11.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split11.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split12.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split12.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split13.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split13.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split14.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split14.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split15.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split15.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split16.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split16.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split17.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split17.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split18.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split18.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split19.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split19.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split20.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split20.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split21.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split21.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split22.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split22.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split23.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split23.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split24.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split24.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split25.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split25.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split26.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split26.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split27.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split27.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split28.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split28.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split29.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split29.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split30.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split30.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split31.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split31.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split32.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split32.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split33.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split33.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split34.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split34.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split35.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split35.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split36.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split36.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split37.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split37.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split38.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split38.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split39.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split39.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split40.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split40.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split41.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split41.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split42.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split42.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split43.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split43.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split44.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split44.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split45.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split45.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split46.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split46.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split47.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split47.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split48.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split48.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split49.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split49.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split50.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split50.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split51.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split51.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split52.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split52.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split53.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split53.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split54.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split54.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split55.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split55.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split56.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split56.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split57.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split57.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split58.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split58.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split59.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split59.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split60.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split60.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split61.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split61.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split62.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split62.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split63.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split63.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split64.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split64.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split65.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split65.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split66.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split66.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split67.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split67.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split68.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split68.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split69.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split69.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split70.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split70.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split71.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split71.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split72.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split72.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split73.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split73.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split74.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split74.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split75.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split75.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split76.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split76.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split77.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split77.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split78.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split78.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split79.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split79.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split80.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split80.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split81.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split81.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split82.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split82.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split83.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split83.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split84.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split84.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split85.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split85.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split86.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split86.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split87.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split87.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split88.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split88.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split89.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split89.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split90.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split90.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split91.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split91.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split92.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split92.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split93.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split93.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split94.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split94.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split95.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split95.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split96.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split96.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split97.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split97.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split98.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split98.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split99.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split99.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split100.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split100.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split101.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split101.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split102.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split102.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split103.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split103.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split104.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split104.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split105.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split105.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split106.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split106.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split107.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split107.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split108.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split108.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split109.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split109.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split110.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split110.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split111.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split111.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split112.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split112.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split113.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split113.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split114.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split114.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split115.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split115.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split116.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split116.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split117.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split117.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split118.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split118.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split119.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split119.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split120.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split120.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split121.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split121.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split122.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split122.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split123.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split123.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split124.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split124.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split125.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split125.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split126.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split126.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split127.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split127.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split128.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split128.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split129.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split129.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split130.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split130.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split131.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split131.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split132.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split132.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split133.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split133.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split134.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split134.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split135.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split135.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split136.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split136.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split137.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split137.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split138.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split138.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split139.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split139.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split140.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split140.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split141.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split141.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split142.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split142.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split143.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split143.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split144.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split144.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split145.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split145.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split146.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split146.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split147.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split147.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split148.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split148.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split149.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split149.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split150.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split150.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split151.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split151.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split152.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split152.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split153.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split153.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split154.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split154.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split155.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split155.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split156.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split156.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split157.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split157.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split158.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split158.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split159.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split159.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split160.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split160.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split161.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split161.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split162.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split162.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split163.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split163.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split164.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split164.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split165.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split165.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split166.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split166.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split167.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split167.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split168.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split168.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split169.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split169.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split170.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split170.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split171.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split171.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split172.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split172.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split173.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split173.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split174.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split174.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split175.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split175.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split176.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split176.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split177.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split177.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split178.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split178.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split179.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split179.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split180.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split180.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split181.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split181.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split182.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split182.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split183.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split183.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split184.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split184.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split185.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split185.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split186.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split186.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split187.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split187.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split188.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split188.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split189.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split189.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split190.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split190.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split191.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split191.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split192.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split192.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split193.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split193.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split194.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split194.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split195.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split195.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split196.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split196.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split197.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split197.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split198.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split198.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split199.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split199.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split200.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split200.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split201.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split201.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split202.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split202.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split203.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split203.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split204.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split204.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split205.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split205.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split206.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split206.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split207.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split207.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split208.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split208.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split209.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split209.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split210.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split210.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split211.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split211.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split212.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split212.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split213.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split213.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split214.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split214.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split215.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split215.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split216.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split216.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split217.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split217.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split218.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split218.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split219.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split219.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split220.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split220.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split221.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split221.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split222.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split222.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split223.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split223.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split224.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split224.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split225.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split225.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split226.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split226.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split227.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split227.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split228.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split228.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split229.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split229.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split230.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split230.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split231.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split231.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split232.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split232.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split233.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split233.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split234.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split234.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split235.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split235.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split236.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split236.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split237.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split237.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split238.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split238.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split239.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split239.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split240.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split240.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split241.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split241.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split242.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split242.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split243.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split243.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split244.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split244.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split245.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/WT_E_L1_R1R2.trim.AssUnFUnR.split245.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting to count rna and dna read positions and contacts\n",
      "Remove PCR duplicates because PCRDupRemoval is set to True\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split0.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split1.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split2.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split3.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split4.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split5.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split6.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split7.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split8.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split9.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split10.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split11.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split12.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split13.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split14.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split15.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split16.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split17.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split18.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split19.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split20.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split21.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split22.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split23.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split24.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split25.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split26.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split27.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split28.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split29.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split30.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split31.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split32.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split33.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split34.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split35.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split36.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split37.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split38.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split39.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split40.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split41.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split42.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split43.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split44.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split45.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split46.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split47.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split48.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split49.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split50.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split51.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split52.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split53.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split54.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split55.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split56.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split57.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split58.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split59.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split60.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split61.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split62.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split63.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split64.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split65.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split66.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split67.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split68.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split69.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split70.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split71.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split72.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split73.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split74.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split75.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split76.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split77.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split78.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split79.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split80.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split81.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split82.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split83.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split84.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split85.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split86.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split87.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split88.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split89.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split90.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split91.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split92.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split93.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split94.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split95.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split96.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split97.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split98.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split99.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split100.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split101.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split102.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split103.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split104.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split105.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split106.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split107.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split108.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split109.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split110.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split111.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split112.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split113.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split114.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split115.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split116.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split117.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split118.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split119.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split120.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split121.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split122.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split123.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split124.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split125.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split126.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split127.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split128.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split129.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split130.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split131.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split132.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split133.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split134.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split135.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split136.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split137.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split138.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split139.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split140.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split141.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split142.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split143.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split144.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split145.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split146.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split147.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split148.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split149.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split150.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split151.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split152.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split153.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split154.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split155.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split156.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split157.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split158.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split159.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split160.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split161.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split162.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split163.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split164.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split165.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split166.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split167.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split168.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split169.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split170.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split171.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split172.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split173.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split174.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split175.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split176.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split177.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split178.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split179.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split180.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split181.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split182.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split183.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split184.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split185.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split186.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split187.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split188.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split189.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split190.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split191.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split192.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split193.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split194.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split195.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split196.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split197.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split198.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split199.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split200.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split201.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split202.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split203.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split204.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split205.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split206.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split207.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split208.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split209.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split210.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split211.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split212.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split213.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split214.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split215.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split216.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split217.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split218.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split219.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split220.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split221.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split222.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split223.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split224.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split225.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split226.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split227.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split228.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split229.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split230.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split231.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split232.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split233.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split234.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split235.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split236.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split237.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split238.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split239.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split240.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split241.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split242.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split243.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split244.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split245.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "Finished counting rna and dna read positions and contacts\n",
      "Starting to summarize and sort RNA DNA raw contacts\n"
     ]
    },
    {
     "ename": "IndexError",
     "evalue": "list index out of range",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mIndexError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-62b5f1a8318e>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m \u001b[0mCharSeqPipeline\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mforwardreadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreversereadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mblastdatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maligndatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile1\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile2\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msteps\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrnamaxlen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mREsequence\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwindowsize\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcontactcutoff\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mMQ\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mhisat2_args\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mgenomefile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mPCRDupRemoval\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mannot_file\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzoomscaffname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzoomwinsize\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mscaffcoordmin\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mscaffcoordmax\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfeatures\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mmain\u001b[0;34m(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, rnamaxlen, REsequence, windowsize, contactcutoff, MQ, sn, alignargs, genomefile, PCRDupRemoval, annot_file, zoomscaffname, zoomwinsize, scaffcoordmin, scaffcoordmax, features)\u001b[0m\n\u001b[1;32m   1226\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Finished counting rna and dna read positions and contacts'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1227\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1228\u001b[0;31m         \u001b[0mcontactfile\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msummarize_raw_rna_dna_contacts\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrawcontactfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mPCRDupRemoval\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1229\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1230\u001b[0m         \u001b[0;31m#for dnasortbam, rnasortbam in zip(dnasortbamfiles, rnasortbamfiles):\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36msummarize_raw_rna_dna_contacts\u001b[0;34m(rawcontactfile, sn, PCRDupRemoval)\u001b[0m\n\u001b[1;32m    478\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    479\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'Starting to summarize and sort RNA DNA raw contacts'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 480\u001b[0;31m     \u001b[0msortoutput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrawcontactfile\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnatsort_raw_rna_dna_contacts\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrawcontactfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mPCRDupRemoval\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    481\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    482\u001b[0m     \u001b[0msntrim\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlower\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m' '\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mnatsort_raw_rna_dna_contacts\u001b[0;34m(filename, sn, PCRDupRemoval)\u001b[0m\n\u001b[1;32m    463\u001b[0m             \u001b[0;31m# iterate over set, sort it by DNA scaffold/position below\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    464\u001b[0m             \u001b[0mlines\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0ml\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstrip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0ml\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mFILE\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreadlines\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 465\u001b[0;31m     \u001b[0msortoutput\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnatsorted\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlines\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mlambda\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    466\u001b[0m     \u001b[0mpath\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mf\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    467\u001b[0m     \u001b[0moutpath\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'RNA.DNA.Contacts.%s.raw.sort.txt'\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0msntrim\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/natsort/natsort.py\u001b[0m in \u001b[0;36mnatsorted\u001b[0;34m(seq, key, reverse, alg)\u001b[0m\n\u001b[1;32m    257\u001b[0m     \"\"\"\n\u001b[1;32m    258\u001b[0m     \u001b[0mkey\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnatsort_keygen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0malg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 259\u001b[0;31m     \u001b[0;32mreturn\u001b[0m \u001b[0msorted\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mseq\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreverse\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mreverse\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    260\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    261\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/natsort/utils.py\u001b[0m in \u001b[0;36mnatsort_key\u001b[0;34m(val, key, string_func, bytes_func, num_func)\u001b[0m\n\u001b[1;32m    207\u001b[0m     \u001b[0;31m# Apply key if needed\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    208\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mkey\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 209\u001b[0;31m         \u001b[0mval\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mval\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    210\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    211\u001b[0m     \u001b[0;31m# Assume the input are strings, which is the most common case\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36m<lambda>\u001b[0;34m(y)\u001b[0m\n\u001b[1;32m    463\u001b[0m             \u001b[0;31m# iterate over set, sort it by DNA scaffold/position below\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    464\u001b[0m             \u001b[0mlines\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0ml\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstrip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0ml\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mFILE\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreadlines\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 465\u001b[0;31m     \u001b[0msortoutput\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnatsorted\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlines\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mlambda\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    466\u001b[0m     \u001b[0mpath\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mf\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    467\u001b[0m     \u001b[0moutpath\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'RNA.DNA.Contacts.%s.raw.sort.txt'\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0msntrim\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mIndexError\u001b[0m: list index out of range"
     ]
    }
   ],
   "source": [
    "## previous cell split AssUnFUnR to chunks\n",
    "## char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1'\n",
    "\n",
    "#altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/pear/WT_L_L1_R1R2.trim.AssUnFUnR.fastq.gz'\n",
    "altreadfile1 = []\n",
    "path ='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/'\n",
    "for i in range(246):\n",
    "    splitfile = os.path.join(path, f'WT_E_L1_R1R2.trim.AssUnFUnR.split{i}.DNA20RNA20RNAmax30.dna.bridgePE.fastq.gz')\n",
    "    altreadfile1.append(splitfile)\n",
    "    \n",
    "#altreadfile2 = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz'\n",
    "altreadfile2 = []\n",
    "path ='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps_OES/pear/'\n",
    "for i in range(246):\n",
    "    splitfile = os.path.join(path, f'WT_E_L1_R1R2.trim.AssUnFUnR.split{i}.DNA20RNA20RNAmax30.rna.bridgePE.fastq.gz')\n",
    "    altreadfile2.append(splitfile)\n",
    "    \n",
    "steps=['hisat2', 'samtools', 'rnadnacontacts', 'normalize']  # , 'heatmap'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'split2chunks', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize', 'heatmap']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "rnamaxlen=30\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "# if mapping to Mac or OES...\n",
    "hisat2_args='-p 2 --no-softclip'\n",
    "# if mapping to Mac+IES\n",
    "#hisat2_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/OES.v1.fa'\n",
    "PCRDupRemoval=True\n",
    "\n",
    "# ptetraurelia_mac_51_annotation_v2.0.gff3\n",
    "annot_file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/internal_eliminated_sequence_PGM_ParTIES.pt_51_with_ies.gff3'\n",
    "features=['internal_eliminated_sequence']\n",
    "\n",
    "\n",
    "zoomscaffname = 'scaffold51_2'\n",
    "zoomwinsize = 50\n",
    "scaffcoordmin, scaffcoordmax = [219999, 354999, 474999, 629999, 839999], [239999, 374999, 484999, 649999, 859999]\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, rnamaxlen, REsequence, windowsize, contactcutoff, MQ, sn, hisat2_args, genomefile, PCRDupRemoval, annot_file, zoomscaffname, zoomwinsize, scaffcoordmin, scaffcoordmax, features)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-03-03 15:57:19.914342\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting to count rna and dna read positions and contacts\n",
      "Remove PCR duplicates because PCRDupRemoval is set to True\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split0.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split1.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split2.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split3.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split4.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split5.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split6.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split7.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split8.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split9.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split10.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split11.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split12.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split13.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split14.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split15.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split16.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split17.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split18.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split19.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split20.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split21.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split22.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split23.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split24.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split25.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split26.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split27.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split28.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split29.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split30.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split31.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split32.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split33.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split34.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split35.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split36.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split37.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split38.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split39.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split40.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split41.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split42.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split43.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split44.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split45.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split46.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split47.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split48.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split49.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split50.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split51.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split52.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split53.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split54.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split55.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split56.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split57.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split58.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split59.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split60.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split61.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split62.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split63.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split64.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split65.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split66.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split67.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split68.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split69.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split70.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split71.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split72.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split73.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split74.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split75.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split76.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split77.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split78.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split79.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split80.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split81.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split82.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split83.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split84.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split85.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split86.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split87.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split88.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split89.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split90.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split91.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split92.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split93.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split94.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split95.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split96.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split97.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split98.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split99.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split100.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split101.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split102.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split103.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split104.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split105.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split106.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split107.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split108.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split109.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split110.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split111.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split112.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split113.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split114.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split115.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split116.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split117.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split118.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split119.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split120.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split121.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split122.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split123.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split124.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split125.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split126.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split127.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split128.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split129.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split130.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split131.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split132.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split133.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split134.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split135.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split136.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split137.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split138.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split139.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split140.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split141.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split142.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split143.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split144.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split145.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split146.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split147.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split148.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split149.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split150.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split151.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split152.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split153.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split154.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split155.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split156.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split157.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split158.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split159.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split160.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split161.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split162.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split163.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split164.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split165.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split166.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split167.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split168.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split169.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split170.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split171.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split172.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split173.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split174.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split175.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split176.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split177.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split178.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split179.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split180.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split181.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split182.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split183.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split184.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split185.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split186.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split187.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split188.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split189.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split190.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split191.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split192.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split193.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split194.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split195.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split196.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split197.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split198.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split199.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split200.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split201.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split202.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split203.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split204.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split205.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split206.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split207.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split208.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split209.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split210.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split211.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split212.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split213.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split214.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split215.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split216.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split217.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split218.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split219.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split220.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split221.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split222.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split223.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split224.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split225.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split226.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split227.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split228.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split229.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split230.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split231.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split232.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split233.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split234.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split235.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split236.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split237.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split238.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split239.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split240.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split241.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split242.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split243.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split244.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/WT_E_L1_R1R2.trim.AssUnFUnR.split245.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam\n",
      "Finished counting rna and dna read positions and contacts\n",
      "Starting to summarize and sort RNA DNA raw contacts\n",
      "Finished summarizing RNA DNA contacts\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51_with_ies.fa\n",
      "Window size: 300\n",
      "RE counts to output file: /media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51_with_ies.fa.GATC.CountsPerWindow.300.tsv\n",
      "\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "Start: Normalize each window of size (300) by restriction enzyme cut frequency\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ10.NormDPNII.txt\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ10.NormDPNIIrounded.txt\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ10.NormDPNIILog2.txt\n",
      "\n",
      "Counting lengths of all scaffolds\n",
      "Summing lengths of all scaffolds\n",
      "Converting coordinates for each scaffold to continuous genome coordinates\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/RNA.DNA.Contacts.pt.wthickness.Win300.MQ10.NormDPNII.txt.gencoords.tsv\n",
      "[0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]\n",
      "[]\n",
      "[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]\n",
      "Plotting scatter plot colored by intensity\n",
      "Finished making scatter plot colored by intensity\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAdoAAAHqCAYAAACjno8CAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzdd5ikVZX48e99U+XYaXKEGWAIiqAruGYUAyoqEmRRQMwY17xrzoFFUVkT/lARFRURdTFgQMyAiOQ8qXumuyt05XrT/f1RPS1NjzIzdFWHOZ/n4fHpW9VvnxF6Tt10jtJaI4QQQojuMOY6ACGEEGIxk0QrhBBCdJEkWiGEEKKLJNEKIYQQXSSJVgghhOgiSbRCCCFEF3Ut0SqlLlJKjSqlbt6D9/6PUurGyX/uVEqVuxWXEEII0UuqW/dolVKPB2rA17TWh+7F950LPFJrfVZXAhNCCCF6qGszWq31NUDxgWNKqfVKqauUUtcrpX6rlDpoN996KnBpt+ISQgghesnq8c/7IvBKrfVdSqnHAJ8HnrzrRaXUamAt8MsexyWEEEJ0Rc8SrVIqCRwDXKaU2jUcedDbTgG+q7UOehWXEEII0U29nNEaQFlr/Yh/8Z5TgNf0KB4hhBCi63p2vUdrXQHuU0qdBKA6jtj1ulJqI5AD/tCrmIQQQohu6+b1nkvpJM2NSqltSqmzgRcDZyul/gbcAjz3Ad9yKvAtLe2EhBBCLCJdu94jhBBCCKkMJYQQQnSVJFohhBCii7py6ri/v1+vWbOmG48WQggh5p3rr79+XGs9sLvXupJo16xZw3XXXdeNRwshhBDzjlJq8z97TZaOhRBCiC6SRCuEEEJ0kSRaIYQQoosk0QohhBBdJIlWCCGE6CJJtEIIIUQXSaIVQgghukgSrRBCCNFFkmiFEEKILpJEK4QQQnSRJFohhBCiiyTRCiGEEF0kiVYIIYToIkm0QgghRBd1pU2eEEII0Utu4KMAUxkYxu7nkGEYEujwX76nGyTRCiGEWHC01nhhgBcGBGGABupeG1MZZCIxAAwUlmkSBAEhsLVWpOI1yThxViRzREy7J7HK0rEQQogFpx343Dh2P4EOaQYem6vjWIZBOhKj6XvUvDbjrRp+EBAAd5RHGGmUqXtthusl7izvwAv8nsQqiVYIIcSCEoQhm6vjGMqg4bWJWQ5JO0LaidHyPbbWitxWGiZlRTCUQqOZcJvTnlFuNwi07km8snQshBAP4LoelXIFgFxfFtM05zgi8WAhmlbg4YUBEdPGVAaD8QwaJpNqg5QdIeFECXQIgKEU4QMSq6l6N8+UGa0QQkwqlya49KLLOPlpL+H0Z72Mn3z/Z1QmqnMdlngQA8VQLI2vQ0yl8MOAtt85DBVqzapkHwfnlsPk1wrFqmQfCuiLJlmd6ueg7FIMpXoUrxBCCABuvuEWPv2hz1MqlBndMcZ73vQhhreOzHVYYpIX+ky0G5TadTKROGvTAxiGwe2lYRK2Q6g1STvKUDxDiCYMQyxlEOqQfDTJUYPrSNpRSu06E24TTW+WjiXRCiEE4HkeP/r+T2eM/+qqa+YgGvFgoQ4ptxuEaLbXS9xc2EbEMLGUgRsG+Dqk2m5ioKh5LdBQ9poEYec6T81rcX91nM3VcSpuk231IrcVh3tyIEoSrRBCAJZlcfgjN80YP+Twg+YgGvFgfhgSNW1uLW6n4jZpBR63l3cQoslHEhgocrEkISEazUijTNKO0go90JqME2e8WZn2zKrX6smBKEm0QggBKKV4+nOfymEPSLb//pRjOOxRM5OvmBsN3yXUGgPFYCzN6lQ/QahZnxrEViZaa/wwIGFFKLRqjDerWIbJn8fuo+G3d3sAqhfbtHLqWAghJvUN5Dn/qx+lVm1gmAaJZJxsLjPXYQnAMkxiloNtmBySX85Ys0qpXWdJJIU2VGceqzVKKSxlsCm/nKrbouw2ABhtVFiV6ufeyujUM1ck8j05fSyJVgghHiDXlyPXl5vrMMSDaY2lTA7MDHF/ZYwJt8kj86vANNCTV3hQChOTW0rDrE8Pko3GafkeAGOtKlHL4fC+ldS9NmknRsS0sYzuX9+SpWMhhBDznhsG3DS+hZjlMOE2SVtxHMsCOkvJWmuCMCQkpOm7NAOXIAxRKAZiKQC21grcWxkjF00QtyPYPbojLTNaIYQQC4KnO7WN+6JJDswsIdABSqvJJWODpt8iYUTYkF3CzsYEWSdO2auTjyRZlepDoQh1SKXdYDDeuy0BSbRCCCHmPY2mP5rEVAarU334oY+pTEI0ylDoMCRlR4FO1SfHsGgHHiknTqhDbisO0/BdElaEg3LLUD0qVgGydCyEEGIBMFCsSw9iGQY76hNsr5c6SRbQGpQyUCiKrRpN3yVuRwi0RuuQO8ojNHwX2zDxwoDbSsO4PWooADKjFUIIsQAYCjSKSruJUood9QmWxjq1qDt7tCFaaUabVdZmBthRL5N2oliGidawKb8cTafwhWNY6B41FABJtEIIIea5UGu0Bq00jmkyYKUZaZS5sbCFoXiaFYk8lmHiaZ+EHWGkXqY/lsJSncNSh+SXcVd5J3W/DYBjWBzRv6pn8cvSsRBCiHktmCyxqLXGMW22VMc5rG8ly5I5TMPE0wGBDrl3YozBeBpLGZgolNIYGDR9l7rfJmFHyEUShDpkR6Pcs1mtzGiFEELMaxowlEFn8VhR89r8vbCVXCSBqQxCrWmHPl4Y4AcB6UiMO8ojpOwY67ODtAOPQ/tW0PI9WoHHimSelu9OFbjoNkm0Yl7RWlMYK1Kr1onGIiSSCVLp5FyHJYSYQwoIdAAo2r7LhuwSbi1tp9Cq8Yj+VdxZ3kHUtNmQXUKgNbeMbwZAq87fKf2xNHeURqh6LQC21Ypsyi/HMHqzqCuJVswrw1tHOPsFr2F0xxhKKc589emc8cpTSWfTcx3aw1av1XHbHulsSpqJC7EXbNOiHfo0vBZ3lHfQH01yWN9KDMDXmqbvMhRLU243pmoXJ6wIeSeORuGFPlWvRdKOsiSewTbMTuP4wMc2u58GZY9WzBu1ap3zPvBZRneMAZ1Pohd97utMlCoP8Z3zm+/7bL53K+9+w4d49YvfyGVfu5xysTzXYQmxoERNm6TTuSc73qrxt/EtaMALAzblV5ByYjimRcqOMRBLsybdT8VrYUz2nD20bwUbs0sptevcPbGTUqvek849IDNaMY+0W23uueO+GeM7hneycu2KOYhodpQLZc54zsupTlQB+Pi7z8fzfE496yQsS2a2QuyJUGvCULMuPcD2epkl8TRKKeKmw2hzgqF4Fq01IZpVyTxeGJCPJAm1xjEs6l6bOyojnV61QLFdxy1tZ1N+BU6XZ7UyoxXzRiqT4inPfMK0MSfisHp9747hd8OW+7dNJdldfnDplUyUJ+YoIiEWHq01w/USDd/l8L4V1Lw2QRji6QBTGVTdBjW/TdVtAnBPZZSk7aCUoh14xG1nKsnuUvPahNKPVuxPHMfmxS87mRf+x/NIphIccNA6vvCtTy/4NmWZ3cQ/MNSPbdlzEI0QC48b+ARaE7FsbMPkvso4hVaNbbUiUdNiIJ6m5rXRWpOyY53Tx2GIaZhU262pDj3Wg1riPfjrbpGlYzGv5PtzvPG/XsvL33AmhmGQ71/47cr6+nM85ZlP5Oqf/BqAaCzKm9/zOtLZ1NwGJsQCEWpNxW3QF00ShCEj9e1Ap6Zxw3cxUFimSSYSxw18lFL0RRL4YUDEslGTz1iXGeSu8g40nZPMazODGD0oeay6cWH3qKOO0tddd92sP1eIhapULDO2Y5yxneMcePB6cn1ZbFtmtGL/VpmoMlGqsGN4J6vXrSSTTROJRma8r+V7bK+VyMcSxEybO8o7aHhtDs4tww19al6bbCROzW3RH0uhVOeubdKO4AcBtmky4TbxwoC005nxduoe++Qiick7ug+PUup6rfVRu3tNZrRC9EAunyWXz7LhkAPmOhQh5oVatc6lF32XL5z3FQBsx+bCb57PkY85YsZ7FbA8maUdeIy3aqxJ9WMohZrsQ9sfTdHw2+SjSQIdYiuTpB1he7XEylSehtcmatpYkzPghueSjcRI2tFZSbIP5SF/glJqo1Lqxgf8U1FKvaHrkQkhhFi06tU6Xzr/q1Nfe67HB9/2MQrjxRnv1cAthe3ErAjtwKPcbuAFAYVWlYjt4JgmbuBjGZ0OPqONKgrIR5NoOnu7pXYd27QwlWIglpy89NObVnkPOaPVWt8BPAJAKWUC24HLuxyXEEKIfVQuTlCt1AiCgHQm9bDPOuyq2NaoN4hEIyRTSRLJ+MN6ZrvdJgzDaWM7tu9EhzO3M0MdYpsWLd9lSTyLFwRoNFWvRdqP45gWmUiMUGu8MCBqWYQa4raNQqEUbK0V2VYrErMcvLDTQP6owbUP68+wp/Z26fgpwD1a683dCEYsXM16k8J4kd/96o+sWruSjZsOXBQHmYRYaIqFEu9+wwf5/a//BMCBB63nc5ecR/9g3z4/c/uWYV5x8usY2b4TyzI59x2v4sRTn00yte/lUeOJOEuXDzGyfefU2NOf+1TiidiM9yqlWJnMYRkmoQ6JWTbt0EdriJg2ng6wDQulFOPNCiuSfaAgCEPMyTKLUdOmFXg0fBfoHKQyejSj3dvF6VOAS3f3glLq5Uqp65RS142NjT38yMSCctvNd/C8x5/Kx/77f3jN6W/i7a9+N6WCVD8Sotdu/dvtU0kW4K7b7+GKb/+YIAj26XnVSo2Pv/v8qYTo+wHnf/BzVCdq+/S8IAgoFUrEE3G+dNlnedLxj2flmhWc8YpTOfftryCemDlTtjCImDZKKYbrE1S8JgaKFck8hgJbmRhKUXNbDMWzBDpEAU3PRaHwA58DMkOYk/uxhlJsyC7BNnpTMGaPZ7RKKQd4DvCO3b2utf4i8EXonDqelejEglAqlvmfD35u2jLQdX/4K4XxIrm+7BxGJsT+545b7poxdvvNd+L7wT7V2G632tx12z3TxrTWjO0cZ+mKJXv1rFKxzI+/exU//v5PWbl6OW/4r9fwvvPeSbvVJplOEonMPHEMYBgGKtC4gUfCjlBpN7FjFkrRSaShj22aRCwbS5lU3AbpSJykEyNEY5s2oe/xyIHVU7NcNfncXtibn/IM4Aat9c6HfOciobXuWb/ChSwMQhq1xozxdrM9B9EIsX974tMeN2PshBc+g0jE2afnJVMJHveUx04bi0SdvU6ynuvx3a//gPM+8FnuuOUufvGTX/MfJ5xDq9mmb6DvnyZZAF8H+NrHNAzy0ThLElluKW6j1Kqj0Uy4DUbqZWpei0q7QdS0QWuUAQaKVuByU3ErN45v4fbyCNeN3sfm6jjBg/aIu2VvEu2p/JNl48XGcz22bRnmvPdfwHnvv4BtW4Zx295chzVvZfMZTn/5KdPGliwbYmjZ4BxFNP9UJ6rsGN7Jlnu3UhibeapSiNkytGyQT3zxg6xcs4LBJQO8+T3ncsRRh+7z86KxKK9809kc/9ynYlkmq9et5MJvnr/XBVcmyhWu+PaPp42VCmV2Dj/03E1rsA2Ttu+jMGgFLof1rWQglsZAkXKiREybutsmHYnjmHan1ywKA2gHPgB+GNCc3KNtBz6h7k2i3aOCFUqpOLAVWKe1fsgCrQu9YMXw1hFe+OTTabU6M7JoNMJlv/wGy1cunePI5q+JcoUb/3IT37/kh6xZv4rTXnYyQ0sHZuW51UqNWqVG/2Af+f5cz5Z7Zku5NMGXP3Mx3/zydwBYvmoZX/rOBSxZPjTHkYnFKgxDSoUyWmuy+QyW9fBLJtSrdZrNVqfq0kB+r79/ojTB6898Gzddf/O08R9ccymr1q78l9/rBh5+GHbuyBoWm6vjrEr1cXtxmINzy6j5bRJ2BD156rjpu2Qi8c7ysFJ4YciNY5tRSjEYSxO1bBJmhHQkNnVY6uH6VwUr9ugnaK0bWuu+PUmyi8Hll145lWQBWq02l3/zh3MY0fyXyaZ5wnGP4yOfey+vfdsrZiXJlksTXPCR/+U5x76I055xFqc8/aUMbx2ZhWh7qzBamEqy0DnBeeGnvkyz0foX3zVdo9GkUv5HY4JSscz2rcOMbN/BRHlhtxEUs88wDPoG8vQP9s1KkgVIpBL0D/btU5KFTs3vt33gjUSi/1jCftbzn75HvaYVCi8McIzOn2V1qo+RerlTXlEpyu0GfhBQatexDIOkHcUPA9BMtsLTHNa/gsP7VhLqkPFmlZrf6tmMVipD7cbuDgyY0s5sj+zuxOC+KowV+f4DPuAUx0t8+sMX8t5PvYNEMjFrP6fbtm7ePmPsrtvuodVsEotH/+X3+n7QScyf/DLjYwVOO+skDjtyE2971bu58S83AZ0rEW993xvk4JmY99ZvWMcV13ybO2+7m6FlgwwM9u9R0xDLMDFVpwoUChQG7cDDNkxCNEPxNBHTotpokY3EsZSBRqOUQqMnyzFGua24nWbQ2Qasei3c0GdNamDWZrX/NP6uPn2Beu4pz+abX/kOtWod6BwGOPGUE+Y4qv3PzuHRGWOb791Cu+VOJdpqpcb2LcP8/MpfcuAhB/DoYx817+7vHrRpA6ZpTrte8ZRnPpFU5qH3uIrjRU5/1tnUJw+bJZMJtty3bSrJAvz0il/wnJOeyWOf8OjZD16IWeREbAaXDjC4Fyte46MFfnv17xndMcbxzzuO/ECeSDTCYCzDvZVRlidymMog1NAXSdLwXSZaTZancoBGASP1Mgdmh6aS7C47GxOsTPZhdrmRnSTa3RgY6ueyX3yd//vBz9Aannni0+h7GJe9xb45YOM6nIiD23anxp7xvOOmDmGEYcgffvNn3v7qd0+9/oijD+dTX/oQub75k2wzuTSf+8an+Mi7PsX4aIETTnomzz/thD1a0rvlxtumkizAspVLufnGW6e9J9+fw3VdPM+TRgViUSmMFTnr+a9i2+ZhAL7ymYu5+IdfZN3Ba7EMg9Xp/s6sdXJ5GCBuRRjXnW2WEFAaDs4tw9/NMrFt9CYFLqxTJT1iWSZDywZ56atP58zXnM7QskEsWTruuUw+w1e+9zkOfcQhDC4Z4OzXncHzTvlHgioVynz+E1+a9j03/uUmJsrV3T1uzsTiMY4+9lF8+buf4/LfXMrr3vHKPf4gkM1PXw6+9W+38finHjP19dNOeDKf+MIH+c3PruUj7/oUm+/dMu2DiRAL2R233DWVZKGzlfKF8y6iVe8cytpRL4PuHHgCMA2ThtdibbozYzaUQqvOMrGhFEvj03+f1mcGe1K0YtHNaMMwpDheYnjrCOlsimwuSza/sBuH768iEYdNRxzMZy7+OL4fkM6mcZx/zNg0erfVbnSP7sbtjX09qbl63UoOf9ShUyc177r9Xh597FGc84aX8sPv/ITTXnYy55z0Wjy3syT24+/9lO/98husWL18VuMXYi74vj9jzHVdAh0yWi9R9VpTM9pttQJLE1mUGWFLtcDSeJaIaWOgcQyTiXaThB3h0PwKmoFL0o7imJ2yjd226BLt8NYRznjOKygXO+X/nnbCU3jbB99ILi8HRRaqB8/qdsnls5z1mv/gA2/72NTYhkMOILMHhysWinx/jvO+/GHuv2cLxfEyhx+5iXx/jrNecwYvftnJXHTB16aSLHTugF9+6ZWc+/ZXzmHUQsyOQw4/iP7BPsZHC1NjZ7/uJVQNHyM0iJkOVbdJyokx2qxS8VockOnsxbYCDy8MiNsOlmGyozFG1WthGSaOYdLyPQ7KLSMa6/52y6JKtI16gws++r9TSRbgZ1dezZmvOV0S7SI0vrNArj/LJ77wQX79s2s56NADOf65x827w1APV74/T75/+mzYsjvVeeK76aCyu6LsYv/RbLRoNZukMqlZu9ozV/oG8nzjx1/me5dcwejIGCe/9AUsXbWU2xo7OSS/nNvbdUyjcwI57cQouw22VAv0R1M4poVlmJMlK4KpJWI/DDpXfwB7H0pS7ouF/W/hQdptd9p6/i47R0bZuOnAOYhIdEuxUOL1Z76VO2+9m7UHrOYRRx9OYay43+2lP/dFz+KbX/nO1B3bTDbNs1/4jDmOSsyVHdt38vlPfpm7b7+Hpz7rSZx46gkL+tqXUorBJQO88k1nEwQhlmUShCEbnCVsrxU5OLcMgLFmlbWZAe4s7cAxLfLRBIEOJ5vDh6A1q1P9RE2LgXimUzVKKSJmb1Lgokq06UyKE056Brf9/Y6psUjUYeOmDXMYleiGVqPFnbfeDcB9d2/mvrs7nRtPfukLWDwLxw+tf6iP7/z8a/zmZ9eilOLxxx27zwUFxMJWGCtyzovOZfuWzmTj9pvvpDBe5Ny3vYJo7F/f157vlFJTH6JNwyAbSZCwo+w6aZywIxgYbMovww0Drh+9j035FVimgVYKrRSVdoOEHeOm8a1oNJZhclh+BRFTlo73immaHP/c4yYrOV1J/2Ce/3zv62XZeBGybItYPEaz0Zway/Vl96k7yUJmmiaDSwY46YwT5zoUMcdq1dpUkt3lyu/8hDNfffqCT7QP5ocB480KKTtKoDXjjSpWwkDTuRu763/XZQbRWmMqg6QTnUqyu55xZ3kHh/atwOnyzHZRJVqYLHB/zsmc8MJnYFkWmdxDl/cSC08mm+YdH3oT733zRwjDzpLSuz/x9j2qMiPEYhSJzux+0z/YhzJ609y8V9zA55biNtakBii7TeKmw/rsIG4Y0PY7BwM3ZJeQsCK0fRfHtHEDb6pK1AM1/Da96M+26BItgGVZsny2yEWiEZ50/BM46pgj2TE8ytLlQ6QzKSx7Uf4n/bA0Gy2ciL3fzfb3N/FEnFPPOolLL7oM6NQDePuH3kx+HhVvmQ0Nr836zBCGUmwpjnP04DpCrdleK9LwXTZml9L0XSzDZFepCEMZmIaBbZh44T+uBOYiCcweXO/Zo+49e2uhd+8RYjEoFctce/Xvufr/fsORjz6CZ7/w+Bmnl8XiMlGaYHy0wNb7t3PQoRvI5NLE4ovnFLoX+LRDn9uLwxyYXULVbXZqGxsm143ex7JEjuWJHO3Ap+o1yUeSWEpR9dvYykQruGdiJ3XPJReJsz4zRNSanT3af9W9Rz7+C7EINRtNvvzpi6dmN9f8/Hdc+8s/8PH//cA/vZe8r7TWFMdLtNsujm2RzWdlZWGOZHIZMrkM6zeum+tQumKsVSXrxGkGHqHW9EWTaDodepYncp0+tGjur46TicSwDINAa6KmTdN3aQU+S+JZoqZNw+8UvugFKcEoxCJUq9b53iVXTBu77g9/pfGAw2OzZct9Wznr+a/i2Y99IS962ku58bqbphXREGI2eL5P1W3h6xDbMIlZDkqpyX6zPksSWRpeC61hVaqPqNFp/q7RBGFIxLK5tzLK3RM7ubm4jXsro2ypjk/dqe0mSbRCzDPF8RLbNm9nx/adVPax16xSasbhGKXUrO/Tlgpl3vna97H1/k4rwHKxzJte9k7Kpf2idXVXuK477TS9AD/w8QkZiKXY0ZhgU245hVZ1qhm8pTr3a5clcxh0VllSTnSypZ5CKYW7m3Ktda9N2IXt0weTRCvEPDI+WuC1//FmnvO4k3nmv72AT73/AkoPqHS2p9LZNK95yznTxp5/2nNmtV8wdGrRPvDeOkCtUqPV3POm9qLD932Gt47wiXefz7te936u/+ONVCu1uQ5rXvB0iB+GmMogZtoYhkHCjlJs1zAwsAwDUykCHeKjiVo2480Kodb4gY+hDBzD5MHHnnLRBKbqfhqUjRQh5gnf9/nOxd/n9pvvnBq78rL/61T32ct9VcexOf65x/GIow/jj9f8hUOP3MTaA1aTSidnNWbLtjj0EYdMa92XyS6uAzi9UiqUOfUZZ1Gd6FT5+vVPf8uFl57PYx632/M1+x0DuL9WwFYmA7EUE36bzdUCXhCyPJml5rvETQc/7JRbLLXrDMYy2KZFiGZ7rcTG3FLuq4zhBj590RQrkvmuN33fFbsQYh5oN9vcetPtM8bvuOWufXpeOptiwyEHcsYrT+PIRx/RlcItuXyWD3/2PRx40HoAliwb4jMXf0LuM++DG//y96kku8vFF15CrSqzWgNFy/dI2VHGWlVGGxVKrTqWMnAsE1A4holpGJRadQylSNixyaXjDjf02VGfYG16kEP7VrIskcFWUutYiP1KPBnnuGc/md//+k/Txh89z2c0K1Yv58JvnY/bdrEsi3x/DqMHs4TFJplK7GYsiSH3nzGVQdyJELMdyu0GNb9Nf6wzI235LgbgmBZe4NMXS1L3XFYkcxiTc8m277IikacVeIQ6xFZ2Zzlahzg9mG/Kb4MQ88SuWsVnvvp04okYA0v6+cjn3svAUN9ch/aQ8n05liwbon+wT5LsPtpwyAGs27B26utI1OGVbz6buCzDExByd3knaDgot5S16QGSdpS/F7aScmJTRSjMySpYlmmypVoABZoQyzS5tbSduOVQ99rcVhpmW62IrwO6UUviwaRghRDzTLvVnjoEsz/Wb96fFcaK3HTDzRTHSxz7pH8j35fHiXS/6P18V27VubsyygGZISKmRbXdpOQ2GG1WOGpwLZV2k0wkjtYhVa9NqEOykTidhWNNqKHUqtEMPEYa/zhcaBsmRw6smZVax1KwQogFJBKN7LZurVj8+gbyPOnpj5/rMOadiGUzGE0TMU2GayWWJnKU3QbQSaWBDqi6DVJOnKQdoeXvusfdmUjWvRbZaIKtha3TnuuFAW7od72pgKzxCCGEmNcswyQXieEGASknypbqOEPxDFHDRqHIR5PE7QghIQpFzHIItSbUnTYCm6vjhFrvtiWe1YMDUZJohRBCzGu2YRKxHNqBR8qJUXYbDNdLHN6/korXYqLdxFImru+jFLSDzozWCwIUimWJHI5hsj4ziPGAJgLLEtmeXO+RpWMhhBDzngKSdudgWNqJMd6qsSrZR8y0sWwTjcYxLbSGQqvG8mRuKonmowkCHbK9VuKwvpW0Aw/HsAi1nlHEohsk0QohhJj3Ah1OVX9amczjBj6WYRBqYLLTrFIKA0VfNEnT94iaNrv2ad3QZ7xVZbxVxTEsfB1gGxZH9K2cbKnXPbJ0LIQQYt5TdBtttEsAACAASURBVOoVK6UwlcHG7FJagQ90UqmBouW5eEFAxLKJWjaB7uzZtgMfx/jHvNIN/ck9W2vaUnK3SKIVQggx7yk692QVimKrRkAIWmOqzpgfBsRsB9vslF8MwhDTMAgnl5RrXouhWHrqeYZSrE8PYnf5xDHI0rEQQogFQKMJdYhSinQkPjkjdQgBUynaOqTebpN0osQtB8vo7Nt2rv+EWIZJNpJgKJ7BCwOSdhSnR3fUZUYrhBBiAVBsrhZQwJZqAdswaQYuhmJqSTkViaGUwjHszoxWGSgUFbeFpUxsw+y0kDRsAh1g9KBzD8iMVgghxCxqt9rsHBnlsq9dTjQW5QUvfi4DS/pnpcKZBnwdTjUZ2LUH64YepjJRWhESorXGMIyp08VpJ8ZfxzZPHpnqOLRvBTOrS3eHJFohhBCzZmT7Tl503Bn4Xueg0rcv/j6X/eJrDC0dfFjPNQ2Dlck8aMhG4uxolFmeyBGEAXHTQSuFQqNQBGGImry6YyiDUqs6LckC7KiXSdsxaZO3O1prxneOc+0vf89ffn8DhbHiXIckhBAC8D2fr3/x0qkkC1Cr1PjVVdc87GdrDRHTohm4ZCNxcpEEO5sVym4TjZpKpAZq6o6sqQxCHe62+pNtWPTgwDGwAGe0O0dGefEzz6ZU6BSGXrdhLV/41qfpG8jPcWRisaiUK7iuSyKZkAbmQuwNxW6XiA3z4c3p/DBga63QWTIOPMy4QdyK4Bh255SxWyPrJAgmDz8pBWGoURiYCpJ2hLjl0PBdoFPScVki17M92gU1o/V9n29+5bKpJAtw7533ccOfbpzDqMRiEYYhW+7byltf+d+cevxZfPI9n97nFZN222Xn8ChX/+TX3PzXW6f9NyvEYmVZFqefcwqRqDM1ls1leOLT/v1hPTfQIVWvxWA8zfJEjqhhsbVWIG7bWIZBqd0gRGMohaEUXhhw58QOQh1MnUpelxnk0NxyjhpYyyP7V2Mo1ZMWebDAZrRBELJzeHTG+OjI2BxEM79UKzWK40VuvekONm46kP7BPtKZ1FyHtaAUx0uc86JzGdsxDsDl3/oRzWaLd330LSSSe3ds4r677uelz3slbrvzCfrxxx3Lez75DnL57KzHLcR8MrRskO9e/Q1+eNlPiMdjHP+8p83KiuPqVD+BDrFNE61heTKHH4ZTCdNWBoEO0apTgrEvkpzqjTxcL7M8mQNTcW9llGKrRsxyODC7hITlYEplqH+IRBxOOfOF08Ys2+KJx+/fbaXa7TY//eEvOPEJp/Guc9/HC598Old8+0c0G825Dm1BadQbU0l2l19edQ3NRmuvnjNRrvDJ931mKskCXPPz31GU8wRiPxCJOCxftYxXvfllvORVL2Zo6cDDPnFsKAMFxCwbA8W2ehE38PF1iK9D1qcH8cMAJpNu3WuTjyamZqxuGACK+6tjFFo1NNDwXf5e2Iqvw4f9Z37I+Lv+E2bZAQet47Nf/xSP+rdH8rgnP5avX/ml/X5/tlKu8ukPfX7a2Oc/8aWp5uFiz0Si0Rl/Iaxcs3zqU/Ge8n2f0nhpxrj8+xBi39iGScy0CTqFjVEo7izvxFIGjmGCgkbgobXmvsoYazODNHx3qmDFuswAoQ4ptxvTnhtqjRsEXY9/wSXaVDrJMU98DJ/60of58Gffy8ZNBxLdz5tk61DTqE+fvbZbLmHY/U9qi0kyleD173o1avIoYiwe4z2ffAf5/txePSeby/DC/zhx+lg+y/LVy2YtViH2J27g0wxc/LCz5zoUz7AknsZAYSrFRLtJqEMCrWn4LlurBfKRBGHYuXNbbNUJ0cQtZ8az7S4vG8MC26N9oHRW9h93icaiPP6px/Kbn187NfboYx9FNBadw6gWnkQyzvNOfjZPfeYTKRZKDAz1k81l9vo5pmnyjBOPIxaP8oNv/aizjPafL6Ovf/9eeRFiX7UDn+F6mSXxLDFlYCmTlBMjVJpQgxcGpCIxbMPk4NwyUNAKPBzTRivdObVcLbI+M8TNhW24oY8C1qYHsXpw8lh149TVUUcdpa+77rpZf6745wrjRS67+HL+eM2fOfKxj+DFZ5+83y+pzzWtNRPlCpGII9eEhHgYGl57qiJU3WuTicSpeS2ipo092fJua3WcpYkcda+NUopsJM5IrciKZD+eDrhxfDMD0RQrknk0YCoDSxlYs1TvWCl1vdb6qN2+Jol28fBcj3q9QSIRx3bsuQ5HCCFmRdv3uKO8o1MZCoiaNhNek5wTRwNBGGAaJg2/jWNYqMlDUW7oE7ccFNAOA4brJRSKZfEstmkRtWbv78l/lWj3aM6slMoqpb6rlLpdKXWbUuqxsxadmDW2Y5PNZSTJCiEWFY1mIJZie71E3W9jGgZVtzn12oTb5L7KKDHLoRW4aN0Zj5kOodbcMzFKy3dZnepneTLHhNfEDX280H+Inzw79nRx+tPAVVrrg4AjgNu6F5IQQgjxD0opEnaEhteiP5qk5btkI4nOlR6tyUTitAOPqtsibkVQSmMbFrcWt6GUouG7lNoNCq0afx3bzH2VMf42voWt1WLnGV32kIlWKZUGHg98BUBr7WqtpcyNEEKInjCVgakUm/pWUnFbeDrAVKpzv1Z1mr6nnBgxy0ajafkebuCTjyYBGIxnGIil2VItTHvu9nqJYJ7co10HjAFfVUr9VSn1ZaVUr7oLCSGE2M9ZhomlTNqBh6EUDc9lR73MeKuKpnPwcHkihxsEmChiVgQv9MlFkygU/dEUEdMi3E1S7UUVxj1JtBZwJHCh1vqRQB14+4PfpJR6uVLqOqXUdWNjUhJRCCHE7AjDkBCNpQySdoRMJE7SiWGbFpV2s1NCUcPORhmtFHWvRcKOTu7jakabE1S9JoOx9LTnZpwYptH9Fj57kmi3Adu01n+a/Pq7dBLvNFrrL2qtj9JaHzUwMDCbMQohhNiPeWFIEGqqXotQa7wwIBuJk7IjRC2bpt8m0JqliRxaM1m8IsBUBqDYXitxz8QoA7E0q1P9ZCNxViX7OCi3DNvofjmJh0y0WusdwFal1MbJoacAt3Y1KiGEeBja7TbjowUKY0WpkLYYqE7H2ZrXxlCK24rD+GHAcK1MxLRJ2lFQGgNo+S6pSKzTv3ZXElWwPj1IzHLIOXGWJ3IMxtIYPWpIu6ep/FzgEqWUA9wLnNm9kIQQYt+VimUuvegyLrv4cpLpJG9+97kcfcyRJFJytGShMlCYhknf5OEmUynur46zITOEG3g0A4+0HcUyTe4vj7Ixt5S2dkk4nfK8jxpYgwbuq4wx3qpOPXdDdgkDPUi4e3S9R2t94+Sy8OFa6+dprWdWTBdCiDmmteaan/+OL3/6YibKFbZvGebN57yTcemctKCZykDrsDNzBVal+ibrHJs0Aw9TGYw0JvCCgMFYCrTG0ApQhDrEDQP8MJyWZKGTeOfF9R4hhFgo6rUGP/3hL6aNaa358++un6OIxGzwdMhtpRFA0/Q9cpEEG3JL+XtxK0k7QtKOYimTiGWTjcQJ0EQth5F6CXOyxZ5m5vHi3Z1C7gZJtEKIRSMScTjosA0zxg88aN0cRCNmS6hDDsgMUXGb2KZJqVVnW7VIO/DRITQ9l6WJDPXJmsg76mU0IRGzUyXPUIqW75GypzdaWZrITR6Y6i5JtEKIRcN2bE478yTWb1w7NfbME5/G6nWr5jCqmdotl/HRAqWC1P7ZE0qDaRjcXy1gKZNsNEE79AAwDIVPQDv0iVkOtmHhmBYKRdqJTfakVbiBz7rMICuTefKRBAdmlrA8kcPcy37T+xa/NBUQQiwyhbEi9VoD27aIJ+NksumH/qYeKRXKXHzhJfzoe1cxMNTPOz78nxy06UCcyMxeqaKj7btoFO3AwzZMNlfH6Y+mMAyDuOXghQFaaxzTwkDRCnyik8lWo6m4LRzTxDJM/NCffJ9JxJq9qz0Pu6mAEGLxC4IAt+3OdRizom8gz6q1K1i6Ysm8SrK+5/O9S67ga1+4lOJ4iTtuuYtzTnot5dLEXIc2r+nJXdZCq4YXBhRadZJOlPsrY4w1K1jKQMPku8APfUATTu7MKtVpHr+1WqDcbhJqULvZs+0WSbRCCEZ3jHHhp77Cu9/4Qa7/41+pTFQf+pvEXqtMVPnZlVdPG/Ncj7tvv3eOIlo4Aq2puE00mqXxDKONCg3fZXO1QKA7VaNavgfoziljrXGDXV/7WIZJfyxFXyxJqPVk8u4NSbT7kcJYkWt/+Qd+/bPfMj5aoBvbBmLhKYwVeMlzXsFFF3yNn135S8456Vz+Iqd0uyISjbBm/T/2iw3D4JgnPoa1B66Zu6AWAEN1rumknRgj9TJLElkafnvq9RvHN6MU3F4aQQPZSHyyOlRnf9YPA0bqZaKmzbZakfsqY1S8Jl7Q/as9sOcFK8QCNz5a4MwTX8X2LcMA9A/28Y0ff5nBJVIuc3+3+d6t7BwZnTb21c9/gyP/7RHk8tk5impxSiTjvP6dr+bG6/6O23b5xBc+xG1/v4Pz3n8BTzr+8Tz2CY+W/893wzEt/DBgaSLLPROjuIFPXzTJeKsGQMS0UShWpvqoeS28MMBR1tTScag1azMD/HVs81S3ngm3wUG5pQzEur+1IIl2P/Gbn187lWShk3i/f8kPecWbzkL1qAyZmJ8ikciMsWg0gtGD05j7o2Url/LNn1xEq9ni4+85n2uv/gMAV//k15xy5gt57dtfQTwem+Mo55dQh2yvlxhv1ViT7O8cgNIma1L9NPw2K5KdBDsQS9EKPLZUixycW0qxXSfQAWknRs1rzWiJt71WIhtJYBtmV+OX36T9xNiO8RljozvGpQ6sYOmKJWzcdODU16Zpcu7bXzmvDhEtJkop+gf7sCxrKsnu8r1LrqBerc9RZPOc7uyqmoaiGXj4YWcpeVWqn1Br0k6Mpu9iKoNsJEYr8Gj7HiYm5VZ9t8nUNsye7NTKjHY/8cznP52vXPA1gsk9CaUUp5z5fEyzu5/kxPyX78/xuW+cxw1/upFtm7fz5Gc8kYHBvrkOa9FThsIwjGkfdm3bkhWm3TCUwfJknqWJHNvrJfKRBCjYWi2yJJGhL5Kk4jZJOzHagU9/NEXUtIgnI2h2XfsxSNlRql5r8pmKVak+rC7PZkES7X5jYEk/X7vyi3zhvIvwPI9zXn8my1ctn+uwxDyR78/x1Gc9aa7D2K/EE3FOPPUEvnfJFVNj57z+paQyyTmMan7SWqO1ZrxVpeI2WZ3q497KGL4OiJoOIZq4HWHCbeBrTdqOUvfbGJNN4PuiKcpug1WpPgId4gUBSSdK0KMDoVKwYj9Tr9XRWpNMyS+zEHOtVChz9x33cOOfb+KxT3wMK9esmFdL9lpriuMltNakM6k5K6rhBh5eELKlVmB5MoelDEbqZYYbZR7ZvxrLMAl1yO3lEdakBjCUwjZMTAyUgnbggVLU3BZJp1OG0Q8DYqZNzJ55RmFf/KuCFTKj3c8kktIqTIj5IteX5ehjHsXRxzxqrkOZoVFvcMOf/sYn3vNpqhNVnn/6c3nxy140J6eiQ91Z6u3cn50gaUcZjGcYa1Vp+h5pp7P82/RdRuol1mUGURo87WMri+HJK0Fe6HPT+BY0kLQjHJzrzaqeHIYSQggxQ3G8xOtf+la23r+NcmmCiy74Glf/5NdzcoBy18prxLIJJwtX3D2xk0Pyy0k6DuV2E1Mp1qYHKLbr3FcZAwV+EAKaVuBhKYMtteJUPaia12ZbrUjQgz+PJFohJpWLE9zyt9u5/NIr2XzPFmoVOf0p9l9//fPfZhS1+ekVV1Or1HofjOrMVgHSTox8NEnMclBKoTW4ocd91QJN3+WI/lUsTWQJQ41G4wUB6zIDU4egHqjqNXvSKk+WjoUAqhNVvvA/X+Hb/+/7U2MfvuA9PPXZT8ay5GS2mKnZaFKt1BgdGWNgST+pTGpR3X9de8CaGWMbDjmAaCw6881dZimTVuDSDjyy0QReELAqmacdeJiWQ8qOMtqs0vDbDNfLrEzkSTkxoqaNYSjQmmwkTtxyWJnsI2J2ilm4vt+T7j0yoxUCaNSbfOfiy6eNffJ9n6FclDZmYibf87nuD3/l2cecxBnPeTnPfuxJ/PYXv1s0TRkAlq9exgknPWPq69XrVvKSV502JweibNOkL5qiGXiEWlN2G3hhMHU31jEt8tE4ljI4tG8FUcshatlsqxdoem0avoeB4qDcMrbWCtxU2MrtxWEs06QXB4/l1LEQwPYtw5xw7IumjTkRhx/9/jL65U6peJDx0QInP+0l0/rJJlMJvverSxgY6p/DyGbXRKlCo97AdT2SqQR9A/me/vxycYIt92/lputv5uhjjiQzlCeVSeGFAQadWtFo8MOQqGVT99tU202G4hkMpah4TWKmg2WYTLh1djYqTLjNqecbKI4aWjvVIP7hkFPHQjyEWDzGwYdv5Lab7pgaO/HUE0gk43MYlZivgiCY0bS9Vq3jef4cRdQdmVyaTG5urhvVq3W++rmv8/Uvfmtq7E3//Vqed/pziFoW2gAvCLANgzYhd5ZHODCzhGQiwkijTH80TdqOAZp26GMb1ox92hDdOQzV5d2h/W7pOPB7061BLCz5/hyf/urHOfM1p3P0MUfytg++iZe/4Uxii2jPTcyeSMThEUcfPm2ss385O3cyBdTrDb75lcumjf3veV+hUaljmAovCCYPMimipo1lmBhKMdaqEjUdUJqG355sKhBS913SzvTfZ0MpTKkMNXsmyhVuu+kOrrzsJ2w64mCe/tyn9nwZRMxv/YN9vPJNZ9NqtUkk41JUX/xT2XyWj37+fZz3gc9ywx9v5LBHHsJb3v8G8n25uQ5t0QiDcKpk7C5u20UDTc/DCzurB1HTxlQGyxLZyVPGPgOJNIV2lbjloFCYyqTUqrEmPYAf7qTmtbENk425pViq+7/n8z7RTpQrWJb1sJbwfN/nqit+wcf+6zwA/u8HP+dH37uKz37jU/KLIaaxHRvbefj7NWLxG1wywH9/7K006k1i8RjJlBSDmU2xeJRjnvgYfv/rP02NPesFx4NjcM/ETjb1rcANA1Cde7YRwybUdK72EOKHIa3AJ2LaOIbJhuxSiq0qa1IDREwLQ6lOe70e1Jaet4m2MlHlhj/dyDe++C3SmTSveevLWblmBU5k7/8SLBcn+H+f+8a0sdtvvpNquSqJVgixzxLJhFRb65JMLsP7/+e/+PH3ruIvv7+Bfz/uWJ5y/BMIYhYEirbvEbXsTgEKpQFFoTFBJppEAf2xFHcUR8jl44Robi1uZ0Uih1JQbNWImg7ReG9OUM/bRPv3G27hTWe/Y+rrP1zzJy7/zaUsWTa0189SSmHZM/+ohilLg0IIMV/l+3Oc9rIXceKLn4PpWAQ6xDYt1qeHqHpNbMsGrTEMRcNzSTpxTKXYXi8xGEuzMbcEL/RpBz4HZpfgBv5kG70EpjLwAh/b7H4anJeZpl6rc+lF35021m65/OX3N+zT87L5DK9+yznTxo4+9lEk01JYXwgh5jPTNEkmE2AotOocbKq4DQIdEoYBanLpeFutiGV0loHdwJ+8AqQot5vELIcgDFFKEWqNFwa4oY+mN9175uWM1rJtBpbMvIvWP7Bv9xlN0+RxT/o3Lr3qq/zix7/ioEM38MhHHzEnxbGFEELsHa01gQ4ZqU+wMpWnP5bi+tH7SNoxVOjjmDZRy6bmtQl0wLrMEHWv1al3rEMMZeCYJpurBQqtTgnJiGlxaH4FtqG7vk87LxNtJOLwsnPP4Jf/9xuqE1UADj5sIxs3HbDPz0xlUmzMpNi46cDZClMIsYg06g0q5Sr33n0/q9asJJNLk5JVr3nBCwO8IGAwlsbEwNMBGii16mQjneXiFck8QagJdIDWIRHTZmejwlA8jaGg6QdTSRagHfhsq5dYnxrANLt7xWdeJlqAJcuH+O7VX+fWm24nlU6yZv0q8v1yHUcIMft8z+ePv72Ot77iv6a607zlfa/neac8W+5SzwNeEBAxLbwwQKtOo4AN2SWd08amiR+GbK8XWZrIM9FuMBBPYwDB5NLwRLtJMNk8IGVHcUyLituk5bvo7h86lhKMe6tRb1CvN4g4EdLZ1FyHI/Yzvu/j+wHRqBRGmE27K6noRBx+eO23GVwyMIeRCYCW71Js1Uk5URzDnFwi1txRHuGI/lU4hsVoc4J8pLMCYRlmZ/9VQzNwqbQbDCWytAOfhtemFXjkogkUiowTm5WlYynBOEvGRwuc94HP8sdr/sLGTQfwzg+/hRWrl/XkHpYQozvGuPSiy9h63zZOesnzOfiwDaQzc1Meb7EJw5BycWLamNt28edhSUXP87Asa7/7e8cNfbwwxDZMYpbDXRM7ATCVgR8G9EVSaDSFVpWkHZt8n03ghQwlsriBz/2VsakyjNvrJQ7OLetJ7PPy1PF8VJ2o8v63fJSrfvBzysUyf/rtdbzqtDdQGC/OdWhiPzA+WuCME17OxRd+k19edQ2vOvUN/PGa62b0CxX7JhaL8vinHjttbOOmA4nGe9sSrt12Gds5zuZ7tzC2cxy37U29Vi5NcO3Vv+fdb/wQl170XQpj+8/fPUoZVN0WgQ7YXitiKmOqc4/WncNMSkHDdxmIpYnbDqAJtCblxPCCAI2eUet4c3UcL+x+WV6Z0e6hdtvld7/647Sx4a0jNOtNkJUl0WVb79/G6I6xaWMXX3gJRx9zJLk+OT3/cKUyKf7rY29l+aql/OE3f+awIzfx6rec09OCNr7nc+Ofb+JNL3sHzUaTRDLO+V/9OI84+jCCIOSKb/2IT3/4QgB+esUv+OkPf8H5F32U3H5QdMcyDJYmssRMm2Qiihv4LE/kqHktQt2Z5YaaqWXg8VYVUxmEYUjEdrAMAzec2eC9Vx9TZUa7h5RSLFu5dNqYE3GIyF6Z6IHd/XcWjUWk6Mos6hvIc+7bX8UXL7uAt33gjT3fmy0Xy7zjte+l2ei0cavXGrzjte+hVChTKVe4+H8vnfb+v99wC/Vao6cxzhWtNZYyOvXHFRTaNSbaDQ7rW0m53UChcAMf0PhhSLFV5/7KOHEnioFiuF7i/7N332GSVmXC/7/nSVX1VK6uDtM9EQYGGLKAAcXVxZzXyBp2BUV9RV1EDGtmDYiuvhh2V3QVww9dXOUV14S6plVXch6YPNMz07lyfMI5vz+qaWgGYWC6q7pnzue6uJg+Xf30PdfVPXeddN8R08a15leCWpXIzc2MF5P+LT1AuXyWSz73gblDKKZp8t6PvVMf/9ceUeAHTE1Mc+uNdzC6cw/lUuVRP2NoeJDjTjpm7mPTNHn7+95COqP3aBdSJOrQl8/15KSx5weUCvNb701PzMztE9vOQ1S3O0waX/gyZKZdm9sqGW+UyUTjFFs1+mNJKl6LmGVTbDeRSpKwI52ax1KigLFGma2lCY7LjbA21c9ALMUJfSvJRROHd63jpUYIwXEnHcMP/+c/mJmaIduXJZFM6KP/2iPasXUX5/7NW+ZmH3973ss5/8JzSaUP/NR6Lp/l8isv4/ab7mTf7jHOesaZuiH9ISYSdVi3fg07tu6aGzv6uPU4EYdUJsVbLz6fj1z0ibnPPeWvn4R7mPRLDmSIbZhMNCuscNNETJtt5Qk2ZIZBgGs6tAKfpB3FNAwG3DTBbMMBUGQclzWpPDsrU1jCJOO4uJbTldks6Os9mraoysUy73zD+7jl+tvnjf/oD1czsro7Jx615WN0514+dOHHuPOWuznptOP56Oc+wMjsllWlXGXv7n38+me/Y+NJx3LC4zYeNk1RmoHHdLNKLpqg1G6QjXTeYOytF1mZyOHLEFN02uHtrEyyNtVPJ8sqBIJASe4u7KUV3n+4bCCW5Mj0INYCJVt9vUfTesT3A/aNju83XiqWdaLV9rNq7Qif+/dLCYIA27ZIZ9Nzn0ulk6RO2MCxJ2zoYYS9IZXEkwH76kVCJUnZURCic4pYgSMMmFsCFoRSMdUsM5LIEiiJVHJekgWYalZZlxroSvyHxwK/pvVIMp3gOS9+xryxVCbJ4Iru/IJry08mlyY/0DcvyR7uhDDYVy9R9poMxVKd8ovtGv3RJGWvgRAGoVKAYk0qTyNoM92qUvfbSNVpofdgEbN7faf1jFbTFlEkEuG1b3oVpmXysx/+kpVrhnn3JRfqKzma9ih4YUDSjrIykcO1I0w1q2ScOEJAK/ARQuCHASECQxhU/BZJJ8bu2gxHpAYwhGBVIsdorXP3WADrM4N6j1bTDiWe51MpVToHWx7FIShN0zolGJuhz5bSBCvcNFGz0/B9c2mMk/vXYIlOwjQEtMKAu2b2sLFvJYVmjQE3xbbyJCsTORzTohV4uHYEUxg4C9iLVu/RalqPOY6tTwlrS1JhpsiNf7yZP//+Rp7+nKey8aRjyCyxFqIK2FwcZ9BN0R9LIYBtlUkUEEoJKIQhCCVIpXBMi3uK+zght4p60GZtqh9DCASdpgJlr0km0r0T2zrRapqmHaYq5Qqf/tD/5efX/gqAa77zI15/wWt549v+ruvlJx/JcDxDfyzJ3YW9HJNdQcKKMEMNx+zUfd5dmSZq2ggh2JhbyVSzgkKRsDt/j7LXwLUcJpsVHMPqWlUoOMDDUEKInUKIO4QQtwoh9JqwpmnaIaBRb80l2ftc9dX/oFqt/YWv6A1TGPRFEzQCD6kkgQzJxRKk7GjnEo9SBEqSjsTojyW5ZXoXFa+FOXsSeXtlkohpc9v0bkZrBbZVJrl1etdsNanF92hmtE9TSk0vWiSapmla1wkh5jWnEA9xQrfXLMMEpfBlSFsG2KbF7dOjHJ0ZRCqFVIpV8RxCCCabFQIZ4pgmrcDHsWyips1YvUh/LMVQPANKoVA0/PaC7tP+Jfp6j6Zp2mHKdWM8+8Vnzxt79fmvXHKlZQ0hCFEk7SgDs3u0ngzYXBrHQBBKiWEYSCWRqlOkYmUiRzsMEEDG5+TnowAAIABJREFUcUnaMdJOjDunR7l9ZpRNxbGulbA8oFPHQogdQJHOnvSXlVJXPNzr9aljTdO05aE4U+KW62/j+j/cxF896ykcs/FoMrmld4e3FfoEQYAwDASwr16iFfockerHFAY7q9MMuimipoNUspOcZ5sRBLMf3zy1c/ZebYdrOZzQt2pBZrULcer4TKXUPiHEAPALIcQ9SqnfPeibnA+cD7B69eqDCljTNE3rjmxfhqc/56k8/TlP7XUoD8sRJpidqz6OabMykaPYqiOVwhTM/t8kUCGhlLRDn4QdRSqJUKBQ85IsdPrXdsMBzZuVUvtm/z8JXAOc8RCvuUIpdZpS6rT+ft2gVdM0TVs44Wwpxahp44U+++pFWtKnHfrsqRdYmcgiENT9NgAVr4llmCg6XY7aYYDzoAIVGcftSveeR0y0Qoi4ECJ535+BZwJ3LnZgmqZpmnYfPwwptRsgOnuYE40yCTuCa0cIws4s1hCdw1yt2dksSiGEIFSSXdVpNmSHSdidVqeZiMtRXaoOdSBLx4PANbNZ3wKuUkr9bFGj0jRN07QHkCgmmhWykTgShWWYmMJAIBhw0+ytF1mXHsC1HBQSCws1273HFIJ60GZbeYLheJaYZc+VbuyGR0y0SqntwEldiEXTNE3THpIlDNal8oRK4poOx2RXYMzuvbpWhKjlMNWoMOCmGK0WyUTipJzOPdtm0GZDZgWbS+NsLU8QMS025lZ2rbGArgylaZqmLXmGELSCgEzExg9DDCFACASCUIUMxlLI2QNP/bEUe2sFUk4UhWJLeZJhN8Mp/Wvuf+Ai1Pn/i7F37TtpmqZp2mMlBO3QRymImBaGYTBW79w6VZ1Pc29xjEB2kvCR6UFqXqdNXtxyyEbj3FMc48bJHdw4uYPxZgU/DLsSuk60mqZp2pJnCgPbMDsFKpRkplWj7Ddn92mh4rVohT4KhW2YBErimBYKwRHpQcbqJWp+a+55e2oFPNmdEow60WqapmlLnkKRjcZBgASqXpNW6BOqTmJthz4CQdR0QCkCGeLNVoZqBh7VByTZ+9x3FWix6T1aTdOWnJmpAlMT09i2TS6fIduX7XVIWg9JJWn4HjWvRSbqYghBLppgulVDKUmAoC+apOq3ULNLyTHLJpQSIQQNv0UuGt8v2Sad7nQo0olW07QlZXpyhvNe+lZGd+4B4KTTTuCfv/IJcnmdbA9XXhiiUGyrTiKqglP6V5N2XFbFc516xQpCFbIu1Y8AQqVoBx5lr8GwmyUdiWMaBs3AZ6pZwTJM1qX6u3KHFvTSsaZpS0gYhvzgqmvnkizAbTfewa033N7DqLReE6i5cokKxZbSBIEMGYqnQXUKWDQCr/MH1SnTmLSjIO+/J1vzW8RMm+P7VrEhs4K043a6AnWBntEukEq5SqvZQghBOpvCcZxeh6Rpy04Yhmy7d8d+4zu27gSWdi1ebfEoIeYauMcshyPTAyBge2WKI1ODgKTYbuAYFqGSJJ0ovpLk3SQApmGweXociYLaDABpx+XY7Aps3SZveShMF7nk4kt5zhl/w0uf/hp+cs11VCtLq3Gypi0HjuPwknNeMG9MCLHkC95ri8sWJrZhcHRmiHWpfhp+m93VGQIpkSpEolibzDPdrGIbVueOrVK0Ag8hBIHsvOaBqn5zv7HFohPtQfI9n6u/8QP++6e/RSlFrVLjknddSnGm1OvQNG1ZOvbEDXzkn/+RtUeuZsPGo/jCNz/DwKBuVHI4Mw0D27DIRFxcyyEVcSm262QiMcpeE4Gg4bcZTmRpBG2agYeic9hJKXV/8n2AlBPDEN1JgXrp+CDV6w3++Js/7ze+6Y57WL1uZQ8i0rTlLZ1J8byXPoszn/YEhCHIzZ44DsOQ4kyJWrWOG48RT8SJJ9weR6t1i2kYmBh4YUDdb+FaESpeiyNT/RhAyonSCHxykTibimNszA7PtsVTGEJwTHaYLaVxfBmSsCMcle5OQwHQifaguXGXUx9/Enfeeve88aOPO6pHEWna8meaJn39uXlju3fs4Y0vv4DCdBHTNLnwg2/lha94HolkvEdRar0g6BxsOio9yJ2FPRjCoBG0acsAEwPHtPBl0Gk8IAS+kgRSMVYvsjE3gm2YSKXwwgDH6FSYWmx66fggOY7Na85/Fac+vtN3wYk4vOMf30JfPvcIX6lp2oEqF8t87D2fojBdBDqz289e8kXq1XqPI9O6RSpJOwxQKPKxFPvqRU7Mr0YqhRAGcStCyolhCkHaiVFo1RAYOIZFxWuyws1iCZO630nKgZT4qjslGPWMdgHkB/r4zBUfp9lsYZkW8VQc1431OixNO2T4XsD2zTvnjUkpKZcqDA4P9CYorWsCGVJsN9haGicbjeNaEVYn89xd2MuRmUGc2dKMjaCNbVisTuaRShIoiYEg5cSwZpu/76kXafhtMpE4q60+LCExF3lWq2e0CySTy7BiZIj+obxOspq2wNxEjLOecea8sUQyrotYHCYCKbmnuI9ASWKmQ380wXijNFf72BIGCjANE8swmWpWsA2TzaUxQGDOHnraVNxHxWsSKMl0q8qu6jSBkosev060y0yr2aJRb/Q6DE3rKjfu8rb3vZlnv/gZRGNRjj5uPVdc/QUy2XSvQ9O6oDVbrOKo9BCDbgpPhkilWJnMUfOahKpz4MkRFhJF2nFpBT4DbhqlJKYwCJXCl/OXigutOqoL7fLEYnyT0047Td14440L/tzDme/57Nszxlf+75VUKzX+7i1/y9HHrSeRTPQ6NE3rmnqtQbPRxDAMPZs9jLQCn/FGiaQdJeFEKbcbxCyHLaVxjs2NEEqJhYFpCtphSKFdp9xucES6H0MYFBpV+twkN07umHdzNm5F2JgbIWIdfAN4IcRNSqnTHupzeo92mZiZLnDOs8+l1ewUxf79r/7I1675V04+7YQeR6Zp3RNPuPpKz2HIMgyG3DShkgShJOHEQCkUYu7zUnVqMVqGQdlrELFs2mFA1LTZXpsmGXE5IjXAtsok0Gm7d1RmcEGS7CPRS8fLxB9+/b9zSfY+3/7yd2g29m/9pGmadii5ryaxAgxDsKdaACBldwpSKGBreQKFQioYdrMk7WhnD1cINmaHEUIQtRxOG1jHKfk1nNq/lritu/doD/BQe1GZvgym1Z0L15qmab1034GmQEoc0wQUq5I5AhliCJOy1ySUnb3amGVz99Rehtw0qxN9uHaUm6d2EM5ulQoEw/EMa5N5eFDFqMWgZ7TLxMmnn8iaI1bNfZxIxnn9/3kNjrP4yx6apmm9JpVCKfBlQD6aRAiD26ZHsQwTAZ1CFCh2V6cpe00AHGGiULRCby7JQqcD0HSrtt/hqMWiZ7TLRF9/jq/+55e489a7qVXqnH7mqeR0UQxN0w4TYrZRQMx0CGYbwW/IDiEQtAKPY7PDmELgmBZxu1O8IhNNIBDYxv6pLtaFvdn76ES7jPT153jqM57c6zA0TdO6rlMBSiAAA4OYbTNWLzEcz2IaBmO1EqtTeQZjaXwVsjrRN3tUqnOPdoWbYazRafZiCYN1yf6uHIQCnWg1TdO0ZaGzdIwQ1LwmSSfKTKvGcDxHzW8RMS1AgVDcPbOHE/pWYxkGoGgGHsPxDENuJwlHDItw9i7ug7v6LAadaDVN07QlzQsDFJ16x4YwcCyLmt8majqEUmIbFsloDD8MsU0LqRQzrSquFSETiVMP2txTGqM/msQ2LArtGhHTYoMd1YehNE3TNC1QIX4YYBgGBuAYFq3QZ3WyD4kkYUcxzU7nHqUkA7E0lXaThB0hlJJsJE4gQ8YaJXbXpqn5LQZiqUWvcXwfPaPVNE3TljQpFYYwqPttEnYEQwgGYkkAqn6b6WaVFfE04ew+7lA8jSVMmmEbS5i0w4BjssOM10vkonHSjkvnbFXn9YtNz2i1AxaGIcVCSdda1jStqxyzU3ii2KoxWisw1axyw+ROxuplYpaDEIJASnZVZxDAltI4Fa9BudUgYlrYhslko8yR6QFK7Qa3z4yyrTxJK/S7UutYJ1rtgJQKJa7+xg+44DUXccnFl7Jn116kXPyuF5qmLU+e51Ot1BYkkZmik2jHmxUGYimmW1UeN7AWiaLQqpKwo0gpyUfjKMC1HCzDpN9NoxQ4psVIIsvW8gSFdp1QScpekztn9nTlLq1eOtYeUeAHfP+qa/nSp64AYNMd93LDH2/mP677BvmBvh5Hp2naUjM5PsU3/+0qtmzaxrNedDZPf/ZZZHKZx/w80zDw5P1/Pio9RKFVY2+9SC4SJ+W4+DIk6XTqYA/Hs1jCoNCu0xdNcufMKCfmV88VsrhPK/QJu9AmTyda7RGVSxV++N3/mjdWnCkxvndCJ1pN0+YpTBd40yvfzq7towDc8MebmRib5LwLXocTcR7zcwWCXCSOKQxqQYtiu7OFtTrZx+0zo6TsGJmIS38syZbSOAOxFAknihBwTHaYQIZETIt2GMw90xCiK9d79NKx9ogsy6Svf/+EmkzrFn2aps1XLdfmkux9vvfN/0elXD2o5woB61ID1LwWrcDHtRyipj239NsXTcwlzWboU/QaxK0I0Fk63lGZ4ojUwNxrBJ3+tpZY/HrxekarPaJ0Ns3FH30H5/7N/8FrdxowP+tFZ5PWTbc1TXsQ+yFmral0cgFO9wqqXgNPBkRMi/5Ycu6qz/G5lURMC4GgEXjErchsAladEo2hT9lrYhkmJ/StIlQSS5jYhtGVKz460WoH5MgNR/DD33+Xe+/cwuDwAIMr+h+yo5B2aCkVy9QqddqtFplchr5+XV9be3jxeIznvuSZ/OSa64BOjeJ3feTt5PLZx/zMUEoKrRpJJ0pKxGiHPmO1IvlYClMI2kGIFwYknRhTzQpHZ1bgqwBPhiiliJo2BoKZVo2ZVg0BDLpp1iTzC/S3fnhiMY42n3baaerGG29c8OdqmtY9xUKJyz74OX5+7a8AGF61gn///pcYXDHQ48i0pa44U2LP7r1s37yTUx9/Mn39Wdy4+5if54U+rTDg9undjMQyrEhmuW16N5Yw2di3klCGeDLAMiwafgvXjrCpsJeNfatAgSGgLUO2lydwrQi5aIKM4+JYCzfXFELcpJQ67aE+p/doNU3bT6VUZdf20bkkC7BvdIyvffFbc9sHmvaXZPsynHDKRl70yuexau3IQSVZ6ByEqvttjs2NkI+n8GWIa0VohB6hkhhGpyqUIQSOaVNqN1ib6qfcbmAaBuONMq7psDG3kpjlMN4oMdYo4z3gYNRi0kvHmqbNUyqU+M7X/pNUJrnf57Zt3kGr1Tqo06Oa9mjZpkXaiVFqN4hFOh13Vif7qBfbbC1PsCGzAkuYTDWrZCIuEdMmYUc6HX+AlBOjLQN2VacptusAlL0mNb/J0dkV2MbiHojSM1pN0+aZGJviW1/+DseffBymOf8foOe+5JkkU/snYE1bbKYwSEdc1Gw3nulmlQ3ZFaxN5lF0tkCF6LTESzuxTvMBwyREUWo3sAxjLsnep9Cud6Xwjk60mvYYVMoVJsYmmZqYptVq9zqcBTW+d4JWq82PvvdTPvmlj7D+mCPID/bx5ovewNOf/dSu1IbVtAczRGf52BQG28qT9LspZhpVhBDU/RYSRdSyEQJ8GRDIEKkkSimG4plOSzzEfs/sBr10rGmPUmG6yCUXX8rvfvkHorEoF7z7fJ7/8ueQSh8aM71jTjga27H5wVXXsn3LTs459+Ucs/Eojtiwjkgk0uvwtMNQqCSh6iRSgLYM2F6eYn1mgNunR+mLJkjYUe4tjvG4/rVIpTq9aIXAVIpCu0E64jKSyDFam5l77upE32zP2sWlZ7Sa9igEQcD3v/3/+N0v/wBAq9niMx/9PNOTM4/wlctHNpfh37//JY4/+Tgm9k0wPTHD0MohnWS1nglkSChDTGEQSMlgLEUm4rKnWmB9epCViRxKwfrUAArYUy9iCAOpFK3ZghZj9RJ90QQn9K1iXaqfk/KrZ1vl6YIVWhcUpovs3jFKpVzl2OM3kOvP7rc3p3XUaw3+9/c37Dd+162bOOKotd0PaAFVy1UmJ6a58U+3cOrjT+IzX/k4hmGQzqSwHbvX4WmHgVKxjBCCdCY1bzyUkm2VKdanBwlVSD6WxDEsJIo91RlS9iCBCrFNk0bQJmY6gAIFkdnTyLtqM0RNi3wsScRMIBBdSbKgE+1hrzBd4B1//x7uum0TAMl0kqt+8u+MrB7ucWTzlYsVLMsknoz3NI543OWMM0/jlutvnzd+3IkbehTRwvA9n1/99LdccvGlc2Ovev1Lecu73qiTrLboqpUat990B1+5/BvYts0F7zmfo45bj+vGAPBkSNlrEMiAZugTNW0sQ1Bst1iVzKNEZ//VMS22V6Y4Kj2IJ0OmGzWGE2l2lic5qW8lILizsJdm4GEKwfr0ILloAkufOtYW0+a7t84lWejMar7y+W/QXiIHfCqlKr/66W+58Lz38v63X8LWe7bjtf2exWPZFi9/3Ys582lPACASdfiHD7yV/sH+nsW0EMqlMpd//F/mjV39jWt072GtK3Zt283bXncxt990Jzf97y2c99K3Mjk2Nfd5qSQrYmmqfuffpVBKDGFS81vYhkHDbxMqyWSjSi4Sn2vNV/U7P795N4UCtlUmaQade+ChUmwujRN04dTxAc9ohRAmcCOwVyn1/MULSeumh9pbnBqfwvcDItHe78nddtMdXHz+++c+/vP/3MA1v/0OQ8ODPYspl8/xscs/RKvZwjAMEqk4sdl33suVUoJGY34LMSklYah7DmsLo1Qs06w3EYbAjbtzhwfDIOTqb/5g3mullPz82l/ypgvPBSBuR3GtCLdO7yJq2qxPDxLIkFw0gQJsw8IyTArtGuvTg9SDNl4YckR6ABAYCAxhUPPnTyAUnRPKURZ31ebRzGjfAWx6xFdpy8ppTzqVSHR+8YFXvf5lJHq8RAtQq9b57tf/c95Yu+Vxwx9v7lFE90tnUwwOD9A/lF/2SRYgnojx4lfOf/98yhknEotFexTR0lGvNZgYm2L75h1MT84QBIvfKPxQU5gu8oG3X8Lznvgynvv4l3LZBz9HcaYEgDAEK1eP7Pc1w6tWzP3ZRODLThWnNck8oZJU2k0SVqSz1yoMBIJ8NEm53aDsNUlHXAR0lokNAxSknfm/qwYCx1j8HdQDSrRCiJXA84CvLm44Wrfl+nJ8+7++ylnPOJOTTz+RT1/xcU563Am9DgsA27boH9p/SbZ/sDuFwA8nbtzlzRedxz9+4l084azTectF53HZv32MbN9jb9Z9KKjXGvz4+z/j+U98GS/769fysr9+DTu27Oh1WMvOf//0t/zxN3+e+/gn11w3t2VlGAYv+dsXzFulWnfUWp701MfPfSxRlNsNhuNZHNOi2KqTjrhsKU/QCFpYhoEnAxJOlHwsSX8shRcGKAUR00YgaAYe61L9c8nWMSyOy40selUoOMCmAkKI/wQ+CSSBdz3S0rFuKrD81Kt1giAknU098ou7aO/ufbz6eedRKXV6WW486Vguv/JT5PK6i8xikFLSrDeJxKJYlj55PjE2xfOf+DLC8P5Z7LEnbuCL3/pnsrnD+03IgfJ9nw9d+HF+/sNfzhs/722v463vPn/u45mpAju27sK2LVauGZnXKcoLA6abneIUuYhLOwyYbFaZblU5OjMEShG1HPzZK0D1oE3acQGFUrCnNsN4s0LacVmXymMZJgLRaa23QEUrHq6pwCPOmYUQzwcmlVI3CSH+6mFedz5wPsDq1asfY6har/T6NO9fMjQyyPd++S3uuXMzqXSSVWtXHlS7Le3hGYaxZH8WeqFRb8xLsgA7t+4mDLpTjL5XgiAk8H2iC7B1YNs2z3je0/ZLtGedfea8j/v6cw/bhjEbjdMKfaRSREybUEqG41mips14s8yIHSUUkp3VKVYn86BU54SyZVPymhybHcY2TBQw1awy6Ka6VuXsQBanzwReKIR4LhAFUkKIbyulXvPAFymlrgCugM6MdsEj1eZIKSlMF9m5bRepdIr+wfwhu8Rnmib9g3m9XKz1RCIZJ5NNUyqW58bOOvtJh8S+/F8yNTHN9755Dds27+CFr3guJ592AumD7D196uNP4vUXvJbvfPVqLMfmTRe+ntVHrHpUz2h4bUIUMdNmtDbDmlSendUpbpsucGR6EEFn5qsAqRSBkiSsCCGSDdkVbCmN05g9cTzkpqGLWepR9aOdndHqpeMe27t7H699/hvnfvmfeNYZfOzzHyTbp2d6Wsd9b8aajSaRaIRUOrkgs5PDTRiG7Ny6i0su/hTbt+zgKWefyTs/eAH5gb5eh7YoZqYKnPfSt7J7x+jc2Ps+fhF/8+oXHnQRm1azTa1aAyCVSeE8ivvZUim8MKAZeERMi6lGFdsyKbTqrE720Qw8MrMdemxhAQKFxJytDrWnXmC8UcYUBpZh0A4DTsmvIeEs3O/EQS0da0tLs9HiX//5q/PeYf/pd9czumuvTrTanF3bR3nzOe9ganyaSNThg5e9h6c98yxi8UN3JrYYTNPkyA1HcPmVnyIIQmJujHji4HqrLmWF6cK8JAvwrSu+w9OfcxZ9/Qf35iIaixCNPbYrg0HYaexuCoNQKfrdJLuqM6xO9jFeL5GLxGn4Po5lIoTAkwFR00LOTiQbvsf69CAxy8YLQ9zZ/dxueVQFK5RSv9F3aHvL8zzG9ozvNz6+b7IH0WhLUalQ4iPv/ART49NA50rUR991KdXZ2YT26GVyGfIDfT1Jsl7bY3pyhpmpAo9mBfKxsO39Z5nRWBQhelvbSKFQUmEKgSEEzcBn0E0x06ox6Kap+W0itkWt3UKhOk0IJBgYgGJNKk/ZazDTqhG1bKp+C8e0upZsdWWoZSaVTvKSv33hvDEn4nDSqcf3KKLHLgxDpiemufn629h273aKhVKvQzokBEHIlk1b5435nk+99tirPAVByNTENHfecjd7du2lUqocbJjaASgWSlxx+ZW84uzXcd5L38qffnv9olbrSmfTnPHk+1c/hRC84/3/Z0kcQLRNCyEMvDAgbkeImjYGAtswsQyDpu+RjroIBJYwEShCFYLovCZUirgV4bbp3WwtT3Dz1E7G6iWCLiTbJb10XKvWKBUq7Ni6kyOOXks6k14ShRR6SQjBk5/+RN5/6cVc/Y1ryObSXPjBC8gsw8NQY3vGed0Lzp9bBn/qM5/Mhy577yF7sKtbItEIj3/K6fz2F/8zN5ZIJUimEo/5mbu37+bvX/IWapXOrPhv3/AKzn/H35PKLK3rYIcSKSW/+K//5mtf+CbQqaz09r+7mB/+/ru48cWZWWf7MnziCx9m0x33sn1zZ096KRxEFAKUlBhGZ49VKslko8JQPE2oVKdW8exkv7M3K0AIJhplhtw0Xhiwwk2zuTR/NXB3dZpBN4XF4l5lW7KJttVq8/Nrf8XH3/vpubFLPvd+nvnCv8ZxnIf5ykNfJpvmxa96Pk971llYlkUqs/z6oDbqDb502RXz9pp/e93/MPaOcZ1oD1IyleB9n7gI3/P50++uZ+2Rq/mnyz9I5jGeHC2XKnzy/Z+dS7IAV331al719y/ViXaRFGdKeJ7Hddf+at64lJJbb7hjUZt+5PJZznzaE+bqeS8Ffhh2mrQrsA2TsXqJFfEs5XadhB2bLULR+XwrDAilJOFEcMxOiotZnW4+D14qVrDoy/GwhBNttVzls5d8cd7YZR++nMc/5fQl8Q6r10zTXBLLOY+V1/bZu3tsv/GJsUmOO/GYHkR0aBkY6ucTX/wInudhGMZB/az4ns++0X37jRdnSqxcs3/pPO2xC4KQ7Zt38OF3fpyTTjuBI45ay81/vm3ea9atX9Oj6HrHNEykkgQqxBEW440yw/EMFb+FaZi4po1pGCg613oilkXVayGl7ByOCgIaoUc+mmC6df8bxrgVwejC/vOS3aMNg5Dmg4qc1yo1ZBc6LWiLL5VJ8uJz5p+ri0QdnWQXUCqTJD/Qd9BvyFLpJM964dnzxhLJOEMjvWvscKgqzhR50yvfzr13beHH3/8ZL3jF81i97v77pi94+XMYXjXUwwh7wxACgUAqRc1vcXRmBaFS5KNJdlWnMUSnBKNUiqhpI6UiatlkZjv5BEqyuzrDcDzLcDxL3IowGEtxXG54bta7mJbsjDYai3DKGSfO6/v5hLNOJxrVdwEPBYZh8PTnPJV2q81/fvuH9OWzXPTht5PNLd9Z+qHKiTi89k3nAPCzH/6S4dUreN/H3qmvky2CWrVOefagWb3W4MMXfoyLPvw2Vq4ZJhKNEk+4+zVFPxyYwqAlfbzQJ+3EKHst/NketUekBqj6TaKWAwiUColaFlvLk7QCjxP7VhEqSdyOcOfMHvKxJINumlboda0y1KMqWHGgFqpgxdTENF+5/Epuuf42Tn/SqZx7wesO2Yvih6swDCkVy9iWvSz3mg8nXtujUq5iO/Zh+Y99N0yOd2orP7BD0HEnHcMXvvmZw7q2slKKZuBxy9RuTulfzabiPtYk82yvTBK3IqxM5OaaA2yrTLI+Pcjm4hhHp4fAEEgpkSi2lieo+W1MYXDUbNN301iYhd2HK1ixpBMtdKqJNOoN3IRLdAn0R9U0TVssjXqDn/3wl1z6gc8S+AGZXIavXP15jtxwRK9D65kgCKmUKpi2ieeAazncUxzDMkxWJXL4YUg6Eus0gzcMbp7ayQm5VRiGIGJYSCUJFYQyxDSMuZZ5ABFr4frQLuvKUAdSTaRerVOt1JgYn2LFyCCpdOoxVyDRtKXK83zarfZBXdPRljY37vLsF53Nk5/+RBr1ZqfWcu7g6gwvZ8VCiR9d/RP+6/s/Y2TVMP/wwbeSGhlgJJ5jS3kcFKQiMaaaVdphwMpElqPSQ53CFnNHkARjjSLFVo2jsyuYrFdphz5D8QymNDpXgxbZkk+0j6TVaPGLH/+aj73nMqRlaiTJAAAgAElEQVSUWLbF5V//FGc8+bSDrs2paUvFxNgk3/y377B9yw5e8LLn8KSnPeExX9fRljY37i7aPdnlJPADvv/tH/Ivn/4KAFvv2c6tN9zOVb/4Otl8lpP7VmMZJg3fYyCWYrpZQSpFK/BIzfacDWaXjMvtBkdmBrm3OEYr9AGYbFY4NjtMPrb4W1ZL9tTxgapUqlz2wc/NnUYO/ICPvOuTFGd0lSHt0HBfoffvfO17/Pn3N/KBd/wTP7r6JwSHeKs27fBWLpa59uofzx8rVRjfM8G+eglfhQRKEjMtQqVIRxKESrKrNkM79FEoTAG2MMhEXKSUc0n2PqO1Gbxw8X+Pln2i9T2fVqs9b2x6YkZfA9IWTHGmxPTkDOUHFNfoppmpAvtG5985/u6V36dU6E08mtYNlm0zMNS/33g2m2G8Uca1HEIVYhgmoNhRmeS+I0dTzersHVqDQClWxDMPeV+2c2lo8S37RBuNRVl31Np5Y0986hn64JR20JRS7Ny2mwtedxHPfNyLeOcb3rdfwuuGyEP8LCdTCb01oh3S0tkUF3/0H+a1d3zOS55JPJMgZUfxZdi5M6skCmiGHorOPVrLMDpJVkosIWgEHo5p4Vrzf5fWJPPYXbhHu+RPHR+IsT3jfOajn+euWzfx+Kecxtve+ybyA7p6lHZwZqYK/P2L38ze3fdXRTrupGP4wjc+09UykaVCiY9efCm/va5Tu9g0Tf7lqs9y+pMe17UYNK0XfM+nWCixffMO8oN53GwCKxElatqUvQb9sSRSKcYbZfpjKartJn2xBHtqBUYSOZqhR8ywubu4j0CFbMisoOq3aIcBg25qNikvzBvWZX2950DVqnVazRZuwsV1dc/N5cRrezQaTeJxF/tRNINebHt37+MFZ75iv/GfXv8DBlcMdDWWwkyRndt2s2v7KKc/6VT6+rK6t6x22PFliC8D/CBAGAaOYRHIkFBJ2oFPwolxb2kfzcDn+NxKQiWJWTabivswhUEmEscSBlW/xXA8Q9JZuN+hZX2950AlkvHDvrPPcjQ9OcO3vvwdbr7+Np541hm88vUvpS+f63VYQKciUiqTpFKqzo2tWDnUkyXbXF+WXF+WU884qevfW9OWDKWoeW1ilo0lzM6BJ8NgZ2mK9ZlBfBnSF02QdlzaoU/cjmJgcHRmBRWvyVSzgm1YjCSyOEb30t+y36PVlq9Socx73vJBvnXFd7nr1k189fPf4BPv+wzVcvWRv7gLMtk0n/ziR3FnZ46pTJJLv/TRZd3MQdOWKy/0qXhNEnYEpaDSbmAKg1BKjsoM4kuJbVgYwuD2mVHuKY0RqBBQVL0mjaDNykSOfCzBaHWGxe/Zc79DZkarLT/NZnNeLWuA3/z89zT/6UKS6d6XY7Qdm1OfcDLX/PY7NBtNYm6MbC6DsUAl2zRNOzBeGLCjMk3KiRIXEWzDwIm41PwWAJbhUPEa9EUT7K7OzH3dRL3EymQfMcuh5re5p7gPyzBZk8zjhT7RBawM9XB0otV6xjRNnIiD1/bmxuLJ+JJKZJGIs+htGX3Pp1qpEXWj+nyBpj0EX4YU23VWJjrbSkIYQKf+8fbKFKcPHDG7lAxSKVYn+hiMpVACTAQVr8lYo1NbIQwDNpfGeVz/2q7Fv3T+RdMOO4lUgrdcdN68sYs+9LbDqpl4YabI1774Td5yzj/wyX/8DON7J3odkqYtOYYQ+DJEIQlmayTcd9oYoBX6ZCIuAji+b4RBN8VUq0rTbyNRFNr1uWdFTZv+WJJW4D/Ut1oUekar9YzrxnjJOS/krGc8mS2btnLM8UeTy2dxltDJ48XUbDT5109/he//f9cCsOWebdx24x18/Zp/o69/aRwI07SlwBIGfZE4KMG+epG1yTyekhizbe7agc90s8KqRB8Rw6bQqhO1Old3BIK4FaHiNTkyNYBjWhTbdTwZ4IXB4d2PVjs8pDJJUpkk69av6XUoXVevNfjxD66bN7Zn1z7qtbpOtJr2ALZpcWR6kMlmhYQdJVQKZ3avdUdlioQTZW+jSIhCAJZhMt2s4VoOg26KITeNbXROKW8qdu7Fj1MmPVvveLGLVuilY03rESEE/YN9+405EadHEWna0hWicO0IuWgciZyrDLUxN8J0s8qwm6XcbmAIcG2HmVaVTCQGCCYaZQbcFKO1wrxnlr0moVr8cr060Wpaj+TyWd7/yYuxrPvv5b7uzecQT+j74Jr2YFJKGn6bPdUCdd/jjpk9TDQrnc8pSToSox36hEphINiYW4klLKBT67gReA/53G5c89FLx5rWI0IITjh1I9f+4Wq2bNrGqjUjZPNZ3W9W0x4klBLTMCh5DVYn81jCIFQS2zAJpGTQTbOzPMVwPEsj8EjYUQrtGgaCFfEsCsVUs8JIPMvO6vTcc1NODPMhmg0sNJ1oNa2HYm6MmBtjaHiw16Fo2pIVqJCZZo3BWJrxeonVyT6OTg+RinSuwymlmG7XWJceoNnyCMyQqWaVqGkzksiiEEw1q6TSMY7NDlNs13GtCPlYoiuHofTSsaZpmrakNQOf3bUZkk4UxzQptRrEnUinsbtSSKWwDZNCq0YqEiNUEsewODYzjFSKQqvOqkSObeVJdlamUCgyEZeIqQtWaJqmaRpR0ybjdO7JZp04zdBnX62IY1ikIjFGawWOyQ6zt1YkH00SCskR6QGUUIBgulUlE3E5Ob8apcA0jK6WYNQzWk3TNG1JswyDNak+fCmJWDZxy6EtA/bUC8Qsh3w0gUCwOtmHRLK1PEnNb9H0PASwMp7FpJNcZ9pVql4Tuwt7s/fRiVbTNE1b0gwhMISBKQTby5NETJuBWAoFVLwmoVLsqxc6d2UVBDJkvF6aK+caNSwils1t07vZUyuypTzBpuI+vDDoTvxd+S5aT8xMFbjuR7/iB1ddy8TYJIHfnR8qTdO0heTLkB2VKQwhiFoOm4r7SDsxjsoMErMcfBmwLjlAxWsgBAy5aYQQRCwLBRimxe4HnDYGqPotfBl2JX69R3uImp6c4fUveQt7d3eqoMTcGN+97kpWrRnpcWSapmmPTtVrYRsmVb/FYCyFJQzuKuxlMJYm4lgMRJMEKkQBdb9NPpogG4kjlcIUBlJJlNp/V1Z1aadWz2gPUTf/+da5JAudurpf/9K38LyHvrStaZq2VDmmxUg8S6FV59bpXTRDn5F4DqUUoZJU/BZlr4FrOUQth2boz5Va7JSkUKyIZzGEIB9NMuSmyTgutjAf7tsuGJ1oD1G1an3/sUodGS5+uTFN07SFFDNtGoFH0oliGyb90UTnEJSbRAET9TIpx2VXdRpDCExhIOh0+IHO9aC0E+XU/rVETQupFGtSecwuteTUifYQdeZfPYHYg3qbvu7N5xCNRXsUkaZp2mNjmxZCCHKRBBtzKxmtFbANg3boYyBIRVyagUc20ilfahsm/bEUZa8BqlP72DRMbp8eZU+9yGSzwm3Tu6l6ra7ErxPtIaqvP8d3r7uSF5/zfM5+3tP45o+uYN2Rh1+HHE3TDg1JK0LDb1PzW6xw00il2F6ZwlMBuWgCUwj6IklQzCblOE3fR4lOR5+K18ST8w+EjtZmunIgSh+GOkRZtsWqNSO892PvRIZSz2Q1TVvWJFDxmwgEA26KZuDjhyGOYaGEouI1UUJhYDDVqpBxXEaSWQSCVujP9a59ICEE+48uPD2jPcQ5jqOTrKZpy54AAinJx5IEYYhCsSqRxUDQDn1cO4IpBCBJ2lF8GXRKNNIpz2gbJjFrfgvKNck8lrH4B6L0jFZ7WDNTBfaO7iMMQlatW0VeNyTXNK0HFIpBN03db5G0Y1jCJGlHkErhS4mUnYTaDnyilk0gQ1qhjylsOmlasDE3QjvwkUrh2g6W0Z0UqBOt9hfNTBV406vewfbNOwAYXrWCr1/zr/QP5nscmaZphxuBoNiqkYsmCGfvxbZnmwe4lo0pTJhtLqCkQgiDqGkjhEEoJTHTxpchU60qrdCnP5okG41jdWFhVy8da3/RH37zv3NJFmDf6BjXfu8nD3nxW9M0bbHlY0lMYTDerGAZJtvKk1jCwDIsDCFQCEzDQBid6z2h6nT3mWiUCZTk7uJexhtlSu0GW8oTTDWqc1eAFpNOtNpfNLpzz35ju7ePIqW+i6tpWvcZdEonxkyLVuBzRKqfQEkCFc6eKFYIBDW/xe3Tu7ENExNB3HLwZUj7QbWNxxolgi6cOtaJVvuLnvviZyIedFLvZa99CabZnWoqmqZpAF4Y0Ap9EAYR0ybtuPgyRAiBMXtyeF+9SM1voegUqvCVZHdtBiEgG01gPURxCmt25rvYHjHRCiGiQojrhRC3CSHuEkJ8tAtxaUvA4PAAV1z9eU4+/USOP/k4Lr/yMtYeubrXYWmadpgJ1X17rAGeDLijsIe4HaHmtTvVipVgyM2QcmL4MiDtxLCFwepEHyAozRauyETcec9dlxrANhf/qNKBfIc28HSlVE0IYQP/I4T4qVLqfxc5Nq3H3LjL455wCp/790+iFGRy6V6HpGnaYUgAIYpCq06gJKEMMYSg2K7TF40jBNS9NtPNGiOJLFWvxSn9awmkRBiCitdkW3mSjbkRVsZzNAOPdMQlYtpdif8RE63qnHypzX5oz/6nT8McRtJZnWA1TeslAUrRCn36ogkipkXda9MXTSCEoB34WIZJNmpTbNWxTZOK1yJuO6AkfdEE440yt8+MEjVtbMMkatnE7UhXoj+gPVohhCmEuBWYBH6hlPrz4oalaZqmaR1CdPZph9w0Bp3yinvqBVJOjNFagahlk3JiRE0bwxDYholrd4pTSEApWJnIYQiBJwPSERf3QcUrFtMBJVqlVKiUOhlYCZwhhDj+wa8RQpwvhLhRCHHj1NTUQsepaZqmHaYipk3EsmkGHjHLpuI1GYimMA2DYrvOTVM7kUpRDzxMYVBqN2ZLVHSS7M7qFKGUHJcbYWNuhFDKzuGqLnlUp46VUiXgN8CzH+JzVyilTlNKndbf379A4WmapmmHO1+GhEoSKsVEs0Iz8MjF4kw3q+SjSVJ2jHrQImbZbC6OkY8l8WWIZZiYQjDopuiPJYkYnSpRw4ksMXMJzWiFEP1CiMzsn2PA2cA9ix2YpmmapgGEUlL2GsQsm+lWlRXxDO0wYKxeJhuJMxRP0w4D2mHAsblhJhsVHNNCSkWIIh9NUWjXuWlqB3fMjHLL1E7aD+rks5gOZEa7Avi1EOJ24AY6e7T/tbhhaZqmaVpHoEJ2VaZxLYfBWKdFXsP3yEXj3FXYS81rk7Aj2IaJNdtAQEpJqCQCgUKxp1aYe55Uiq2lCbywO8n2QE4d3w6c0oVYNE3TNG0/jmExEEsx0azQF0kw0aiQdKLk7SS2YZKLxvFliCd9UnaUmGUz1igzEE0SNU38B1Szi5gWwewererSBRrdVEDTNE1b2hQMJ7LcPLUTW5iEStIMPBzDpD+W5PaZUXwZkosmSNsxpls1+mPJudNQAuiPJhlOZGkFfmdZWUmMLhVH1IlW0zRNW9LqQRtTCIbj2bnG73fOjDLkZjAMs3PwSRjETBuFImbZmMKg4rewDQsvCBhOZLljZnSuicBgLE3c7k6vbl3rWFu2GvUG+0bH+M11v2fntt1UStVeh6Rp2iIIlaTYrtMfTbK3UUQgOLV/LQOxVGe2Gktycv8afBkSKEkmEkcIgWOYKAGOZbG9PDmvU89Es4xU3WmQome02rIUBCF/+t0NvPtNH5hr2/fWd7+Rc859OW7cfYSv1jRtOUk6MfbWCrhWhKMzQ1TbDUzTJG5HSNgRYpbD5tI4/bEkgQypeS3STpyIYdMKPGKWM9vdZ76gS53I9Iy2RwrTRXZtH2X3jlGKM6Veh7PslAolLn3/P8/rjfvlz36NWrW+32uVUpSKZeq1RjdD1DRtgTiGyYbsCpJOlJbvkY52Kjt5gU+o1FzfWVMYNAKfhBMDAY5pEbciBGFAPpqc90zbMHG60FAA9Iy2J2amCvzDue/hrls3AXDKGSfx6S9/jFw+2+PIlg+lFKVCed5YEIT4/vxqL+VShT/+5s/8x5Xfp68/x9ve8yZWrhnBsvWPvqYtF512eAaNoE3MdgikpNCuE7cixITAl5JcJM5Ms8q69ABbSxMMJ7JsqU5zRHKAsteYaxo/3aoSNW1WJ/NYXWmSp2e0PfHrn/1uLskC3HL9bfz59zf0MKLlJ+bGeMbznzZvbMPGo4jFYnMfK6X402//zPvf9lFuv+lOfv2z3/Hq572Bwkyx2+FqmnaQQiVp+h6WYVL327RDn0CFBFJSatVJOTGy0TjtMGAkkWW0OsP69CC1oEU+lqTqtWiFPsPxLH3RBLZhdK23tk60XaaU4s4HJNn73H3HvT2IZvlKJOO86yPv4NwLXsvRx63npa9+IZdfedm8VYFKqcp/XPmDeV/XbDTnvcnRNG35yEbjhFJS8hoMxNIEYYiBoN9N0/C9uc4+UcthKJ5hV2WKfDSBALZXJplsVthbK7C3XmRvvYjs0h6tXj/rMiEEL3j5s7n26h/PG3/2C8/uUUTLVy6f5fwLz+XVb3gFMTdGNDb/qL7tWPQP5vf/uv5ct0LUNG2BGAgaQbuz7JvooxG06Yt19l23lsdZm+wnlAqFoh0GuKbNymQfBoJq0CZm2qzPDNEOfZRSxO0IgQxxjMWfb+oZbQ+sP+ZI3n/pxQwNDzK8agWXfO79rFq3stdhLUuOY5Pty84l2WKhxNTkNK1mCzfu8tZ3v5F44v5TyKc+/mRWrRnpVbiapj1WAmzDQiG4q7CXdhhgCkHT91iXGmCiUcFXIUIIdlSnsA2LiGEhUURNm6OzK9hSGmdzaZwt5QnuKuztWmN1PaPtgXQmxYte+Xz+6hlPQQlFNpfp2l7BoSrwA3Zs3cnH3/tpdu/cyzOe9zTe9M5zGVk9wg9+cxWb7riXbF+GkVXD+tCZpi1DAoFEIZCEKqQdBkilSDgR7i7sox0GrEr+/+zdd5zdVZ34/9f5tNvb9EkmIYGEELpSFJEVC4qo2BDrggVZFQXFFWR3wV0b9r6siouirgXFAougfhF+gLgovQQIhLSZZNrt/VPO+f1xh4FhgglkZu6U8/SRx8M5c+/n836QybzvOZ9z3u8OAimpeU2yjTI9kSQKsAyDXKMypTWeJwOGawVWJroQYnY3RelE2yaWZdLZo5cwZ0o+V+A9bzxr8njPL370awzT4Jx/+QDdvV27XELWNG3hMBAMVwsYQrA+vYxa4DJcK9IfS1P1mzy3ax8ETHT3yUxWfVK0kuquzsw256ipgF461haFsZHxaWdor7/2RirlSpsi0jRtJgkhSDtRuiNJwrbD5tIYUcsGoDMUwzRMpFIoBf3RFFHbQU4sDpsIuiKJaYd5lsUysz6bBT2j1RaJdEdq2tjK1SuwHbsN0YDneRRyRXYODtPZ3UEilSCZSuz+jZq2BGTHcgRBgGVbdHTu2aMc0zBIhiL4slWcQgG2sFBKsSrZTSAlCOiNpRivleiOJDCECSgQsLNa4KCOAXZU80gU/dE0lqGP92jaHksk4vzTR941+ek0lU7yL5/9Z1LpZFvieWzjZl7/orfyzte9j9cceyo//K+fUC7qWsza0hYEARsffJQzTjmLE496PR98x0fZvmVoj98vFShalaAc0yJsOxhC4EmfRuCipGR7eZyOcByEQCo5MYsV7KwV2FQcIWaHSToRtpazMEfbocSTS9jNlCOPPFLdfvvtM35dTft7yqUKlXKFcrFCR1eGTGd7NpkVcgXOPv087r97w5Txa/7yS/oH+uY8Hk2bL8ZHs7z9Ve9hbHh8cuyAg/fnWz/+8h7NbGteEwVsKo6wJtVLya2TCkXJ1StkwjEMIRis5OiPpTGFQdltkHDCSAWPFIcpufXJa0Uth0M6V8xYGUYhxB1KqSN39T29dKwtGolknEQyTn+bT+/4fsDQ9h3TxsvlCv1tiEfT5otmozklyQI8dP9GfHf3m5J8GVD1msSdMBWviVSKLeVxDnNW0hGJI5VCKkVPJEW+UaUrkiTuRFBKkW9U2D/dx1AlR9hyCJk2STs8Z7WO9dKxps2weCLGy1/z0ilj6Y70Hj+L0rTFygk50x7nrFy9AtPa/cpToBRRO4SUkgMzyybGWjuJC40qhhD4SmIaBvlmDVAE0scQkApFkVLRF01TaFbZWS2QbVTw9K5jTVuYwpEw7z3nnbzjzDfTv7yX5x13JJf96hIynel2h6ZpbZXuSPGlSz8zmWy7ejv5wrc/ReceVGsTgJQSyzAxDRNX+iyLppEokk6UQElqvkvD90g6ERRgChNfSQwhEAKyjQorE130x1LEnTBlr8FsPD6dFrt+Rqtps6PZcCmXyjghR+841rQJj+/IbzabhMPhPd5L4QU+UrZ2EG8tj1Ny66zvWE7Va5B0WruRS16NpB2lHniEDBNTmFiGQaFZJWw5FN3axCaoVuI+ILOMlBPBnoEl5L/3jHbRzWgr5Srjo1nqtfruX6xpsygUdujq6dRJVtOexLZtunu7GFi5nK6ezj1Ksm7g80BuiGyzglIK27SoBx65epm67wECBaScKKP1Eg/ld2CI1qwXAY5h45gW2yaSLLT2Gz9WGp08azubFlWi3TE4zEUf/jT/+Or38sWLvsb4aHb3b9I0TdPmtZrfpOw1Wku9ArojCWzDJBOJ0xGOAQpfBgzXilOaDkQsh0C2GgioibO3T9YMfMQc9KRdNLuOs2M5znr7R9j62HYAfvPza8hlC3zqa/9GQs8oNE3TFqzHSyWGTIuG5xKxHA7vWkm20aoG1xmKT7TOS2IIA0/6jNbL2IZB2HTIN6tEbIewaU+pd9wZjmPMQWWoRTOjrdfqk0n2cTdffyuNRrNNEWmapmkzIR2KIhB4MiDqhMg1qzQCD18GdIRj+ASknMhko4EtpXFSTgSpFEIIpFJYwmRdpp/OcJywadMbTbFfqmdOqkMtmkTrhByckDNlrLe/B8OY/U8rmqZp2uyxhMnh3StbXygwhWC42iqz2Aw8DARRy8ESBqYQmMIg6USIWA4oRdR2GK0XsYRBTzjB2nQvK+Od2LoE4zOTSMb55098aLIEn+3Y/PuXLyCjzy5qmqYtaKZhELfDrE5240lJ1AoRsixqXpOQYSMQ7KgUiFitRgKrkl1sK41jCgFCUPWaxOwQGwvDFNwabuATKIkh5iYFLppntJFohBNf93KOffExjA6P0T/QRzKdxDAWzWcJTVsQ8rkChWyBRqNJd18XXd2d7Q7pGWk2XGrVKrF4bNoqmdZeUipMQ1BsNumLphgs50iGolS9Jnm3yoDooNpsYAiDlYlOqr5L2LRJOhE2FobZL9WDaRigWpWm1MTS8mxbNIkWWhV54omYrieraW2Sz+a58MOf5tYbbwOgf6CPy351Cb39PW2ObM+Mj2b5/n/+mDtvu5vnHXcU/3jmW/aomII2+9zAp+TVafoeCSfcaocXT08my0wohicD4k4UqQIagd/6HoKm3+TAjmU8kBui5rsA9EfTRCxnTsow6umepmkzZtPGzZNJFmDn4DA//u7P8Fzv77xrfijkCpz//gv56WW/4OEHHuGH3/4Jn/zY5ygVSu0ObclzJzY5ld06mXAMU5iU3Drj9TK+kriBT180NVHlSSGEQb5ZIWaHEUIQcyJsr+QmkyzAzloBV5dg1DRtoXnqzn+AzY9uxW26u3j1/NKoN7nrr/dOGbvlT3+hUdcnF9qt4XugFN3RJFIpmtIl16yQCUURwM5qnrrvMVwr0vBdSs0ahjAwAKkkvvSnJNnHVf25+btdVEvH2sLSqDfJ5/Lceds9DKxcxsrVA3rz2gL3vBceiWmaBEEwOfbaN7+KWCLWxqj2jGmahMOhKUcCY4kYQp9caDsxUau44XlYE+3vYlYIEJTcGusyy6h6DTrCMcKmjW1aSKXwVYApDKQSpEPRKW3yAJJOZE7i1zNarW02Pvgor33hm7nwnE/xrte/n3//6MXkc4V2h6Xthc6uDr79s6+x/pB1rFg1wHmf/AhHH3tEu8PaI4lUnLMveP+UsY/9+9mkMsmneYc2V0KmhQRsw2RnrYBj2mTCcUxhYAqTLaVRim6DqOUglWK8XuaRwk4sYQECyzBIO1H6oikMBI5hsT6zbM6O9+imAlpbFHIFznnn+dx31wNTxq/8049ZvXZVe4LSZkw+W0BKSbojtUe1bOeLcqlCPltg08OPsXb9GtIdKeILYDa+FLiBR6AUm4qjDMQy2KbFpuIoa9O93DW2lcO6VrQKVCAYrZcIWzYJq9VzdqTeOnPrSR/bsFAKhICQac9YZSjd+F2bd4JAUsgXp41Xq7U2RDN/FfNFSsUyxUKJ3mU9dHRmFkTiWqgtARPJOIlknJWrB9odivYUjmnjBwFrUr1IJRlvVChOnIldk+ql4XsYwmBHLc+KeOdE8wETBQQyoOI18WXA5tIQgZIk7DAHZJYRtuxZj10vHe+hYr7I2Oi47go0Q1KZFKee/oYpY129nfQt621TRPNPIV/km5/7Dq897i2c9pozOfWlpzG4dUe7w9K0tjENA0MILMOk6jVaS8eGQcSyGaoWCJmt2er2cpaIaVPzXASQDsUImxaPFkcmm8WXvQabS6P4Mvj7N50BOtHuhu/7PLZxMx95zwW8+YR38qVPfJ3sWK7dYS14lmXyqje+gk9+9V854vnP4Q1vO5nLf/sdfWbxSfLZAr/6yVWTXxcLJb7yqW9SKVfaGJW2EJSKZUaHxxgbGafZXBy7pgMZUGhWeaQ4Qt136QjHOSDTz1i1hC8lRbeGKQTLYxlyzSqBanXt8ZUkajm4u0ioJbdBMAeN3/XS8W4UckXOOOWDk8ucv/7Z/xIoxXn/cQ7RWLTN0S1s6UyKV73xRP7hZccSCocIhUPtDmleGR8ZnzY2uGWIZtMlrhtSaU8jN57nMxd8kRt/fzORaJhz/k8frdMAACAASURBVOUDnPjaly34LmauDLg/N8S+yR42l8ZYEe/AlwFjzTJ98TQdThRfKVzp8ZyufVAoTGEABr5sJdunSjmRVpnGWaZntLtRKpSmPUv807X/H7WqXkKeCUIIkumkTrK7sM9++xCOhKeMvfL1LyeV0rtgtV3zPZ9f/Og33HDdTSilqFXrXPyvX14Uq3CFZmv/hm2YeDJgtF6i4jVbHXukZG26n2bgYRsWUkmUgkBJFIoAyXi9wgGZfqyJ+sYJO8yqZLfu3jMfxBKxafWS99l3BaY1/zekaAtbpiPFD37zbZ5z9KH0D/Tx3g+/kze+/bVYtl6I0natWqnyfzf9ddr4A/c+1IZoZlbUckiHoiScMMujGQbinXSEY8SsEIYQuNInZNoUGlVqvodlGAhhYGDgBQE1v8l4vcz6jmUc2rmCgXjHxIx39ulEuxvxeIwPffyfJgtPx5NxLvz8+WQ6FuauysWoWqmRHcstugo+tmOz/4Fr+Mr3Luby336HMz50+oLdzavNjWgsylEveO608fUH79+GaGZWxHQYiLYK2mQiMTYVRwibNgdk+ql6Lr4KkEqyLJ5hU3GEmu+iUEgktmnSH0sTs8Nsr+QouDXidhh7jnbw64/GuxFLxHjjO17LK05+GcVCic6uDGmdZOeN4aERvvrpb/HgfQ/zguOfz3vPeeei21CVyqTaHYK2QNiOzZvf+Ubuu+sBbrv5dkJhh3869z109SysDkq7ogSU/QYhy6bo1lmb7mNHtUB/NI1jmggEg5U8KxMdSBRbyuMcnFlOoGC8XiJk2nSEYnSG45TcuT1GqAtWaAtWdizHe9/0QbZs2jY59tKTjueiL36cRDLexsg0rb0KuSKNegPDNEgk40Sic1NqcDZV3AauDAgZJq70cUybitcg5UQZrhbpjSbwlEQqxcP5nbjS55ie/Sh4dWJ2iM2lMbKNCo5hsSbVQ8wOEd7FBqln6+8VrNBLx9qCVa81piRZgBuuu4lGvdGmiDRtfkh3pOhb3ktPX/eiSLIAtmniGAZSKcKWg1Ryolaxoi+WIu/WeDi/Ez/wObhzOd3hBEooopbDvePbWBHr4Mie1RzSuYJAyTnZBPU4vXQ8x6qVGtVKFSUVkWiYZFrvIH22nJCNZVv43hOtrnr6uuekkbOmaXPLEibSaJVYVEpiCoOm7xGxQlTdBnE7TCPweLCwkwPSy1iZ6AIECjisax+yjQrDxQKWYbI62c1srOY+nd3OaIUQK4QQNwghHhRCPCCEOGcuAluMCrkCl37t+7z6mFN45fPewCfP+xy58fys3S83nuPqX1zLlz/5TR66byPl0uIqdBBPxDjnX54oAm9ZJhd+4Ty9YUjTFiFPBlS9Js3Aw1cSgaDiNQBFxHHwZMCyaIbDO1cQt0OMN8oopbAwKDSrPFYaxTEseqMp6r6LnMNEu9tntEKIfqBfKXWnECIB3AG8Tim14eneo5/R7trdf7uXd7/hA1PGzv/0uZx62utnfBaWy+Y55/TzeOCeByfHvvDtT/PSk140o/fKjuXYtHEzgR+wdv1+c77polyqUCqW2bF9JytXryCZThB5ytlTTdMWtkBK6r7LI4VhDuxYTq5ZpeI26I+lMYTAwGBrZZwV8dZGyFbzgRHWpZeBUGwsDJNyooQtm6FKDqkUA/EOusJxbHNmFnb3qqmAUmonsHPi/5eFEA8Cy4GnTbTart152z3Txm67+W+89tSTphUm2FuFbGFKkgX4ry9/j+c+71A6umZmV+74WJYz3vhBtm1uNfvu7uviR1d/j56+rhm5/p54vAj88hX9c3ZPTdPmVqAknvRZl+mjGbTOyPbGUhNVnQQSyfJ4hkBJDGEglcQ2TJrSJWI6xKwQ6VCU+7LbJ6/5aHGEiOWQnqFE+/c8o81QQohVwHOA23bxvTOFELcLIW4fGxubmegWmaNfOL0v53EvfcGsVEXa1bKIDAJmcrXkluv/MplkAcaGx/nNT6+e02cfmqYtfo8XlmgEPhuLI8TtMG7gkW1UsQwDNwhQSlF069R9F1cGCMCcSHErEh3kG9MfnY3UinOyhLzHiVYIEQeuBD6slCo99ftKqe8qpY5USh3Z3d09kzEuGiv2GeADH3sv4XAI0zQ5+dSTOP7lx83K5p2Ozgxr16+ZMvbec945o88vh4dGpo3tGNyJlHLG7qFpmtbq0BPCkwH7JDpp/cYUbCmPI5UkbFnUPJekHWFLaYyIabMi0TnRqUfR8H1Cu2iHF7NDzMXWyT06RyuEsIH/BX6vlPrK7l6vn9E+vUa9QblUQSlFLBYlNotNpbNjOf54zQ1sfOARXvvmV7F67SqSM1hYfOumbZzy0n8kCJ7oivHja77HgYceMGP30DRNA3ADn+FKgZ5YEhAMVnJUvAb7JLoImxZyoonAjmqBkltnVbKLmBlCCdhaHqc7kmBLaZyy1zr+F7UcDulcgTMHz2j3ZDOUAC4HckqpD+/JDXWinV+UUrMya65X62zauJlLvnQpvudzxjnv5MBDDiCenL0PD5qmLT2BlFT9BmEzhCd9lFJU/SagSDpRPBlgCQOJouF5xJzWTFUpMIRBrllha3mc/VI92IaFQuEYFjF75h7b7W2ifSFwM3Af8Pia4L8opX73dO/RiXZpKZcqKKlIphd2Gy7t6blNF9MyMZ9SG7ZcLLNjcJib/ngLBx62nvWHrqOjM9OmKLXFqhl4+EGAGwREbYd8s0bcDuFKn6jlUPNcJIpCs8bOWoGucJxViW48GUzOWB/M70AqiSVMBLCuo5+QOX05+dna213Ht8CcLGNrC5Qud7h4FQslNtzzEL/+6dWsOWBfXv/Wk+nubR3h8v2AG/9wM58497OTr3/xif/AhV84n7Suz6zNIKkUtcCl6jWJOQ6WMCh7DVzpEbUcQpZN2a1TD1z2TXbTEY6Daj3blUrS8H3WZ5bhqwAUWIaBJeauMpQuwahp2i4FQcAN193EWe84l/93zQ18+8v/zZmnfnCyt2kxX+Cbn//OlPfccN1NulezNuMEgqbvk7QjNHyPuBMi6UToCMVRAEphCEFvJEnSiRDIVh/aTYURFIqY3Sposa2c5b7cII8WR3Glv7vbzhidaDVN26VCrsgPLvmfKWNbH9s+mWiVAq/pTXuffNLmOE2bGYqI7RCzQzimzUP5HRgILMPEDXxMwyBk2qScKKYwEUIghCCg1QDeV5LHSqNkGxUMBA3fZUNuiIY//ed3NuhEq2naLgkhdnnG+/HG88l0knec+ZYp3zv48AOJxqNzEp+2dAhau4R9FaCUZF26n521Ap4MqHgNpFI4psVgJYcQrQ2go7USa9N9BEoigIbvcXDHAAd2LOfAjuWs71g2Z89EdVMBTdN2qaMrw9kXvI+zT//YZBGS5xx9GJmJfsyOY/PGt7+WffdfzbW//gOHPvcgXvn6l+vNUNosEAgFJiZVv4EChqp5LMNAqtbScsmtM1wvsjyeoezWSToRco0qneE4hjBYn+lHCIOt5XHyzSoRy2Fdug9bWRiz3IhE96PVNO1pVctVRkfGuOG6m9l37SoOPeJgOrqmJ1K36WLZFoahF8m0mecFATWv1fR9qJLHV5LRegmB4LCuFVjCIO/W2FQc5aCO5UQm2uiBwBSCuudimSbbKjlyT6oQZQqDI3pWzcju473adaxp2tIVS8RYnYix+oOr/u7rnNDMNdDWtKeyTZOwcpBKIVGt8691UChQIFF0hGIMWw4hw2KwnGO4XiRsWBzcOYAwDIQQFJrVKdcNlMSXktAsb0DWiVbTNE2b9yzDxJcBvdEUljAohGrkm61axw3pU6yUObBjAIViuF5kWSxDbyRJI/CJTJRfjFmhycpQ0Hr2a83BKoxe59GWnHy2wL133s8frr6e4R0jNBvNdoekadpuSCUpNKuEhEWuUWVFLMNRPftiGgZu4NMZjePJAE8GrEp0YU5UhIpYNoFSNHyXteleHKM1fRUI1qb75uQ8rZ7RaktKPlfgE+d+hlv+9BcAbMfmsl9dwkGHrW9zZPNbbjyH5/nYtjVjbRY17ZkIJrrzpEKtXe2WaVH3XaJWiLgdJt+sknTCCASpUJR7xrexb7IbgUAIMAyTHZU8B3UMAKAA2zAw9YxW02ZWbiw3mWQBPNfjK5/8JsXCtIZUGiClZNPDj/HeN32IVx79Bt731g+zZdO2doelLUEKxbJYBgVkwjGUUmwqjdKUHoVmlYrXwBIGvpST52MNWudplVKU3TrD9RJ3jW/lrvGt3D2+lcFKfmLT1OzSiVZbUirl6rSxQq6I789dlZiFJJ8tcM67zmfzo1sBePShx/joGReQHc+1OTJtqTGFIFCtAhQCGGuUqfsulmgVq0jaEQwMPBkQsmxCpk1HOI6UCsswJlrmTfV4ScbZphOttqQsX7lsWk/eN532el2b92k0m012bN85ZWzzo1vx3LmpqKNpTxDYhsXOagFTGFOqOsXtMLZpcn9ukIQTJlcvc0Cmn3yz2srKCrojyWnnZZfHOubkSJp+RqstKZ3dHfzo6kv5zlcvY3DrEK97y6s57mXHTutKo7U4jkNXTyfjo9nJsf6BPixL/+rQ5k5r6be1NNwVjqGA3miSkltHCEHRrZNrVDCEQdVr0hNNYgoT7Mk8y/ZylsM6VzJUbS0X98cykxujZpsuWKEtSY16g2bTJZlKzEqv3sUiCAI23PMQH373+eSzBbp6Ovn697/AuoPX6uIU2pxxA5/t5Sw90RSOYTJUzZMJxWgELulQjO3lHLZpMlTJc2T3KprSxxImVb9JzA4hleLu8a3YhklXOIEhBMVmjfUdywlbM9MqTxes0LSnCEfChCPhGb+u7wdUSmUi0Sih8MIv4mCaJusPPYCf//4HNJsuoXCITGdaJ1ltzvXH0tR8F8cwkUpyf26QgzsHCKTElz59sRSdThw58dA116wQNp0pZ2U9GbCzVgAg6UR0rWNtZtSqNSqlKq7nEYmE6eye2aMZSilKxTKO4xCJznziWkjy2Ty/+dn/csN1N7P+0HW850On0dPX3e6w9pplmXT1drU7DG0Jsw2Tpu9iGSZSKXqiKYpuHS8I2FjaySGdK6gHHhHLIQgCar6LJwO6I6GJzVOCVYkutpbHJ471mOyX7JlsCj/bdKJdxCrlClf/4jq+9pn/xHM9Vu23kkv+56v0Le+dkesX80Vuvv5Wfv3Tq+kf6OP9Hz2D/oG+JTnbqdXqXPLFS7nyf64C4P67N3DnbXfznZ99XZ871bS9JIQAIWh4TQxbMForcUBmGWMTHXrqvtea6cpWd5+EHSZuh1GAEDBWL9EdSdAZjuMriSVMzIlWenNh6f1GXEJKhTJf/MTXJneIbtm0ja986ptUK9OPuDxTQRDwx2tu4KKPfIa7/novv/vVH/jHV7+X3Hh+r6+9ENUqNa7+5XVTxjY9vFk3Qde0GRIybeJOmGyjgicDSs06PdEkO6p5XOlT8RtYholt2kglUUq1/gDpUJSH8jvZUh5npFZkuFbc5XGf2bLoE20hX6Rcquz+hYvQzqGRaWMP3reReq2xi1c/M8V8iSsu/9WUsUK+yNbHlmYxAyGYbB/3OMMwsJ2Z2WihaUudY1rYwqQrFGd1sgvHslBAXzSNAcTtCErBSL2EovVc1hACA4Pxepmy1yDbqDBcKzJUzeHKuTs7v2gTbblY5qY//pkPv+t8PvZP/8YDdz9Ivb73CWYhGdhnGZY1dfv6MS86mngivtfXtixrWmIBSKYSe33thSjdkebjnz53ylLU29/7ZqIx3QRd+/sKuSI7B4cZ2TFKqVhudzjzWsiyEQKkgrFaCYXCkwHjjQpu4CGAQrOKJwMErcpQvgqo+i4AYdNm32RPq6PPnG2FWsSJ9pGHNvHhd5/PvXfcz19vuZ13vf59ZJ90FnApSKaTfP0HX6S3vwfDMHjxK/6BMz/yLsKR0AxcO8G5F31oSnu0Fxz/PLp6luamGdM0OerY53LVn3/O5y75D6744+W8+6x3kEju/YcabfHKjee54IOf4FXHnMJJz38jX/73r5PPFtod1rz1eI/Z7ZVxYnYYW5iU3Dr5Zo2QaRMoSW80xYP5HTSl35rZIugKx4laDusy/Yw3yjyQHWRreXxK0YvZtCg3QzWbLj/7/pVTxnw/4IbrbuIf/+mtbYpq7kUiYY5+4RH86H8vRSlFOBKe0V/8q9bsw29v/hn33HE/fct6GNhn+bSqS0tJNBYlGouyfMWydoeiLQBKKf7wv3/itptvn/z66l9exyvf8Aqef9xRbY5u/lFKEUjJcK3AeKPCEd1dbCtnSYeijNZLFJo1Sl6dvmiK/ZI9KNXqNyuEIGI5rEn18khhmNrE7Lbo1tmQG+LgzoFZ3328KBOtaZosX9k/bbx/oK8N0bSXaZp09XTOyrVDIYfe/h5e/uqXzMr1NW0x81yPO//v7mnj995xv060TxFISdVvTs5A5UShpeF6kc5InK5wAlf6COCB3BAdoRiZUAwx8dr7soMc3r1yMsk+ruo3J681mxbl0rFlmbzlXadMSTBrD9iP5xx9WBujWlry2QK3/OkvfO0zl3DnbXdTyBfbHZKmzStOyOFlr3rxtPFjX/z8NkQzv3kyoNCsMVov0RNNIRAoIG6HeDC/g7gdoi+aYkWis5VcaXUeKLg1DGGwPJbGwMB6SslFxzCn1T+eDYtyRgvQ09fNT669jE0bNxMOh1ixark+zzhHysUyX//sJVz7mz/y8le/hEce3EStWuPIFxxBOLz3z4c1bbE4+tjnctr73sYVP7gSy7F537nvYWCf5e0Oa96p+c2J3rMOg5Uch3atQClYk+qj4tbJhOPUfRcEHN69D5YwAAFK4UufnmiKQrPKunQfW0pjRK0QzcBjn2Q39hzUO9a1jrUZN7JjlDe+5B189bLP8ec//YW/3PQ39lu3mg+edybLVkxf0te0paxRa1CpVABBMp3E0UfCpqn7LhtyOzi4czkb88OUvQb90RTLYhl8JRmrl+iJJHmkOML+qT4qfrPVTEBJ0qEoI7UyUduBib3G2UaFVChC0o4QmoNaxzrRajNueMcIP//+leRzBa664neT4ytWDXDZry6Z8TKQ2twr5ApUq3VQimg8usujXpo2UzwZ4AY+Na+JpwJiVghDCJqBT8wK4avW902j1ZvWlwGeDIiaDkIIar5LzXdxpc9g5Yleyt3hBPule2dkVvv3Eu2ifEa7mNWqNcZHs/O6CEckGuGlJx3PH666fsr49i2D1Kq1NkWlzZTWkZR/5zUveBOvOfZUPv7+i5ZsRTBtbthGq2TizlqBTcVR7s1uZ7CSp+I1MITADXwMIXAMC0Fr3jpSK+IrhUIRtiySTpgd1ak/p2ON8pxUiNKJdgEZGxnn0+d/gTe//HQuOOsTDG3b0e6QdimVTtLd20VX79TdzoZhTDl3qy1M/3fTXyePpAD87dY7ueVPf2ljRNpSIBBTNjPV/CYRy0EIJpd/pVJUPBepFDErxOP7nASiNWvd1QLu7G861ol2oSgVSnzi3M9w3W//H/lsgVtvvI2z3vFRsmO53b+5DXr6u/mXz35sSkP1097/VmJxXSlpobv3zgemj91xfxsi0ZYShWJFvHNyl3DNd7ENE09KDFpnZRuBh20YWIZB1HYwaCXfHZXWTHZZPDPlmp3hOKbedaw9rtl0+b+b/jZlbNvm7fO2rKQQgsOOPJir/3wFjzz4KAOrBujoysxI+UetvV5x8kun1bk+8XUntCkabakwhEHDb/Dc7lWU3DpRy8EyzCdKKSpJ3A4zWMkRtmw6wnEMxMTSsYMnA5JOmP3TfRSaNeJ2mM5wHHsOWuXpGe0CYRgGfcumtrcLh0PzeodiJBqhb3kvx73sWFav2YdUOtnukLQZsO/+q/n4p8+ls7uDTGeaj/3HOaw9YL92h9V2lXKF8dEsRX1mfFY4pkXSiaJoNW0vunVGayVcGeBJn7xbJ1AST/qETZtsvQwIlIJUKErd93isOIYhBMtjGeJ2iNFakUAGsx673nW8QARBwJ233cPZp/8zzYaLaZpc9MWPc8KrX0w4srQbrmtzz/d9CrlWQklnUlj20l4cGx0e4/P/9hX++uc7WHfw/lz0hY+zYtXyOet3ulQUmzWqfpOo6VAPPGpek95oElOYVPwGNd+lO5yg7DVI2GHuy27n8K59UEDRrRGxHDYVR6l4jVbz91QPKSc6IyUY9fGeRaLZaFIqlBkbHaezq4NEKq67w2hamxULJc5/34X89c93TI4tW9HP5b/9jj7KNoPcwOfB3A7WZfrYVsmBUvTH0uysFlgR70QIwVAlx/J4Bl8G5Jo1hqsFDuxYjmkYoOCx0ii55hP9uAWCo3r3JTTLiXZpfwxdYELhEN19Ibr7lmaHHE2bj9ymOyXJAuzYvpN6rd6miBYnqRQrEq0PLoVGhQM7B7hrbCsAA/EOdlbyDMQ7MIQg79UwhUE6FMMyTNzAxzZanX6eTKHwAn9GEu3foxOtpmnznpSS7FiOO/7vbmzb4rAjD5m1ZhnPlGEYLF+5bMpxu3AkrI+y0Tr3X6vWEUKQ7khNOYXwTAnAMSxAsE+im6HKE2diFdAdTRIoiZo4yhO2bBJOGKUUjmmSb9ZIOBHyT5nR2nsR057SiVbTtHlvbHict574rsnmFP3Le7n8qu/Oi2Tb0ZXh01+/kA+8/VzqtTqWbXHhF84nmVram//y2TyXfPFSrv7ldWQ60lzw2Y9y5DHPedaPuxQKQ7T276ZDUUpea3YaMm1MBJJWBaiEHSZkWFjCYKhWoC+aZEc5z4pEF3E7jBv4VP0mljBYm+7DEjrRapq2xEkpueKHv5rSAWrn0Ag3/uFmTnnH69oYWYsQgvWHHMBvbvoppWKZRDJOIhEnHFm6DTR83+dXP7maK//nKgBGdo5y7nsu4Ko///xZJ1qBQAgFKMpuk/5omrBpE7VDgMIPfBJ2mJJbJ+lEGG+UMYWBQNAXS1P1mmwujbIi0UnYtJEois0a6dDs73PRiVbTtHlNSkU+W5g2Pp+O0Tghm+7eLrp79f4JgHKpwg2/v2nKmJSSDfc8xLKBZ99YxEBMtsALpMQxLTbkhjiqZzUhy8YNfFzpAxCzw5PPXjcXx+iNpqgHHhsLw5PXS9hhlscyu7zXTNLnaJeoIAgoFkq4TXf3L9a0NrIsk7e8+5QpR2Us2+IVJ+siGfNVJBJh/cH7TxtftWafZ3U9XwZ4KqDs1RFA2LJxLIvtlRz7JrtpBj71wKPs1umOtI73+DIg36gigKJbJ2za05oHDMQ75qRghZ7RzpJyqUK9VscwjHl3zjCfLXDtr//AH6+5gQMO3p93nfUOevq62x2Wpj2tgZXL+OHV3+Wyb/4IJ+Rwxtmn093b/uez2q6FIyHOOOdd3PnXe9n8yBYMw+AdZ76Z7mf5TD1QEqUUQrSO6ZjCQCqJKQwSTgQQCCVJhWM0Ax838Kn4TRq+SyYcI+NE8aXk8K598GWARGEb5kTf2tmnz9HOguxYjov/9cvccN1NpDIpLvjMubzg+OcRi8faHRqNeoNvfeG7/OR7V0yOrV67ikuv+AYdXfPjzN/4aJZbrr+VXLbAy1/zErp6u3TDeA2AWq2OIYQu0rJA5MZzVKt1HMcmGouSSD67EqzNwKfiNWj4LrYwSIaiSKXwZUDN94hYNmHTZme1QFckgQIqXgNfSbpCcYSAbeUsXZEED+eHsQyDQEkOyPSTdKKT9ZP3hi5YMYdc1+O/vvQ9Lv+v/5kcE0Jw9Z+vmBdNz8dHs7z2uLdMO+P325t/zopVy9sU1RPGR7O86/XvnzwqYdkWP/ndf7NGl/jTtCUrkJLRWpFUKErZa5AJxRitl8iEYvgyQKEIGTaeapVTtISJLwOEaB0JGqzm6AjFKTRrdEbi1HyXiGm3lpqjqVkvWLHbebMQ4jIhxKgQQrfn2APVcoVbb7xtyphSikce2tSmiKYStM6zPZXtzI+l7QfueXDKeUTf8/nOV78/L/vYVis1hodGuPO2exjZMUqtqgsUzCf5XIGH7t/IjX+4mZGdozT1foQFyzQMuqOt41LFZg2pJA3fw5cBbuARMR0C2RrbkBtCiNZyc2uJWVHxmoRNm4hlc8/4Nh4pDHNvdjsShZon/Wh/AJw4y3EsGpFolEOfe9C08dXPchPATMt0pTn/Ux+ZsrHkTae9ft6UcvRcb9pYs9FkNlZe9kaz0eT6393Iq445hTNOOYtXv+BN/PWW2/E9v92haUAhV+Bz//oV3vbKd3Puey7gtce9hcc2bm53WNpeULQ2Ra2Id1CdqHFc913iTgRX+piGIN+s4smA0VqJiOXgBq1EnAnFEEKwpTw+5Zrbyjlg9utR7zbRKqVuAuZn09N5KBwJceZH3sX6Q9cBYDs25/zrB8h0pNscWYthGBx5zHO46paf85lvXMRPrv1v3v/RM0imEu0ODYDDjjyE9JP+WwkhOOPs0+fF8+0nKxZKfP7Cr05+AAiCgE+e93kK+enHUBaqcrHMhnsf4quf+ha/+9UfyI3nd/+meaKQK/LH//3T5Ndu0+WLn/j6lLO42sIiaG2CQggKbp2a3yRqhVpLxwpG62UyodbviWyjQjPwCFsWtmnRGW6Ne0/p1KMm/jfb5sd64SLT3dvFN3/4JRq1BpZtEU/E5s2MESAaixKNRVm+clm7Q5mms7uDn157GVf88FeMj+Z467tPYcWqgXaHNY3v+dOecxdyBYJg9peh5kIQBNxyw//xrx/6j8mxw486lC9f+hkynbN/7nBvlUuVaWPZsRyBP/st0bTZYRkmgSkJpERJSdltEgrbraIUAkbqRdam+uiLJCl5DVzpYyCwTBOloOzV6QjHyTWe+NmI2aFW8p7t2GfqQkKIM4EzAVauXDlTl12wOjozoE8fPGOGYdC7rIezzjsTKSWWNT8/C4ajYQ44eH8eun/j5NjRxx6xaHZHa+TH4gAAIABJREFUF3JFvv3l700Zu/tv91IqlhdEou0f6CPdkaaQe2KF4Q1vP5lUZmmXRVzIvImZqxCC3lgKY6Lqk6C1D+bAjuU8WhihIxTjoEQnEqh7LrZpIoBHCyMc2rWSsGlRaNZI2BFWJjpnpEXe7szYHZRS3wW+C61dxzN1XW1pMgwDw5i/9VQ6OjN89bLP8fXPXMI9d9zPUS94LmeddyapzPSNZgvVrmbnUi6Mf9odXRl+eNV3+c8vfIftW4c4+U0n8fLXvGRefXCr1+qUSxXGhsfo7usmnogTjUXaHda85MuAbK2MaRjEnTCPlcZIO1E6JpaKLdPEVIK16T4MIRis5Cm6NQ7sWD5RuhGidmuZuSMUozMUxzGtOXg62zJ/fuo0bYHp7e/h3z5/HrVqnVg8SiS6eH5JpjtSvOdDp/Gp8z4/Obb+kHWk0gtjRmgYBgP7LOPCL5yP23RJpBJ71Tlmpnmux20338557/s3fD/AskwuvuSTHPeSF+CE7HaHN+/4SuIjGa4W2d/pp+TWiZg2CLANC4Ui36iSicQQCEbrRdak+jAUSKEwEOyf7uOB3BB1v7X7XACHd+1DiNn/773bc7RCiJ8CxwNdwAjwCaXUf/+99yzlc7SatlgUCyUeffgxrr7iWtYfuo6XnXS8bmQ+Q8ZGxjn1ZadRLJQmxxKpBL+8/ke6XvIu1Lwm9cBlvF6mP5bh/uwgEcvhgEw/XhAQMi0U8FB+B+sy/TT8idMLSiGEwDEtKl6Thws7p1y3IxRjXaYfy9j7D2F71fhdKfXWvY5A02aA53kMDg7SaDTaHcqSEU2GOPU9r0UIwej4CKPjI+0Oaa+Ew2EGBgaw7fbOGh+vNf5k5WIZ39fHw3bFMkzCOOyT6AYU+6V6iFgOOyo5+ieaAozWi0ilaPoeYcvmjtEtrE52kw61Kj/JXZyXlXN0bFAvHWsLxuDgIIlEglWrVk05B6xpe0IpRTabZXBwkNWrV7c1llAoxMGHH8j9d2+YHDvwsAMIh3VpyV0xhUEgJJ70Kbl1QqbD9nIWAKkklmFQ810agYdtmNR8F4VCAJYwcGVAwgljG+aUIz4D8Y4Zmc3uzvzdbaJpT9FoNOjs7NRJVntWhBB0dnbOixWRTGeaL37305PL8S955Yv48nc/S6Zzfpy3n28M0dpdbAiDstvAFIKy18AyTMpeE19KusMJAiXJNqpETJvuSBImzsj6MkBJOKxrJctjGbrDCQ7tXEHCmZsPNnpGqy0oOslqe2M+/fz09vdw0ZcuoFFrEI6GiSfmV1GW+USI1kGeQPp0R5JYhkHaidEfTSGEwJOSmB1i32QPbuBhIBiIZRiuFelS4JgWzcBHeS7LYq0PMwZiTmazrXtp2hKgpJqxMo7XXXcd69atY82aNXzuc5972tf98pe/RAiB3hioPZ14IkZXb6dOsnvAVxIhBDE7xIO5IVYmOhipl/ClJGSaFN06XZE4CSeMovVcN2o7SBSFZpWIZbOzVuL20c1syO2gKYM5K+2qE622qAV+QLVSY8fgTrJjOby9rEUcBAFnnXUW1157LRs2bOCnP/0pGzZsmPa6crnMN77xDZ73vOft1f00TWsxhcHD+WECJWnKgLrv4gcBUikCJYlaIQYrOWJWCF9JxuolBK3ZcMi02VQcJd+sooCq3+SB3OC0koyzRSdabdEaqRb529hj3FncxhZRYltujK2btu1V4f+//vWvrFmzhn333RfHcXjLW97Cb3/722mvu/DCCznvvPP05hZNmwFu4BMoyZpUD4YQmMIgbDl0RRLYZqslHijG6xUQgqFqju2VHDErjEAQt8Pkm9Up1/RkgD8HnXtAJ1ptkRqtFXm0OIKnJAiBNAXVBJRw92pWOzQ0xIoVKya/HhgYYGhoaMpr7rrrLrZv386rX/3qZ30fTdOeUPWalNw6vpQopTi4Y4BCs0rUbpU8rfkubuC3+soqhS8lgZJsL4+12gYoScyaWh5VIDDn6Jm93gz1LFTKFarlGoEMiEQj86Yzj/aELeVx5FO7cghBPabYm39bu3qm8+QNNlJKPvKRj/CDH/zg2d9E07QphIDHSqMc1rkCIQRVv0muWUUgyIRi5BpV+iJJ9kl2ESjF8liGvmiKhB1GothcHmffVA8P5ne0GsIjWJvqxRJzsxlKJ9pnqJArcOnXL+fnP7gSKSXPed5hfOG/PqUr5swzzWDXs1ZpiL2qdzswMMD27dsnvx4cHGTZsie6IJXLZe6//36OP/54AIaHhzn55JO58sorOerIo7DshfFPTimFDCSGacyrnbra0hS1QvRH0ghhUHLrSClJ2GGidohCs8o+yS58GVDzXLAcLMPE9fyJD8YKqSTbyuOsz/RP1D4WWMLAnKN66nrp+BnavnUHP73sF0jZWtu/67Z7+NVPrsLX7bfmldDTdORwTGuvkt1RRx3FI488wubNm3Fdl5/97GecfPLJk99PpVKMj4+zZcsWHnvsMY466ii+8dVvkYpl2PrYNlzXfdb3niu+5zM+mmX71iHGR7O6mb3Wdo5psSyewZMBjxZGsE2L3mgKUwgSToRHi6M0fI/OcAxfBSgUCSdMI/CoBx6rkt1UvAb3ZQfZUh6n6jaw57BpiU60z9CGex6cNnb33+6lWW//IXjtCasSXRhPmYkZQrA6uXd1ZC3L4lvf+haveMUrWL9+PaeeeioHHXQQF110EVddddWU1wZBQLPRnPxQ1my67Nw+PK97ogZ+wM6hYcZGxqlVa4yNjLNjcFh/kNTazlcBxWaNzkichws72ZAbwjRaKy5lt95Kqr5H3ApjCIEhDBQQtRyEajUQOKJ7FX3RFHm3xki9jPs0K18zbWGsY80jRx7z3GljL37FPxDR7a3mlZ5oq13dlvI4zYlNEqsSXZPje+Okk07ipJNOmjL2yU9+ctrrZCD5wX//cMpYrVZHKonJ/Okk82RSymlN0yvlCkpKmKcxa0uDbVg0A49ViS4cw6Lo1mj4rZKLpjDIhOMI1dqDIaWiKT3Cpt3aQAU0vCYlr86OaqtHcbZRIdeocEBm2az3pNUz2meop6+Lf/v8eSTTCWzH5pR3vI6Xvur4ed07danqiaY4unc/jlu2jqN795uRJPtM7Or5ZjQWxRDz+GdFTK+eJIRgzhp3atrTsITBykQXd4xuxZUB3ZEkUimy9Qpr072tBvBC0JQBg5UcbhAgadVBHq0XiTlhhmvFKdcsuvU5aSygZ7TPUCKV4DWnvJJ/eNmxAERjEaKxaJuj0uYj0zRZuWqAoe078P2AcCRM/0AfpjV/Z4amadLT183IztHJse7eLsw5KlWnaU9HCEHNbyJpFaMYr0NXOMGyWAbHtFBK4asAIQTZRoUV8Q5cGRA1TXoiqVatZMS00whz8RlSJ9pnwXZsuno62x3GNLVqDRBE9TL2vGAYBtF4lNVrV7f6Yu7ljue5YBgG6UyKWCJGvVYnEglj2zaGOY9n4dqSEMgAe+IDX9qJsjrVTbFZxzEttpbG6I2lCJs22UaFAzJ92IZFSIBUYBkGg5UcA/EOtpTHJ6/ZG0nOyWrk/P5Xr+2RerXO5ke38t2vfR/DMHjvh9/Jqv1WEonqhNtuQgjsBXKk53GmZWJaJuFwaPcv1rQ5YgiDZuCzKtFFZzjB3eNbyYSixO0QmXCcB3M7OKhzgEwoRs1rUnRrREybkGnjyYDhWpHVyW4O6lhO0a0Tt0Iknchk8p5NC+s3gLZLO3eMcNrJZ07ubr35+lv5xfU/YtV+K9scmaZp2swwDYOEEyZmhRis5ifqGzs4psV92UEUChQYhmC4XqQrnMQyLEBNPvrYXBrDMUxidpiG75J05mYyoteDaBWhuOmPt/DvH72YP1x9Pflsod0hPSNX/vg3k0kWWsdKfv3Tq9sYkaZp2sxzDAtDCIKJZgDNiRrICkXaieKYFq4MCJkW6VAEWukXoRT90VYFP1cGFN0ay2KZOZnNgk601Gp1vv+fP+bD7/44V11xDR//wCf40n98g1Kx3O7Q9tiunhd3z8NnyIvFnrTJu+KKKzjwwAM56KCDeNvb3jY5fvnll7N27VrWrl3L5ZdfPjl+xx13cMghh7BmzRrOPvvsyVKPuVyOE044gbVr13LCCSeQz+eBVuWms88+mzVr1nDooYdy5513Lvl7/OIXv+Cggw7CMAzdmnARcgOf8UYZhaInmgSg4jUxhcHqeCf7JntQSrZ60cY7UQrKXgMQbCvn6I+lOKxrJeszyzi8ayUh05670yJKqRn/c8QRR6iFYnR4TB297/HqOQPHTv557ooXquEdI+0ObY+NDo+pE444eTL+Vxz5OjU6PNbusGbchg0b2h2C8n1f7bvvvmrTpk2q2WyqQw89VD3wwANTXrNx40Z1+OGHq1wup5RSamSk9bOUzWbV6tWrVTabVblcTq1evXryNUcddZS69dZblZRSnXjiiep3v/udUkqpj33sY+riiy9WSil18cUXq/POO08ppdQ111yjTjzxRCWlVH/5y1/U0UcfveTvsWHDBvXQQw+pF73oRepvf/vb0/4dzoefI+2Z8QJflZt1NV4vq4rbUIPlrMrWy+qB7KCqeU1V91y1pTiqdpRzqu421cO5Hari1tV4rayanqe2FMfU7SOPqaFyTo3XyurR/IhqeO6Mxgjcrp4mJy75GS201vQXsq6eTn7yu8v40nc/w1e+91l+fM336O7duwpIi8HDo2N8//Y7+Mafb+X7t9/Bw6Nje33NW2+9lf32249Vq1Y9bZu8Sy+9lLPOOotMJgNAT08PAL///e854YQT6OjoIJPJcMIJJ3Ddddexc+dOSqUSxxxzDEIITjvtNH7zm98A8Nvf/pbTTz8dgNNPP33K+GmnnYYQguc///kUCgV27ty5pO+xfv161q1bt9d/x9r8I5Wk6NZ4tDBC3W+SCcUYrhZYHstgCgOJwpMSy7BQKGq+y7Zyjohl4ytJbzRFdzjBjmqeHbU8fbHUnJ5nX/KJNp6I87Yz3jxl7DVveuWCOhsrhKCrp5OXvPJFHP+Kf5iXR4/m2sOjY1y/aRPlZhOAcrPJ9Zs2PetkK6WkWqlxz133kkqmWjWAfX+XbfI2btzIxo0bOfbYY3n+85/PddddBzx9i72hoSEGBgamjQOMjIzQ398PQH9/P6Ojo7u91lK9h7Z3CvkiD973MNdc+Xu2bxmiWqm1O6RJUkHZbXBQx3Lidpi657JvqhehQCqFAKpeg4QTwpOSjnCM8UaZbKOCKQRSBXRHUxzUOcDaVC8mAtucu7PhS37XcaPe4A1vfQ3POepQbr3xNo5+4REcdsQhJJLxdoem7YVbt23Dl1ObOvtScuu2bazr6X7G1wv8gG2PbcNzPZRUjI9mMU0TpdS0Skq+7/PII49w4403Mjg4yHHHHcf999//tC32nm7873mm11rK99B2r1Qs819fvJRf/Ki1OiCE4Cvfu5gXvvSY/7+9ew+Oq74SPP49t99S6y3ZliUbSzY4xo4JjJNAqFCJk7F5DSaVwYNrgWFITUgwkwdVGbKzgQmzsLtMlgmTYLIQbMCbxB5IQsFSJvHkQU0YMIFAEsAYG2JHli1ZUust9fue/aPbioVlI1vd6lbrfKq6bN1u3XtuSerT93d/v3PwTGNCOpmmcC1ex6E7OkRDqILhZIywL5AtOCEsrcl00VLVsapRQY8PF0XE4a2+wwwlY5ma5xVzqBMh4PVNS+yz+oq2L9LPP95yF39x4Xruvet+otEYS5efSU2d9Zed6Y5eyU52+3uJjsZQYO7cuXR0dgIw2D/IwYMHx7XJg8xV1rp16/D5fLS0tLB06VL27dt3whZ7zc3NtLe3H7edo8fr6ACgo6NjbBj6ZPuarccwp290eHQsyUImWf3z7d+kv3fgJN81fTySadKecl3KfQHSqgS9PpLpNI4IKTdNWl1A8Xk87OvrZF5ZdaZSFELbUCQ7MSpzBfzO4JHj+1Xn0axOtB2HOnnuFy8A8Ie9+3ly+9N885/uK6ohE3N6KgITF1s40fb34g9kPvmuWP5+2tr+SHt7O+IIjz322Lg2eQBXXnklv/zlLwHo6elh7969tLa2snbtWnbu3ElfXx99fX3s3LmTtWvX0tjYSEVFBbt27UJV2bp1K+vWrQPgiiuuGJtx++ijj47bvnXrVlSVXbt2UVVVRWNj46w+hjl9yQlaIfb3DU44glAICTcTnyOCN9tEoDc2jOM4pFXxOk6m1jGQVpegz0/7cC/lvgCgDCWix+0zlprGlpUnmiU1lcdMmXX87M5fjZttfG7zhXrt5X+rvZG+QodmJnAqs0X3HOnSTc+/oP/63H+OPTY9/4LuOdJ1WsdOJpN6qO2wvvG7N/U79/0fXXTGIm1tadU777xTVVVvu+02ffLJJ1VV1XVd/fKXv6zLli3TFStW6LZt28b2s3nzZl28eLEuXrxYt2zZMrb9pZde0uXLl2tra6tu3LhRXddVVdWenh5dvXq1LlmyRFevXq2RSGTsGDfddJO2trbqihUrxs2yna3H+PGPf6xNTU3q9/t1zpw5umbNmgl/ljbr+Hg9XRH99Oprxr0XfuPr/6rR0WihQ1NV1Z7RIY0l4zoYH9WRZEx7R4f0nf4jemCgS6PJuA7FRjWaTGg0Gdf2oV7tjQ7pK137dSA+qrFkQt/qPaz/cWjPuMdoMp7TGDnJrGPRPHxiWbVqlc6EdWxdnd1cedEGYsf0kr3tn29l3V9dZt14itCbb77JsmXLJv36t7q6eb6tjaF4nIpAgI8sXHha92ePSqXSuG4a11U8Hs+MK61oMk7192i26O7s5pHv/IA9r7/FJy/7OBev+/OiuY0WTydxFXZH2mmpbCDo86OqvNp9gA/Un0F3dIj55VWkVUm6aXyOw5HRIWoDZfi9PgRhX38n/YlRvOKwuGoOlf4QQa8/ZzGKyG9UddVEz83qd4qaumq2/r8Hufe/b6KnO8Knr1nH6osvmpVJdnBgkP7eQY50dHFG6wKqaqoIBHL3S1gIS+c0TCmxvpvX68F6sppS1TCvgS/+w+eJRmOEK8qLZhIUgEc8pDWJ3+tl70AnjWXV1Acr+LOGFhCYX16Nm60C9VZfB8tqm6gLhTPNPIC2oR5qg2EWVtThqtITG6LKP30rS2Z1ovX5fCxZ2sr/vP8OkokkVTWVRfXLNV2GBoZ45Dvf55FN3wfAH/Dz4GPfZuV5ywscmTFmOvkDfvxF+AHbAVJpl5aKzAdnRfE4Qk9smLAvgEc8OCK4rsuZ1fM4NNJLf3yUs2ubAOFIdBCigwB4xCHo9ZGexvvPs+/SbQIVlWFq62tmZZIFGBkZ5dH7fzD2dSKe4K6vfoPenr4CRmWMMRmO4+BxnEydY3WJppKAgCp+x0sincQjDm3DEXyOh3g6RdJNs7evA8gs8wFYGK5jeW0Tc0OVuOqSzNZMzrdZfUVrMkZHosfNLuw8fGRcowJjjCmkgMdLJDoMAgHHh6pSEygnmkpQ7guQSCcJ+4J4xKEhVEHA46WxrBoBllTNpSs6iMdx+H3kT8vGzqiop6m8eqy7T77YFa2hsqqC+rnjq0ldsu6ThMPlBYrImNKTTqfp6Yqwd/fbdB46wtDgcKFDmlE8klnCU+4L0BMbAoHBZIyQ10/aVVyUhlAF8XSSan85zeFaOkb7UYTO0X4WVtTRNhQZt8+2oQgpzf8FhSVaQ11DLVt+dD8fW/NRFrYs4G82XsNnv3wDwbJgoUMzpmQceKeNv/zENVy99nouu+Av2bblcUu2p0BRKgIhHBy6svdbu6ODmeFkXEBQBb/XR/twL50jA1RnJzz1xUdIuW62qMX4fU7HrVpLtAYRofmMJv7p3v/GQz+8jxtv+Qy19TWFDqtovVebvLa2Nj7+8Y9z7rnnsnLlSnbs2FGAKE0xGegb4K6vfoPB/kz7TVXlgX/ZwvCQJdrJSLlpBuJRhuJRHIFQtnRiVaCMhJumLz6C3/HgdRwS6RQpTVMXDFMTKEeAlXULcQTqguNL61b5Q3imoamMJVozJlwRpn5OHX7/9NT/nInS6TQbN27kmWeeYffu3Wzbto3du3ePe82dd97J+vXrefXVV9m+fTs33XRTgaI1xSKZTNG2/+C4barKQN9ggSKaWZJumrSmSalL21CEM6vmAlDm9eMRoT5Ywd7+Iwwmo3gdh5FknJDXz+8ibQB0RQdJq9JUXkNzuJYKX5D55dW0VM6ZlitamwxlStbzL7/D40+/TKRvhLqacq66fBUfWbV4Svv89a9/zZIlS2htbQUYa5N39tlnj71GRBgczLyBDgwMWC1eQ7iinE9c8jEe/79P/GlbZZi5jQ30dEXo6YoQKgtSVV1FdW1VASMtTvFUkoDHR8gb4Eh0gCOjg5T5AiTdFKpQ5vOT0jRp12UwEaWlsoG++DANwQoU6IoOUB0oZ3fvIepDYeqCYUZTCfb0HuachoV5j98SrSlJz7/8Dlu2P0cimZm+H+kbYcv25wCmlGwnat/24osvjnvN17/+ddasWcO3v/1tRkZG+NnPfnbaxzOlIRgKcuMtN6Cq/PyZZ1mwqJnb7r6VeDzBDZ+6ic7DRwD42NqP8rW7/57aOrt1c6yA18eh4V5qAmHqQ5XEU0kGE1HKvH4q/EGSbppYKkGZL8DuyCHOqV9IXzzFvPJqFMURB0cY6/7TTWYIvzlci3ca+tLa0LEpSY8//fJYkj0qkUzz+NNTKw06UcnSd7dp27ZtG9dffz3t7e3s2LGDa6+91pZKGWrra7jl9pvZvvMR7t3yv1iwqImH7/veWJIFePanv+Lg/vaT7GV2EoSg10/7SC9lXj/zyqpoDtfyx6EIiXSKoUSMRRUN7B/s5v11zbgotcEw0WRirNBFz+gQ59Qt5Oya+aysW8CK2maaymumpRKgJVpTkiJ9I6e0fbJO1L7tWJs3b2b9+vUAXHDBBcRiMXp6eqZ0XFMagqEgDXPqqa6tJh6Ls//tA8e95uABS7Tv5nc8BBwvlb4QQY+PaCpBf3yUlfULSLku1YEQaXVpqWzAcRwEYSgZZTgZw+tx8Hu8zA/XkHRTtA338lqknUMjfdPWncgSrSlJdTUTrwE+0fbJ+uAHP8i+ffvYv38/iUSC7du3H9cmb+HChfz85z8HMgXsY7EYDQ25q7lsSkO4MsxfXHXJuG0ej4fzPvyBAkVUvBzHoSpQxrzyKlQzPWf3D3bzeqQdn+OQcl3ahiP8vqcNVxUR6BjppzZYjqtKwOMl4aZ5s+8ww8kYitIXH2HvQOe0VIeyRGtK0lWXr8LvG1/txe/zcNXlEzbXmDSv18t9993H2rVrWbZsGevXr2f58uXcfvvtPPXUUwDcc889fPe73+Wcc85hw4YNPPLII8cNLxsjInz0kxfypa9tZP6CRpa9fykP/Nu37P7sCfg9XgTB4zi4qnhEMvWNj2ngroCHzBKfkNePz/GSTruZ5Ask3pVU++Oj03JVO6vb5JmZ5VTbm+Vj1rGZ+YqtTV4qlWKgbxDHcYqmLV0xUlViqQRxN81QfBTH8RBNJagNlhNNJYkm48wP1+CIMJKKU+YNMJSIUuUPkdJMsv19z8FxiTnk9bOybgF+z9TnBVubPDMrfWTVYkuspuh5vV7qGmoLHcZJpdNp+iL9xKIxAsEA1bVV+HzTu94+6aZwVYmnktSFMst2qvwhFGXfcCfn1C8krUpaXcq9Afb0dVDm8VPpD+EVhzRKa9Uc3hnoQlG84rC0el5Okux7sURrjDHmhFSVvbvf5gt//RUi3b1UVVdyz0P/g5XnrcDrm54Uoqok3DQecagMhHhnoIu6YJiEmyKRSrKidgEKODi4uCTcNEPJGGdVz8OD0J+I4vN4qPKHWDWnhbS6CJlGBdNhUvdoReRiEXlLRN4Wka/mOyhjjDHFobenj6/c+DUi3b0ADPQP8pUbv0Z/b/+0xZBy04wk4oym4iTdNPXBMIdH+ugeHaQhVIlHBBQSbhJHZKyzjwAqQsJN8Vqkndci7bze284r3Qc4MNQDTM/cifdMtCLiATYBlwBnAxtE5OyTf5cxxphSkEqmOHywY9y2vkg/8URi2mKIpVOE/UG6ooN4xUHJrK2NppMk3BTd2W4+AjgIXsehubwGyTYaqA6UI2QSbjSVibuxrBrPNKyhhcld0X4IeFtV/6CqCWA7sC6/YRljjCkGvoCPM983fq5DY/M8AsHAtMXgdRz646NU+cuIpTJ9Z5vDmfvajjh0jPTjqovf42MkFef1SDsKpFVBMkt5VtQ1URsop8pfxtLqxum6mM3EOInXNAHHVsNuz24bR0Q+KyIvi8jL3d3duYrPGGNMAdXW1fC/v3sX71txFgCLl7bwrUe/Ma3LkLziYTgZyxag8OAVodIf4ryGRTgilPkC7OnrJOmmCHp8JNw0r/e2M5yKg0KlP8S+/iOU+wJUB8roGOkj6Jm+yVyTSbQT5f3j1gSp6oOqukpVV9nifFOqbrjhBubMmcOKFSsmfF5V+cIXvsCSJUtYuXIlr7zyyjRHaEzuLVjUzKbv3cNPXnqCB7Z/i8VntUxL6cKjfB4PrVUNNIVrSblpHMfB5/FQ7gtQGwyzJNvN57c9bQwlY8wNZRoz7O3voDc6jEeEs6rnMZpKMJqKs6R6Xua+7jSZzJSrdmDBMV83A4fzE44xxe3666/n5ptv5rrrrpvw+WeeeYZ9+/axb98+XnzxRT7/+c8f13TAmJmopsCFNHyOF5/jpdx3/JB10OvL1DhWRUSoCZSzoKKWZDpFwOvPTpBK0lRWjYiD1/FMy7KeoybzkeQl4EwRaRERP3A18FR+wzJm6nY8sZPLzv80f7bwo1x2/qfZ8cTOKe/zoosuorb2xGsen3zySa677jpEhPPPP5/+/n46OjpO+HpjTG74PV6CXh8Bjxe/x0vI66cyUEbA48XneAj7g1QGyqgMhKZtWc9R73k0VU2JyM3ATwEPsEXp0QAYAAAGm0lEQVRV38h7ZMZMwY4ndnLnrXcTi8YB6Dh0hDtvvRuASz+1Jm/HnaiN3qFDh2hsbMzbMY0xk1OoUqiTGmRX1R2qepaqLlbVu/IdlDFTtenuB8aS7FGxaJxNdz+Q1+NOpo2eMWZ2saYCpiR1Hu46pe25Mpk2esaY2cUSrSlJ8+bPOaXtuXLFFVewdetWVJVdu3ZRVVVlw8bGzHJW69iUpI233jjuHi1AMBRg4603Tmm/GzZs4Nlnn6Wnp4fm5mbuuOMOkskkAJ/73Oe49NJL2bFjB0uWLKGsrIyHH354Ssczxsx8lmhNSTo64WnT3Q/QebiLefPnsPHWG6c8EWrbtm0nfV5E2LRp05SOYYwpLZZoTcm69FNr8jrD2BhjJsPu0RpjjDF5ZInWGGOMySNLtGZGmWidqjGTZb8/phAs0ZoZIxgMEolE7M3SnBZVJRKJEAwGCx2KmWVsMpSZMZqbm2lvb8faMJrTFQwGaW5uLnQYZpaxRGtmDJ/PR0tLS6HDMMaYU2JDx8YYY0weWaI1xhhj8sgSrTHGGJNHko8ZnCLSDfwxB7uqB3pysJ9iY+c1s9h5zRyleE5g5zUTnKGqDRM9kZdEmysi8rKqrip0HLlm5zWz2HnNHKV4TmDnNdPZ0LExxhiTR5ZojTHGmDwq9kT7YKEDyBM7r5nFzmvmKMVzAjuvGa2o79EaY4wxM12xX9EaY4wxM1rRJloRuVhE3hKRt0Xkq4WOJxdEZIuIdInI64WOJVdEZIGI/FJE3hSRN0Tki4WOKRdEJCgivxaR32XP645Cx5RLIuIRkVdF5OlCx5IrInJARF4Tkd+KyMuFjidXRKRaRH4oInuyf2cXFDqmqRKRpdmf09HHoIh8qdBx5UtRDh2LiAfYC/w50A68BGxQ1d0FDWyKROQiYBjYqqorCh1PLohII9Coqq+ISAXwG+DKEvhZCVCuqsMi4gOeA76oqrsKHFpOiMgtwCqgUlUvL3Q8uSAiB4BVqloq6zIBEJFHgV+p6kMi4gfKVLW/0HHlSvb9/hDwYVXNRf2FolOsV7QfAt5W1T+oagLYDqwrcExTpqr/AfQWOo5cUtUOVX0l+/8h4E2gqbBRTZ1mDGe/9GUfxfep9DSISDNwGfBQoWMxJycilcBFwGYAVU2UUpLN+gTwTqkmWSjeRNsEHDzm63ZK4M271InIIuBc4MXCRpIb2eHV3wJdwL+rakmcF3Av8PeAW+hAckyBnSLyGxH5bKGDyZFWoBt4ODvU/5CIlBc6qBy7GthW6CDyqVgTrUywrSSuJkqViISBHwFfUtXBQseTC6qaVtUPAM3Ah0Rkxg/3i8jlQJeq/qbQseTBhap6HnAJsDF7q2am8wLnAd9R1XOBEaAk5qwAZIfCrwAeL3Qs+VSsibYdWHDM183A4QLFYt5D9h7mj4Dvq+qPCx1PrmWH6p4FLi5wKLlwIXBF9n7mdmC1iHyvsCHlhqoezv7bBTxB5hbUTNcOtB8zmvJDMom3VFwCvKKqRwodSD4Va6J9CThTRFqyn3iuBp4qcExmAtlJQ5uBN1X1XwodT66ISIOIVGf/HwI+CewpbFRTp6r/VVWbVXURmb+rX6jqNQUOa8pEpDw7GY/s0OoaYMbP7lfVTuCgiCzNbvoEMKMnGr7LBkp82BgywxJFR1VTInIz8FPAA2xR1TcKHNaUicg24GNAvYi0A/+oqpsLG9WUXQhcC7yWvZ8J8A+quqOAMeVCI/BodkakAzymqiWzFKYEzQWeyHzuwwv8QFV/UtiQcubvgO9nLzr+APxNgePJCREpI7Oy5MZCx5JvRbm8xxhjjCkVxTp0bIwxxpQES7TGGGNMHlmiNcYYY/LIEq0xxhiTR5ZojTHGzFqn0uxFRL55TCOEvSIyqXKYNuvYGGPMrHW6zV5E5O+Ac1X1hvd6rV3RGmOMmbUmavYiIotF5CfZutm/EpH3TfCtky62UZQFK4wxxpgCehD4nKruE5EPA/cDq48+KSJnAC3ALyazM0u0xhhjTFa2QcpHgMezlcYAAu962dXAD1U1PZl9WqI1xhhj/sQB+rOdu07kamDjqezQGGOMMUC2zed+EbkKMo1TROSco89nGzzUAC9Mdp+WaI0xxsxa2WYvLwBLRaRdRD4D/BfgMyLyO+ANYN0x37IB2K6nsGTHlvcYY4wxeWRXtMYYY0weWaI1xhhj8sgSrTHGGJNHlmiNMcaYPLJEa4wxxuSRJVpjjDEmjyzRGmOMMXlkidYYY4zJo/8PToR54WrNZWYAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 576x576 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Plotting scatter plot colored by intensity\n"
     ]
    },
    {
     "ename": "TypeError",
     "evalue": "'ListedColormap' object is not iterable",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-3-2fa38026236b>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m \u001b[0mCharSeqPipeline\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mforwardreadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreversereadfile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mblastdatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maligndatabase\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile1\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maltreadfile2\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msteps\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdnalen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrnamaxlen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mREsequence\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwindowsize\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcontactcutoff\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mMQ\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msn\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mhisat2_args\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mgenomefile\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mPCRDupRemoval\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mannot_file\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzoomscaffname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzoomwinsize\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mscaffcoordmin\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mscaffcoordmax\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfeatures\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mmain\u001b[0;34m(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, rnamaxlen, REsequence, windowsize, contactcutoff, MQ, sn, alignargs, genomefile, PCRDupRemoval, annot_file, zoomscaffname, zoomwinsize, scaffcoordmin, scaffcoordmax, features)\u001b[0m\n\u001b[1;32m   1276\u001b[0m         \u001b[0;31m#seaborn_scatter_color_intensity(x[-1000:], y[-1000:], logintensities[-1000:])\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1277\u001b[0m         \u001b[0mseaborn_scatter_color_intensity\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mintensities\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1278\u001b[0;31m         \u001b[0mseaborn_scatter_color_intensity\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlogintensities\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m2500\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1279\u001b[0m         \u001b[0;31m#seaborn_scatter_color_intensity(x[-5000:], y[-5000:], intensities[-5000:])\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1280\u001b[0m         \u001b[0;31m# seaborn_scatter_color_intensity(x, y, intensities, outpath=gencoordcontactsfile + '.pdf')\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/pygentoolbox/CharSeqPipeline.py\u001b[0m in \u001b[0;36mseaborn_scatter_color_intensity\u001b[0;34m(x, y, intensities, outpath)\u001b[0m\n\u001b[1;32m    258\u001b[0m     \u001b[0mfig\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxes\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mplt\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msubplots\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfigsize\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m8\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m8\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    259\u001b[0m     \u001b[0mcmap\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msns\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcubehelix_palette\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstart\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m.5\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrot\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m.5\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mas_cmap\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# lighter values have lower hue values\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 260\u001b[0;31m     \u001b[0msns\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mscatterplot\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mhue\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mintensities\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0max\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpalette\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcmap\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    261\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    262\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0moutpath\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/seaborn/relational.py\u001b[0m in \u001b[0;36mscatterplot\u001b[0;34m(x, y, hue, style, size, data, palette, hue_order, hue_norm, sizes, size_order, size_norm, markers, style_order, x_bins, y_bins, units, estimator, ci, n_boot, alpha, x_jitter, y_jitter, legend, ax, **kwargs)\u001b[0m\n\u001b[1;32m   1333\u001b[0m         \u001b[0mx_bins\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mx_bins\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my_bins\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0my_bins\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1334\u001b[0m         \u001b[0mestimator\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mestimator\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mci\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mci\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mn_boot\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mn_boot\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1335\u001b[0;31m         \u001b[0malpha\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0malpha\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mx_jitter\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mx_jitter\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0my_jitter\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0my_jitter\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlegend\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mlegend\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1336\u001b[0m     )\n\u001b[1;32m   1337\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/seaborn/relational.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, x, y, hue, size, style, data, palette, hue_order, hue_norm, sizes, size_order, size_norm, dashes, markers, style_order, x_bins, y_bins, units, estimator, ci, n_boot, alpha, x_jitter, y_jitter, legend)\u001b[0m\n\u001b[1;32m    857\u001b[0m         )\n\u001b[1;32m    858\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 859\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparse_hue\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mplot_data\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"hue\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpalette\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mhue_order\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mhue_norm\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    860\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparse_size\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mplot_data\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"size\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msizes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msize_order\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msize_norm\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    861\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparse_style\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mplot_data\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"style\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmarkers\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstyle_order\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/seaborn/relational.py\u001b[0m in \u001b[0;36mparse_hue\u001b[0;34m(self, data, palette, order, norm)\u001b[0m\n\u001b[1;32m    365\u001b[0m             \u001b[0mlimits\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    366\u001b[0m             levels, palette = self.categorical_to_palette(\n\u001b[0;32m--> 367\u001b[0;31m                 \u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0morder\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpalette\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    368\u001b[0m             )\n\u001b[1;32m    369\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/seaborn/relational.py\u001b[0m in \u001b[0;36mcategorical_to_palette\u001b[0;34m(self, data, order, palette)\u001b[0m\n\u001b[1;32m    222\u001b[0m                 \u001b[0mcolors\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpalette\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    223\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 224\u001b[0;31m                 \u001b[0mcolors\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcolor_palette\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpalette\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mn_colors\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    225\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    226\u001b[0m             \u001b[0mpalette\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdict\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlevels\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcolors\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/seaborn/palettes.py\u001b[0m in \u001b[0;36mcolor_palette\u001b[0;34m(palette, n_colors, desat)\u001b[0m\n\u001b[1;32m    238\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    239\u001b[0m     \u001b[0;31m# Always return as many colors as we asked for\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 240\u001b[0;31m     \u001b[0mpal_cycle\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcycle\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpalette\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    241\u001b[0m     \u001b[0mpalette\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mnext\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpal_cycle\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0m_\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrange\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mn_colors\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    242\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: 'ListedColormap' object is not iterable"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAesAAAHWCAYAAABXF6HSAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAR50lEQVR4nO3dX4jl91nH8c/TxFistRWzgmQTE3FrXYKQOsSKoJVWSXKxuaklgVIroQtqKmgpRJQq8cqKCEK0rliqgo3RC13Klgg1paU0JVuqoUkJrGltlhSS/jE3oU2jjxcz1el0due3mzO7z+55vWDg/M75zpmH7w7zzu/MmV+quwMAzPWyiz0AAHB2Yg0Aw4k1AAwn1gAwnFgDwHBiDQDD7Rnrqnp/VT1TVZ89w+NVVX9aVaeq6tGqet3qxwSA9bXkzPoDSW45y+O3Jjm09XE0yZ+/9LEAgG/ZM9bd/bEkXz3LktuT/E1vejjJq6vqh1Y1IACsu1X8zvqaJE9tOz69dR8AsAJXruA5apf7dr2GaVUdzeZL5XnFK17xk6997WtX8OUBYL5Pf/rTX+7uA+fzuauI9ekk1247Ppjk6d0WdvexJMeSZGNjo0+ePLmCLw8A81XVf57v567iZfDjSd629a7w1yd5rru/tILnBQCy4My6qj6Y5A1Jrq6q00l+L8l3JUl3vy/JiSS3JTmV5Pkkv7JfwwLAOtoz1t195x6Pd5JfX9lEAMC3cQUzABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWC4RbGuqluq6omqOlVV9+zy+HVV9VBVfaaqHq2q21Y/KgCspz1jXVVXJLkvya1JDie5s6oO71j2u0ke6O6bktyR5M9WPSgArKslZ9Y3JznV3U929wtJ7k9y+441neT7tm6/KsnTqxsRANbblQvWXJPkqW3Hp5P81I41v5/kX6rqnUlekeRNK5kOAFh0Zl273Nc7ju9M8oHuPpjktiR/W1Xf8dxVdbSqTlbVyWefffbcpwWANbQk1qeTXLvt+GC+82Xuu5I8kCTd/ckkL09y9c4n6u5j3b3R3RsHDhw4v4kBYM0sifUjSQ5V1Q1VdVU230B2fMeaLyZ5Y5JU1Y9nM9ZOnQFgBfaMdXe/mOTuJA8m+Vw23/X9WFXdW1VHtpa9K8k7qurfk3wwydu7e+dL5QDAeVjyBrN094kkJ3bc955ttx9P8jOrHQ0ASFzBDADGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFgOLEGgOHEGgCGE2sAGE6sAWA4sQaA4cQaAIYTawAYblGsq+qWqnqiqk5V1T1nWPOWqnq8qh6rqr9b7ZgAsL6u3GtBVV2R5L4kv5DkdJJHqup4dz++bc2hJL+d5Ge6+2tV9YP7NTAArJslZ9Y3JznV3U929wtJ7k9y+44170hyX3d/LUm6+5nVjgkA62tJrK9J8tS249Nb9233miSvqapPVNXDVXXLqgYEgHW358vgSWqX+3qX5zmU5A1JDib5eFXd2N3/9W1PVHU0ydEkue666855WABYR0vOrE8nuXbb8cEkT++y5p+7+5vd/fkkT2Qz3t+mu49190Z3bxw4cOB8ZwaAtbIk1o8kOVRVN1TVVUnuSHJ8x5p/SvLzSVJVV2fzZfEnVzkoAKyrPWPd3S8muTvJg0k+l+SB7n6squ6tqiNbyx5M8pWqejzJQ0ne3d1f2a+hAWCdVPfOXz9fGBsbG33y5MmL8rUB4EKrqk9398b5fK4rmAHAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Awy2KdVXdUlVPVNWpqrrnLOveXFVdVRurGxEA1tuesa6qK5Lcl+TWJIeT3FlVh3dZ98okv5HkU6seEgDW2ZIz65uTnOruJ7v7hST3J7l9l3V/kOS9Sb6+wvkAYO0tifU1SZ7adnx6677/U1U3Jbm2uz+0wtkAgCyLde1yX//fg1UvS/InSd615xNVHa2qk1V18tlnn10+JQCssSWxPp3k2m3HB5M8ve34lUluTPLRqvpCktcnOb7bm8y6+1h3b3T3xoEDB85/agBYI0ti/UiSQ1V1Q1VdleSOJMe/9WB3P9fdV3f39d19fZKHkxzp7pP7MjEArJk9Y93dLya5O8mDST6X5IHufqyq7q2qI/s9IACsuyuXLOruE0lO7LjvPWdY+4aXPhYA8C2uYAYAw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADLco1lV1S1U9UVWnquqeXR7/rap6vKoeraqPVNUPr35UAFhPe8a6qq5Icl+SW5McTnJnVR3esewzSTa6+yeS/GOS9656UABYV0vOrG9Ocqq7n+zuF5Lcn+T27Qu6+6Hufn7r8OEkB1c7JgCsryWxvibJU9uOT2/ddyZ3JfnwSxkKAPh/Vy5YU7vc17surHprko0kP3eGx48mOZok11133cIRAWC9LTmzPp3k2m3HB5M8vXNRVb0pye8kOdLd39jtibr7WHdvdPfGgQMHzmdeAFg7S2L9SJJDVXVDVV2V5I4kx7cvqKqbkvxFNkP9zOrHBID1tWesu/vFJHcneTDJ55I80N2PVdW9VXVka9kfJfneJP9QVf9WVcfP8HQAwDla8jvrdPeJJCd23PeebbfftOK5AIAtrmAGAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAwn1gAwnFgDwHBiDQDDiTUADCfWADCcWAPAcGINAMOJNQAMJ9YAMJxYA8BwYg0Aw4k1AAy3KNZVdUtVPVFVp6rqnl0e/+6q+vutxz9VVdevelAAWFd7xrqqrkhyX5JbkxxOcmdVHd6x7K4kX+vuH03yJ0n+cNWDAsC6WnJmfXOSU939ZHe/kOT+JLfvWHN7kr/euv2PSd5YVbW6MQFgfS2J9TVJntp2fHrrvl3XdPeLSZ5L8gOrGBAA1t2VC9bsdobc57EmVXU0ydGtw29U1WcXfH3O39VJvnyxh1gD9nn/2eP9Z4/334+d7ycuifXpJNduOz6Y5OkzrDldVVcmeVWSr+58ou4+luRYklTVye7eOJ+hWcYeXxj2ef/Z4/1nj/dfVZ08389d8jL4I0kOVdUNVXVVkjuSHN+x5niSX966/eYk/9rd33FmDQCcuz3PrLv7xaq6O8mDSa5I8v7ufqyq7k1ysruPJ/mrJH9bVaeyeUZ9x34ODQDrZMnL4OnuE0lO7LjvPdtufz3JL53j1z52jus5d/b4wrDP+88e7z97vP/Oe4/Lq9UAMJvLjQLAcPsea5cq3X8L9vi3qurxqnq0qj5SVT98Mea8lO21x9vWvbmquqq8q/Y8LNnnqnrL1vfzY1X1dxd6xkvdgp8X11XVQ1X1ma2fGbddjDkvZVX1/qp65kx/nlyb/nTr3+DRqnrdnk/a3fv2kc03pP1Hkh9JclWSf09yeMeaX0vyvq3bdyT5+/2c6XL7WLjHP5/ke7Zu/6o9Xv0eb617ZZKPJXk4ycbFnvtS+1j4vXwoyWeSfP/W8Q9e7LkvpY+Fe3wsya9u3T6c5AsXe+5L7SPJzyZ5XZLPnuHx25J8OJvXKHl9kk/t9Zz7fWbtUqX7b8897u6Huvv5rcOHs/m38iy35Ps4Sf4gyXuTfP1CDncZWbLP70hyX3d/LUm6+5kLPOOlbsked5Lv27r9qnzndTXYQ3d/LLtca2Sb25P8TW96OMmrq+qHzvac+x1rlyrdf0v2eLu7svlfdCy35x5X1U1Jru3uD13IwS4zS76XX5PkNVX1iap6uKpuuWDTXR6W7PHvJ3lrVZ3O5l8BvfPCjLZWzvXn9rI/3XoJVnapUs5o8f5V1VuTbCT5uX2d6PJz1j2uqpdl8/829/YLNdBlasn38pXZfCn8Ddl8hejjVXVjd//XPs92uViyx3cm+UB3/3FV/XQ2r6FxY3f/z/6PtzbOuXv7fWZ9LpcqzdkuVcoZLdnjVNWbkvxOkiPd/Y0LNNvlYq89fmWSG5N8tKq+kM3fQR33JrNztvTnxT939ze7+/NJnshmvFlmyR7fleSBJOnuTyZ5eTavG87qLPq5vd1+x9qlSvffnnu89RLtX2Qz1H7Hd+7Ousfd/Vx3X93d13f39dl8X8CR7j7v6wCvqSU/L/4pm2+YTFVdnc2XxZ+8oFNe2pbs8ReTvDFJqurHsxnrZy/olJe/40netvWu8Ncnea67v3S2T9jXl8HbpUr33cI9/qMk35vkH7beu/fF7j5y0Ya+xCzcY16ihfv8YJJfrKrHk/x3knd391cu3tSXloV7/K4kf1lVv5nNl2bf7gTq3FTVB7P5q5qrt373/3tJvitJuvt92XwvwG1JTiV5Psmv7Pmc/g0AYDZXMAOA4cQaAIYTawAYTqwBYDixBoDhxBoAhhNrABhOrAFguP8FP+e+/VegSywAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 576x576 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "## previous cell split AssUnFUnR to chunks\n",
    "## char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/WT_E_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/WT_E_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES'\n",
    "\n",
    "#altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/pear/WT_L_L1_R1R2.trim.AssUnFUnR.fastq.gz'\n",
    "altreadfile1 = []\n",
    "path ='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/'\n",
    "for i in range(246):\n",
    "    splitfile = os.path.join(path, f'WT_E_L1_R1R2.trim.AssUnFUnR.split{i}.DNA20RNA20RNAmax30.dna.bridgePE.sort.bam')\n",
    "    altreadfile1.append(splitfile)\n",
    "    \n",
    "#altreadfile2 = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/pear/WT_E_L1_R1R2.trim.Ass50Million.fastq.gz'\n",
    "altreadfile2 = []\n",
    "path ='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_sRNA20to30bps/hisat2/'\n",
    "for i in range(246):\n",
    "    splitfile = os.path.join(path, f'WT_E_L1_R1R2.trim.AssUnFUnR.split{i}.DNA20RNA20RNAmax30.rna.bridgePE.sort.bam')\n",
    "    altreadfile2.append(splitfile)\n",
    "    \n",
    "steps=['rnadnacontacts', 'normalize', 'heatmap']\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'split2chunks', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize', 'heatmap']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "rnamaxlen=30\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=1\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "# if mapping to Mac\n",
    "# hisat2_args='-p 2 --no-softclip'\n",
    "# if mapping to Mac+IES\n",
    "hisat2_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n",
    "genomefile = '/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51_with_ies.fa'\n",
    "PCRDupRemoval=True\n",
    "\n",
    "# ptetraurelia_mac_51_annotation_v2.0.gff3\n",
    "annot_file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/internal_eliminated_sequence_PGM_ParTIES.pt_51_with_ies.gff3'\n",
    "features=['internal_eliminated_sequence']\n",
    "\n",
    "\n",
    "zoomscaffname = 'scaffold51_2'\n",
    "zoomwinsize = 50\n",
    "scaffcoordmin, scaffcoordmax = [219999, 354999, 474999, 629999, 839999], [239999, 374999, 484999, 649999, 859999]\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, rnamaxlen, REsequence, windowsize, contactcutoff, MQ, sn, hisat2_args, genomefile, PCRDupRemoval, annot_file, zoomscaffname, zoomwinsize, scaffcoordmin, scaffcoordmax, features)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-08 13:20:05.287738\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import TrimFileToNumberLines\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "On line: 50000000\n",
      "On line: 100000000\n",
      "On line: 150000000\n",
      "On line: 200000000\n",
      "Finished\n",
      "On line: 5000000\n",
      "On line: 10000000\n",
      "On line: 15000000\n",
      "On line: 20000000\n",
      "Finished\n"
     ]
    }
   ],
   "source": [
    "# first lets trim the assembled PEAR reads to 50 million reads\n",
    "f = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.fastq'\n",
    "limit = 200000000\n",
    "\n",
    "TrimFileToNumberLines.main(f,limit)\n",
    "# i then compressed this file with gzip before running the next cell\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start Charseq Bridge Removal on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.fastq.gz\n",
      "python2 /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/char_bridge_trackall.py --FASTQGZ /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.fastq.gz --NAME /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20. --minRNA 20 --minDNA 20\n",
      "Finished Charseq Bridge Removal, output files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Starting samtools\n",
      "Finished with Samtools\n",
      "\n",
      "Finished\n",
      "Finished2\n",
      "Finished3\n",
      "Start: counting number of restriction sites for each genomic window.\n",
      "Genome:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Seqs/ptetraurelia_mac_51.fa\n",
      "Window size: 300\n",
      "Finished counting restriction sites per window\n",
      "\n",
      "start restriction enzyme nut frequency normalization per window of size 300\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNII.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, rounded, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIIrounded.tsv\n",
      "\n",
      "Finished normalized RNA-DNA contact support by restriction counts per window, log2 of normalized support, output at:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/RNA.DNA.Contacts.pt.wthickness.NormDPNIILog2.tsv\n",
      "\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# skipping BLAST, running fastq2fasta for gzip decompression of dnafile and rnafile after charbridge\n",
    "\n",
    "\n",
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/WT_L_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/WT_L_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/NotUsed.fastq.gz'\n",
    "\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['charbridge', 'fastq2fasta', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2020-06-08 17:27:27.411344\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "from pygentoolbox import TrimFileToNumberLines\n",
    "\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "On line: 500000\n",
      "On line: 1000000\n",
      "On line: 1500000\n",
      "On line: 2000000\n",
      "Finished\n"
     ]
    }
   ],
   "source": [
    "\n",
    "# lets trim the RNA reads to 0.5 million reads\n",
    "# this we can try to BLAST\n",
    "f = '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq'\n",
    "limit = 2000000\n",
    "\n",
    "TrimFileToNumberLines.main(f,limit)\n",
    "\n",
    "# CharSeqPipeline.run_gzip_compress()\n",
    "# i then compressed this file with gzip before running the next cell\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start packing files:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq\n",
      "gzip -f -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq\n",
      "Finished packing:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq.gz\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq.gz'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# i then compressed this file with gzip before running the next cell\n",
    "CharSeqPipeline.run_gzip_compress('/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa\n",
      "\n",
      "Start calculations of percent free-floating human RNA contamination\n",
      "Expected percent of human-spiked-in of total RNA is 1 percent\n",
      "Start BLASTn on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa\n",
      "To Database:\n",
      "/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn  already exists\n",
      "Pipe step 1.1\n",
      "blastn -query /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa -db /media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa -outfmt 6\n",
      "Pipe step 1.2\n",
      "Pipe step 1.3\n",
      "Pipe step 2.1\n",
      "Pipe step 2.2\n",
      "Pipe step 2.3\n",
      "Pipe step 3.1\n",
      "Pipe step 3.2\n",
      "Pipe step 3.3\n",
      "Finished BLASTn, output results to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn/best_bit_score_per_query.blastn.RNA.tsv\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn/human.rna.freefloating.tsv\n",
      "\n",
      "Measured percent of free-floating human RNA results: 0.049788\n",
      "Expected percent of free-floating RNA from Paramecium tetraurelia \"contaminating\" results: 4.978803\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_L_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_L_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz'\n",
    "# altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['fastq2fasta', 'blast']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-09 18:28:35.686306\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.SeqsPerFeatureAndFeatureTypeAndGOTerms import main\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading Gff3 file: /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gene.gff3\n",
      "Number of scaffolds: 539\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.sam\n",
      "Number of scaffolds: 697\n",
      "Total number of lines in sam file (excluding header): 9784929\n",
      "Number of GO term entries (lines in file) = 86461\n",
      "Counting reads for all feature and Count reads per feature type (ex: exon, gene, CDS, etc.)\n",
      "scaffold51_1\n",
      "scaffold51_10\n",
      "scaffold51_100\n",
      "scaffold51_101\n",
      "scaffold51_102\n",
      "scaffold51_103\n",
      "scaffold51_104\n",
      "scaffold51_105\n",
      "scaffold51_106\n",
      "scaffold51_107\n",
      "scaffold51_108\n",
      "scaffold51_109\n",
      "scaffold51_11\n",
      "scaffold51_110\n",
      "scaffold51_111\n",
      "scaffold51_112\n",
      "scaffold51_113\n",
      "scaffold51_114\n",
      "scaffold51_115\n",
      "scaffold51_116\n",
      "scaffold51_117\n",
      "scaffold51_118\n",
      "scaffold51_119\n",
      "scaffold51_12\n",
      "scaffold51_120\n",
      "scaffold51_121\n",
      "scaffold51_122\n",
      "scaffold51_123\n",
      "scaffold51_124\n",
      "scaffold51_125\n",
      "scaffold51_126\n",
      "scaffold51_127\n",
      "scaffold51_128\n",
      "scaffold51_129\n",
      "scaffold51_13\n",
      "scaffold51_130\n",
      "scaffold51_131\n",
      "scaffold51_132\n",
      "scaffold51_133\n",
      "scaffold51_134\n",
      "scaffold51_135\n",
      "scaffold51_136\n",
      "scaffold51_137\n",
      "scaffold51_138\n",
      "scaffold51_139\n",
      "scaffold51_14\n",
      "scaffold51_140\n",
      "scaffold51_141\n",
      "scaffold51_142\n",
      "scaffold51_143\n",
      "scaffold51_144\n",
      "scaffold51_145\n",
      "scaffold51_146\n",
      "scaffold51_147\n",
      "scaffold51_148\n",
      "scaffold51_149\n",
      "scaffold51_15\n",
      "scaffold51_150\n",
      "scaffold51_151\n",
      "scaffold51_152\n",
      "scaffold51_153\n",
      "scaffold51_154\n",
      "scaffold51_155\n",
      "scaffold51_156\n",
      "scaffold51_157\n",
      "scaffold51_158\n",
      "scaffold51_159\n",
      "scaffold51_16\n",
      "scaffold51_160\n",
      "scaffold51_161\n",
      "scaffold51_162\n",
      "scaffold51_163\n",
      "scaffold51_164\n",
      "scaffold51_165\n",
      "scaffold51_166\n",
      "scaffold51_167\n",
      "scaffold51_168\n",
      "scaffold51_169\n",
      "scaffold51_17\n",
      "scaffold51_170\n",
      "scaffold51_171\n",
      "scaffold51_172\n",
      "scaffold51_173\n",
      "scaffold51_174\n",
      "scaffold51_175\n",
      "scaffold51_176\n",
      "scaffold51_177\n",
      "scaffold51_178\n",
      "scaffold51_179\n",
      "scaffold51_18\n",
      "scaffold51_180\n",
      "scaffold51_181\n",
      "scaffold51_182\n",
      "scaffold51_183\n",
      "scaffold51_184\n",
      "scaffold51_185\n",
      "scaffold51_186\n",
      "scaffold51_187\n",
      "scaffold51_188\n",
      "scaffold51_189\n",
      "scaffold51_19\n",
      "scaffold51_190\n",
      "scaffold51_191\n",
      "scaffold51_192\n",
      "scaffold51_193\n",
      "scaffold51_194\n",
      "scaffold51_195\n",
      "scaffold51_196\n",
      "scaffold51_197\n",
      "scaffold51_198\n",
      "scaffold51_199\n",
      "scaffold51_2\n",
      "scaffold51_20\n",
      "scaffold51_200\n",
      "scaffold51_201\n",
      "scaffold51_202\n",
      "scaffold51_203\n",
      "scaffold51_204\n",
      "scaffold51_205\n",
      "scaffold51_206\n",
      "scaffold51_207\n",
      "scaffold51_208\n",
      "scaffold51_209\n",
      "scaffold51_21\n",
      "scaffold51_210\n",
      "scaffold51_211\n",
      "scaffold51_212\n",
      "scaffold51_213\n",
      "scaffold51_214\n",
      "scaffold51_215\n",
      "scaffold51_216\n",
      "scaffold51_217\n",
      "scaffold51_218\n",
      "scaffold51_219\n",
      "scaffold51_22\n",
      "scaffold51_220\n",
      "scaffold51_221\n",
      "scaffold51_222\n",
      "scaffold51_223\n",
      "scaffold51_224\n",
      "scaffold51_225\n",
      "scaffold51_226\n",
      "scaffold51_227\n",
      "scaffold51_228\n",
      "scaffold51_229\n",
      "scaffold51_23\n",
      "scaffold51_230\n",
      "scaffold51_231\n",
      "scaffold51_232\n",
      "scaffold51_233\n",
      "scaffold51_234\n",
      "scaffold51_235\n",
      "scaffold51_236\n",
      "scaffold51_237\n",
      "scaffold51_238\n",
      "scaffold51_239\n",
      "scaffold51_24\n",
      "scaffold51_240\n",
      "scaffold51_241\n",
      "scaffold51_242\n",
      "scaffold51_243\n",
      "scaffold51_244\n",
      "scaffold51_245\n",
      "scaffold51_246\n",
      "scaffold51_248\n",
      "scaffold51_249\n",
      "scaffold51_25\n",
      "scaffold51_250\n",
      "scaffold51_251\n",
      "scaffold51_252\n",
      "scaffold51_253\n",
      "scaffold51_254\n",
      "scaffold51_255\n",
      "scaffold51_256\n",
      "scaffold51_257\n",
      "scaffold51_258\n",
      "scaffold51_259\n",
      "scaffold51_26\n",
      "scaffold51_261\n",
      "scaffold51_262\n",
      "scaffold51_263\n",
      "scaffold51_264\n",
      "scaffold51_265\n",
      "scaffold51_266\n",
      "scaffold51_267\n",
      "scaffold51_268\n",
      "scaffold51_269\n",
      "scaffold51_27\n",
      "scaffold51_270\n",
      "scaffold51_271\n",
      "scaffold51_272\n",
      "scaffold51_273\n",
      "scaffold51_274\n",
      "scaffold51_275\n",
      "scaffold51_276\n",
      "scaffold51_278\n",
      "scaffold51_279\n",
      "scaffold51_28\n",
      "scaffold51_280\n",
      "scaffold51_281\n",
      "scaffold51_282\n",
      "scaffold51_283\n",
      "scaffold51_284\n",
      "scaffold51_285\n",
      "scaffold51_286\n",
      "scaffold51_287\n",
      "scaffold51_288\n",
      "scaffold51_289\n",
      "scaffold51_29\n",
      "scaffold51_290\n",
      "scaffold51_291\n",
      "scaffold51_293\n",
      "scaffold51_294\n",
      "scaffold51_295\n",
      "scaffold51_296\n",
      "scaffold51_297\n",
      "scaffold51_298\n",
      "scaffold51_299\n",
      "scaffold51_3\n",
      "scaffold51_30\n",
      "scaffold51_300\n",
      "scaffold51_301\n",
      "scaffold51_302\n",
      "scaffold51_303\n",
      "scaffold51_304\n",
      "scaffold51_305\n",
      "scaffold51_306\n",
      "scaffold51_307\n",
      "scaffold51_308\n",
      "scaffold51_309\n",
      "scaffold51_31\n",
      "scaffold51_310\n",
      "scaffold51_311\n",
      "scaffold51_312\n",
      "scaffold51_313\n",
      "scaffold51_314\n",
      "scaffold51_315\n",
      "scaffold51_316\n",
      "scaffold51_317\n",
      "scaffold51_318\n",
      "scaffold51_319\n",
      "scaffold51_32\n",
      "scaffold51_320\n",
      "scaffold51_321\n",
      "scaffold51_322\n",
      "scaffold51_323\n",
      "scaffold51_325\n",
      "scaffold51_326\n",
      "scaffold51_327\n",
      "scaffold51_328\n",
      "scaffold51_329\n",
      "scaffold51_33\n",
      "scaffold51_331\n",
      "scaffold51_332\n",
      "scaffold51_333\n",
      "scaffold51_334\n",
      "scaffold51_335\n",
      "scaffold51_336\n",
      "scaffold51_337\n",
      "scaffold51_338\n",
      "scaffold51_339\n",
      "scaffold51_34\n",
      "scaffold51_340\n",
      "scaffold51_341\n",
      "scaffold51_342\n",
      "scaffold51_343\n",
      "scaffold51_344\n",
      "scaffold51_345\n",
      "scaffold51_347\n",
      "scaffold51_348\n",
      "scaffold51_349\n",
      "scaffold51_35\n",
      "scaffold51_350\n",
      "scaffold51_351\n",
      "scaffold51_352\n",
      "scaffold51_353\n",
      "scaffold51_354\n",
      "scaffold51_355\n",
      "scaffold51_356\n",
      "scaffold51_357\n",
      "scaffold51_358\n",
      "scaffold51_359\n",
      "scaffold51_36\n",
      "scaffold51_361\n",
      "scaffold51_363\n",
      "scaffold51_364\n",
      "scaffold51_365\n",
      "scaffold51_366\n",
      "scaffold51_368\n",
      "scaffold51_37\n",
      "scaffold51_370\n",
      "scaffold51_371\n",
      "scaffold51_372\n",
      "scaffold51_373\n",
      "scaffold51_374\n",
      "scaffold51_375\n",
      "scaffold51_377\n",
      "scaffold51_378\n",
      "scaffold51_379\n",
      "scaffold51_38\n",
      "scaffold51_380\n",
      "scaffold51_381\n",
      "scaffold51_382\n",
      "scaffold51_383\n",
      "scaffold51_384\n",
      "scaffold51_385\n",
      "scaffold51_386\n",
      "scaffold51_388\n",
      "scaffold51_389\n",
      "scaffold51_39\n",
      "scaffold51_390\n",
      "scaffold51_391\n",
      "scaffold51_392\n",
      "scaffold51_393\n",
      "scaffold51_394\n",
      "scaffold51_395\n",
      "scaffold51_396\n",
      "scaffold51_397\n",
      "scaffold51_4\n",
      "scaffold51_40\n",
      "scaffold51_400\n",
      "scaffold51_401\n",
      "scaffold51_402\n",
      "scaffold51_404\n",
      "scaffold51_406\n",
      "scaffold51_409\n",
      "scaffold51_41\n",
      "scaffold51_410\n",
      "scaffold51_411\n",
      "scaffold51_414\n",
      "scaffold51_415\n",
      "scaffold51_417\n",
      "scaffold51_418\n",
      "scaffold51_42\n",
      "scaffold51_420\n",
      "scaffold51_421\n",
      "scaffold51_422\n",
      "scaffold51_423\n",
      "scaffold51_424\n",
      "scaffold51_425\n",
      "scaffold51_426\n",
      "scaffold51_427\n",
      "scaffold51_428\n",
      "scaffold51_429\n",
      "scaffold51_43\n",
      "scaffold51_431\n",
      "scaffold51_433\n",
      "scaffold51_434\n",
      "scaffold51_435\n",
      "scaffold51_436\n",
      "scaffold51_438\n",
      "scaffold51_44\n",
      "scaffold51_440\n",
      "scaffold51_442\n",
      "scaffold51_443\n",
      "scaffold51_444\n",
      "scaffold51_445\n",
      "scaffold51_446\n",
      "scaffold51_448\n",
      "scaffold51_45\n",
      "scaffold51_450\n",
      "scaffold51_451\n",
      "scaffold51_452\n",
      "scaffold51_453\n",
      "scaffold51_454\n",
      "scaffold51_458\n",
      "scaffold51_459\n",
      "scaffold51_46\n",
      "scaffold51_460\n",
      "scaffold51_461\n",
      "scaffold51_462\n",
      "scaffold51_463\n",
      "scaffold51_464\n",
      "scaffold51_465\n",
      "scaffold51_466\n",
      "scaffold51_467\n",
      "scaffold51_468\n",
      "scaffold51_469\n",
      "scaffold51_47\n",
      "scaffold51_472\n",
      "scaffold51_473\n",
      "scaffold51_474\n",
      "scaffold51_475\n",
      "scaffold51_476\n",
      "scaffold51_477\n",
      "scaffold51_478\n",
      "scaffold51_48\n",
      "scaffold51_480\n",
      "scaffold51_483\n",
      "scaffold51_485\n",
      "scaffold51_488\n",
      "scaffold51_489\n",
      "scaffold51_49\n",
      "scaffold51_490\n",
      "scaffold51_491\n",
      "scaffold51_492\n",
      "scaffold51_493\n",
      "scaffold51_495\n",
      "scaffold51_497\n",
      "scaffold51_498\n",
      "scaffold51_499\n",
      "scaffold51_5\n",
      "scaffold51_50\n",
      "scaffold51_501\n",
      "scaffold51_504\n",
      "scaffold51_505\n",
      "scaffold51_507\n",
      "scaffold51_51\n",
      "scaffold51_511\n",
      "scaffold51_514\n",
      "scaffold51_515\n",
      "scaffold51_516\n",
      "scaffold51_52\n",
      "scaffold51_522\n",
      "scaffold51_523\n",
      "scaffold51_526\n",
      "scaffold51_528\n",
      "scaffold51_53\n",
      "scaffold51_530\n",
      "scaffold51_533\n",
      "scaffold51_535\n",
      "scaffold51_536\n",
      "scaffold51_539\n",
      "scaffold51_54\n",
      "scaffold51_540\n",
      "scaffold51_541\n",
      "scaffold51_543\n",
      "scaffold51_548\n",
      "scaffold51_55\n",
      "scaffold51_550\n",
      "scaffold51_554\n",
      "scaffold51_556\n",
      "scaffold51_56\n",
      "scaffold51_562\n",
      "scaffold51_564\n",
      "scaffold51_565\n",
      "scaffold51_566\n",
      "scaffold51_567\n",
      "scaffold51_57\n",
      "scaffold51_572\n",
      "scaffold51_575\n",
      "scaffold51_579\n",
      "scaffold51_58\n",
      "scaffold51_580\n",
      "scaffold51_582\n",
      "scaffold51_584\n",
      "scaffold51_585\n",
      "scaffold51_589\n",
      "scaffold51_59\n",
      "scaffold51_591\n",
      "scaffold51_592\n",
      "scaffold51_593\n",
      "scaffold51_596\n",
      "scaffold51_597\n",
      "scaffold51_599\n",
      "scaffold51_6\n",
      "scaffold51_60\n",
      "scaffold51_600\n",
      "scaffold51_602\n",
      "scaffold51_609\n",
      "scaffold51_61\n",
      "scaffold51_610\n",
      "scaffold51_611\n",
      "scaffold51_612\n",
      "scaffold51_613\n",
      "scaffold51_614\n",
      "scaffold51_616\n",
      "scaffold51_617\n",
      "scaffold51_62\n",
      "scaffold51_625\n",
      "scaffold51_63\n",
      "scaffold51_633\n",
      "scaffold51_636\n",
      "scaffold51_637\n",
      "scaffold51_64\n",
      "scaffold51_641\n",
      "scaffold51_643\n",
      "scaffold51_648\n",
      "scaffold51_65\n",
      "scaffold51_651\n",
      "scaffold51_652\n",
      "scaffold51_655\n",
      "scaffold51_658\n",
      "scaffold51_659\n",
      "scaffold51_66\n",
      "scaffold51_661\n",
      "scaffold51_663\n",
      "scaffold51_665\n",
      "scaffold51_667\n",
      "scaffold51_668\n",
      "scaffold51_67\n",
      "scaffold51_671\n",
      "scaffold51_674\n",
      "scaffold51_675\n",
      "scaffold51_678\n",
      "scaffold51_68\n",
      "scaffold51_681\n",
      "scaffold51_685\n",
      "scaffold51_687\n",
      "scaffold51_688\n",
      "scaffold51_689\n",
      "scaffold51_69\n",
      "scaffold51_690\n",
      "scaffold51_693\n",
      "scaffold51_696\n",
      "scaffold51_697\n",
      "scaffold51_7\n",
      "scaffold51_70\n",
      "scaffold51_71\n",
      "scaffold51_72\n",
      "scaffold51_73\n",
      "scaffold51_74\n",
      "scaffold51_75\n",
      "scaffold51_76\n",
      "scaffold51_77\n",
      "scaffold51_78\n",
      "scaffold51_79\n",
      "scaffold51_8\n",
      "scaffold51_80\n",
      "scaffold51_81\n",
      "scaffold51_82\n",
      "scaffold51_83\n",
      "scaffold51_84\n",
      "scaffold51_85\n",
      "scaffold51_86\n",
      "scaffold51_87\n",
      "scaffold51_88\n",
      "scaffold51_89\n",
      "scaffold51_9\n",
      "scaffold51_90\n",
      "scaffold51_91\n",
      "scaffold51_92\n",
      "scaffold51_93\n",
      "scaffold51_94\n",
      "scaffold51_95\n",
      "scaffold51_96\n",
      "scaffold51_97\n",
      "scaffold51_98\n",
      "scaffold51_99\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.allfeatures.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.allfeatures.sorted.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.allfeatures.sorted.RPKM.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.featuretypes.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.GOtermFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.GOtermDefinitionFromAnnotationFile.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.GOterm.counts.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.GOtermDefinition.counts.tsv\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.sam\n"
     ]
    }
   ],
   "source": [
    "#GOfile only has GO terms for genes and therefore we must limit the Gff3file to only genes\n",
    "\n",
    "Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3'\n",
    "samfilelist = ['/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.sort.sam', '/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.sam']\n",
    "GOfile = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/Ptetraurelia51_GOTerms.tsv'\n",
    "\n",
    "# count number of reads per feature\n",
    "# count number of reads per feature type (exon, mRNA, 5' UTR, etc.) and sort so counts are decreasing\n",
    "\n",
    "main(Gff3file, samfilelist, GOfile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-11 08:08:09.614334\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox.SeqsPerFeatureAndFeatureTypeFixed import main\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading Gff3 file: /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.tRNAFam.gff3\n",
      "Number of scaffolds: 234\n",
      "Reading sam file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.sam\n",
      "Number of scaffolds (in header): 697\n",
      "Total number of lines in sam file (excluding header): 9072103\n",
      "Mapped reads, so calculating seqs per feature and feature type\n",
      "Counting reads for all feature and Count reads per feature type (ex: exon, gene, CDS, etc.)\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.allfeatures.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.allfeatures.sorted.fixed.tsv\n",
      "Writing out file: /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.featuretypes.fixed.tsv\n"
     ]
    }
   ],
   "source": [
    "#GOfile only has GO terms for genes and therefore we must limit the Gff3file to only genes\n",
    "\n",
    "#Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.gff3'\n",
    "Gff3file = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/ptetraurelia_mac_51_annotation_v2.0.tRNAFam.gff3'\n",
    "samfilelist = ['/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/hisat2/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.sort.sam']\n",
    "#GOfile = '/media/sf_LinuxShare/Ciliates/Genomes/Annotations/Ptetraurelia51_GOTerms.tsv'\n",
    "\n",
    "# count number of reads per feature\n",
    "# count number of reads per feature type (exon, mRNA, 5' UTR, etc.) and sort so counts are decreasing\n",
    "\n",
    "main(Gff3file, samfilelist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2020-06-09 19:37:23.221568\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "import os\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import CharSeqPipeline\n",
    "from pygentoolbox import TrimFileToNumberLines\n",
    "\n",
    "# from pygentoolbox.Tools import read_interleaved_fasta_as_noninterleaved\n",
    "# from pygentoolbox.Tools import make_circos_karyotype_file\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq\n",
      "\n",
      "converting fastq to fasta\n",
      "Converted fastq to fasta:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa\n",
      "\n",
      "Start calculations of percent free-floating human RNA contamination\n",
      "Expected percent of human-spiked-in of total RNA is 1 percent\n",
      "Start BLASTn on file:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa\n",
      "To Database:\n",
      "/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn  already exists\n",
      "Pipe step 1.1\n",
      "blastn -query /media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fa -db /media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa -outfmt 6\n",
      "Pipe step 1.2\n",
      "Pipe step 1.3\n",
      "Pipe step 2.1\n",
      "Pipe step 2.2\n",
      "Pipe step 2.3\n",
      "Pipe step 3.1\n",
      "Pipe step 3.2\n",
      "Pipe step 3.3\n",
      "Finished BLASTn, output results to:\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn/best_bit_score_per_query.blastn.RNA.tsv\n",
      "/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/blastn/human.rna.freefloating.tsv\n",
      "\n",
      "Measured percent of free-floating human RNA results: 0.049788\n",
      "Expected percent of free-floating RNA from Paramecium tetraurelia \"contaminating\" results: 4.978803\n",
      "##########\n",
      "    Fin\n",
      "##########\n"
     ]
    }
   ],
   "source": [
    "# char_bridge_tools.py requires python2 i'm running python 2.7.17\n",
    "# pip install Python-Levenshtein\n",
    "# pip install biopython\n",
    "# RecordRNADNAContacts.py uses python3 i'm running python 3.7 anaconda\n",
    "# pip install natsort\n",
    "# forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_E_L1_R1.fastq.gz'\n",
    "forwardreadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_L_L1_R1.fastq.gz'\n",
    "reversereadfile='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_E_50MilReads/WT_L_L1_R2.fastq.gz'\n",
    "blastdatabase='/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabase='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "# altreadfile='/path/to/altreads.fastq'\n",
    "altreadfile1='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.dna.bridgePE.fastq.gz'\n",
    "altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.fastq.gz'\n",
    "# altreadfile2='/media/sf_LinuxShare/Projects/Lyna/CharSeqPipe/EV_L_50MilReads/pear/WT_L_L1_R1R2.trim.assembled.lines200000000.DNA20RNA20.rna.bridgePE.lines2000000.fastq.gz'\n",
    "# steps=['fastp', 'fastqc', 'pear', 'mergefiles', 'charbridge', 'fastq2fasta', 'blast', 'hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "steps=['hisat2', 'samtools', 'rnadnacontacts', 'normalize']\n",
    "rnalen=20\n",
    "dnalen=20\n",
    "REsequence='GATC'\n",
    "windowsize=300\n",
    "contactcutoff=2\n",
    "MQ=10\n",
    "sn='Paramecium tetraurelia'\n",
    "\n",
    "\n",
    "CharSeqPipeline.main(forwardreadfile, reversereadfile, blastdatabase, aligndatabase, altreadfile1, altreadfile2, steps, rnalen, dnalen, REsequence, windowsize, contactcutoff, MQ, sn)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
