{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-01-21 16:31:33.278518\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import os\n",
    "import glob\n",
    "import datetime\n",
    "import subprocess\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import ReadMapping\n",
    "from pygentoolbox import ReadMapping_v2\n",
    "from pygentoolbox import SplitFastqFileBySeqLength\n",
    "\n",
    "from pygentoolbox import SeqsPerFeatureAndFeatureTypeFixed\n",
    "from pygentoolbox import CalculateRelativeBaseCoverageOfFeatureType\n",
    "\n",
    "# need pygentoolbox.Tools also\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "On sequence: 0\n",
      "On sequence: 1000000\n",
      "On sequence: 2000000\n",
      "On sequence: 3000000\n",
      "On sequence: 4000000\n",
      "On sequence: 5000000\n",
      "On sequence: 6000000\n",
      "On sequence: 7000000\n",
      "On sequence: 8000000\n",
      "On sequence: 9000000\n",
      "On sequence: 10000000\n",
      "On sequence: 11000000\n",
      "On sequence: 12000000\n",
      "On sequence: 13000000\n",
      "On sequence: 14000000\n",
      "On sequence: 15000000\n",
      "On sequence: 16000000\n",
      "On sequence: 17000000\n",
      "On sequence: 18000000\n",
      "On sequence: 19000000\n",
      "On sequence: 20000000\n",
      "On sequence: 21000000\n",
      "On sequence: 22000000\n",
      "On sequence: 23000000\n",
      "On sequence: 24000000\n",
      "On sequence: 25000000\n",
      "On sequence: 26000000\n",
      "On sequence: 27000000\n",
      "Made one fastq file for each sequence length:\n",
      "Input file: /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early.fastq\n",
      "Example output filename: /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early25bp25bp.fastq\n",
      "Number of lines in input file: 108524048\n",
      "On sequence: 0\n",
      "On sequence: 1000000\n",
      "On sequence: 2000000\n",
      "On sequence: 3000000\n",
      "On sequence: 4000000\n",
      "On sequence: 5000000\n",
      "On sequence: 6000000\n",
      "On sequence: 7000000\n",
      "On sequence: 8000000\n",
      "On sequence: 9000000\n",
      "On sequence: 10000000\n",
      "On sequence: 11000000\n",
      "On sequence: 12000000\n",
      "On sequence: 13000000\n",
      "On sequence: 14000000\n",
      "On sequence: 15000000\n",
      "On sequence: 16000000\n",
      "On sequence: 17000000\n",
      "On sequence: 18000000\n",
      "On sequence: 19000000\n",
      "On sequence: 20000000\n",
      "On sequence: 21000000\n",
      "On sequence: 22000000\n",
      "On sequence: 23000000\n",
      "On sequence: 24000000\n",
      "On sequence: 25000000\n",
      "On sequence: 26000000\n",
      "On sequence: 27000000\n",
      "On sequence: 28000000\n",
      "On sequence: 29000000\n",
      "On sequence: 30000000\n",
      "On sequence: 31000000\n",
      "On sequence: 32000000\n",
      "On sequence: 33000000\n",
      "On sequence: 34000000\n",
      "On sequence: 35000000\n",
      "On sequence: 36000000\n",
      "On sequence: 37000000\n",
      "On sequence: 38000000\n",
      "On sequence: 39000000\n",
      "On sequence: 40000000\n",
      "On sequence: 41000000\n",
      "On sequence: 42000000\n",
      "On sequence: 43000000\n",
      "On sequence: 44000000\n",
      "On sequence: 45000000\n",
      "On sequence: 46000000\n",
      "On sequence: 47000000\n",
      "On sequence: 48000000\n",
      "On sequence: 49000000\n",
      "On sequence: 50000000\n",
      "On sequence: 51000000\n",
      "On sequence: 52000000\n",
      "On sequence: 53000000\n",
      "On sequence: 54000000\n",
      "On sequence: 55000000\n",
      "On sequence: 56000000\n",
      "On sequence: 57000000\n",
      "On sequence: 58000000\n",
      "On sequence: 59000000\n",
      "On sequence: 60000000\n",
      "On sequence: 61000000\n",
      "On sequence: 62000000\n",
      "On sequence: 63000000\n",
      "On sequence: 64000000\n",
      "On sequence: 65000000\n",
      "On sequence: 66000000\n",
      "On sequence: 67000000\n",
      "On sequence: 68000000\n",
      "On sequence: 69000000\n",
      "On sequence: 70000000\n",
      "On sequence: 71000000\n",
      "On sequence: 72000000\n",
      "On sequence: 73000000\n",
      "On sequence: 74000000\n",
      "On sequence: 75000000\n",
      "On sequence: 76000000\n",
      "On sequence: 77000000\n",
      "On sequence: 78000000\n",
      "On sequence: 79000000\n",
      "On sequence: 80000000\n",
      "On sequence: 81000000\n",
      "On sequence: 82000000\n",
      "On sequence: 83000000\n",
      "On sequence: 84000000\n",
      "On sequence: 85000000\n",
      "On sequence: 86000000\n",
      "On sequence: 87000000\n",
      "On sequence: 88000000\n",
      "On sequence: 89000000\n",
      "On sequence: 90000000\n",
      "On sequence: 91000000\n",
      "On sequence: 92000000\n",
      "On sequence: 93000000\n",
      "On sequence: 94000000\n",
      "On sequence: 95000000\n",
      "On sequence: 96000000\n",
      "On sequence: 97000000\n",
      "On sequence: 98000000\n",
      "On sequence: 99000000\n",
      "On sequence: 100000000\n",
      "On sequence: 101000000\n",
      "On sequence: 102000000\n",
      "On sequence: 103000000\n",
      "On sequence: 104000000\n",
      "On sequence: 105000000\n",
      "On sequence: 106000000\n",
      "On sequence: 107000000\n",
      "On sequence: 108000000\n",
      "On sequence: 109000000\n",
      "On sequence: 110000000\n",
      "On sequence: 111000000\n",
      "On sequence: 112000000\n",
      "On sequence: 113000000\n",
      "On sequence: 114000000\n",
      "On sequence: 115000000\n",
      "On sequence: 116000000\n",
      "On sequence: 117000000\n",
      "On sequence: 118000000\n",
      "On sequence: 119000000\n",
      "On sequence: 120000000\n",
      "On sequence: 121000000\n",
      "On sequence: 122000000\n",
      "On sequence: 123000000\n",
      "On sequence: 124000000\n",
      "On sequence: 125000000\n",
      "On sequence: 126000000\n",
      "On sequence: 127000000\n",
      "On sequence: 128000000\n",
      "On sequence: 129000000\n",
      "On sequence: 130000000\n",
      "On sequence: 131000000\n",
      "On sequence: 132000000\n",
      "Made one fastq file for each sequence length:\n",
      "Input file: /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early.fastq\n",
      "Example output filename: /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early25bp25bp.fastq\n",
      "Number of lines in input file: 529756000\n"
     ]
    }
   ],
   "source": [
    "fastqfile = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early.fastq'\n",
    "minlength=25\n",
    "maxlength=25\n",
    "onefile=True\n",
    "SplitFastqFileBySeqLength.main(fastqfile, minlength, maxlength, onefile)\n",
    "\n",
    "fastqfile = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early.fastq'\n",
    "minlength=25\n",
    "maxlength=25\n",
    "onefile=True\n",
    "SplitFastqFileBySeqLength.main(fastqfile, minlength, maxlength, onefile)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "forward = ['Ptiwi09_RIP_Early25bp25bp.fastq.gz']\n",
    "reverse = ['']\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-01-21 18:37:27.238867\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early25bp25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early25bp25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/hisat2  Created \n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz\n",
      "hisat2 -q -p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/Ptiwi09_RIP_Early25bp25bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/hisat2/Ptiwi09_RIP_Early25bp25bp.trim.Pt_51_MacAndIES.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/fastp/hisat2/Ptiwi09_RIP_Early25bp25bp.trim.Pt_51_MacAndIES.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2021-01-21 19:30:43.411441\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "forward = ['Ptiwi0109KD_total_sRNA_Early25bp25bp.fastq.gz']\n",
    "reverse = ['']\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-01-21 19:30:43.486899\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early25bp25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early25bp25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/hisat2  Created \n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz\n",
      "hisat2 -q -p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/hisat2/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.Pt_51_MacAndIES.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/fastp/hisat2/Ptiwi0109KD_total_sRNA_Early25bp25bp.trim.Pt_51_MacAndIES.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2021-01-21 19:35:07.617522\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
