{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-03-27 17:12:25.236036\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import os\n",
    "import glob\n",
    "import datetime\n",
    "import subprocess\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import ReadMapping\n",
    "from pygentoolbox import ReadMapping_v2\n",
    "from pygentoolbox import SplitFastqFileBySeqLength\n",
    "\n",
    "from pygentoolbox import SeqsPerFeatureAndFeatureTypeFixed\n",
    "from pygentoolbox import CalculateRelativeBaseCoverageOfFeatureType\n",
    "\n",
    "# need pygentoolbox.Tools also\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "fastqfile = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt0109KD_Total_sRNA/Early/Pt_51_MacAndIES/Ptiwi0109KD_total_sRNA_Early.fastq'\n",
    "minlength=25\n",
    "maxlength=25\n",
    "onefile=True\n",
    "SplitFastqFileBySeqLength.main(fastqfile, minlength, maxlength, onefile)\n",
    "\n",
    "fastqfile = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/Pt09_RIP/Early/Pt_51_MacAndIES/Ptiwi09_RIP_Early.fastq'\n",
    "minlength=25\n",
    "maxlength=25\n",
    "onefile=True\n",
    "SplitFastqFileBySeqLength.main(fastqfile, minlength, maxlength, onefile)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "forward = ['SeroA_sRNA.23bp.fastq', 'SeroB_sRNA.23bp.fastq', 'SeroD_sRNA.23bp.fastq', 'SeroH_sRNA.23bp.fastq']\n",
    "# forward = []\n",
    "# for i in range(15, 42): # align each seq length 15-41 and then 50bps # this is EV_late_totalsRNA\n",
    "#     name = f'190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_{i}bp.fastq.gz'\n",
    "#     forward.append(name)\n",
    "# forward.append('190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_50bp.fastq.gz')\n",
    "reverse = [''] * len(forward)\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p2'  # --no-spliced-alignment # --no-softclip # max-intronlen 6000\n",
    "#hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-03-27 17:13:26.168756\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroA_sRNA.23bp.fastq\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroA_sRNA.23bp.fastq -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2  Created \n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroA_sRNA.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroA_sRNA.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroA_sRNA.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroB_sRNA.23bp.fastq\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroB_sRNA.23bp.fastq -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroB_sRNA.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroB_sRNA.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroB_sRNA.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroD_sRNA.23bp.fastq\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroD_sRNA.23bp.fastq -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroD_sRNA.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroD_sRNA.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroD_sRNA.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroH_sRNA.23bp.fastq\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/SeroH_sRNA.23bp.fastq -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/SeroH_sRNA.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroH_sRNA.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/ENA/fastp/hisat2/SeroH_sRNA.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2022-03-27 17:19:43.992300\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "forward = ['EV_L_R1_23bp.fastq.gz']\n",
    "# forward = []\n",
    "# for i in range(15, 42): # align each seq length 15-41 and then 50bps # this is EV_late_totalsRNA\n",
    "#     name = f'190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_{i}bp.fastq.gz'\n",
    "#     forward.append(name)\n",
    "# forward.append('190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_50bp.fastq.gz')\n",
    "reverse = [''] * len(forward)\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p2'  # --no-spliced-alignment # --no-softclip # max-intronlen 6000\n",
    "#hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-03-18 16:52:24.479173\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/EV_L_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/EV_L_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/hisat2  already exists\n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/EV_L_R1_23bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/hisat2/EV_L_R1_23bp.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/Hen1/fastp/hisat2/EV_L_R1_23bp.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2022-03-18 16:54:24.261155\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define variables:\n",
    "# cell 1\n",
    "forward = ['InvertedClusters_L_R1_23bp.fastq.gz']\n",
    "# forward = []\n",
    "# for i in range(15, 42): # align each seq length 15-41 and then 50bps # this is EV_late_totalsRNA\n",
    "#     name = f'190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_{i}bp.fastq.gz'\n",
    "#     forward.append(name)\n",
    "# forward.append('190118_NB501473_A_L1-4_ADPF-56_AdapterTrimmed_R1_50bp.fastq.gz')\n",
    "reverse = [''] * len(forward)\n",
    "#forward = ['H3K4me3_Rep1_R1.fastq.gz', 'H3K4me3_Rep2_R1.fastq.gz', 'H3K9me3_Rep1_R1.fastq.gz', 'H3K9me3_Rep2_R1.fastq.gz', 'H3K27me3_Rep1_R1.fastq.gz', 'H3K27me3_Rep2_R1.fastq.gz']\n",
    "#reverse = ['H3K4me3_Rep1_R2.fastq.gz', 'H3K4me3_Rep2_R2.fastq.gz', 'H3K9me3_Rep1_R2.fastq.gz', 'H3K9me3_Rep2_R2.fastq.gz', 'H3K27me3_Rep1_R2.fastq.gz', 'H3K27me3_Rep2_R2.fastq.gz']\n",
    "directory='/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters'\n",
    "cleanreads=False  # are .fastq.gz reads already clean? if True skip fastp\n",
    "paired=False\n",
    "reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "\n",
    "skipsteps=[]\n",
    "hisat_args='-p2'  # --no-spliced-alignment # --no-softclip # max-intronlen 6000\n",
    "#hisat_args='-p 2 --no-softclip --pen-noncansplice 0 --known-splicesite-infile /media/sf_LinuxShare/Ciliates/Genomes/Annotations/ss.IES.pt_51_MacAndIES.tsv'\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-03-17 17:20:11.235502\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/InvertedClusters_L_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/InvertedClusters_L_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/fastqc\n",
      "\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/hisat2  Created \n",
      "Starting Hisat2 single end: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz\n",
      "hisat2 -q -p2 -x /media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac -U /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/InvertedClusters_L_R1_23bp.trim.fastq.gz -S /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/hisat2/InvertedClusters_L_R1_23bp.trim.Pt_51_Mac.sam\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools, returning /media/sf_LinuxShare/Projects/Theresa/Hisat2/InvertedP08Clusters/fastp/hisat2/InvertedClusters_L_R1_23bp.trim.Pt_51_Mac.sort.bam\n",
      "\n",
      "##########\n",
      "###FIN####\n",
      "##########\n",
      "2022-03-17 17:24:56.629133\n"
     ]
    }
   ],
   "source": [
    "#fileextension='.fastq.gz'\n",
    "#directory='/media/sf_LinuxShare/Projects/Ryuma/hisat2/2018-09-19_L1-4_ADPF-29_inserts'\n",
    "#cleanreads=True  # are .fastq.gz reads already clean? if True skip fastp\n",
    "#reference='/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac'\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "ReadMapping_v2.main(forward, reverse, directory, hisat_args, skipsteps, paired, reference)\n",
    "\n",
    "print(datetime.datetime.now())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
