{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2022-03-08 13:07:44.178643\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import Tools\n",
    "from pygentoolbox import ReadSubtractionByGenomeMappingColorsProportion as readsubtract\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/media/sf_LinuxShare/PyPacks/PyGenToolbox/notebooks/Theresa\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.getcwd())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 54247\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1484\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 49144\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 5176\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 98\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 63610\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 939\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 57083\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 96\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 44791\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1531\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 35746\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 9263\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 35636\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 110\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 27621\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 21443\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 27615\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 22\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 27606\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 9\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 42646\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 859\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 28593\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 14618\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 28496\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 17683\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 25852\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 17652\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 37\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 17648\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 4\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 45737\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 969\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26513\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 20508\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26423\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 13414\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 25064\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 13318\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 103\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13313\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 7\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 52325\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 722\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 29502\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 23284\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 29409\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 93\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 14741\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 26648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 14548\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 202\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 14542\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 6\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 128252\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 566\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 81814\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 47344\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 81694\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 120\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 41714\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 61408\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 40973\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 887\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 40944\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 30\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 378858\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 875\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 189392\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 193271\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 189221\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 171\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 31633\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 201273\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 29740\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2026\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 29658\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 83\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3361812\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1288\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 1431194\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1943523\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 1430813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 381\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 149739\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1313005\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 139375\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 10899\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 138880\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 510\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 184331\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 84813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 100062\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 84666\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 147\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 21186\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 100550\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 19730\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1520\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 19679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 51\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 62181\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 626\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 41005\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 21230\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 40908\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 9090\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 76248\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 8621\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 483\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 27\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 37487\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 387\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26846\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 10648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26740\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 106\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 6986\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 35704\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 314\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 6601\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 80\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 20619\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 522\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 13788\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6834\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 13698\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5589\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 11085\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5331\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 267\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5302\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 29\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 19544\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 587\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14887\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4659\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 14779\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 108\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 6675\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 18893\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6488\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 197\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5798\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 690\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 12489\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 403\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 10962\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1527\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 10894\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5819\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 13503\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5696\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 127\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 60\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8192\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 492\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7149\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7081\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5225\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 3805\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5125\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 120\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5110\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 16\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 308\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5863\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 731\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5793\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 70\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4356\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2536\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4286\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 75\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4284\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8735\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 184\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 8191\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 544\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 8107\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 84\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5146\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 8851\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5123\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 26\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5111\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 18\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 7713\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 104\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7234\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 479\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7198\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 36\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4424\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 8745\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4408\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 25\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4400\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5783\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5417\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 366\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5365\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 52\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3788\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2585\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3777\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 16\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3768\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 12\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 4347\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 45\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 3983\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 364\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 3925\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 58\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2653\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2765\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2642\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 11\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mic2.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "wrote out table of counts aligned to each genome to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/ReadsMappedToEachReference.tsv\n",
      "\n",
      "Output Stacked Barplot to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors_Proportion_testMIC.pdf\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlMAAAGtCAYAAAAyMfEcAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzdfXzPdf////vTnGfI8PkpxQhhJ+/N21nClJM5DiFRTg5HKmdFJ/ocMkcn5JNyxKci5KijrLRMVHIIOS1FsckwVmyMhqOcxGeLsen5+wPv78bGe3u9Z8Pternsctnr9Xq+ns/Hay3uXq/n+/U01loBAACgcEoVdwEAAABXM8IUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOFC6uAauXr26rVu3bnENDwAA4LVNmzYdttbWyOtYsYWpunXrKj4+vriGBwAA8JoxZm9+x3jMBwAA4ABhCgAAwAHCFAAAgAPFNmcKAABvZWVlKS0tTZmZmcVdCq5x5cuXV+3atVWmTBmvzyFMAQBKvLS0NPn7+6tu3boyxhR3ObhGWWt15MgRpaWlKTAw0OvzeMwHACjxMjMzFRAQQJBCkTLGKCAgoMB3QAlTAICrAkEKV0Jhfs8IUwAAFBM/Pz+5XC4FBQXpnnvu0bFjxy57zrRp09S4cWMNGDDgClQIbxCmAAAoJhUqVFBCQoISExNVrVo1zZgx47LnzJw5U0uWLFFMTIxXY2RnZzstE5dBmAIAoARo3bq19u/f79mePHmymjdvrpCQEI0bN06SNHz4cO3evVvdu3fX66+/rt9//10PP/ywmjdvrrCwMH3++eeSpOjoaPXp00f33HOPOnfunG9/qampaty4sYYMGaKmTZuqc+fOOnnypCQpOTlZHTt2VGhoqMLDw5WSkpJvP9c7whQAAMXszJkzWrVqlbp37y5JWr58uXbt2qWNGzcqISFBmzZt0tq1azVr1izddNNNWrNmjUaNGqWJEyfqrrvuUlxcnNasWaPRo0fr999/lyR99913ev/997V69ep8+5OkXbt2acSIEdq+fbuqVq2qTz75RJI0YMAAjRgxQlu2bNH69etVq1atS/ZzPePVCAAAFJOTJ0/K5XIpNTVVzZo1U6dOnSSdDVPLly9XWFiYJCkjI0O7du1Su3btcp2/fPlyLVq0SFOmTJF09lOP+/btkyR16tRJ1apVu2R/t956qwIDA+VyuSRJzZo1U2pqqtLT07V//37de++9ks6+e6kgdV1vCFMAABST83Omjh8/rm7dumnGjBl64oknZK3V2LFjNWzYsEueb63VJ598okaNGuXav2HDBt1www252uXVX2pqqsqVK+fZ9vPz08mTJ2WtzXc8b+q63vCYDwCAYlalShVNmzZNU6ZMUVZWlrp06aL33ntPGRkZkqT9+/fr119/vei8Ll266M033/SEn82bN+fZv7f9nVe5cmXVrl1bCxculCSdOnVKJ06cKHA/1wvuTAEAUAKEhYUpNDRUsbGxGjhwoJKSktS6dWtJUqVKlfThhx+qZs2auc55/vnn9dRTTykkJETWWtWtW1eLFy++qO/OnTvn2Z+fn1++9cyZM0fDhg3TCy+8oDJlymj+/Pn59nNhXdcbk9+tvKLmdrttfHx8sYwNALi6JCUlqXHjxsVdBq4Tef2+GWM2WWvdebXnzhRwjXvxxRcLdR4feQYA7zBnCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAXHWMMT798kalSpU83y9ZskQNGjTwrINXlLKyshQVFaUGDRooKChILVq00NKlS4t0zJzXmpdjx45p5syZXvX1yy+/qH///qpXr56aNWum1q1b67PPPvNFmSUGYQoAgAJYtWqVHn/8cS1btky33nprkY/3/PPP6+DBg0pMTFRiYqL+/e9/Kz09vcjHvRRvw5S1Vj179lS7du20e/dubdq0SbGxsUpLS7sCVV45hCkAALz0zTffaMiQIfriiy9Uv359SdKgQYM0fPhwtW3bVg0bNvQs5xIdHa1evXopMjJSDRo00DPPPOPpZ/ny5WrdurXCw8PVp08fz1p3Fzpx4oTeeecdvfnmm54Fif/rv/5L999/vyRp7ty5Cg4OVlBQkMaMGeM5r1KlShozZoyaNWumjh07auPGjYqIiFC9evW0aNEiT309evRQZGSkGjVqlO8LfidPnqzmzZsrJCTE8zLfqKgopaSkyOVyafTo0fm2W716tcqWLavhw4d7+qtTp44ef/xxSdKZM2c0evRoz3n//Oc/JUlfffWVIiIi1Lt3b91+++0aMGCAZ/3BVatWKSwsTMHBwXr44Yd16tQpSVLdunX197//Xa1bt5bb7dYPP/ygLl26qH79+po1a9Ylr8cpwhQAAF44deqUevTooYULF+r222/PdSw1NVVff/21vvjiCw0fPlyZmZmSpISEBM2bN0/btm3TvHnz9PPPP+vw4cN66aWXtHLlSv3www9yu9167bXX8hwzOTlZt956qypXrnzRsQMHDmjMmDFavXq1EhISFBcX51mY+Pfff1dERIQ2bdokf39/Pffcc1qxYoU+++wzvfDCC54+Nm7cqJiYGCUkJGj+/Pm6cJm35cuXa9euXdq4caMSEhK0adMmrV27VpMmTVL9+vWVkJCgyZMn59tu+/btCg8Pz/dn+u6776pKlSqKi4tTXFyc3nnnHe3Zs0fS2UWb33jjDe3YsUO7d+/WunXrlJmZqUGDBnl+ptnZ2Xrrrbc8/d1yyy367rvv1LZtWw0aNEgLFizQ999/77nm/Op0iuVkAADwQpkyZXTHHXfo3Xff1dSpU3Mdu//++1WqVCk1aNBA9erV048//ihJuvvuu1WlShVJUpMmTbR3714dO3ZMO3bsUJs2bSRJp0+f9iwcXBBxcXGKiIhQjRo1JEkDBgzQ2rVr1bNnT5UtW1aRkZGSpODgYJUrV05lypRRcHCwUlNTPX106tRJAQEBkqRevXrp22+/ldv9/5afW758uZYvX66wsDBJUkZGhnbt2nXR48382l1oxIgR+vbbb1W2bFnFxcVp+fLl2rp1qxYsWCBJOn78uHbt2qWyZcuqRYsWql27tiTJ5XIpNTVV/v7+CgwMVMOGDSVJDz74oGbMmKGnnnpKktS9e3fPNWdkZMjf31/+/v4qX768jh07lm+d7dq1K/DPPyfCFAAAXihVqpQ+/vhjdezYUS+//LL+/ve/e45dOIn9/Pb5R3OS5Ofnp+zsbFlr1alTJ82dO/eyY952223at2+f0tPT5e/vn+vY+cdeeSlTpoynhlKlSnnqKFWqlLKzsy9bd84xxo4dq2HDhuXanzOQXardqlWr9Mknn3i2Z8yYocOHD3sCm7VWb775prp06ZLrvK+++irfn92l5LzOnOefv+786nSKx3wAAHipYsWKWrx4sWJiYvTuu+969s+fP19//PGHUlJStHv3bjVq1CjfPlq1aqV169YpOTlZ0tl5UTt37sx3vEceeURPPPGETp8+LUk6ePCgPvzwQ7Vs2VJff/21Dh8+rDNnzmju3Llq3759ga5nxYoVOnr0qE6ePKmFCxd67pad16VLF7333nueOV379+/Xr7/+Kn9//1yT4PNrd9dddykzMzPXo7gTJ07kOu+tt95SVlaWJGnnzp36/fff86339ttvV2pqqudnN2fOnAJdc351OsWdKQDAVedydyiKUrVq1bRs2TK1a9dO1atXlyQ1atRI7du31y+//KJZs2apfPny+Z5fo0YNRUdHq1+/fp7J0y+99JLn0dWFXnrpJT333HNq0qSJypcvrxtuuEETJkxQrVq19Morr6hDhw6y1upPf/qTevToUaBrufPOOzVw4EAlJyerf//+uR7xSVLnzp2VlJTkeQxZqVIlffjhh6pfv77atGmjoKAgde3aVZMnT86zXc2aNbVw4UKNGjVKr776qmrUqKEbbrhB//jHPyRJgwcPVmpqqsLDw2WtVY0aNTzzvvJSvnx5zZ49W3369FF2draaN2+ea3L75eR3PTVr1izQz+1Cprh+Id1ut71wohsA36s/pX6hzkv5W4qPKwEKLykpSY0bNy7uMvI0aNAgdevWTb179y7uUgokOjpa8fHxmj59enGXUuLk9ftmjNlkrXXn1Z47U8A1LrsMoQgAihJhCgAAB6Kjo33Sz7333ut5LcB5//jHPy6anO0rgwYN0qBBg4qk7+sNYQoAgBLgWlti5XrCp/kAAAAc8CpMGWMijTE/GWOSjTFReRx/3RiTcO5rpzHmmO9LBQAAKHku+5jPGOMnaYakTpLSJMUZYxZZa3ecb2OtHZWj/eOSwoqgVgAAgBLHmztTLSQlW2t3W2tPS4qVdKkXWfSTdPnXugIAAFwDvJmAfrOkn3Nsp0lqmVdDY0wdSYGSVudzfKikoZIuWtcHAABvzTCJPu1vhA265PGIiAiNHTs21yfr3njjDe3cuVMzZ870epyFCxeqYcOGatKkSaFrPW/8+PF65513PGvzRUZGatKkSYqIiNCUKVMuegGnN/J699SF/W3evFnh4eFatmzZZT9p6Ofnp+DgYGVlZal06dJ68MEH9dRTT6lUqfzv5eQcLyIiQgcPHlSFChUknV0D0OkLNouCN3emTB778nvTZ19JC6y1Z/I6aK1921rrtta6z//HBwCgpOvXr59iY2Nz7YuNjVW/fv0K1M/ChQu1Y8eOyzfMIedaehcaNWqUEhISlJCQoEmTJhWo38KaO3eu7rzzTq/WFqxQoYISEhK0fft2rVixQkuWLNGLL75YoPFiYmI811gSg5TkXZhKk3RLju3akg7k07aveMQHALjG9O7dW4sXL/Ys/5KamqoDBw7ozjvv1OTJk9W8eXOFhIRo3LhxnnM++OADhYSEKDQ0VAMHDtT69eu1aNEijR49Wi6XSykpKUpISFCrVq0UEhKie++9V7/99puks3dn/v73v6t9+/aaOnVqoet+9NFH5Xa71bRp01y1LVmyRLfffrvuvPNOPfHEE+rWrZtX/VlrtWDBAkVHR2v58uXKzMz0upaaNWvq7bff1vTp02Wt1ZkzZ/S3v/1NwcHBCgkJ0Ztvvlng6yspvHnMFyepgTEmUNJ+nQ1M/S9sZIxpJOlGSd/5tEIAAIpZQECAWrRooWXLlqlHjx6KjY3VAw88oBUrVmjXrl3auHGjrLXq3r271q5dq4CAAE2cOFHr1q1T9erVdfToUVWrVk3du3fPtfTM+RDRvn17vfDCC3rxxRf1xhtvSJKOHTumr7/++pJ1vf766/rwww8l5f2Cz4kTJ6patWo6c+aM7r77bm3dulUNGzbUsGHDtHbtWgUGBl50d23evHn69ttvPdvnFxWWpHXr1ikwMFD169dXRESElixZol69enn9c6xXr57++OMP/frrr/r000+1Z88ebd68WaVLl9bRo0fzPOehhx6Sn5+f7rvvPj333HMyJq8HZsXrsnemrLXZkkZK+lJSkqSPrbXbjTETjDHdczTtJynWFufqkwAAFJGcj/rOP+Jbvny5li9frrCwMIWHh+vHH3/Url27tHr1avXu3duzEHK1atUu6u/48eM6duyY2rdvL0l68MEHtXbtWs/xBx544LI15XzMl9f8pY8//ljh4eEKCwvT9u3btWPHDv3444+qV6+eAgMDPdeV0wMPPODpMyEhIdfcq7lz56pv376SpL59+3r1qO9C52PCypUrNXz4cJUuffa+Tl4/o5iYGG3btk3ffPONvvnmG82ZM6fA410JXr0B3Vq7RNKSC/a9cMH2eN+VBQBAydKzZ089/fTT+uGHH3Ty5EmFh4crJiZGY8eO1bBhw3K1nTZtmuM7KDfccIOj8/fs2aMpU6YoLi5ON954owYNGqTMzEwV9p7HmTNn9Mknn2jRokWaOHGirLU6cuSI0tPT5e/v71Ufu3fvlp+fn2rWrClr7WV/RjfffLMkyd/fX/3799fGjRv117/+tVD1FyXegA4AgBcqVaqkiIgIPfzww567OV26dNF7772njIwMSdL+/fv166+/6u6779bHH3+sI0eOSJLnEZa/v7/S09MlSVWqVNGNN96ob775RpI0Z84cz10qX/i///s/3XDDDapSpYp++eUXLV26VJJ0++23a/fu3UpNTZV09rGeN1auXKnQ0FD9/PPPSk1N1d69e3Xfffdp4cKFXp1/6NAhDR8+XCNHjpQxRp07d9asWbM8E+wvfMyXnZ2tw4cPS5KysrK0ePFiBQVd+lOXxYW1+QAAV53LvcqgqPTr10+9evXyPO7r3LmzkpKS1Lp1a0lnA9eHH36opk2b6tlnn1X79u3l5+ensLAwRUdHq2/fvhoyZIimTZumBQsW6P3339fw4cN14sQJ1atXT7Nnz/ZZraGhoQoLC1PTpk1Vr149tWnTRtLZT9jNnDlTkZGRql69ulq0aOFVf3PnztW9996ba999992nt956SwMHDszznJMnT8rlcnlejTBw4EA9/fTTkqTBgwdr586dCgkJUZkyZTRkyBCNHDnSc+6pU6fUpUsXZWVl6cyZM+rYsaOGDBlSmB9FkTPFNcXJ7Xbb+Pj4YhkbuJ7UKeQHgfY+6ds6ACeSkpLUuHHj4i7jmpGRkaFKlSrJWqsRI0aoQYMGGjVq1OVPvE7k9ftmjNlkrc3z5V085gMA4DrzzjvvyOVyqWnTpjp+/PhFc75QMDzmAwCgBJs4caLmz5+fa1+fPn307LPPFrrPUaNG+exO1JEjR3T33XdftH/VqlUKCAjwyRglHWEKAIAS7Nlnn3UUnIpaQECAEhISiruMYsVjPgAAAAcIUwAAAA4QpgAAABxgzhQA4OrzkY/XZ+vPSmgoPO5MAQDgBT8/P7lcLgUFBalPnz46ceKEjh07ppkzZ/qk/4ULF2rHjh259r322mu6/fbbFRwcrNDQUD399NPKysryyXjwHcIUAABeqFChghISEpSYmKiyZctq1qxZRRqmZs2apeXLl+v777/Xtm3bFBcXp5o1a+rkyZM+GQ++Q5gCAKCA2rZtq+TkZEVFRSklJUUul0ujR4/Os+1XX32l9u3b6/7771fDhg0VFRWlmJgYtWjRQsHBwUpJSdH69eu1aNEijR49Wi6XSykpKZo4caLeeustVa1aVZJUtmxZRUVFqXLlypKk5cuXq3Xr1goPD1efPn086wPWrVtX48aNU3h4uIKDg/Xjjz9KOrv2Xc+ePRUSEqJWrVpp69atkqTx48frwQcfVOfOnVW3bl19+umneuaZZxQcHKzIyEjPnbBNmzapffv2atasmbp06aKDBw8W6c/4akKYAgCgALKzs7V06VIFBwdr0qRJql+/vhISEjR58uR8z9myZYumTp2qbdu2ac6cOdq5c6c2btyowYMH680339Qdd9yh7t27a/LkyUpISFDNmjWVkZGhwMDAPPs7fPiwXnrpJa1cuVI//PCD3G63XnvtNc/x6tWr64cfftCjjz6qKVOmSJLGjRunsLAwbd26VS+//LL++te/etqnpKToiy++0Oeff66//OUv6tChg7Zt26YKFSroiy++UFZWlh5//HEtWLBAmzZt0sMPP1yi3311pTEBHQAAL5xftFc6e2fqkUce0YEDB7w6t3nz5qpVq5YkqX79+urcubMkKTg4WGvWrLmovbVWxvy/SfZffvmlxowZo2PHjumjjz7S0aNHtWPHDs/ixadPn/YstixJvXr1kiQ1a9ZMn376qSTp22+/1SeffCJJuuuuu3TkyBEdP35cktS1a1eVKVNGwcHBOnPmjCIjIz31paam6qefflJiYqI6deokSTpz5oznekCYAgDAK+fnTBVGuXLlPN+XKlXKs12qVCllZ2df1L5y5cq64YYbtGfPHgUGBqpLly7q0qWLunXrptOnT8taq06dOmnu3LmXHM/Pz8/Tv7UXf2LxfGDLWU+ZMmU8+8/XZ61V06ZN9d133xXq+q91hCkAwNWnhLzKwN/fX+np6UXS19ixY/Xoo48qNjZWVatWlbVWmZmZkqRWrVppxIgRSk5O1m233aYTJ04oLS1NDRs2zLf/du3aKSYmRs8//7y++uorVa9e3TP/6nIaNWqkQ4cO6bvvvlPr1q2VlZWlnTt3qmnTps4u+hrBnCkAAAopICBAbdq0UVBQUL4T0L3Vt29fTZ48WWFhYUpJSdGjjz6qjh07qmXLlgoJCVGbNm0UFhamsLAw1ahRQ9HR0erXr59nQvn5ieb5GT9+vOLj4xUSEqKoqCi9//77XtdWtmxZLViwQGPGjFFoaKhcLpfWr1/v6HqvJSav235XgtvttvHx8cUyNnA9qTO1cOftfdK3dQBOJCUlqXHjxsVdBq4Tef2+GWM2WWvdebXnzhQAAIADzJkCAMAHtm3bpoEDB+baV65cOW3YsKGYKsKVQpgCAMAHgoODC/1pP1zdeMwHAADgAGEKAADAAcIUAACAA8yZAgBcdQr7yo/88CoQOMGdKQAAvODn5yeXy6WgoCD16dNHJ06c0LFjxzRz5kyvz3W5XOrevbtn//Tp03XbbbfJGKPDhw9fso/o6GgZY7Rq1SrPvs8++0zGGC1YsECSNHjwYO3YsSPfPmJiYhQSEqKQkBDdcccd2rJly2Vrx+URpgAA8ML5tfkSExNVtmxZzZo1y+swdf7chIQELVq0yLO/TZs2WrlyperUqeNVDcHBwbnW44uNjVVoaKhn+1//+peaNGmS7/mBgYH6+uuvtXXrVj3//PMaOnSoV+Pi0ghTAAAUUNu2bZWcnKyoqCilpKTI5XIVajmZsLAw1a1bt0Djbty4UVlZWcrIyFBycrJcLpfneEREhM6vLrJs2TKFh4crNDRUd999tyTpjjvu0I033ijp7Pp+aWlpBa4ZF2POFAAABZCdna2lS5cqMjJSXbt2VWJi4mXfL5WZmSm3263SpUsrKipKPXv2LNTYxhh17NhRX375pY4fP67u3btrz549F7U7dOiQhgwZorVr1yowMFBHjx69qM27776rrl27FqoO5EaYAgDACydPnvTcBWrbtq0eeeQRHThwwKtz9+3bp5tuukm7d+/WXXfdpeDgYNWvX79QdfTt21fTpk3T8ePH9b//+796+eWXL2rz/fffq127dgoMDJQkVatWLdfxNWvW6N1339W3335bqBqQG2EKAAAvnJ/3VBg33XSTJKlevXqKiIjQ5s2bCx2mWrRoocTERFWoUEENGzbMs421VsaYPI9t3bpVgwcP1tKlSxUQEFCoGpAbYQoAcNUpKa8y8Pf3V3p6+iXb/Pbbb6pYsaLKlSunw4cPa926dXrmmWccjfvKK6+ofPny+R5v3bq1RowYoT179nge81WrVk379u1Tr169NGfOnHyDGAqOCegAABRSQECA2rRpo6CgoHwnoCclJcntdis0NFQdOnRQVFSU5xN306ZNU+3atZWWlqaQkBANHjzYq3G7du2qDh065Hu8Ro0aevvtt9WrVy+FhobqgQcekCRNmDBBR44c0WOPPSaXyyW3213AK0ZejLW2WAZ2u932/CcOABSdwr7csKT8yx+QzgaSxo0bF3cZuE7k9ftmjNlkrc0zfXJnCgAAwAHmTAEA4APbtm3TwIEDc+0rV66cNmzYUKB+Zs+eralTc99SbtOmjWbMmOG4RhQNwhQAAD4QHBxc6E/75fTQQw/poYce8kFFuFJ4zAcAAOCAV2HKGBNpjPnJGJNsjInKp839xpgdxpjtxpiPfFsmAABAyXTZx3zGGD9JMyR1kpQmKc4Ys8hauyNHmwaSxkpqY639zRhTs6gKBgAAKEm8mTPVQlKytXa3JBljYiX1kLQjR5shkmZYa3+TJGvtr74uFACA8+pPKdzbw/OT8rcUn/aH64s3j/lulvRzju20c/tyaiipoTFmnTHme2NMpK8KBACgJPDz85PL5VJQUJD69OmjEydO6NixY5o5c6bX57pcLnXv3t2zf/r06brttttkjNHhw4cv2Ud0dLSMMVq1apVn32effSZjjBYsWFDo63r99ddVvnx5HT9+vMDnRkdHa+TIkZKk8ePHa8qUKZKkQYMGKTAw0HPNd9xxhyTpl19+Ubdu3RQaGqomTZroT3/6U6HrLkm8CVN5Le5z4Zs+S0tqIClCUj9J/zLGVL2oI2OGGmPijTHxhw4dKmitAAAUm/Nr8yUmJqps2bKaNWuW12Hq/LkJCQlatGiRZ3+bNm20cuVK1alTx6sagoODNXfuXM92bGysQkNDC34xOcydO1fNmzfXZ5995qifC02ePNlzzevXr5ckvfDCC+rUqZO2bNmiHTt2aNKkST4ds7h4E6bSJN2SY7u2pAuXyU6T9Lm1Nstau0fSTzobrnKx1r5trXVba901atQobM0AABSrtm3bKjk5WVFRUUpJSZHL5cp3OZlLCQsLU926dQs07saNG5WVlaWMjAwlJyfL5XJ5jk+YMEHNmzdXUFCQhg4dqvOrnCQnJ6tjx44KDQ1VeHi4UlLOPtZMSUlRRkaGXnrppVwhLTo6Wr169VJkZKQaNGiQay3B2bNnq2HDhmrfvr3WrVtXoOs9ePCgateu7dkOCQkp0PkllTdhKk5SA2NMoDGmrKS+khZd0GahpA6SZIyprrOP/Xb7slAAAEqC7OxsLV26VMHBwZo0aZLq16+vhIQETZ48Od9zMjMz5Xa71apVKy1cuLDQYxtj1LFjR3355Zf6/PPPcz0ylKSRI0cqLi5OiYmJOnnypBYvXixJGjBggEaMGKEtW7Zo/fr1qlWrlqSzd6X69euntm3b6qefftKvv/6/Kc8JCQmaN2+etm3bpnnz5unnn3/WwYMHNW7cOK1bt04rVqzQjh07lJ/Ro0d7HvMNGDBAkjRixAg98sgj6tChgyZOnKgDBy68N3N1umyYstZmSxop6UtJSZI+ttZuN8ZMMMac/6/4paQjxpgdktZIGm2tPVJURQMAcKWdPHnSszjwrbfeqkceecTrc/ft26f4+Hh99NFHeuqppzx3hgqjb9++io2NVWxsrPr165fr2Jo1a9SyZUsFBwdr9erV2r59u9LT07V//37de++9kqTy5curYsWKks4+Juzbt69KlSqlXv5OxOIAACAASURBVL16af78+Z6+7r77blWpUkXly5dXkyZNtHfvXm3YsEERERGqUaOGypYt61lAOS85H/PFxMRIkrp06aLdu3dryJAh+vHHHxUWFqZrYdqPV29At9YukbTkgn0v5PjeSnr63BcAANec8/OeCuOmm26SJNWrV08RERHavHmz6tcv3CcSW7RoocTERFWoUEENGzb07M/MzNRjjz2m+Ph43XLLLRo/frwyMzM9j/outHXrVu3atUudOnWSJJ0+fVr16tXTiBEjJJ1dCuc8Pz8/ZWdnSzp7d8yJatWqqX///urfv7+6deumtWvX6r777nPUZ3FjORkAwFWnpLzKwN/fX+np6Zds89tvv6lixYoqV66cDh8+rHXr1uWag1QYr7zyisqXL59rX2ZmpiSpevXqysjI0IIFC9S7d29VrlxZtWvX1sKFC9WzZ0+dOnVKZ86c0dy5czV+/HiNHTvW00dgYKD27t2b77gtW7bUk08+qSNHjqhy5cqaP39+gSbAr169Wq1atVLFihWVnp6ulJQU3XrrrQW8+pKH5WQAACikgIAAtWnTRkFBQflOQE9KSpLb7VZoaKg6dOigqKgoNWnSRJI0bdo01a5dW2lpaQoJCdHgwYO9Grdr167q0KFDrn1Vq1bVkCFDFBwcrJ49e6p58+aeY3PmzNG0adMUEhKiO+64Q//5z38UGxvrefR33r333qvY2Nh8x61Vq5bGjx+v1q1bq2PHjgoPD8+3bc45Uy6XS6dPn9amTZvkdrsVEhKi1q1ba/DgwbnqvFqZ/G7/FTW3223j4+OLZWzgelJn6uXb5GXvk76tA3AiKSlJjRs3Lu4ycJ3I6/fNGLPJWuvOqz13pgAAABxgzhQAAD6wbds2DRw4MNe+cuXKacOGDQXqZ/bs2Zo6Nfct5TZt2mjGjBmOa0TRIEwBAOADwcHBhf60X04PPfSQHnroIR9UhCuFx3wAAAAOEKYAAAAcIEwBAAA4wJwpAMDVx+FbuC9STK8JwrWBO1MAAHjBz89PLpdLQUFB6tOnj06cOKFjx45p5syZXp/rcrlyLU48ffp03XbbbTLG6PDhw5fsIzo6WsYYrVq1yrPvs88+kzFGCxYsKPyFwTHCFAAAXji/Nl9iYqLKli2rWbNmeR2mzp+bkJCgRYsWefa3adNGK1euVJ06dbyqITg4WHPnzvVsx8bGFmg5FxQNwhQAAAXUtm1bJScnKyoqSikpKXK5XPkuJ3MpYWFhqlu3boHG3bhxo7KyspSRkaHk5GS5XC7P8QkTJqh58+YKCgrS0KFDPYscJycnq2PHjgoNDVV4eLhSUkrG2obXCsIUAAAFkJ2draVLlyo4OFiTJk1S/fr1lZCQoMmTJ+d7TmZmptxut1q1aqWFCxcWemxjjDp27Kgvv/xSn3/+ea5HhpI0cuRIxcXFKTExUSdPntTixYslSQMGDNCIESO0ZcsWrV+/XrVq1Sp0DbgYYQoAAC+cPHlSLpdLbrdbt956qx555BGvz923b5/i4+P10Ucf6amnnnJ0Z6hv376KjY1VbGys+vXrl+vYmjVr1LJlSwUHB2v16tXavn270tPTtX//fs+ixuXLl1fFihULPT4uxqf5AADwwvl5T4Vx0003SZLq1auniIgIbd68WfXr1y9UXy1atFBiYqIqVKighg0bevZnZmbqscceU3x8vG655RaNHz9emZmZnkd9KDrcmQIAXH2s9e1XIfn7+ys9Pf2SbX777TedOnVKknT48GGtW7dOTZo0KfSYkvTKK6/o5ZdfzrUvMzNTklS9enVlZGR4PuFXuXJl1a5d2/N48dSpUzpx4oSj8ZEbYQoAgEIKCAhQmzZtFBQUlO8E9KSkJLndboWGhqpDhw6KioryhKlp06apdu3aSktLU0hIiAYPHuzVuF27dlWHDh1y7atataqGDBmi4OBg9ezZU82bN/ccmzNnjqZNm6aQkBDdcccd+s9//lPIK0ZeTHHd/nO73TY+Pr5YxgauJ3WmXr5NXvY+6ds6ACeSkpLUuHHj4i4D14m8ft+MMZuste682nNnCgAAwAEmoAMA4APbtm3TwIEDc+0rV66cNmzYUKB+Zs+eralTc99SbtOmjWbMmOG4RhQNwhQAAD4QHBxc6E/75fTQQw/poYce8kFFuFJ4zAcAAOAAYQoAAMABwhQAAIADzJkCAFx1XnzxRZ/2N27cOJ/2h+sLd6YAAPBCamqqgoKCcu0bP368pkyZUkwVFV50dLRGjhyZ7/Hx48fr5ptvlsvlUpMmTTR37lyfjV23bl0dPnzYZ/2VBIQpAABwkVGjRikhIUGff/65hg0bpqysrOIuqcQiTAEA4FBERITGjBmjFi1aqGHDhvrmm28knb0D1LNnT91zzz0KDAzU9OnT9dprryksLEytWrXS0aNHJUnvvPOOmjdvrtDQUN13332etfMGDRqk4cOHq23btmrYsKEWL17s6bdHjx6KjIxUo0aNcj32/PDDD9WiRQu5XC4NGzZMZ86ckXT2/VUNGzZU+/bttW7dOq+vrUGDBqpYsaJ+++03SVJKSooiIyPVrFkztW3bVj/++KMk6d///rdatmypsLAwdezYUb/88osk6ciRI+rcubPCwsI0bNgwz8LLv//+u/785z8rNDRUQUFBmjdvXqF//sWNMAUAgA9kZ2dr48aNeuONN3KFm8TERH300UfauHGjnn32WVWsWFGbN29W69at9cEHH0iSevXqpbi4OG3ZskWNGzfWu+++6zk/NTVVX3/9tb744gsNHz7cs6Dxxo0bFRMTo4SEBM2fP1/x8fFKSkrSvHnztG7dOiUkJMjPz08xMTE6ePCgxo0bp3Xr1mnFihXasWOH19f1ww8/qEGDBqpZs6YkaejQoXrzzTe1adMmTZkyRY899pgk6c4779T333+vzZs3q2/fvnr11VclnZ3fduedd2rz5s3q3r279u3bJ0latmyZbrrpJm3ZskWJiYmKjIx08NMvXkxABwDAC8aYS+7v1auXJKlZs2ZKTU31HO/QoYP8/f3l7++vKlWq6J577pF09iWfW7dulXQ2cD333HM6duyYMjIy1KVLF8/5999/v0qVKqUGDRqoXr16njtBnTp1UkBAgGfsb7/9VqVLl9amTZs8ixyfPHlSNWvW1IYNGxQREaEaNWpIkh544AHt3Lnzktf7+uuv65133tHu3bu1bNkySVJGRobWr1+vPn36eNqdOnVKkpSWlqYHHnhABw8e1OnTpxUYGChJWrt2rT799FNJ0p///GfdeOONnuv/29/+pjFjxqhbt25q27btJespybgzBQCAFwICAjyPus47evSoqlevLuns0jGS5Ofnp+zsbE+b8/slqVSpUp7tUqVKedoNGjRI06dP17Zt2zRu3DjP3Sfp4hB3fjuv/dZaPfjgg0pISFBCQoJ++uknjR8/Ps/2lzNq1Cj99NNPmjdvnv76178qMzNTf/zxh6pWrerpPyEhQUlJSZKkxx9/XCNHjtS2bdv0z3/+85LXIEkNGzbUpk2bFBwcrLFjx2rChAkFqq8k4c4UAOCqUxyvMqhUqZJq1aqlVatW6e6779bRo0e1bNkyPfnkk5o9e7ajvtPT01WrVi1lZWUpJiZGN998s+fY/Pnz9eCDD2rPnj3avXu3GjVqpM2bN2vFihU6evSoKlSooIULF+q9995TxYoV1aNHD40aNUo1a9bU0aNHlZ6erpYtW+rJJ5/UkSNHVLlyZc2fP1+hoaFe1darVy+9//77ev/99zVs2DAFBgZq/vz56tOnj6y12rp1q0JDQ3X8+HFP3e+//77n/Hbt2ikmJkbPPfecli5d6gmkBw4cULVq1fSXv/xFlSpVUnR0tKOfYXEiTAEA4KUPPvhAI0aM0H//939LOhvq6tev77jf//mf/1HLli1Vp04dBQcHKz093XOsUaNGat++vX755RfNmjVL5cuXl3R2jtLAgQOVnJys/v37y+12S5Jeeuklde7cWX/88YfKlCmjGTNmqFWrVho/frxat26tWrVqKTw83DMx3RsvvPCC+vfvryFDhigmJkaPPvqoXnrpJWVlZalv374KDQ3V+PHj1adPH918881q1aqV9uzZ4/kZ9evXT+Hh4Wrfvr1uvfVWSWcXhh49erRKlSqlMmXK6K233nL8cywu5vys+ivN7Xbb+Pj4YhkbuJ7UmXr5NnnZ+6Rv6wCcSEpKUuPGjYu7jCtu0KBB6tatm3r37p1rf3R0tOLj4zV9+vRiquzaltfvmzFmk7XWnVd75kwBAAA4wGM+AABKqPzmEQ0aNEiDBg1y3P/EiRM1f/78XPv69OmjZ5991nHf1xPCFADgqmCtLfAn0nBpzz77LMHpAoWZ/sRjPgBAiVe+fHkdOXKkUH/RAd6y1urIkSOeSf7e4s4UAKDEq127ttLS0nTo0KHiLgXXuPLly6t27doFOserMGWMiZQ0VZKfpH9ZayddcHyQpMmS9p/bNd1a+68CVQIAQD7KlCnjeaM2UNJcNkwZY/wkzZDUSVKapDhjzCJr7YUL+8yz1o4sghoBAABKLG/mTLWQlGyt3W2tPS0pVlKPoi0LAADg6uBNmLpZ0s85ttPO7bvQfcaYrcaYBcaYW/LqyBgz1BgTb4yJ57k3AAC4FngTpvL6HOqFH6f4t6S61toQSSslvX/xKZK19m1rrdta6z6/cjUAAMDVzJswlSYp552m2pIO5GxgrT1irT11bvMdSc18Ux4AAEDJ5k2YipPUwBgTaIwpK6mvpEU5GxhjauXY7C4pyXclAgAAlFyX/TSftTbbGDNS0pc6+2qE96y1240xEyTFW2sXSXrCGNNdUrako5IGFWHNAAAAJYZX75my1i6RtOSCfS/k+H6spLG+LQ0AAKDkYzkZAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcKB0cRcAoGg981Ri4U58Msi3hQDANYo7UwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOeBWmjDGRxpifjDHJxpioS7TrbYyxxhi370oEAAAouS4bpowxfpJmSOoqqYmkfsaYJnm085f0hKQNvi4SAACgpPLmzlQLScnW2t3W2tOSYiX1yKPd/0h6VVKmD+sDAAAo0bwJUzdL+jnHdtq5fR7GmDBJt1hrF1+qI2PMUGNMvDEm/tChQwUuFgAAoKTxJkyZPPZZz0FjSkl6XdJ/X64ja+3b1lq3tdZdo0YN76sEAAAoobwJU2mSbsmxXVvSgRzb/pKCJH1ljEmV1ErSIiahAwCA64E3YSpOUgNjTKAxpqykvpIWnT9orT1ura1ura1rra0r6XtJ3a218UVSMQAAQAly2TBlrc2WNFLSl5KSJH1srd1ujJlgjOle1AUCAACUZKW9aWStXSJpyQX7XsinbYTzsgAAAK4OvAEdAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHDAqzBljIk0xvxkjEk2xkTlcXy4MWabMSbBGPOtMaaJ70sFAAAoeS4bpowxfpJmSOoqqYmkfnmEpY+stcHWWpekVyW95vNKAQAASiBv7ky1kJRsrd1trT0tKVZSj5wNrLX/l2PzBknWdyUCAACUXKW9aHOzpJ9zbKdJanlhI2PMCElPSyor6S6fVAcAAFDCeXNnyuSx76I7T9baGdba+pLGSHouz46MGWqMiTfGxB86dKhglQIAAJRA3oSpNEm35NiuLenAJdrHSuqZ1wFr7dvWWre11l2jRg3vqwQAACihvAlTcZIaGGMCjTFlJfWVtChnA2NMgxybf5a0y3clAgAAlFyXnTNlrc02xoyU9KUkP0nvWWu3G2MmSIq31i6SNNIY01FSlqTfJD1YlEUDAACUFN5MQJe1domkJRfseyHH90/6uC4AAICrAm9ABwAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA44FWYMsZEGmN+MsYkG2Oi8jj+tDFmhzFmqzFmlTGmju9LBQAAKHkuG6aMMX6SZkjqKqmJpH7GmCYXNNssyW2tDZG0QNKrvi4UAACgJPLmzlQLScnW2t3W2tOSYiX1yNnAWrvGWnvi3Ob3kmr7tkwAAICSyZswdbOkn3Nsp53bl59HJC3N64AxZqgxJt4YE3/o0CHvqwQAACihvAlTJo99Ns+GxvxFklvS5LyOW2vftta6rbXuGjVqeF8lAABACVXaizZpkm7JsV1b0oELGxljOkp6VlJ7a+0p35QHAABQsnlzZypOUgNjTKAxpqykvpIW5WxgjAmT9E9J3a21v/q+TAAAgJLpsmHKWpstaaSkLyUlSfrYWrvdGDPBGNP9XLPJkipJmm+MSTDGLMqnOwAAgGuKN4/5ZK1dImnJBfteyPF9Rx/XBQAAcFXgDegAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgANehSljTKQx5idjTLIxJiqP4+2MMT8YY7KNMb19XyYAAEDJdNkwZYzxkzRDUldJTST1M8Y0uaDZPkmDJH3k6wIBAABKstJetGkhKdlau1uSjDGxknpI2nG+gbU29dyxP4qgRgAAgBLLm8d8N0v6Ocd22rl9AAAA1z1vwpTJY58tzGDGmKHGmHhjTPyhQ4cK0wUAAECJ4k2YSpN0S47t2pIOFGYwa+3b1lq3tdZdo0aNwnQBAABQongTpuIkNTDGBBpjykrqK2lR0ZYFAABwdbjsBHRrbbYxZqSkLyX5SXrPWrvdGDNBUry1dpExprmkzyTdKOkeY8yL1tqmRVo5cJV78cUXC3zOuHHjiqASAIAT3nyaT9baJZKWXLDvhRzfx+ns4z8AAIDrCm9ABwAAcMCrO1MAfO+9qgV/ZMdDPgAoebgzBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgQOniLgDAtePFF18s8Dnjxo0rgkoA4MrhzhQAAIADhCkAAAAHeMwHFJO9NUwhzrI+r8OX3qta8Ed2POQDcLUjTAHFZMaAbQU+Z0T/IigEAOAIj/kAAAAcIEwBAAA4QJgCAABwgDlTQA68JwkAUFDcmQIAAHCAMAUAAOAAYQoAAMAB5kwBOXxwwwcFPmccr50EgOsaYQrIIbtMSnGXcFV75qnEgp/0ZFCBTynMBwUkPiwAoGgQpgBcdQqzbI3E0jUAigZzpgAAABwgTAEAADhwXT/mY97F1YOXaQIASqoSGaauxb84r9Q1XcmAeC3+dwIuxO85gMspkWEKV4fCTAIu7F8xV2qsK/VpNFw9ruTvOYCrE2HqGsOnnAAAuLK8ClPGmEhJUyX5SfqXtXbSBcfLSfpAUjNJRyQ9YK1N9W2pVzf+dQugxDCm4OdY6/s6gGvEZcOUMcZP0gxJnSSlSYozxiyy1u7I0ewRSb9Za28zxvSV9A9JDxRFwbi0Qj2mkgr1qOpKPhLj8RtyupK/51dSnakFP2fvk4UY542CB6O9BR/m7FhX6JqA4uTNnakWkpKttbslyRgTK6mHpJxhqoek8ee+XyBpujHGWFu4f8pUH39fwU8qxG2cQo1TyLEIA8DViX80XB2uxQ/fFCaISoUM2Fcw9F6LH+owl8s7xpjekiKttYPPbQ+U1NJaOzJHm8RzbdLObaeca3P4gr6GShp6brORpJ8KWG91SYcv28o3rsWxuKarY6xr8Zqu5Fhc09Ux1rV4TVdyLK7pyo9Vx1pbI68D3tyZyuvh+oUJzJs2sta+LeltL8bMuxBj4q217sKef72PxTVdHWNdi9d0Jcfimq6Osa7Fa7qSY3FNJWssb96AnibplhzbtSUdyK+NMaa0pCqSjvqiQAAAgJLMmzAVJ6mBMSbQGFNWUl9Jiy5os0jSg+e+7y1pdWHnSwEAAFxNLvuYz1qbbYwZKelLnX01wnvW2u3GmAmS4q21iyS9K2mOMSZZZ+9I9S2iegv9iJCxrug4V3IsromximucKzkW18RYxTXOlRzrqr2my05ABwAAQP68ecwHAACAfBCmAAAAHCBMAQAAOECYAlDkjDE1i7sGXzPGBBR3DQBKBsJUPviDsuQxxlQxxkwyxvxojDly7ivp3L6qV7COpT7sq7Ix5hVjzBxjTP8Ljs301Tjn+vv/jDFvGWNmGGMCjDHjjTHbjDEfG2Nq+XCcahd8BUjaaIy50RhTzVfjnBsrMsf3VYwx7xpjthpjPjLG/JcPx5lkjKl+7nu3MWa3pA3GmL3GmPa+Gudc/z8YY54zxtT3Zb/5jOU2xqwxxnxojLnFGLPCGHPcGBNnjAnz4TiVjDETjDHbz/V/yBjzvTFmkK/GyDFWaWPMMGPMsnO/C1uMMUuNMcONMWV8PV4+Nfj2k2LG+J27pv8xxrS54NhzPhynojHmGWPMaGNMeWPMIGPMImPMq8aYSr4a5xLj7yyifkNyfF/m3P9fi4wxLxtjKvpijBIbpq7UH5Ln+r/m/qC8Un9InhvrSv1B+bGk3yRFWGsDrLUBkjqc2zfflwMZY8Lz+WomyeXDoWbr7AoCn0jqa4z5xBhT7tyxVj4cR5KidXZNzZ8lrZF0UtKfJX0jaZYPxzksaVOOr3hJN0v64dz3vvRyju//V9JBSffo7Pvx/unDcf6cY3msyZIesNbeprMLwP+vD8eRpBslVZW0xhiz0Rgzyhhzk4/HOG+mpFclfSFpvaR/WmurSIo6d8xXYiTtltRF0ouSpkkaKKmDMeblS51YCHN09v/R8ZL+pLO/4y9KCpX0oa8GyeMfDTn/8fAnX41zzj/1/7d37kFXVWUY/z0ikZmX0oK00Y9EipJkFKMmMTWHsDFpVCJyxlITohktlcYKw7HRvBGYjjWVmpcSJ63RqZBqJOkmiunHZVRoDDR1aJoywbRR4OmPtQ6zPX4foGed7fkO729mz9l7rbP3s999Wefd71pnv/AR4F/AVZLmVupOKKhzAzAUGE66JsYCc0ht1PcK6iBpg6T1edogaQNwQKO8pBbJrgaXAiNI9+0ulGr7bHfkBDxYmb8WuAjYHzgbuKOw1orK/O+Aw/L8SNK7tEpqrSFdnE8A92d79mnD8bsfOBaYSvrxPCmXfxS4t7DWncDnSG/HPwf4BnAgcCPwrYI6q15L3WvU2gQsytdD8/RCQZ3epuVZwJ+Avar3QCGthyrzT2xtP1rUmQksBEZXytaUtKWy3Wo70XwsS9r0KLBznl/SVLeilE4fNo0nOTXr8rU3rcZr4qGCOsualpfmz52ARwvbtLV2YnVBnU0kB3FNZWosv1jYpuWV+Z1J70j6OTCk8HnqzZ/K15wqy8tL6eRtXg3cBAytlK0pqVHZbvU67wUGl7arYyNTTYy1fb7tx23PA3oKb3+wUhocgF1sLwWwvZp0sZbkGdszbe8HnEtyOh7MUaRp21j31TDY9l225wO2fTtp5m7gjQV1AHps32D7SdtzgeNt/xU4lbJPTY/nEPSWyKSkoZLOIzmMJXkEmG77qOaJsok4h0jach/avpjUUP6e5FCVpHq/39RUN6iUiO05wOeB2ZLmStqNPnJ1FuLtks6RdC6wu6RqntCS7ds1wAJJRwMLJV0p6QhJF5Ia57Zg+w+2v0iK7F0GfKiwxP8kTZA0GbCkTwLkiPymgjr/lXR43vYnyOnGbG+m79yurfCMpMnV+0rSTpKmkKLYpfgbKUo+vDK9y/Zw4B8FdQDe0JixvdH2NGAZ6YGvePebk6exIH82lovew7bPBL4DzJd0Vj5f7Won9pB0gqQTgSG2X8r7UMyuTnam6mokoTsbyroaSaivoZxCcjAWS3pG0r+Be4C3Ap8qqAOpi6C/6+zMgjq/0BUy7QAABgBJREFUAI6uFti+keRov1hQB+DOxrgH21vGWUgaAawqKZQd68mkaMpvgSLjEvrgh8BupB+UG0mZ4JE0jIL3ru2rSV2K04FJpAjvV4GngNNK6WReMW7E9ibbC22fWljrC6Rr7TRSF9xRkv5DioadVVBnBjA3b/u8xrYlvY3U/pbk06S0Zuskrc7jcNaRHuxKZue4ktQl2xeXF9QBeKA69AXA9oWkYQI9hXUabcSW6zoPS9lQUIes8RfgmLy4mPIP+g0Wk7r/jwOWNB7IcztR5OG4Y9+ALumCpqLv2v5nNv5y26cU1juSdMOPJIVR/w7cQUqfs7Ggzq2225Vup6pzMOmG3kzqSpxByp/4FHCG7T8X1Ho/qSt2JLASOM326txQTrV9VUGt95C6E5fYfq5SPtH2wlI6Fa19gfvaqbUVnWNtFxvsvg2tttlEct4PsL2yS8/TgLUpb3MUsE+7tbLOvtRz744jRRweA0aRxh8+bHtBYZ0PkAIcSyW9F5hI6rYsqlOnVj86q6hEqtqkNZ40BvaBNh2/ccDmth2/En2FdU/AqaHV+TqltUhPs6tITu5aYFKlrvT4olq0SFGuumyqRavm8xQ2tW7XozUdv7br5O1dACwh/dnhEuBuYDap63xWG3UWtUOnTq3X2aa2nKe6tIrtbJ0TTQMlQ6szdUprASuAN+f5nnxjfCkvFxuEWadW2DQwtLrRpi4/foNIXcvrgd1z+S4UHERdl07YNDC0GoOuOw5Jy/urIv11M7Q6QKdmrUHO3QO21+au2dsl7U/5Qax1aYVNA0OrG22qU6tOmzba3gQ8L+kx2+uz7guSNg9AnTq1utGmWrQ61pki/Qh/jFf++0Kk96GEVmfo1Km1TtIY270Atp+TdBxwPTC6oE6dWmHTwNDqRpvq1KrTphclvcn288ChjUJJe5DGkA40nTq1utGmerRKhtIKh+WuAw7vp+6W0OoMnZpteicwrJ+6Dxe2qRatsGlgaHWjTV18/Ib0U743lXefDRSdsGlgaHXsv/mCIAiCIAgGAp38nqkgCIIgCIKOJ5ypIAiCIAiCFghnKgiCIAiCoAXCmQqC4HVF0j2SVklaJmmppDGVurWSflZZPknSDU3r3ynp3u3QeXfW6pX0iKQf5PKxkoq9pT8Igh2PcKaCIOgETrZ9MCkn3BVNdWMlva+vlSTtCRwC7Clp+DY0rgLm2R5jexQpaz22H7BdMg9dEAQ7GOFMBUFQG5J2lfSrHIVaKWlK01fuJeVvqzIH+Ho/mzyRlCz6VradxPYdwJONBdsr8j4dKemXeX5Bjlz1SnpW0mclDZJ0RY6aLZc0ffusDYJgRyGcqSAI6mQi8LTtg20fBDQnuJ1Iyt9W5afAIZJG9LG9qcD8PE3dhvY8YJGkuySdnaNaL8P2x22PAU4HHs/7cjrwrO3DgMOAM7YjChYEwQ5EOFNBENTJCuAYSZdJGm/72Vz+E0lPAueRu98qbCJ1/X2tWihpKDAC+KPt1cBGSQf1J2z7R8Ao4DbgSGCJpCHN35O0N3Az8Jm8fxOAUyT1AvcBewEHvjqzgyDoZsKZCoKgNrLTcyjJqbpE0uxcdTIwHLgFuKaPVW8GjgD2q5RNAd4CrJG0lpRAd6tdfbaftn297UnARuBlzpekQaQuw2/aXtkoBs7MY63G2B5u+zfbaXIQBDsA4UwFQVAbkvYBnrf9Y9JYqEMadbZfAs4HPihpVHW9XDcP+HKleCow0XaP7R6Sk9avMyVpoqTBeX4YKcL0VNPXLiVlkb+1UvZrYEZl3ZGSdt1+q4Mg6HbCmQqCoE5GA/fnLrNZwEXVStsvAN8GZvax7nXk5OySekhRqiWVddcA6yWN60d7ArBS0jKSg/QV2+uavjMTmFAZhH48cC3wMPCgpJXA9+nsJPFBENRM5OYLgiAIgiBogYhMBUEQBEEQtECEqoMg6CokzQImNxXfZvvi12N/giDofqKbLwiCIAiCoAWimy8IgiAIgqAFwpkKgiAIgiBogXCmgiAIgiAIWiCcqSAIgiAIghYIZyoIgiAIgqAF/g/J3dpWZo8R3AAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 720x504 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "totalreads = 4554180  # ran wc -l *.fastq (=18216720)on the cleaned reads in the fastp folder (number of cleaned reads from insterts 15-35)\n",
    "\n",
    "fileextension='.fastq.gz'\n",
    "directory = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later'\n",
    "outplotname = 'Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors_Proportion_testMIC.pdf'\n",
    "startseqlength = 15\n",
    "cleanreads=False\n",
    "blastdatabase = '/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabaselist=['/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES']  # '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1',\\\n",
    "\n",
    "colors = ['black', \\\n",
    "          'darkviolet', \\\n",
    "          'orange', \\\n",
    "          'dodgerblue', \\\n",
    "          'forestgreen', \\\n",
    "          'red', \\\n",
    "          'grey'] # this is length of align databases + 1 because we need a color for unmapped\n",
    "    \n",
    "# Vector, color=‘darkviolet’\n",
    "# MAC, color=‘forestgreen’\n",
    "# IES, color=‘red’\n",
    "# OES, color=‘dodgerblue’\n",
    "# Mitochondrial, color=‘orange’\n",
    "# Klebsiella, color=‘black’\n",
    "# Unmapped, color=‘grey’\n",
    "\n",
    "readsubtract.main(totalreads, fileextension, directory, outplotname, startseqlength, cleanreads, blastdatabase, aligndatabaselist, colors)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
