{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-03-08 13:25:21.685004\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import Tools\n",
    "from pygentoolbox import ReadSubtractionByGenomeMappingColorsProportion as readsubtract\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/media/sf_LinuxShare/PyPacks/PyGenToolbox/notebooks/Theresa\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.getcwd())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 54247\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1484\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 49144\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 5176\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 98\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 63610\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 939\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 57083\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 96\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 44791\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1531\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 35746\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 9263\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 35636\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 110\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 28156\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 18918\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 28032\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 133\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 27606\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1118\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 42646\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 859\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 28593\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 14618\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 28496\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 18208\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21941\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 18097\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 111\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 17648\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 837\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 45737\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 969\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26513\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 20508\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26423\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13953\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21846\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 13776\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 181\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 13313\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 824\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 52325\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 722\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 29502\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 23284\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 29409\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 93\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 15520\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 23043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 15315\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 210\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 14542\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1178\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 128252\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 566\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 81814\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 47344\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 81694\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 120\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 42688\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 56915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 42345\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 349\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 40944\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1922\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 378858\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 875\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 189392\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 193271\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 189221\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 171\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 33530\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 193913\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 32875\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 29658\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 4555\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3361812\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1288\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 1431194\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1943523\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 1430813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 381\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 152687\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 1306246\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 150602\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2112\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 138880\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 14607\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 184331\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 84813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 100062\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 84666\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 147\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 25585\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 85713\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 25049\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 543\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 19679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 5819\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 62181\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 626\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 41005\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 21230\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 40908\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13228\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 58401\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 12211\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1027\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 8594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 4664\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 37487\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 387\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26846\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 10648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26740\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 106\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 9227\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 30058\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 7882\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1357\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 6601\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1700\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 20619\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 522\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 13788\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6834\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 13698\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8080\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8088\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6177\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5302\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1075\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 19544\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 587\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14887\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4659\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 14779\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 108\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8016\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 17255\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6422\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1609\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5798\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 836\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 12489\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 403\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 10962\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1527\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 10894\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 6680\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 11981\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5937\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 750\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 413\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8192\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 492\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7149\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7081\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5375\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3620\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5251\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 124\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5110\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 198\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 308\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5863\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 731\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5793\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 70\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4402\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2505\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4356\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 46\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4284\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 73\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8735\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 184\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 8191\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 544\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 8107\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 84\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5231\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8842\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5218\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 13\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5111\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 117\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 7713\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 104\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7234\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 479\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7198\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 36\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4470\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8819\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4458\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 12\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4400\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 73\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5783\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5417\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 366\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5365\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 52\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3807\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2649\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3799\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 8\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3768\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 33\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 4347\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 45\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 3983\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 364\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 3925\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 58\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2681\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2738\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 44\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "wrote out table of counts aligned to each genome to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/ReadsMappedToEachReference.tsv\n",
      "\n",
      "Output Stacked Barplot to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors_Proportion.pdf\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlMAAAGtCAYAAAAyMfEcAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzdfXzPdf////vTnGfI8PkplVkIO3lv3s4SppzMcQiJnHQ4Ujkr6kifZB2dWD4pR3w6EXLUUVZaJio5hJyWothkGCs2loajnMRni7Hp+fsD7+82G+/t9WbD7Xq5uFz2er2er+fz8Xq3dO/5er5fL2OtFQAAAEqmXGkXAAAAcDkjTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIAD5Utr4Nq1a9sGDRqU1vAAAABe27hx40FrbZ3CjpVamGrQoIESExNLa3gAAACvGWN+KuoYt/kAAAAcIEwBAAA4QJgCAABwoNTWTAEA4K2cnBxlZGQoOzu7tEvBFa5y5cqqX7++KlSo4PU5hCkAQJmXkZEhf39/NWjQQMaY0i4HVyhrrQ4dOqSMjAwFBgZ6fR63+QAAZV52drYCAgIIUriojDEKCAgo9gwoYQoAcFkgSOFSKMnvGWEKAIBS4ufnJ5fLpeDgYN155506cuTIBc+ZOnWqmjZtqnvvvfcSVAhvEKYAACglVapUUVJSkpKTk1WrVi1Nnz79gufMmDFDixcvVlxcnFdj5ObmOi0TF0CYAgCgDGjbtq327t3r2Z48ebJatmyp0NBQjR8/XpI0cuRI7dq1Sz179tSrr76q33//XQ888IBatmyp8PBwffbZZ5Kk2NhY9evXT3feeae6du1aZH/p6elq2rSphg0bpubNm6tr1646fvy4JCk1NVWdO3dWWFiYIiIilJaWVmQ/VzvCFAAApezUqVNauXKlevbsKUlatmyZdu7cqQ0bNigpKUkbN27UmjVrNHPmTF133XVavXq1xowZo4kTJ+r2229XQkKCVq9erbFjx+r333+XJH377bd67733tGrVqiL7k6SdO3dq1KhR2rZtm2rWrKmPP/5YknTvvfdq1KhR2rx5s9atW6d69eqdt5+rGY9GAACglBw/flwul0vp6elq0aKFunTpIul0mFq2bJnCw8MlSVlZWdq5c6c6dOiQ7/xly5Zp4cKFmjJliqTT33rcs2ePJKlLly6qVavWefu78cYbFRgYKJfLJUlq0aKF0tPTlZmZqb179+quu+6SdPrZS8Wp62pDmAIAoJScXTN19OhR9ejRQ9OnT9ejjz4qa62eeuopjRgx4rznW2v18ccfq0mTJvn2r1+/Xtdcc02+doX1l56erkqVKnm2/fz8dPz4cVlrixzPm7quNtzmAwCglNWoUUNTp07VlClTlJOTo27duundd99VVlaWJGnv3r369ddfzzmvW7dueuONNzzhZ9OmTYX2721/Z1WvXl3169fXggULJEknTpzQsWPHit3P1YKZKQAAyoDw8HCFhYUpPj5egwcPVkpKitq2bStJqlatmj744APVrVs33znPPvusHnvsMYWGhspaqwYNGmjRokXn9N21a9dC+/Pz8yuyntmzZ2vEiBF67rnnVKFCBc2bN6/IfgrWdbUxRU3lXWxut9smJiaWytgAgMtLSkqKmjZtWtpl4CpR2O+bMWajtdZdWHtmpoAr3PPPP1+i8/jKMwB4hzVTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFADgsmOM8ekfb1SrVs3z8+LFi9WoUSPPe/AuppycHEVHR6tRo0YKDg5Wq1attGTJkos6Zt5rLcyRI0c0Y8YMr/r65ZdfNGjQIDVs2FAtWrRQ27Zt9emnn/qizDKDMAUAQDGsXLlSjzzyiJYuXaobb7zxoo/37LPPav/+/UpOTlZycrL+/e9/KzMz86KPez7ehilrrXr37q0OHTpo165d2rhxo+Lj45WRkXEJqrx0CFMAAHjp66+/1rBhw/T5558rKChIkjRkyBCNHDlS7du3V+PGjT2vc4mNjVWfPn0UFRWlRo0a6cknn/T0s2zZMrVt21YRERHq16+f5113BR07dkxvv/223njjDc8Lif/rv/5L99xzjyRpzpw5CgkJUXBwsMaNG+c5r1q1aho3bpxatGihzp07a8OGDYqMjFTDhg21cOFCT329evVSVFSUmjRpUuQDfidPnqyWLVsqNDTU8zDf6OhopaWlyeVyaezYsUW2W7VqlSpWrKiRI0d6+rvpppv0yCOPSJJOnTqlsWPHes775z//KUn68ssvFRkZqb59++qWW27Rvffe63n/4MqVKxUeHq6QkBA98MADOnHihCSpQYMG+vvf/662bdvK7Xbr+++/V7du3RQUFKSZM2ee93qcIkwBAOCFEydOqFevXlqwYIFuueWWfMfS09P11Vdf6fPPP9fIkSOVnZ0tSUpKStLcuXO1detWzZ07Vz///LMOHjyoF154QStWrND3338vt9utV155pdAxU1NTdeONN6p69ernHNu3b5/GjRunVatWKSkpSQkJCZ4XE//++++KjIzUxo0b5e/vr2eeeUbLly/Xp59+queee87Tx4YNGxQXF6ekpCTNmzdPBV/ztmzZMu3cuVMbNmxQUlKSNm7cqDVr1mjSpEkKCgpSUlKSJk+eXGS7bdu2KSIiosjP9J133lGNGjWUkJCghIQEvf3229q9e7ek0y9tfu2117R9+3bt2rVLa9euVXZ2toYMGeL5THNzc/Xmm296+rvhhhv07bffqn379hoyZIjmz5+v7777znPNRdXpFK+TAQDACxUqVNCtt96qd955R6+//nq+Y/fcc4/KlSunRo0aqWHDhvrhhx8kSXfccYdq1KghSWrWrJl++uknHTlyRNu3b1e7du0kSSdPnvS8OLg4EhISFBkZqTp16kiS7r33Xq1Zs0a9e/dWxYoVFRUVJUkKCQlRpUqVVKFCBYWEhCg9Pd3TR5cuXRQQECBJ6tOnj7755hu53f/v9XPLli3TsmXLFB4eLknKysrSzp07z7m9WVS7gkaNGqVvvvlGFStWVEJCgpYtW6YtW7Zo/vz5kqSjR49q586dqlixolq1aqX69etLklwul9LT0+Xv76/AwEA1btxYknTfffdp+vTpeuyxxyRJPXv29FxzVlaW/P395e/vr8qVK+vIkSNF1tmhQ4dif/55EaYAAPBCuXLl9NFHH6lz58568cUX9fe//91zrOAi9rPbZ2/NSZKfn59yc3NlrVWXLl00Z86cC4558803a8+ePcrMzJS/v3++Y2dvexWmQoUKnhrKlSvnqaNcuXLKzc29YN15x3jqqac0YsSIfPvzBrLztVu5cqU+/vhjz/b06dN18OBBT2Cz1uqNN95Qt27d8p335ZdfFvnZnU/e68x7/tnrLqpOp7jNBwCAl6pWrapFixYpLi5O77zzjmf/vHnz9McffygtLU27du1SkyZNiuyjTZs2Wrt2rVJTUyWdXhe1Y8eOIsd78MEH9eijj+rkyZOSpP379+uDDz5Q69at9dVXX+ngwYM6deqU5syZo44dOxbrepYvX67Dhw/r+PHjWrBggWe27Kxu3brp3Xff9azp2rt3r3799Vf5+/vnWwRfVLvbb79d2dnZ+W7FHTt2LN95b775pnJyciRJO3bs0O+//15kvbfccovS09M9n93s2bOLdc1F1ekUM1MAgMvOhWYoLqZatWpp6dKl6tChg2rXri1JatKkiTp27KhffvlFM2fOVOXKlYs8v06dOoqNjdXAgQM9i6dfeOEFz62rgl544QU988wzatasmSpXrqxrrrlGEyZMUL169fTSSy+pU6dOstbqT3/6k3r16lWsa7nttts0ePBgpaamatCgQflu8UlS165dlZKS4rkNWa1aNX3wwQcKCgpSu3btFBwcrO7du2vy5MmFtqtbt64WLFigMWPG6OWXX1adOnV0zTXX6B//+IckaejQoUpPT1dERISstapTp45n3VdhKleurFmzZqlfv37Kzc1Vy5Yt8y1uv5Cirqdu3brF+twKMqX1C+l2u23BhW4AfO+m1y/cpjA//c23dQBOpKSkqGnTpqVdRqGGDBmiHj16qG/fvqVdSrHExsYqMTFR06ZNK+1SypzCft+MMRutte7C2jMzBVzhyucElfDMNJ/WAQBXKsIUAAAOxMbG+qSfu+66y/NYgLP+8Y9/nLM421eGDBmiIUOGXJS+rzaEKQAAyoAr7RUrVxO+zQcAAOCAV2HKGBNljPnRGJNqjIku5PirxpikM392GGOO+L5UAACAsueCt/mMMX6SpkvqIilDUoIxZqG1dvvZNtbaMXnaPyIp/CLUCgAAUOZ4MzPVSlKqtXaXtfakpHhJ53uQxUBJF36sKwAAwBXAmwXo10v6Oc92hqTWhTU0xtwkKVDSqiKOD5c0XNI57/UBAMBb002yT/sbZYPPezwyMlJPPfVUvm/Wvfbaa9qxY4dmzJjh9TgLFixQ48aN1axZsxLXelZMTIzefvttz7v5oqKiNGnSJEVGRmrKlCnnPIDTG4U9e6pgf5s2bVJERISWLl16wW8a+vn5KSQkRDk5OSpfvrzuu+8+PfbYYypXrui5nLzjRUZGav/+/apSpYqk0+8AdPqAzYvBm5kpU8i+op70OUDSfGvtqcIOWmvfsta6rbXus//wAQAo6wYOHKj4+Ph8++Lj4zVw4MBi9bNgwQJt3779wg3zyPsuvYLGjBmjpKQkJSUladKkScXqt6TmzJmj2267zat3C1apUkVJSUnatm2bli9frsWLF+v5558v1nhxcXGeayyLQUryLkxlSLohz3Z9SfuKaDtA3OIDAFxh+vbtq0WLFnle/5Kenq59+/bptttu0+TJk9WyZUuFhoZq/PjxnnPef/99hYaGKiwsTIMHD9a6deu0cOFCjR07Vi6XS2lpaUpKSlKbNm0UGhqqu+66S7/99puk07Mzf//739WxY0e9/noJX2Mg6aGHHpLb7Vbz5s3z1bZ48WLdcsstuu222/Too4+qR48eXvVnrdX8+fMVGxurZcuWKTs72+ta6tatq7feekvTpk2TtVanTp3SE088oZCQEIWGhuqNN94o9vWVFd7c5kuQ1MgYEyhpr04HpkEFGxljmki6VtK3Pq0QAIBSFhAQoFatWmnp0qXq1auX4uPj1b9/fy1fvlw7d+7Uhg0bZK1Vz549tWbNGgUEBGjixIlau3atateurcOHD6tWrVrq2bNnvlfPnA0RHTt21HPPPafnn39er732miTpyJEj+uqrr85b16uvvqoPPvhAUuEP+Jw4caJq1aqlU6dO6Y477tCWLVvUuHFjjRgxQmvWrFFgYOA5s2tz587VN99849k++1JhSVq7dq0CAwMVFBSkyMhILV68WH369PH6c2zYsKH++OMP/frrr/rkk0+0e/dubdq0SeXLl9fhw4cLPef++++Xn5+f7r77bj3zzDMyprAbZqXrgjNT1tpcSaMlfSEpRdJH1tptxpgJxpieeZoOlBRvS/PtkwAAXCR5b/WdvcW3bNkyLVu2TOHh4YqIiNAPP/ygnTt3atWqVerbt6/nRci1atU6p7+jR4/qyJEj6tixoyTpvvvu05o1azzH+/fvf8Ga8t7mK2z90kcffaSIiAiFh4dr27Zt2r59u3744Qc1bNhQgYGBnuvKq3///p4+k5KS8q29mjNnjgYMGCBJGjBggFe3+go6GxNWrFihkSNHqnz50/M6hX1GcXFx2rp1q77++mt9/fXXmj17drHHuxS8egK6tXaxpMUF9j1XYDvGd2UBAFC29O7dW48//ri+//57HT9+XBEREYqLi9NTTz2lESNG5Gs7depUxzMo11xzjaPzd+/erSlTpighIUHXXnuthgwZouzsbJV0zuPUqVP6+OOPtXDhQk2cOFHWWh06dEiZmZny9/f3qo9du3bJz89PdevWlbX2gp/R9ddfL0ny9/fXoEGDtGHDBv31r38tUf0XE09ABwDAC9WqVVNkZKQeeOABz2xOt27d9O677yorK0uStHfvXv3666+644479NFHH+nQoUOS5LmF5e/vr8zMTElSjRo1dO211+rrr7+WJM2ePdszS+UL//d//6drrrlGNWrU0C+//KIlS5ZIkm655Rbt2rVL6enpkk7f1vPGihUrFBYWpp9//lnp6en66aefdPfdd2vBggVenX/gwAGNHDlSo0ePljFGXbt21cyZMz0L7Ave5svNzdXBgwclSTk5OVq0aJGCg8//rcvSwrv5AACXnQs9yuBiGThwoPr06eO53de1a1elpKSobdu2kk4Hrg8++EDNmzfX008/rY4dO8rPz0/h4eGKjY3VgAEDNGzYME2dOlXz58/Xe++9p5EjR+rYsWNq2LChZs2a5bNaw8LCFB4erubNm6thw4Zq166dpNPfsJsxY4aioqJUu3ZttWrVyqv+5syZo7vuuivfvrvvvltvvvmmBg8eXOg5x48fl8vl8jwaYfDgwXr88cclSUOHDtWOHTsUGhqqChUqaNiwYRo9erTn3BMnTqhbt27KycnRqVOn1LlzZw0bNqwkH8VFZ0priZPb7baJiYmlMjZwNQmaElSi89KeSPNxJUDJpaSkqGnTpqVdxhUjKytL1apVk7VWo0aNUqNGjTRmzJgLn3iVKOz3zRiz0Vpb6MO7uM0HAMBV5u2335bL5VLz5s119OjRc9Z8oXi4zQcAQBk2ceJEzZs3L9++fv366emnny5xn2PGjPHZTNShQ4d0xx13nLN/5cqVCggI8MkYZR1hCgCAMuzpp592FJwutoCAACUlJZV2GaWK23wAAAAOEKYAAAAcIEwBAAA4wJopAMDl50Mfv59tEG9CQ8kxMwUAgBf8/PzkcrkUHBysfv366dixYzpy5IhmzJjhk/4XLFig7du359v3yiuv6JZbblFISIjCwsL0+OOPKycnxyfjwXcIUwAAeKFKlSpKSkpScnKyKlasqJkzZ17UMDVz5kwtW7ZM3333nbZu3aqEhATVrVtXx48f98l48B3CFAAAxdS+fXulpqYqOjpaaWlpcrlcGjt2bKFtv/zyS3Xs2FH33HOPGjdurOjoaMXFxalVq1YKCQlRWlqa1q1bp4ULF2rs2LFyuVxKS0vTxIkT9eabb6pmzZqSpIoVKyo6OlrVq1eXJC1btkxt27ZVRESE+vXr53k/YIMGDTR+/HhFREQoJCREP/zwg6TT777r3bu3QkND1aZNG23ZskWSFBMTo/vuu09du3ZVgwYN9Mknn+jJJ59USEiIoqKiPDNhGzduVMeOHdWiRQt169ZN+/fvv6if8eWEMAUAQDHk5uZqyZIlCgkJ0aRJkxQUFKSkpCRNnjy5yHM2b96s119/XVu3btXs2bO1Y8cObdiwQUOHDtUbb7yhW2+9VT179tTkyZOVlJSkunXrKisrS4GBgYX2d/DgQb3wwgtasWKFvv/+e7ndbr3yyiue47Vr19b333+vhx56SFOmTJEkjR8/XuHh4dqyZYtefPFF/fWvf/W0T0tL0+eff67PPvtMf/nLX9SpUydt3bpVVapU0eeff66cnBw98sgjmj9/vjZu3KgHHnigTD/76lJjAToAAF44+9Je6fTM1IMPPqh9+/Z5dW7Lli1Vr149SVJQUJC6du0qSQoJCdHq1avPaW+tlTH/b5H9F198oXHjxunIkSP68MMPdfjwYW3fvt3z8uKTJ096XrYsSX369JEktWjRQp988okk6ZtvvtHHH38sSbr99tt16NAhHT16VJLUvXt3VahQQSEhITp16pSioqI89aWnp+vHH39UcnKyunTpIkk6deqU53pAmAIAwCtn10yVRKVKlTw/lytXzrNdrlw55ebmntO+evXquuaaa7R7924FBgaqW7du6tatm3r06KGTJ0/KWqsuXbpozpw55x3Pz8/P07+1535j8Wxgy1tPhQoVPPvP1metVfPmzfXtt9+W6PqvdIQpAMDlp4w8ysDf31+ZmZkXpa+nnnpKDz30kOLj41WzZk1Za5WdnS1JatOmjUaNGqXU1FTdfPPNOnbsmDIyMtS4ceMi++/QoYPi4uL07LPP6ssvv1Tt2rU9668upEmTJjpw4IC+/fZbtW3bVjk5OdqxY4eaN2/u7KKvEKyZAgCghAICAtSuXTsFBwcXuQDdWwMGDNDkyZMVHh6utLQ0PfTQQ+rcubNat26t0NBQtWvXTuHh4QoPD1edOnUUGxurgQMHehaUn11oXpSYmBglJiYqNDRU0dHReu+997yurWLFipo/f77GjRunsLAwuVwurVu3ztH1XklMYdN+l4Lb7baJiYmlMjZwNQmaElSi89KeSPNxJUDJpaSkqGnTpqVdBq4Shf2+GWM2WmvdhbVnZgoAAMAB1kwBAOADW7du1eDBg/Ptq1SpktavX19KFeFSIUwBAOADISEhJf62Hy5v3OYDAABwgDAFAADgAGEKAADAAdZMAQAuOyV95EdReBQInGBmCgAAL/j5+cnlcik4OFj9+vXTsWPHdOTIEc2YMcPrc10ul3r27OnZP23aNN18880yxujgwYPn7SM2NlbGGK1cudKz79NPP5UxRvPnzy/5hcExwhQAAF44+26+5ORkVaxYUTNnzvQ6TJ09NykpSQsXLvTsb9eunVasWKGbbrrJqxpCQkLyvY8vPj5eYWFhxb8Y+BRhCgCAYmrfvr1SU1MVHR2ttLQ0uVyuEr1OJjw8XA0aNCjWuBs2bFBOTo6ysrKUmpoql8vlOT5hwgS1bNlSwcHBGj58uOflxqmpqercubPCwsIUERGhtDRua/oSYQoAgGLIzc3VkiVLFBISokmTJikoKEhJSUmaPHlykedkZ2fL7XarTZs2WrBgQYnHNsaoc+fO+uKLL/TZZ5/lu2UoSaNHj1ZCQoKSk5N1/PhxLVq0SJJ07733atSoUdq8ebPWrVunevXqlbgGnIswBQCAF44fPy6XyyW3260bb7xRDz74oNfn7tmzR4mJifrwww/12GOPOZoZGjBggOLj4xUfH6+BAwfmO7Z69Wq1bt1aISEhWrVqlbZt26bMzEzt3btXd911lySpcuXKqlq1aonHx7n4Nh8AAF44u+6pJK677jpJUsOGDRUZGalNmzYpKKhk30hs1aqVkpOTVaVKFTVu3NizPzs7Ww8//LASExN1ww03KCYmRtnZ2Z5bfbh4CFMAgMtOWXmUgb+/vzIzM8/b5rffflPVqlVVqVIlHTx4UGvXrtWTTz7paNyXXnpJlStXzrcvOztbklS7dm1lZWVp/vz56tu3r6pXr6769etrwYIF6t27t06cOKFTp04xO+VD3OYDAKCEAgIC1K5dOwUHBxe5AD0lJUVut1thYWHq1KmToqOj1axZM0nS1KlTVb9+fWVkZCg0NFRDhw71atzu3burU6dO+fbVrFlTw4YNU0hIiHr37q2WLVt6js2ePVtTp05VaGiobr31Vv3nP/8p4RWjMKa0pv/cbrdNTEwslbGBq0lJH25YVv7PH5BOB5KmTZuWdhm4ShT2+2aM2WitdRfWnpkpAAAAB1gzBQCAD2zdulWDBw/Ot69SpUpav359sfqZNWuWXn/99Xz72rVrp+nTpzuuERcHYQoAAB8ICQkp8bf98rr//vt1//33+6AiXCrc5gMAAHDAqzBljIkyxvxojEk1xkQX0eYeY8x2Y8w2Y8yHvi0TAACgbLrgbT5jjJ+k6ZK6SMqQlGCMWWit3Z6nTSNJT0lqZ639zRhT92IVDAAAUJZ4MzPVSlKqtXaXtfakpHhJvQq0GSZpurX2N0my1v7q2zIBAMjDGN/+ARzwJkxdL+nnPNsZZ/bl1VhSY2PMWmPMd8aYqMI6MsYMN8YkGmMSDxw4ULKKAQAoBX5+fnK5XAoODla/fv107NgxHTlyRDNmzPD6XJfLle/lxNOmTdPNN98sY4wOHjx43j5iY2NljNHKlSs9+z799FMZYzR//vwSX9err76qypUr6+jRo8U+NzY2VqNHj5YkxcTEaMqUKZKkIUOGKDAw0HPNt956qyTpl19+UY8ePRQWFqZmzZrpT3/6U4nrLku8CVOFRfaCT/osL6mRpEhJAyX9yxhT85yTrH3LWuu21rrr1KlT3FoBACg1Z9/Nl5ycrIoVK2rmzJleh6mz5yYlJWnhwoWe/e3atdOKFSt00003eVVDSEiI5syZ49mOj49XWFhY8S8mjzlz5qhly5b69NNPHfVT0OTJkz3XvG7dOknSc889py5dumjz5s3avn27Jk2a5NMxS4s3YSpD0g15tutL2ldIm8+stTnW2t2SftTpcAUAwBWnffv2Sk1NVXR0tNLS0uRyuYp8ncz5hIeHq0GDBsUad8OGDcrJyVFWVpZSU1Plcrk8xydMmKCWLVsqODhYw4cP97zkODU1VZ07d1ZYWJgiIiKUlnb6DQdpaWnKysrSCy+8kC+kxcbGqk+fPoqKilKjRo3yvUtw1qxZaty4sTp27Ki1a9cW63r379+v+vXre7ZDQ0OLdX5Z5U2YSpDUyBgTaIypKGmApIUF2iyQ1EmSjDG1dfq23y5fFgoAQFmQm5urJUuWKCQkRJMmTVJQUJCSkpI0efLkIs/Jzs6W2+1WmzZttGDBghKPbYxR586d9cUXX+izzz7Ld8tQkkaPHq2EhAQlJyfr+PHjWrRokSTp3nvv1ahRo7R582atW7dO9erVk3R6VmrgwIFq3769fvzxR/366/9b8pyUlKS5c+dq69atmjt3rn7++Wft379f48eP19q1a7V8+XJt375dRRk7dqznNt+9994rSRo1apQefPBBderUSRMnTtS+fQXnZi5PFwxT1tpcSaMlfSEpRdJH1tptxpgJxpiz/xS/kHTIGLNd0mpJY621hy5W0QAAXGrHjx+Xy+WS2+3WjTfeqAcffNDrc/fs2aPExER9+OGHeuyxxzwzQyUxYMAAxcfHKz4+XgMHDsx3bPXq1WrdurVCQkK0atUqbdu2TZmZmdq7d6/uuusuSVLlypVVtWpVSadvEw4YMEDlypVTnz59NG/ePE9fd9xxh2rUqKHKlSurWbNm+umnn7R+/XpFRkaqTp06qlixomxegOwAACAASURBVPr3719knXlv88XFxUmSunXrpl27dmnYsGH64YcfFB4erithDbVXT0C31i6WtLjAvufy/GwlPX7mDwAAV5yz655K4rrrrpMkNWzYUJGRkdq0aZOCgkr2EvJWrVopOTlZVapUUePGjT37s7Oz9fDDDysxMVE33HCDYmJilJ2d7bnVV9CWLVu0c+dOdenSRZJ08uRJNWzYUKNGjZJ0+lU4Z/n5+Sk3N1fS6dkxJ2rVqqVBgwZp0KBB6tGjh9asWaO7777bUZ+ljSegAwAuP9b69k8J+fv7KzMz87xtfvvtN504cUKSdPDgQa1du1bNmjUr8ZiS9NJLL+nFF1/Mty87O1uSVLt2bWVlZXm+4Ve9enXVr1/fc3vxxIkTOnbsmObMmaOYmBilp6crPT1d+/bt0969e/XTTz8VOW7r1q315Zdf6tChQ8rJyck3k+WNVatW6dixY5KkzMxMpaWl6cYbbyxWH2URYQoAgBIKCAhQu3btFBwcXOQC9JSUFLndboWFhalTp06Kjo72hKmpU6eqfv36ysjIUGhoqIYOHerVuN27d1enTp3y7atZs6aGDRumkJAQ9e7dWy1btvQcmz17tqZOnarQ0FDdeuut+s9//qP4+HjPrb+z7rrrLsXHxxc5br169RQTE6O2bduqc+fOioiIKLJt3jVTLpdLJ0+e1MaNG+V2uxUaGqq2bdtq6NCh+eq8XJmipv8uNrfbbRMTE0tlbOBqEjSlZLcS0p4o+ZoOwNdSUlLUtGnT0i4DV4nCft+MMRutte7C2jMzBQAA4IBXC9ABAMD5bd26VYMHD863r1KlSlq/fn2x+pk1a5Zef/31fPvatWun6dOnO64RFwdhCgAAHwgJCSnxt/3yuv/++3X//ff7oCJcKtzmAwAAcIAwBQAA4ABhCgAAwAHWTAEALjs3vX7hNsXx09982x+uLsxMAQDgBT8/P7lcLgUHB6tfv346duyYjhw5ohkzZnh9rsvlyvdy4mnTpunmm2+WMUYHDx48bx+xsbEyxmjlypWefZ9++qmMMZ6nnQ8dOvS8Lx+Oi4tTaGio5+GdmzdvvmDtuDDCFAAAXjj7br7k5GRVrFhRM2fO9DpMnT03KSlJCxcu9Oxv166dVqxYoZtuusmrGkJCQjRnzhzPdnx8vMLCwjzb//rXv877qprAwEB99dVX2rJli5599lkNHz7cq3FxfoQpAACKqX379kpNTVV0dLTS0tLkcrmKfJ3M+YSHh6tBgwbFGnfDhg3KyclRVlaWUlNT5XK5PMcjIyN19u0iS5cuVUREhMLCwnTHHXdIkm699VZde+21kqQ2bdooIyOj2DXjXKyZAgCgGHJzc7VkyRJFRUWpe/fuSk5OvuDzpbKzs+V2u1W+fHlFR0erd+/eJRrbGKPOnTvriy++0NGjR9WzZ0/t3r37nHYHDhzQsGHDtGbNGgUGBurw4cPntHnnnXfUvXv3EtWB/AhTAAB44fjx455ZoPbt2+vBBx/Uvn37vDp3z549uu6667Rr1y7dfvvtCgkJUVBQyd6bOWDAAE2dOlVHjx7V//7v/+rFF188p813332nDh06KDAwUJJUq1atfMdXr16td955R998802JakB+hCkAALxwdt1TSVx33XWSpIYNGyoyMlKbNm0qcZhq1aqVkpOTVaVKFTVu3LjQNtZaGWMKPbZlyxYNHTpUS5YsUUBAQIlqQH6EKQDAZaesPMrA399fmZmZ523z22+/qWrVqqpUqZIOHjyotWvX6sknn3Q07ksvvaTKlSsXebxt27YaNWqUdu/e7bnNV6tWLe3Zs0d9+vTR7NmziwxiKD4WoAMAUEIBAQFq166dgoODi1yAnpKSIrfbrbCwMHXq1EnR0dGeb9xNnTpV9evXV0ZGhkJDQzV06FCvxu3evbs6depU5PE6derorbfeUp8+fRQWFqb+/ftLkiZMmKBDhw7p4YcflsvlktvtLuYVozDGWlsqA7vdbnv2GwcALp6gKSW7lZD2RJqPKwFKLiUlRU2bNi3tMnCVKOz3zRiz0VpbaPpkZgoAAMAB1kwBAOADW7du1eDBg/Ptq1SpktavX1+sfmbNmqXXX8//vpx27dpp+vTpjmvExUGYAgDAB0JCQkr8bb+87r//ft1///0+qAiXCrf5AAAAHCBMAQAAOECYAgAAcIA1UwCAy87zzz/v0/7Gjx/v0/5wdWFmCgAAL6Snpys4ODjfvpiYGE2ZMqWUKiq52NhYjR49usjjMTExuv766+VyudSsWTPNmTPHZ2M3aNBABw8e9Fl/ZQFhCgAAnGPMmDFKSkrSZ599phEjRignJ6e0SyqzCFMAADgUGRmpcePGqVWrVmrcuLG+/vprSadngHr37q0777xTgYGBmjZtml555RWFh4erTZs2Onz4sCTp7bffVsuWLRUWFqa7775bx44dkyQNGTJEI0eOVPv27dW4cWMtWrTI02+vXr0UFRWlJk2a5Lvt+cEHH6hVq1ZyuVwaMWKETp06Jen086saN26sjh07au3atV5fW6NGjVS1alX99ttvkqS0tDRFRUWpRYsWat++vX744QdJ0r///W+1bt1a4eHh6ty5s3755RdJ0qFDh9S1a1eFh4drxIgROvvmld9//11//vOfFRYWpuDgYM2dO7fEn39pI0wBAOADubm52rBhg1577bV84SY5OVkffvihNmzYoKefflpVq1bVpk2b1LZtW73//vuSpD59+ighIUGbN29W06ZN9c4773jOT09P11dffaXPP/9cI0eOVHZ2tiRpw4YNiouLU1JSkubNm6fExESlpKRo7ty5Wrt2rZKSkuTn56e4uDjt379f48eP19q1a7V8+XJt377d6+v6/vvv1ahRI9WtW1eSNHz4cL3xxhvauHGjpkyZoocffliSdNttt+m7777Tpk2bNGDAAL388suSTq9vu+2227Rp0yb17NlTe/bskSQtXbpU1113nTZv3qzk5GRFRUU5+PRLFwvQAQDwgjHmvPv79OkjSWrRooXS09M9xzt16iR/f3/5+/urRo0auvPOOyWdfsjnli1bJJ0OXM8884yOHDmirKwsdevWzXP+Pffco3LlyqlRo0Zq2LChZyaoS5cuCggI8Iz9zTffqHz58tq4caNatmwpSTp+/Ljq1q2r9evXKzIyUnXq1JEk9e/fXzt27Djv9b766qt6++23tWvXLi1dulSSlJWVpXXr1qlfv36edidOnJAkZWRkqH///tq/f79OnjypwMBASdKaNWv0ySefSJL+/Oc/69prr/Vc/xNPPKFx48apR48eat++/XnrKcuYmQIAwAsBAQGeW11nHT58WLVr15Z0+tUxkuTn56fc3FxPm7P7JalcuXKe7XLlynnaDRkyRNOmTdPWrVs1fvx4z+yTdG6IO7td2H5rre677z4lJSUpKSlJP/74o2JiYgptfyFjxozRjz/+qLlz5+qvf/2rsrOz9ccff6hmzZqe/pOSkpSSkiJJeuSRRzR69Ght3bpV//znP897DZLUuHFjbdy4USEhIXrqqac0YcKEYtVXljAzBQC47JTGowyqVaumevXqaeXKlbrjjjt0+PBhLV26VH/72980a9YsR31nZmaqXr16ysnJUVxcnK6//nrPsXnz5um+++7T7t27tWvXLjVp0kSbNm3S8uXLdfjwYVWpUkULFizQu+++q6pVq6pXr14aM2aM6tatq8OHDyszM1OtW7fW3/72Nx06dEjVq1fXvHnzFBYW5lVtffr00Xvvvaf33ntPI0aMUGBgoObNm6d+/frJWqstW7YoLCxMR48e9dT93nvvec7v0KGD4uLi9Mwzz2jJkiWeQLpv3z7VqlVLf/nLX1StWjXFxsY6+gxLE2EKAAAvvf/++xo1apT++7//W9LpUBcUFOS43//5n/9R69atddNNNykkJESZmZmeY02aNFHHjh31yy+/aObMmapcubKk02uUBg8erNTUVA0aNEhut1uS9MILL6hr1676448/VKFCBU2fPl1t2rRRTEyM2rZtq3r16ikiIsKzMN0bzz33nAYNGqRhw4YpLi5ODz30kF544QXl5ORowIABCgsLU0xMjPr166frr79ebdq00e7duz2f0cCBAxUREaGOHTvqxhtvlHT6xdBjx45VuXLlVKFCBb355puOP8fSYs6uqr/U3G63TUxMLJWxgatJ0JSS/UWf9kSajysBSi4lJUVNmzYt7TIuuSFDhqhHjx7q27dvvv2xsbFKTEzUtGnTSqmyK1thv2/GmI3WWndh7VkzBQAA4AC3+QAAKKOKWkc0ZMgQDRkyxHH/EydO1Lx58/Lt69evn55++mnHfV9NCFMAgMuCtbbY30jD+T399NMEpwJKsvyJ23wAgDKvcuXKOnToUIn+Qwd4y1qrQ4cOeRb5e4uZKQBAmVe/fn1lZGTowIEDpV0KrnCVK1dW/fr1i3WOV2HKGBMl6XVJfpL+Za2dVOD4EEmTJe09s2uatfZfxaoEAIAiVKhQwfNEbaCsuWCYMsb4SZouqYukDEkJxpiF1tqCL/aZa60dfRFqBAAAKLO8WTPVSlKqtXaXtfakpHhJvS5uWQAAAJcHb8LU9ZJ+zrOdcWZfQXcbY7YYY+YbY24orCNjzHBjTKIxJpH73gAA4ErgTZgq7HuoBb9O8W9JDay1oZJWSHrv3FMka+1b1lq3tdZ99s3VAAAAlzNvwlSGpLwzTfUl7cvbwFp7yFp74szm25Ja+KY8AACAss2bMJUgqZExJtAYU1HSAEkL8zYwxtTLs9lTUorvSgQAACi7LvhtPmttrjFmtKQvdPrRCO9aa7cZYyZISrTWLpT0qDGmp6RcSYclDbmINQMAAJQZXj1nylq7WNLiAvuey/PzU5Ke8m1pAAAAZR+vkwEAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHypd2AQAursfHflayE5/wbR0AcKViZgoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAa/ClDEmyhjzozEm1RgTfZ52fY0x1hjj9l2JAAAAZdcFw5Qxxk/SdEndJTWTNNAY06yQdv6SHpW03tdFAgAAlFXezEy1kpRqrd1lrT0pKV5Sr0La/Y+klyVl+7A+AACAMs2bMHW9pJ/zbGec2edhjAmXdIO1dtH5OjLGDDfGJBpjEg8cOFDsYgEAAMoab8KUKWSf9Rw0ppykVyX994U6sta+Za11W2vdderU8b5KAACAMsqbMJUh6YY82/Ul7cuz7S8pWNKXxph0SW0kLWQROgAAuBp4E6YSJDUyxgQaYypKGiBp4dmD1tqj1tra1toG1toGkr6T1NNam3hRKgYAAChDLhimrLW5kkZL+kJSiqSPrLXbjDETjDE9L3aBAAAAZVl5bxpZaxdLWlxg33NFtI10XhYAAMDlgSegAwAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOeBWmjDFRxpgfjTGpxpjoQo6PNMZsNcYkGWO+McY0832pAAAAZc8Fw5Qxxk/SdEndJTWTNLCQsPShtTbEWuuS9LKkV3xeKQAAQBnkzcxUK0mp1tpd1tqTkuIl9crbwFr7f3k2r5FkfVciAABA2VXeizbXS/o5z3aGpNYFGxljRkl6XFJFSbf7pDoAAIAyzpuZKVPIvnNmnqy10621QZLGSXqm0I6MGW6MSTTGJB44cKB4lQIAAJRB3oSpDEk35NmuL2nfedrHS+pd2AFr7VvWWre11l2nTh3vqwQAACijvAlTCZIaGWMCjTEVJQ2QtDBvA2NMozybf5a003clAgAAlF0XXDNlrc01xoyW9IUkP0nvWmu3GWMmSEq01i6UNNoY01lSjqTfJN13MYsGAAAoK7xZgC5r7WJJiwvsey7Pz3/zcV0AAACXBZ6ADgAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwwKswZYyJMsb8aIxJNcZEF3L8cWPMdmPMFmPMSmPMTb4vFQAAoOy5YJgyxvhJmi6pu6RmkgYaY5oVaLZJkttaGyppvqSXfV0oAABAWeTNzFQrSanW2l3W2pOS4iX1ytvAWrvaWnvszOZ3kur7tkwAAICyyZswdb2kn/NsZ5zZV5QHJS0p7IAxZrgxJtEYk3jgwAHvqwQAACijvAlTppB9ttCGxvxFklvS5MKOW2vfsta6rbXuOnXqeF8lAABAGVXeizYZkm7Is11f0r6CjYwxnSU9LamjtfaEb8oDAAAo27yZmUqQ1MgYE2iMqShpgKSFeRsYY8Il/VNST2vtr74vEwAAoGy6YJiy1uZKGi3pC0kpkj6y1m4zxkwwxvQ802yypGqS5hljkowxC4voDgAA4IrizW0+WWsXS1pcYN9zeX7u7OO6AAAALgs8AR0AAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwgTAEAADhAmAIAAHCAMAUAAOAAYQoAAMABwhQAAIADhCkAAAAHCFMAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcMCrMGWMiTLG/GiMSTXGRBdyvIMx5ntjTK4xpq/vywQAACibLhimjDF+kqZL6i6pmaSBxphmBZrtkTRE0oe+LhAAAKAsK+9Fm1aSUq21uyTJGBMvqZek7WcbWGvTzxz74yLUCAAAUGZ5c5vvekk/59nOOLMPAADgqudNmDKF7LMlGcwYM9wYk2iMSTxw4EBJugAAAChTvAlTGZJuyLNdX9K+kgxmrX3LWuu21rrr1KlTki4AAADKFG/CVIKkRsaYQGNMRUkDJC28uGUBAABcHi64AN1am2uMGS3pC0l+kt611m4zxkyQlGitXWiMaSnpU0nXSrrTGPO8tbb5Ra0cuMw9//zzxT5n/PjxF6ESAIAT3nybT9baxZIWF9j3XJ6fE3T69h8AAMBVhSegAwAAOODVzBQA33u3ZvFv2XGTDwDKHsIUUErK5wSV4Kw0n9cBAHCG23wAAAAOEKYAAAAcIEwBAAA4QJgCAABwgDAFAADgAGEKAADAAcIUAACAA4QpAAAABwhTAAAADhCmAAAAHCBMAQAAOECYAgAAcIAwBQAA4ABhCgAAwAHCFAAAgAOEKQAAAAcIUwAAAA4QpgAAABwoX9oFALhyPP/888U+Z/z48RehEgC4dAhTQCl5fOxnxT/pCd/XAQBwhjAFwGfev+b9Yp8zXsxMAbi8sWYKAADAAcIUAACAA4QpAAAAB1gzBeTBt9EAAMXFzBQAAIADhCkAAAAHCFMAAAAOEKYAAAAcYAE6kAcPnXSGp7oDuBoRpgBcdkryrUuJb14CuDgIUwAuO+NjYkp4ImEKgO8RpgBcdoImNyzReWk+rgMApKs8THGr4PLBwzQBAGVVmQxTV+J/OC/VNV3KgHgl/nMCAKC4ymSYwuXh3ZrFD0YljVKX6lt2fBsNBd30evHP+elvvq8DQNlFmLrClCR0SCULHuVzgkowEqtWcHnh9xzAhXgVpowxUZJel+Qn6V/W2kkFjleS9L6kFpIOSepvrU33bamXN55fBKCsYLYN8K0LhiljjJ+k6ZK6SMqQlGCMWWit3Z6n2YOSfrPW3myMGSDpH5L6X4yCcX4luk0llehW1aW8JcbtN+R1KX/PL6WgKcWfBUt7ovizYJdyto21lbgaeDMz1UpSqrV2lyQZY+Il9ZKUN0z1khRz5uf5kqYZY4y11pakqNoxdxf/pBL8u1eicUo4FmEAuDzxPw3OXKpZ+SvxyzclCddSyQL2pQry0pUZsM2F8o4xpq+kKGvt0DPbgyW1ttaOztMm+UybjDPbaWfaHCzQ13BJw89sNpH0YzHrrS3p4AVb+caVOBbXdHmMdSVe06Uci2u6PMa6Eq/pUo7FNV36sW6y1tYp7IA3M1OmkH0FE5g3bWStfUvSW16MWXghxiRaa90lPf9qH4trujzGuhKv6VKOxTVdHmNdidd0KcfimsrWWOW8aJMh6YY82/Ul7SuqjTGmvKQakg77okAAAICyzJswlSCpkTEm0BhTUdIASQsLtFko6b4zP/eVtKqk66UAAAAuJxe8zWetzTXGjJb0hU4/GuFda+02Y8wESYnW2oWS3pE02xiTqtMzUgMuUr0lvkXIWJd0nEs5FtfEWKU1zqUci2tirNIa51KOddle0wUXoAMAAKBo3tzmAwAAQBEIUwAAAA4QpgAAABwgTAG46IwxdUu7Bl8zxgSUdg0AygbCVBH4i7LsMcbUMMZMMsb8YIw5dOZPypl9NS9hHUt82Fd1Y8xLxpjZxphBBY7N8NU4Z/r7/4wxbxpjphtjAowxMcaYrcaYj4wx9Xw4Tq0CfwIkbTDGXGuMqeWrcc6MFZXn5xrGmHeMMVuMMR8aY/7Lh+NMMsbUPvOz2xizS9J6Y8xPxpiOvhrnTP/fG2OeMcaU7F0ixRvLbYxZbYz5wBhzgzFmuTHmqDEmwRgT7sNxqhljJhhjtp3p/4Ax5jtjzBBfjZFnrPLGmBHGmKVnfhc2G2OWGGNGGmMq+Hq8Imrw7TfFjPE7c03/Y4xpV+DYMz4cp6ox5kljzFhjTGVjzBBjzEJjzMvGmGq+Guc84++4SP2G5vm5wpl/vxYaY140xlT1xRhlNkxdqr8kz/R/xf1Fean+kjwz1qX6i/IjSb9JirTWBlhrAyR1OrNvni8HMsZEFPGnhSSXD4eapdNvEPhY0gBjzMfGmEpnjrXx4TiSFKvT79T8WdJqSccl/VnS15Jm+nCcg5I25vmTKOl6Sd+f+dmXXszz8/9K2i/pTp1+Pt4/fTjOn/O8HmuypP7W2pt1+gXw/+vDcSTpWkk1Ja02xmwwxowxxlzn4zHOmiHpZUmfS1on6Z/W2hqSos8c85U4SbskdZP0vKSpkgbr/2/v3GPtqKow/vsotSLyUNAiGLiVUq2CbaCIRkBAUgtBMECtlQSlSBETUKAGtQhieFNaLEGDAvISiKCBqKVqQPDV0tZy+wjQGmxBIDVGkRaLgbaff+x9muFwb1s4+wznnq5fMrkze5+Zb9acPfuuWXufWXC4pEs3teMb4DbSPfod4GhSG78IGAXcXkqkj4eG6sPD0aV0MtcDnwD+BcyUNL1Sd3xBnZuBocAwUpsYA0wj9VE/KKiDpDWSVudljaQ1wN6N8pJaJLsaXA4MJ92321Gq77PdkQuwsLJ+A3AxsBdwNnBvYa0llfXfAQfm9RGkd2mV1FpBapxPA/OyPbu34frNA44CJpL+eZ6Yyz8JzCmsdR/wRdLb8c8Bvg3sA9wCXFpQZ9kbqXuDWuuBB3N7aF5eKqjT27Q9FfgTsEv1Hiik9Whl/elNnUeLOlOA2cB+lbIVJW2pHLfaTzRfy5I2PQFsm9fnNtUtKaXTh02HkJyaVbntTa6xTTxaUGdR0/b8/Hcb4InCNm2qn1heUGc9yUFcUVka2y8XtmlxZX1b0juSfg4MKfw99ea/ym1Ole3FpXTyMa8FbgWGVspWlNSoHLfaznuBwaXt6tjIVBNjbJ9v+ynbM4CewscfrJQGB2A72/MBbC8nNdaSPG97iu09gXNJTsfCHEWavJl9Xw+Dbd9v+07Atu8hrTwAvLWgDkCP7ZttP2N7OnCs7b8Cp1D2qempHILeGJmUNFTSeSSHsSSPA6fbPrx5oWwiziGSNt6Hti8hdZS/JzlUJane77c21Q0qJWJ7GvAl4AJJ0yXtQB+5OgvxbknnSDoX2FFSNU9oyf7tOmCWpCOA2ZKukXSopItInXNbsP0H218hRfauAD5WWOJ/ksZKGg9Y0mcAckR+fUGd/0o6OB/70+R0Y7Y30Hdu11Z4XtL46n0laRtJE0hR7FL8jRQlH1ZZ3md7GPCPgjoAb2ms2F5nezKwiPTAV3z4zcnTmJX/NraL3sO2zwS+B9wp6az8fbWrn9hJ0vGSTgCG2H4ln0MxuzrZmaqrk4Tu7Cjr6iShvo5yAsnBeFjS85L+DTwEvBP4bEEdSEME/bWzMwvq/AI4olpgwWHntgAABfxJREFU+xaSo/1yQR2A+xrzHmxvnGchaTiwrKRQdqzHk6IpvwWKzEvogx8BO5D+odxCygSPpN0oeO/avpY0pHg6cBwpwvsN4FlgUimdzGvmjdheb3u27VMKa32Z1NYmkYbgDpf0H1I07KyCOmcA0/Oxz2scW9K7SP1vST5HSmu2StLyPA9nFenBrmR2jmtIQ7J9cWVBHYAF1akvALYvIk0T6Cms0+gjNrbrPC1lTUEdssZfgCPz5sOUf9Bv8DBp+P8YYG7jgTz3E0Uejjv2DeiSLmwq+r7tf2bjr7R9cmG9w0g3/AhSGPXvwL2k9DnrCurcZbtd6XaqOqNIN/QG0lDiGaT8ic8Cp9n+c0GtD5OGYkcAS4FJtpfnjnKi7ZkFtT5AGk6ca/vFSvk427NL6VS09gAeaafWJnSOsl1ssvtmtNpmE8l539v20i79ngasTfmYI4Hd262Vdfagnnv3IFLE4UlgJGn+4WO2ZxXW+QgpwDFf0geBcaRhy6I6dWr1o7OMSqSqTVqHkObALmjT9TsI2NC261dirLDuBTgltDpfp7QW6Wl2GcnJXQkcV6krPb+oFi1SlKsum2rRqvl7Cptat+uJmq5f23Xy8S4E5pJ+7HAZ8ABwAWnofGobdR5sh06dWm+yTW35nurSKnaydS40TZQMrc7UKa0FLAHentd78o3x1bxdbBJmnVph08DQ6kabuvz6DSINLa8Gdszl21FwEnVdOmHTwNBqTLruOCQt7q+K9NPN0OoAnZq1BjkPD9hemYdm75G0F+UnsdalFTYNDK1utKlOrTptWmd7PbBW0pO2V2fdlyRtGIA6dWp1o021aHWsM0X6J/wpXvvrC5HehxJanaFTp9YqSaNt9wLYflHSMcBNwH4FderUCpsGhlY32lSnVp02vSzpbbbXAgc0CiXtRJpDOtB06tTqRpvq0SoZSisclrsROLifujtCqzN0arbpvcBu/dR9vLBNtWiFTQNDqxtt6uLrN6Sf8l2pvPtsoOiETQNDq2N/zRcEQRAEQTAQ6OT3TAVBEARBEHQ84UwFQRAEQRC0QDhTQRAEQRAELRDOVBAEbyqSHpK0TNIiSfMlja7UrZT0s8r2iZJubtr/PklztkDn/VmrV9Ljkn6Yy8dIKvaW/iAItj7CmQqCoBM4yfYoUk64q5rqxkj6UF87SdoZ2B/YWdKwzWjMBGbYHm17JClrPbYX2C6Zhy4Igq2McKaCIKgNSdtL+lWOQi2VNKHpI3NI+duqTAO+1c8hTyAli76LzSexfQ/wTGPD9pJ8TodJ+mVen5UjV72SXpD0BUmDJF2Vo2aLJZ2+ZdYGQbC1EM5UEAR1Mg54zvYo2/sCzQlux5Hyt1X5KbC/pOF9HG8icGdeJm5GewbwoKT7JZ2do1qvwvbRtkcDpwJP5XM5FXjB9oHAgcBpWxAFC4JgKyKcqSAI6mQJcKSkKyQdYvuFXP4TSc8A55GH3yqsJw39fbNaKGkoMBz4o+3lwDpJ+/YnbPvHwEjgbuAwYK6kIc2fk7QrcBvw+Xx+Y4GTJfUCjwC7APu8PrODIOhmwpkKgqA2stNzAMmpukzSBbnqJGAYcAdwXR+73gYcCuxZKZsAvANYIWklKYHuJof6bD9n+ybbxwHrgFc5X5IGkYYMv2t7aaMYODPPtRpte5jt32yhyUEQbAWEMxUEQW1I2h1Ya/t20lyo/Rt1tl8Bzgc+Kmlkdb9cNwP4WqV4IjDOdo/tHpKT1q8zJWmcpMF5fTdShOnZpo9dTsoif1el7NfAGZV9R0jafsutDoKg2wlnKgiCOtkPmJeHzKYCF1crbb8EXA1M6WPfG8nJ2SX1kKJUcyv7rgBWSzqoH+2xwFJJi0gO0tdtr2r6zBRgbGUS+rHADcBjwEJJS4Hr6ewk8UEQ1Ezk5guCIAiCIGiBiEwFQRAEQRC0QISqgyDoKiRNBcY3Fd9t+5I343yCIOh+YpgvCIIgCIKgBWKYLwiCIAiCoAXCmQqCIAiCIGiBcKaCIAiCIAhaIJypIAiCIAiCFghnKgiCIAiCoAX+D2jJ9d8afMJzAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 720x504 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "totalreads = 4554180  # ran wc -l *.fastq (=18216720)on the cleaned reads in the fastp folder (number of cleaned reads from insterts 15-35)\n",
    "\n",
    "fileextension='.fastq.gz'\n",
    "directory = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later'\n",
    "outplotname = 'Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors_Proportion.pdf'\n",
    "startseqlength = 15\n",
    "cleanreads=False\n",
    "blastdatabase = '/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabaselist=['/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2']  # '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1',\\\n",
    "\n",
    "colors = ['black', \\\n",
    "          'darkviolet', \\\n",
    "          'orange', \\\n",
    "          'forestgreen', \\\n",
    "          'red', \\\n",
    "          'dodgerblue', \\\n",
    "          'grey'] # this is length of align databases + 1 because we need a color for unmapped\n",
    "    \n",
    "# Vector, color=‘darkviolet’\n",
    "# MAC, color=‘forestgreen’\n",
    "# IES, color=‘red’\n",
    "# OES, color=‘dodgerblue’\n",
    "# Mitochondrial, color=‘orange’\n",
    "# Klebsiella, color=‘black’\n",
    "# Unmapped, color=‘grey’\n",
    "\n",
    "readsubtract.main(totalreads, fileextension, directory, outplotname, startseqlength, cleanreads, blastdatabase, aligndatabaselist, colors)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
