{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-12-08 13:40:01.046174\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import Tools\n",
    "from pygentoolbox import ReadSubtractionByGenomeMappingColors as readsubtract\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/media/sf_LinuxShare/PyPacks/PyGenToolbox/notebooks/Theresa\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.getcwd())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 54247\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1484\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 49144\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 5176\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 98\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 63610\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 939\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 57083\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 96\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 44791\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1531\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 35746\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 9263\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 35636\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 110\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 28156\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 18918\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 28032\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 133\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 27606\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1118\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 42646\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 859\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 28593\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 14618\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 28496\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 18208\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21941\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 18097\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 111\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 17648\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 837\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 45737\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 969\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26513\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 20508\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26423\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13953\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21846\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 13776\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 181\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 13313\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 824\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 52325\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 722\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 29502\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 23284\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 29409\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 93\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 15520\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 23043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 15315\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 210\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 14542\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1178\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 128252\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 566\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 81814\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 47344\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 81694\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 120\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 42688\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 56915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 42345\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 349\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 40944\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1922\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 378858\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 875\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 189392\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 193271\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 189221\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 171\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 33530\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 193913\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 32875\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 29658\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 4555\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3361812\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1288\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 1431194\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1943523\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 1430813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 381\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 152687\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 1306246\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 150602\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2112\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 138880\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 14607\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 184331\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 84813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 100062\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 84666\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 147\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 25585\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 85713\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 25049\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 543\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 19679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 5819\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 62181\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 626\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 41005\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 21230\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 40908\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13228\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 58401\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 12211\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1027\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 8594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 4664\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 37487\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 387\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26846\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 10648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26740\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 106\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 9227\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 30058\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 7882\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1357\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 6601\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1700\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 20619\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 522\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 13788\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6834\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 13698\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8080\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8088\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6177\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5302\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1075\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 19544\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 587\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14887\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4659\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 14779\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 108\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8016\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 17255\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6422\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1609\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5798\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 836\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 12489\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 403\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 10962\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1527\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 10894\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 6680\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 11981\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5937\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 750\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 413\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8192\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 492\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7149\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7081\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5375\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3620\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5251\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 124\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5110\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 198\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 308\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5863\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 731\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5793\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 70\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4402\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2505\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4356\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 46\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4284\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 73\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8735\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 184\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 8191\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 544\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 8107\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 84\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5231\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8842\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5218\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 13\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5111\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 117\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 7713\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 104\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7234\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 479\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7198\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 36\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4470\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8819\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4458\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 12\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4400\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 73\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5783\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5417\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 366\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5365\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 52\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3807\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2649\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3799\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 8\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3768\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 33\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 4347\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 45\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 3983\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 364\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 3925\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 58\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2681\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2738\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 44\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "wrote out table of counts aligned to each genome to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/ReadsMappedToEachReference.tsv\n",
      "\n",
      "Output Stacked Barplot to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors.pdf\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAm8AAAGtCAYAAABA/pOqAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzdeVhWZeL/8fct7rnlNj/TTCQ1leUBH7fMxNxwMre0tMbUXEubqflmYpvWaDmj02JqTn1LzExMK3PMNVssKxUThaQSlAx1yv0rKQp2//7g8AwYyCImBz+v6+KS5z73dp6L6+rTfc59jrHWIiIiIiLuUOZyT0BERERECk7hTURERMRFFN5EREREXEThTURERMRFFN5EREREXEThTURERMRFyl7uCfxeateubRs1anS5pyEiIiKSr23bth221tbJ7dgVE94aNWpETEzM5Z6GiIiISL6MMT/kdUyXTUVERERcROFNRERExEXyDW/GmIrGmC3GmB3GmG+MMU855VHGmL3GmFjnx+OUG2PMLGNMojFmpzEmLFtfQ40xu52fodnKWxlj4pw2s4wxximvaYxZ79Rfb4y5Or8xREREREqzgtzzdga4xVqbaowpB3xujFntHJtgrV12Xv2eQBPnpy3wMtDWGFMTmAx4AQtsM8assNYec+qMBr4CVgERwGogEthgrZ1ujIl0Pk/Ma4yifAEiIiLnS09PJyUlhbS0tMs9FSnlKlasSIMGDShXrlyB2+Qb3mzmm+tTnY/lnJ8Lvc2+D/CG0+4rY0wNY0w9IBxYb609CmCMWQ9EGGM+AapZa790yt8A+pIZ3vo47QAWAJ+QGd5yHcNae7CA5y0iIpKnlJQUqlatSqNGjXAuBokUO2stR44cISUlBX9//wK3K9A9b8YYP2NMLPAzmQFss3NomnPZ8nljTAWnrD7wY7bmKU7ZhcpTcikH+ENWIHP+rZvPGCIiIhctLS2NWrVqKbjJJWWMoVatWoVe4S1QeLPWnrPWeoAGQBtjTCAwCbgBaA3UJHNFDCC3v3RbhPILKVAbY8xoY0yMMSbm0KFD+XQpIiLyXwpu8nsoyt9ZoXabWmuPk3npMsJae9BmOgPMB9o41VKAa7M1awAcyKe8QS7lAD85l1xx/v05nzHOn+8r1lqvtdZbp06uz7kTERG5Yvj5+eHxeAgMDOS2227j+PHj+baZNWsWzZs35+677/4dZigFUZDdpnWMMTWc3ysBXYFvs4UqQ+Y9avFOkxXAPc6O0HbACeeS51qguzHmamfXaHdgrXPspDGmndPXPcD72frK2pU69Lzy3MYQERGRPFSqVInY2Fji4+OpWbMmc+bMybfN3LlzWbVqFYsWLSrQGBkZGRc7TclHQVbe6gEfG2N2AlvJvOdtJbDIGBMHxAG1galO/VXAHiAReBW4H8DZqPA3p4+twNNZmxeA+4D/ddokkblZAWA60M0Ysxvo5nzOcwwREREpmPbt27N//37f5xkzZtC6dWuCg4OZPHkyAGPHjmXPnj307t2b559/nl9++YV7772X1q1bExoayvvvZ66pREVFMXDgQG677Ta6d++eZ3/Jyck0b96cUaNG0bJlS7p3787p06cBSExMpGvXroSEhBAWFkZSUlKe/VzxrLVXxE+rVq2siIhIQezatetyT+GSuOqqq6y11mZkZNgBAwbY1atXW2utXbt2rR01apT99ddf7blz5+ytt95qP/30U2uttdddd509dOiQtdbaSZMm2YULF1prrT127Jht0qSJTU1NtfPnz7f169e3R44cuWB/e/futX5+fnb79u3WWmsHDhzo669Nmzb23XfftdZae/r0afvLL79ccF6lSW5/b0CMzSPTXDHvNhUREbnSnT59Go/HQ3JyMq1ataJbt24ArFu3jnXr1hEaGgpAamoqu3fv5uabb87Rft26daxYsYKZM2cCmbty9+3bB0C3bt2oWbPmBftr2LAh/v7+eDweAFq1akVycjInT55k//799OvXD8h89llh5nWlUXgTERG5QmTd83bixAl69erFnDlz+POf/4y1lkmTJjFmzJgLtrfW8s4779CsWbMc5Zs3b+aqq67KUS+3/pKTk6lQoYLvs5+fH6dPnyZzoSn38QoyryuN3m0qIiJyhalevTqzZs1i5syZpKen06NHD15//XVSUzOfyb9//35+/vnn37Tr0aMHL730ki9sbd++Pdf+C9pflmrVqtGgQQOWL18OwJkzZzh16lSh+7lSaOVNRETkChQaGkpISAjR0dEMGTKEhIQE2rdvD0CVKlV48803qVu3bo42TzzxBA8++CDBwcFYa2nUqBErV678Td/du3fPtT8/P78857Nw4ULGjBnDk08+Sbly5Vi6dGme/Zw/ryuNyWupsrTxer02Jibmck9DRERcICEhgebNm1/uacgVIre/N2PMNmutN7f6WnkTkWL11FNPFamdHgEgIlIwuudNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNREQkH8aYYv0piCpVqvh+X7VqFU2aNPG9R/RSSk9PJzIykiZNmhAYGEibNm1YvXr1JR0z+7nm5vjx48ydO7dAff3000/cddddNG7cmFatWtG+fXvee++94phmiaHwJiIiUoJt2LCBBx54gDVr1tCwYcNLPt4TTzzBwYMHiY+PJz4+nn//+9+cPHnyko97IQUNb9Za+vbty80338yePXvYtm0b0dHRpKSk/A6z/P0ovImIiJRQn332GaNGjeKDDz4gICAAgGHDhjF27Fg6duxI06ZNfa+nioqKon///kRERNCkSRMeeeQRXz/r1q2jffv2hIWFMXDgQN+7Qs936tQpXn31VV566SXfC+T/8Ic/cMcddwCwePFigoKCCAwMZOLEib52VapUYeLEibRq1YquXbuyZcsWwsPDady4MStWrPDNr0+fPkRERNCsWbM8H+g9Y8YMWrduTXBwsO/h3ZGRkSQlJeHxeJgwYUKe9T766CPKly/P2LFjff1dd911PPDAAwCcO3eOCRMm+Nr961//AuCTTz4hPDycAQMGcMMNN3D33Xf73t+6YcMGQkNDCQoK4t577+XMmTMANGrUiEcffZT27dvj9Xr5+uuv6dGjBwEBAcybN++C53OxFN5ERERKoDNnztCnTx+WL1/ODTfckONYcnIyn376KR988AFjx44lLS0NgNjYWJYsWUJcXBxLlizhxx9/5PDhw0ydOpUPP/yQr7/+Gq/Xy3PPPZfrmImJiTRs2JBq1ar95tiBAweYOHEiH330EbGxsWzdutX3IvlffvmF8PBwtm3bRtWqVXn88cdZv3497733Hk8++aSvjy1btrBo0SJiY2NZunQp57+2ct26dezevZstW7YQGxvLtm3b2LhxI9OnTycgIIDY2FhmzJiRZ71vvvmGsLCwPL/T1157jerVq7N161a2bt3Kq6++yt69ewHYvn07L7zwArt27WLPnj1s2rSJtLQ0hg0b5vtOMzIyePnll339XXvttXz55Zd07NiRYcOGsWzZMr766ivfOec1z4ul12OJiIiUQOXKlePGG2/ktdde48UXX8xx7I477qBMmTI0adKExo0b8+233wLQpUsXqlevDkCLFi344YcfOH78OLt27aJDhw4AnD171vei98LYunUr4eHh1KlTB4C7776bjRs30rdvX8qXL09ERAQAQUFBVKhQgXLlyhEUFERycrKvj27dulGrVi0A+vfvz+eff47X+9/Xd65bt45169YRGhoKQGpqKrt37/7N5eK86p1v3LhxfP7555QvX56tW7eybt06du7cybJlywA4ceIEu3fvpnz58rRp04YGDRoA4PF4SE5OpmrVqvj7+9O0aVMAhg4dypw5c3jwwQcB6N27t++cU1NTqVq1KlWrVqVixYocP348z3nefPPNhf7+s1N4ExERKYHKlCnD22+/TdeuXXnmmWd49NFHfcfO3/SQ9TnrUieAn58fGRkZWGvp1q0bixcvznfM66+/nn379nHy5EmqVq2a41jWZcTclCtXzjeHMmXK+OZRpkwZMjIy8p139jEmTZrEmDFjcpRnD4AXqrdhwwbeeecd3+c5c+Zw+PBhX0C01vLSSy/Ro0ePHO0++eSTPL+7C8l+ntnbZ513XvO8WLpsKiIiUkJVrlyZlStXsmjRIl577TVf+dKlS/n1119JSkpiz549NGvWLM8+2rVrx6ZNm0hMTAQy72v7/vvv8xxvxIgR/PnPf+bs2bMAHDx4kDfffJO2bdvy6aefcvjwYc6dO8fixYvp1KlToc5n/fr1HD16lNOnT7N8+XLfamCWHj168Prrr/vuydu/fz8///wzVatWzbFpIq96t9xyC2lpaTkubZ46dSpHu5dffpn09HQAvv/+e3755Zc853vDDTeQnJzs++4WLlxYqHPOa54XSytvIiIi+chvBeZSqlmzJmvWrOHmm2+mdu3aADRr1oxOnTrx008/MW/ePCpWrJhn+zp16hAVFcXgwYN9N9tPnTrVdynwfFOnTuXxxx+nRYsWVKxYkauuuoqnn36aevXq8eyzz9K5c2estfzxj3+kT58+hTqXm266iSFDhpCYmMhdd92V45IpQPfu3UlISPBd1q1SpQpvvvkmAQEBdOjQgcDAQHr27MmMGTNyrVe3bl2WL1/OQw89xD/+8Q/q1KnDVVddxd///ncARo4cSXJyMmFhYVhrqVOnju++vdxUrFiR+fPnM3DgQDIyMmjdunWOzRD5yet86tatW6jv7Xzmcv5B/p68Xq89/8ZIESl+172Yf53c/PCX4p2HyMVISEigefPml3sauRo2bBi9evViwIABl3sqhRIVFUVMTAyzZ8++3FMpcXL7ezPGbLPWenOrr5U3ESlWZdMDitgyqVjnISJSWim8iYiIuEhUVFSx9NOvXz/fYzKy/P3vf//NzfzFZdiwYQwbNuyS9H2lUXgTERG5ApW2V0ZdSbTbVERERMRFFN5EREREXEThTURERMRFFN5EREREXEQbFkRERPIxx8QXa3/jbOAFj4eHhzNp0qQcOz9feOEFvv/+e+bOnVvgcZYvX07Tpk1p0aJFkeeaZcqUKbz66qu+d5tGREQwffp0wsPDmTlz5m8euFsQuT377fz+tm/fTlhYGGvWrMl3J6yfnx9BQUGkp6dTtmxZhg4dyoMPPkiZMnmvVWUfLzw8nIMHD1KpUiUg8x2qF/tA3UtBK28iIiIlzODBg4mOjs5RFh0dzeDBgwvVz/Lly9m1a1eh2mR/F+n5HnroIWJjY4mNjWX69OmF6reoFi9ezE033VSgd7NWqlSJ2NhYvvnmG9avX8+qVat46qmnCjXeokWLfOdYEoMbKLyJiIiUOAMGDGDlypW+11klJydz4MABbrrpJmbMmEHr1q0JDg5m8uTJvjZvvPEGwcHBhISEMGTIEL744gtWrFjBhAkT8Hg8JCUlERsbS7t27QgODqZfv34cO3YMyFx9evTRR+nUqRMvvljE16QA9913H16vl5YtW+aY26pVq7jhhhu46aab+POf/0yvXr0K1J+1lmXLlhEVFcW6detIS0sr8Fzq1q3LK6+8wuzZs7HWcu7cOR5++GGCgoIIDg7mpZdeKvT5lRS6bCoiIlLC1KpVizZt2rBmzRr69OlDdHQ0d955J+vXr2f37t1s2bIFay29e/dm48aN1KpVi2nTprFp0yZq167N0aNHqVmzJr17987xKq2s0NKpUyeefPJJnnrqKV544QUAjh8/zqeffnrBeT3//PO8+eabQO4P9J02bRo1a9bk3LlzdOnShZ07d9K0aVPGjBnDxo0b8ff3/83q4ZIlS/j88899n7NeAg+wadMm/P39CQgIIDw8nFWrVtG/f/8Cf4+NGzfm119/5eeff+bdd99l7969bN++nbJly3L06NFc2wwfPhw/Pz9uv/12Hn/8cYwxBR7v96KVNxERkRIo+6XTrEum69atY926dYSGhhIWFsa3337L7t27+eijjxgwYIDvxfU1a9b8TX8nTpzg+PHjdOrUCYChQ4eyceNG3/E777wz3zllv2ya2/1nb7/9NmFhYYSGhvLNN9+wa9cuvv32Wxo3boy/v7/vvLK78847fX3GxsbmuHdu8eLFDBo0CIBBgwYV6NLp+bLe4f7hhx8yduxYypbNXLfK7TtatGgRcXFxfPbZZ3z22WcsXLiw0OP9HhTeRERESqC+ffuyYcMGvv76a06fPk1YWBjWWiZNmuQLOomJiYwYMQJr7UWvEF111VUX1X7v3r3MnDmTDRs2sHPnTm699VbS0tJ84amwzp07xzvvvMPTTz9No0aNeOCBB1i9ejUnT54scB979uzBz8+PunXrFug7ql+/PgBVq1blrrvuYsuWLUWa+6Wm8CYiIlICValShfDwcO69917falWPHj14/fXXSU1NBWD//v38/PPPdOnShbfffpsjR44A+C4JVq1a1Rd2qlevztVXX81nn30GwMKFC32rcMXh//7v/7jqqquoXr06P/30E6tXrwbghhtuYM+ePSQnJwOZl0kL4sMPPyQkJIQff/yR5ORkfvjhB26//XaWL19eoPaHDh1i7NixjB8/HmMM3bt3Z968eb4NGedfNs3IyODw4cMApKens3LlSgIDL7wr+HLRPW8iIiL5yO/RHpfK4MGD6d+/v+/yaffu3UlISKB9+/ZAZsB78803admyJY899hidOnXCz8+P0NBQoqKiGDRoEKNGjWLWrFksW7aMBQsWMHbsWE6dOkXjxo2ZP39+sc01JCSE0NBQWrZsSePGjenQoQOQuQN07ty5REREULt2bdq0aVOg/hYvXky/fv1ylN1+++28/PLLDBkyJNc2p0+fxuPx+B4VMmTIEP76178CMHLkSL7//nuCg4MpV64co0aNYvz48b62Z86coUePHqSnp3Pu3Dm6du3KqFGjivJVXHKmqMuZbuP1em1MTMzlnoZIqRcwM6BI7ZIeTirmmYgUXUJCAs2bN7/c0yg1UlNTqVKlCtZaxo0bR5MmTXjooYcu97RKjNz+3owx26y1uT48T5dNRURE5JJ69dVX8Xg8tGzZkhMnTjBmzJjLPSVX02VTERER8Zk2bRpLly7NUTZw4EAee+yxIvf50EMPFdtK25EjR+jSpctvyjds2ECtWrWKZYySTuFNREREfB577LGLCmqXWq1atYiNjb3c07isdNlURERExEUU3kRERERcROFNRERExEV0z5uIiEh+3irm91vedWU8pksuDa28iYiIlEB+fn54PB4CAwMZOHAgp06d4vjx48ydO7dY+l++fDm7du3KUfbcc89xww03EBQUREhICH/9619JT08vlvGk+OQb3owxFY0xW4wxO4wx3xhjnnLK/Y0xm40xu40xS4wx5Z3yCs7nROd4o2x9TXLKvzPG9MhWHuGUJRpjIrOVF3oMERGR0qBSpUrExsYSHx9P+fLlmTdv3iUNb/PmzWPdunV89dVXxMXFsXXrVurWrcvp06eLZTwpPgVZeTsD3GKtDQE8QIQxph3wd+B5a20T4Bgwwqk/Ajhmrb0eeN6phzGmBTAIaAlEAHONMX7GGD9gDtATaAEMdupS2DFERERKo44dO5KYmEhkZCRJSUl4PB4mTJiQa91PPvmETp06cccdd9C0aVMiIyNZtGgRbdq0ISgoiKSkJL744gtWrFjBhAkT8Hg8JCUlMW3aNF5++WVq1KgBQPny5YmMjKRatWoArFu3jvbt2xMWFsbAgQN971dt1KgRkydPJiwsjKCgIL799lsg892hffv2JTg4mHbt2rFz504ApkyZwtChQ+nevTuNGjXi3Xff5ZFHHiEoKIiIiAjfSt+2bdvo1KkTrVq1okePHhw8ePCSfsdukm94s5lSnY/lnB8L3AIsc8oXAH2d3/s4n3GOdzHGGKc82lp7xlq7F0gE2jg/idbaPdbas0A00MdpU9gxRERESpWMjAxWr15NUFAQ06dPJyAggNjYWGbMmJFnmx07dvDiiy8SFxfHwoUL+f7779myZQsjR47kpZde4sYbb6R3797MmDGD2NhY6tatS2pqKv7+/rn2d/jwYaZOncqHH37I119/jdfr5bnnnvMdr127Nl9//TX33XcfM2fOBGDy5MmEhoayc+dOnnnmGe655x5f/aSkJD744APef/99/vSnP9G5c2fi4uKoVKkSH3zwAenp6TzwwAMsW7aMbdu2ce+995boZ8/93gq0YcFZHdsGXE/mKlkScNxam+FUSQHqO7/XB34EsNZmGGNOALWc8q+ydZu9zY/nlbd12hR2jMPnzXs0MBqgYcOGBTlVERGREiHrJeuQufI2YsQIDhw4UKC2rVu3pl69egAEBATQvXt3AIKCgvj4449/U99aS/Y1kLVr1zJx4kSOHz/OW2+9xdGjR9m1a5fvZfNnz56lffv2vvr9+/cHoFWrVrz77rsAfP7557zzzjsA3HLLLRw5coQTJ04A0LNnT8qVK0dQUBDnzp0jIiLCN7/k5GS+++474uPj6datGwDnzp3znY8UMLxZa88BHmNMDeA9ILe39WZtncltBcxeoDy31b8L1b/QGDkLrH0FeAUyX0yfSxsREZESKeuet6KoUKGC7/cyZcr4PpcpU4aMjIzf1K9WrRpXXXUVe/fuxd/fnx49etCjRw969erF2bNnsdbSrVs3Fi9efMHx/Pz8fP1b+9v/7GYFxOzzKVeunK88a37WWlq2bMmXX35ZpPMv7Qr1qBBr7XFjzCdAO6CGMaasszLWAMj634EU4FogxRhTFqgOHM1WniV7m9zKDxdhDBERkeJXQh7tUbVqVU6ePHlJ+po0aRL33Xcf0dHR1KhRA2staWlpALRr145x48aRmJjI9ddfz6lTp0hJSaFp06Z59n/zzTezaNEinnjiCT755BNq167tu38uP82aNePQoUN8+eWXtG/fnvT0dL7//ntatmx5cSddShRkt2kdZ8UNY0wloCuQAHwMDHCqDQXed35f4XzGOf6RzYzfK4BBzk5Rf6AJsAXYCjRxdpaWJ3NTwwqnTWHHEBERKbVq1apFhw4dCAwMzHPDQkENGjSIGTNmEBoaSlJSEvfddx9du3albdu2BAcH06FDB0JDQwkNDaVOnTpERUUxePBg3waErI0JeZkyZQoxMTEEBwcTGRnJggULLlg/u/Lly7Ns2TImTpxISEgIHo+HL7744qLOtzQx+WUeY0wwmZsD/MgMe29ba582xjQmc3NBTWA78Cdr7RljTEVgIRBK5mrYIGvtHqevx4B7gQzgQWvtaqf8j8ALzhivW2unOeWFHiMvXq/XxsTEFOrLEZHCC5gZUKR2SQ8nFfNMRIouISGB5s1zu0NIpPjl9vdmjNlmrfXmVj/fy6bW2p1khqTzy/eQuVP0/PI0YGAefU0DpuVSvgpYVRxjiIiIiJRmej2WiIiIC8XFxTFkyJAcZRUqVGDz5s2XaUbye1F4ExERcaGgoKAi70YVd9O7TUVERERcROFNRERExEUU3kRERERcRPe8iYiI5KOoj8DJix6NIxdDK28iIiIlkJ+fHx6Ph8DAQAYOHMipU6c4fvw4c+fOLXBbj8dD7969feWzZ8/m+uuvxxjD4cOHL9ADREVFYYxhw4YNvrL33nsPYwzLli0r+onJRVN4ExERKYGy3m0aHx9P+fLlmTdvXoHDW1bb2NhYVqxY4Svv0KEDH374Idddd12B5hAUFJTjfabR0dGEhIQU/mSkWCm8iYiIlHAdO3YkMTGRyMhIkpKS8Hg8RXo9VmhoKI0aNSrUuFu2bCE9PZ3U1FQSExPxeDy+408//TStW7cmMDCQ0aNH+15Gn5iYSNeuXQkJCSEsLIykJF0mLk4KbyIiIiVYRkYGq1evJigoiOnTpxMQEEBsbCwzZszIs01aWhper5d27dqxfPnyIo9tjKFr166sXbuW999/P8clWIDx48ezdetW4uPjOX36NCtXrgTg7rvvZty4cezYsYMvvviCevXqFXkO8lsKbyIiIiXQ6dOn8Xg8eL1eGjZsyIgRIwrcdt++fcTExPDWW2/x4IMPXtTK16BBg4iOjiY6OprBgwfnOPbxxx/Ttm1bgoKC+Oijj/jmm284efIk+/fvp1+/fgBUrFiRypUrF3l8+S3tNhURESmBsu5bK4prrrkGgMaNGxMeHs727dsJCCjajtk2bdoQHx9PpUqVaNq0qa88LS2N+++/n5iYGK699lqmTJlCWlqa79KpXDoKbyIiIvkoKY/2qFq1KidPnrxgnWPHjlG5cmUqVKjA4cOH2bRpE4888shFjfvss89SsWLFHGVpaWkA1K5dm9TUVJYtW8aAAQOoVq0aDRo0YPny5fTt25czZ85w7tw5rb4VI102FRERcYlatWrRoUMHAgMD89ywkJCQgNfrJSQkhM6dOxMZGUmLFi0AmDVrFg0aNCAlJYXg4GBGjhxZoHF79uxJ586dc5TVqFGDUaNGERQURN++fWndurXv2MKFC5k1axbBwcHceOON/Oc//yniGUtuzJWyvOn1em1MTMzlnoZIqVfUh5mWlJUNEcgMQM2bN7/c05ArRG5/b8aYbdZab271tfImIiIi4iK6501ERMSF4uLiGDJkSI6yChUqsHnz5kL1M3/+fF588cUcZR06dGDOnDkXPUe5NBTeREREXCgoKKjIu1GzGz58OMOHDy+GGcnvRZdNRURERFxE4U1ERETERRTeRERERFxE97yJiIjkx5ji7e8KeUyXXBpaeRMRESmB/Pz88Hg8BAYGMnDgQE6dOsXx48eZO3dugdt6PJ4cL5OfPXs2119/PcYYDh8+fME+oqKiMMawYcMGX9l7772HMYZly5YV+byef/55KlasyIkTJwrdNioqivHjxwMwZcoUZs6cCcCwYcPw9/f3nfONN94IwE8//USvXr0ICQmhRYsW/PGPfyzyvEsShTcREZESKOvdpvHx8ZQvX5558+YVOLxltY2NjWXFihW+8g4dOvDhhx9y3XXXFWgOQUFBLF682Pc5OjqakJCQwp9MNosXL6Z169a89957F9XP+WbMmOE75y+++AKAJ598km7duuLCoagAACAASURBVLFjxw527drF9OnTi3XMy0XhTUREpITr2LEjiYmJREZGkpSUhMfjyfP1WBcSGhpKo0aNCjXuli1bSE9PJzU1lcTERDwej+/4008/TevWrQkMDGT06NG+l9InJibStWtXQkJCCAsLIykp8w0qSUlJpKamMnXq1ByhMCoqiv79+xMREUGTJk1yvIt1/vz5NG3alE6dOrFp06ZCne/Bgwdp0KCB73NwcHCh2pdUCm8iIiIlWEZGBqtXryYoKIjp06cTEBBAbGwsM2bMyLNNWloaXq+Xdu3asXz58iKPbYyha9eurF27lvfffz/HJViA8ePHs3XrVuLj4zl9+jQrV64E4O6772bcuHHs2LGDL774gnr16gGZq26DBw+mY8eOfPfdd/z888++vmJjY1myZAlxcXEsWbKEH3/8kYMHDzJ58mQ2bdrE+vXr2bVrV55znTBhgu+y6d133w3AuHHjGDFiBJ07d2batGkcOHCgyN9FSaLwJiIiUgKdPn0aj8eD1+ulYcOGjBgxosBt9+3bR0xMDG+99RYPPvigb+WrKAYNGkR0dDTR0dEMHjw4x7GPP/6Ytm3bEhQUxEcffcQ333zDyZMn2b9/P/369QOgYsWKVK5cGci87Dpo0CDKlClD//79Wbp0qa+vLl26UL16dSpWrEiLFi344Ycf2Lx5M+Hh4dSpU4fy5ctz55135jnP7JdNFy1aBECPHj3Ys2cPo0aN4ttvvyU0NJRDhw4V+bsoKbTbVEREpATKum+tKK655hoAGjduTHh4ONu3bycgIKBIfbVp04b4+HgqVapE06ZNfeVpaWncf//9xMTEcO211zJlyhTS0tJ8l07Pt3PnTnbv3k23bt0AOHv2LI0bN2bcuHFA5qu9svj5+ZGRkQFkrv5djJo1a3LXXXdx11130atXLzZu3Mjtt99+UX1eblp5ExERyY+1xftTRFWrVuXkyZMXrHPs2DHOnDkDwOHDh9m0aRMtWrQo8pgAzz77LM8880yOsrS0NABq165NamqqbwdqtWrVaNCgge9y7ZkzZzh16hSLFy9mypQpJCcnk5yczIEDB9i/fz8//PBDnuO2bduWTz75hCNHjpCenp5jpa4gPvroI06dOgXAyZMnSUpKomHDhoXqoyRSeBMREXGJWrVq0aFDBwIDA/PcsJCQkIDX6yUkJITOnTsTGRnpC2+zZs2iQYMGpKSkEBwczMiRIws0bs+ePencuXOOsho1ajBq1CiCgoLo27cvrVu39h1buHAhs2bNIjg4mBtvvJH//Oc/REdH+y6lZunXrx/R0dF5jluvXj2mTJlC+/bt6dq1K2FhYXnWzX7Pm8fj4ezZs2zbtg2v10twcDDt27dn5MiROebpViav5c3Sxuv12piYmMs9DZFSL2Bm0S7NJD1c9HtyRIpbQkICzZs3v9zTkCtEbn9vxpht1lpvbvW18iYiIiLiItqwICIi4kJxcXEMGTIkR1mFChXYvHlzofqZP38+L774Yo6yDh06MGfOnIueo1waCm8iIiIuFBQUVOTdqNkNHz6c4cOHF8OM5Peiy6YiIiIiLqLwJiIiIuIiCm8iIiIiLqJ73kRERPJx3Yv51ymMH/5SvP3JlUUrbyIiIiWQn58fHo+HwMBABg4cyKlTpzh+/Dhz584tcFuPx5PjZfKzZ8/m+uuvxxjD4cOHL9hHVFQUxhg2bNjgK3vvvfcwxvjepjBy5MgLvix+0aJFBAcH+x7Wu2PHjnznLvlTeBMRESmBst5tGh8fT/ny5Zk3b16Bw1tW29jYWFasWOEr79ChAx9++CHXXXddgeYQFBTE4sWLfZ+jo6MJCQnxff7f//3fC756y9/fn08//ZSdO3fyxBNPMHr06AKNKxem8CYiIlLCdezYkcTERCIjI0lKSsLj8eT5eqwLCQ0NpVGjRoUad8uWLaSnp5OamkpiYiIej8d3PDw8nKy3F61Zs4awsDBCQkLo0qULADfeeCNXX301AO3atSMlJaXQc5bf0j1vIiIiJVhGRgarV68mIiKCnj17Eh8fn+/z3dLS0vB6vZQtW5bIyEj69u1bpLGNMXTt2pW1a9dy4sQJevfuzd69e39T79ChQ4waNYqNGzfi7+/P0aNHf1Pntddeo2fPnkWah+Sk8CYiIlICnT592rfK1bFjR0aMGMGBAwcK1Hbfvn1cc8017Nmzh1tuuYWgoCACAor23uFBgwYxa9YsTpw4wT//+U+eeeaZ39T56quvuPnmm/H39wegZs2aOY5//PHHvPbaa3z++edFmoPkpPAmIiJSAmXdt1YU11xzDQCNGzcmPDyc7du3Fzm8tWnThvj4eCpVqkTTpk1zrWOtxRiT67GdO3cycuRIVq9eTa1atYo0B8kp3/BmjLkWeAP4f8CvwCvW2heNMVOAUcAhp+qj1tpVTptJwAjgHPBna+1apzwCeBHwA/7XWjvdKfcHooGawNfAEGvtWWNMBWfsVsAR4E5rbfKFxhARESluJeXRHlWrVuXkyZMXrHPs2DEqV65MhQoVOHz4MJs2beKRRx65qHGfffZZKlasmOfx9u3bM27cOPbu3eu7bFqzZk327dtH//79WbhwYZ7BTwqvIBsWMoD/sdY2B9oB44wxWVtLnrfWepyfrODWAhgEtAQigLnGGD9jjB8wB+gJtAAGZ+vn705fTYBjZIYynH+PWWuvB5536uU5RpG/BREREReoVasWHTp0IDAwMM8NCwkJCXi9XkJCQujcuTORkZG+HaGzZs2iQYMGpKSkEBwczMiRIws0bs+ePencuXOex+vUqcMrr7xC//79CQkJ4c477wTg6aef5siRI9x///14PB68Xm8hz1hyY6y1hWtgzPvAbKADkGqtnXne8UkA1tpnnc9rgSnO4SnW2h7Z6wHTyVy9+3/W2gxjTPuselltrbVfGmPKAv8B6gCRuY1hrf0yr3l7vV6btSNGRC6dgJlFuzST9HBSMc9EpOgSEhJo3rz55Z6GXCFy+3szxmyz1uaadgv1qBBjTCMgFNjsFI03xuw0xrxujLnaKasP/JitWYpTlld5LeC4tTbjvPIcfTnHTzj18+rr/PmONsbEGGNiDh06dP5hEREREdcp8IYFY0wV4B3gQWvt/xljXgb+Bljn338C9wK53bFoyT0o2gvU5wLHLtTmvwXWvgK8Apkrb7m0ERERcaW4uDiGDBmSo6xChQps3rw5jxa5mz9/Pi++mPP9Xx06dGDOnDkXPUe5NAoU3owx5cgMbouste8CWGt/ynb8VWCl8zEFuDZb8wZA1t7m3MoPAzWMMWWd1bXs9bP6SnEum1YHjuYzhoiISKkXFBRU5N2o2Q0fPpzhw4cXw4zk95LvZVOTuff3NSDBWvtctvJ62ar1A+Kd31cAg4wxFZxdpE2ALcBWoIkxxt8YU57MDQcrbOZNdx8DA5z2Q4H3s/U11Pl9APCRUz+vMURERERKtYKsvHUAhgBxxpisiP8ombtFPWRerkwGxgBYa78xxrwN7CJzp+o4a+05AGPMeGAtmY8Ked1a+43T30Qg2hgzFdhOZljE+XehMSaRzBW3QfmNISIiIlKa5RverLWfk/s9Zqsu0GYaMC2X8lW5tbPW7gHa5FKeBgwszBgiIiIipZnesCAiIpKPp556qlj7mzx5crH2J1eWQj0qRERERH4fycnJBAYG5iibMmUKM2fOzKNFyRUVFcX48ePzPD5lyhTq16+Px+OhRYsWLF68uNjGbtSoEYcPHy62/koChTcRERG57B566CFiY2N5//33GTNmDOnp6Zd7SiWWwpuIiIjLhIeHM3HiRNq0aUPTpk357LPPgMwVrr59+3Lbbbfh7+/P7Nmzee655wgNDaVdu3YcPXoUgFdffZXWrVsTEhLC7bffzqlTpwAYNmwYY8eOpWPHjjRt2pSVK1f6+u3Tpw8RERE0a9Ysx2XkN998kzZt2uDxeBgzZgznzmXuH5w/fz5NmzalU6dObNq0qcDn1qRJEypXrsyxY8cASEpKIiIiglatWtGxY0e+/fZbAP7973/Ttm1bQkND6dq1Kz/9lPkEsyNHjtC9e3dCQ0MZM2YMWW+S+uWXX7j11lsJCQkhMDCQJUuWFPn7v9wU3kRERFwoIyODLVu28MILL+QIU/Hx8bz11lts2bKFxx57jMqVK7N9+3bat2/PG2+8AUD//v3ZunUrO3bsoHnz5rz22mu+9snJyXz66ad88MEHjB07lrS0NAC2bNnCokWLiI2NZenSpcTExJCQkMCSJUvYtGkTsbGx+Pn5sWjRIg4ePMjkyZPZtGkT69evZ9euXQU+r6+//pomTZpQt25dAEaPHs1LL73Etm3bmDlzJvfffz8AN910E1999RXbt29n0KBB/OMf/wAy70+86aab2L59O71792bfvn0ArFmzhmuuuYYdO3YQHx9PRETERXz7l5c2LIiIiJRAmY9Zzbu8f//+ALRq1Yrk5GTf8c6dO1O1alWqVq1K9erVue2224DMh/ru3LkTyAx4jz/+OMePHyc1NZUePXr42t9xxx2UKVOGJk2a0LhxY99KV7du3ahVq5Zv7M8//5yyZcuybds2WrduDcDp06epW7cumzdvJjw8nDp16gBw55138v3331/wfJ9//nleffVV9uzZw5o1awBITU3liy++YODA/z544syZMwCkpKRw5513cvDgQc6ePYu/vz8AGzdu5N133wXg1ltv5eqrr/ad/8MPP8zEiRPp1asXHTt2vOB8SjKtvImIiJRAtWrV8l06zHL06FFq164NZL4KC8DPz4+MjAxfnaxygDJlyvg+lylTxldv2LBhzJ49m7i4OCZPnuxbXYPfhsasz7mVW2sZOnQosbGxxMbG8t133zFlypRc6+fnoYce4rvvvmPJkiXcc889pKWl8euvv1KjRg1f/7GxsSQkJADwwAMPMH78eOLi4vjXv/51wXMAaNq0Kdu2bSMoKIhJkybx9NNPF2p+JYlW3kRERPJxOR7tUaVKFerVq8eGDRvo0qULR48eZc2aNfzlL39h/vz5F9X3yZMnqVevHunp6SxatIj69ev7ji1dupShQ4eyd+9e9uzZQ7Nmzdi+fTvr16/n6NGjVKpUieXLl/P6669TuXJl+vTpw0MPPUTdunU5evQoJ0+epG3btvzlL3/hyJEjVKtWjaVLlxISElKgufXv358FCxawYMECxowZg7+/P0uXLmXgwIFYa9m5cychISGcOHHCN+8FCxb42t98880sWrSIxx9/nNWrV/sC8IEDB6hZsyZ/+tOfqFKlClFRURf1HV5OCm8iIiIl1BtvvMG4ceP4n//5HyAzRAYEBFx0v3/7299o27Yt1113HUFBQZw8edJ3rFmzZnTq1ImffvqJefPmUbFiRSDzHrMhQ4aQmJjIXXfdhdfrBWDq1Kl0796dX3/9lXLlyjFnzhzatWvHlClTaN++PfXq1SMsLMy3kaEgnnzySe666y5GjRrFokWLuO+++5g6dSrp6ekMGjSIkJAQpkyZwsCBA6lfvz7t2rVj7969vu9o8ODBhIWF0alTJxo2bAhAXFwcEyZMoEyZMpQrV46XX375or/Hy8Vk7cIo7bxer42Jibnc0xAp9QJmFu0/LEkPJxXzTESKLiEhgebNm1/uafzuhg0bRq9evRgwYECO8qioKGJiYpg9e/ZlmlnpltvfmzFmm7XWm1t93fMmIiIi4iK6bCoiIiIAed4HNmzYMIYNG3bR/U+bNo2lS5fmKBs4cCCPPfbYRfd9JVF4ExERyYW1ttA7JuXCHnvsMQW18xTl9jVdNhURETlPxYoVOXLkSJH+wypSUNZajhw54tsUUlBaeRMRETlPgwYNSElJ4dChQ5d7KlLKVaxYkQYNGhSqjcKbiIjIecqVK+d7Yr9ISaPLpiIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iIKbyIiIiIuovAmIiIi4iL5hjdjzLXGmI+NMQnGmG+MMX9xymsaY9YbY3Y7/17tlBtjzCxjTKIxZqcxJixbX0Od+ruNMUOzlbcyxsQ5bWYZY0xRxxAREREpzQqy8pYB/I+1tjnQDhhnjGkBRAIbrLVNgA3OZ4CeQBPnZzTwMmQGMWAy0BZoA0zOCmNOndHZ2kU45YUaQ0RERKS0yze8WWsPWmu/dn4/CSQA9YE+wAKn2gKgr/N7H+ANm+kroIYxph7QA1hvrT1qrT0GrAcinGPVrLVfWmst8MZ5fRVmDBEREZFSrVD3vBljGgGhwGbgD9bag5AZ8IC6TrX6wI/ZmqU4ZRcqT8mlnCKMcf58RxtjYowxMYcOHSrMqYqIiIiUSAUOb8aYKsA7wIPW2v+7UNVcymwRyi84nYK0sda+Yq31Wmu9derUyadLERERkZKvQOHNGFOOzOC2yFr7rlP8U9alSuffn53yFODabM0bAAfyKW+QS3lRxhAREREp1Qqy29QArwEJ1trnsh1aAWTtGB0KvJ+t/B5nR2g74IRzyXMt0N0Yc7WzUaE7sNY5dtIY084Z657z+irMGCIiIiKlWtkC1OkADAHijDGxTtmjwHTgbWPMCGAfMNA5tgr4I5AInAKGA1hrjxpj/gZsdeo9ba096vx+HxAFVAJWOz8UdgwRERGR0i7f8Gat/Zzc7zED6JJLfQuMy6Ov14HXcymPAQJzKT9S2DFERERESrOCrLyJiBTYXye8n3+l3DxcvPMQESmt9HosERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERfJN7wZY143xvxsjInPVjbFGLPfGBPr/Pwx27FJxphEY8x3xpge2cojnLJEY0xktnJ/Y8xmY8xuY8wSY0x5p7yC8znROd4ovzFERERESruCrLxFARG5lD9vrfU4P6sAjDEtgEFAS6fNXGOMnzHGD5gD9ARaAIOdugB/d/pqAhwDRjjlI4Bj1trrgeedenmOUbjTFhEREXGnfMObtXYjcLSA/fUBoq21Z6y1e4FEoI3zk2it3WOtPQtEA32MMQa4BVjmtF8A9M3W1wLn92VAF6d+XmOIiIiIlHoXc8/beGPMTuey6tVOWX3gx2x1UpyyvMprAcettRnnlefoyzl+wqmfV18iIiIipV5Rw9vLQADgAQ4C/3TKTS51bRHKi9LXbxhjRhtjYowxMYcOHcqtioiIiIirFCm8WWt/staes9b+CrzKfy9bpgDXZqvaADhwgfLDQA1jTNnzynP05RyvTubl27z6ym2er1hrvdZab506dYpyqiIiIiIlSpHCmzGmXraP/YCsnagrgEHOTlF/oAmwBdgKNHF2lpYnc8PBCmutBT4GBjjthwLvZ+trqPP7AOAjp35eY4iIiIiUemXzq2CMWQyEA7WNMSnAZCDcGOMh83JlMjAGwFr7jTHmbWAXkAGMs9aec/oZD6wF/IDXrbXfOENMBKKNMVOB7cBrTvlrwEJjTCKZK26D8htDREREpLQzmYtZpZ/X67UxMTGXexoipd6c/z4SslDG2cBinomIiHsZY7ZZa725HdMbFkRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEUU3kRERERcROFNRERExEXyDW/GmNeNMT8bY+KzldU0xqw3xux2/r3aKTfGmFnGmERjzE5jTFi2NkOd+ruNMUOzlbcyxsQ5bWYZY0xRxxAREREp7Qqy8hYFRJxXFglssNY2ATY4nwF6Ak2cn9HAy5AZxIDJQFugDTA5K4w5dUZnaxdRlDFERERErgT5hjdr7Ubg6HnFfYAFzu8LgL7Zyt+wmb4Cahhj6gE9gPXW2qPW2mPAeiDCOVbNWvultdYCb5zXV2HGEBERESn1inrP2x+stQcBnH/rOuX1gR+z1Utxyi5UnpJLeVHGEBERESn1invDgsmlzBahvChj/LaiMaONMTHGmJhDhw7l062IiIhIyVfU8PZT1qVK59+fnfIU4Nps9RoAB/Ipb5BLeVHG+A1r7SvWWq+11lunTp1CnaCIiIhISVTU8LYCyNoxOhR4P1v5Pc6O0HbACeeS51qguzHmamejQndgrXPspDGmnbPL9J7z+irMGCIiIiKlXtn8KhhjFgPhQG1jTAqZu0anA28bY0YA+4CBTvVVwB+BROAUMBzAWnvUGPM3YKtT72lrbdYmiPvI3NFaCVjt/FDYMURERESuBCZzk2fp5/V6bUxMzOWehkipN+e/j4QslHE2sJhnIiLiXsaYbdZab27H9IYFERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERdReBMRERFxEYU3ERERERe5qPBmjEk2xsQZY2KNMTFOWU1jzHpjzG7n36udcmOMmWWMSTTG7DTGhGXrZ6hTf7cxZmi28lZO/4lOW3OhMURERERKu+JYeetsrfVYa73O50hgg7W2CbDB+QzQE2ji/IwGXobMIAZMBtoCbYDJ2cLYy07drHYR+YwhIiIiUqpdisumfYAFzu8LgL7Zyt+wmb4Cahhj6gE9gPXW2qPW2mPAeiDCOVbNWvultfb/t3f3UXbU9R3H3x+SGHlMIEiQB0kMoGnBpBCEHkTBhzTiAx6BAnrkUYO0ggVDsYUSggoREAQOWCjE8CBwQCqJNQQ5gtRWQhIwJJGHVAgPMaIINGChBxK+/WN+lzO53U127/7u7M7dz+uce3buzM585rd77+x3f7+ZOwFc37StrjLMzMzMOlpfi7cAfirpQUlT07zREfE7gPR1uzR/R+DZ0rqr0rwNzV/VxfwNZaxH0lRJiyUtfv7551tsopmZmdnAMbSP6+8fEaslbQfcLemxDXyvupgXLczvsYi4GrgaYNKkSb1a18zMzGwg6lPPW0SsTl//APyI4py136chT9LXP6RvXwXsXFp9J2D1Rubv1MV8NpBhZmZm1tFaLt4kbS5py8Y0MBlYDswFGleMHgPMSdNzgaPTVaf7AWvSkOddwGRJW6cLFSYDd6Vlr0jaL11lenTTtrrKMDMzM+tofRk2HQ38KH16x1DgpoiYL2kRcKukE4BngMPT988DDgZ+A7wKHAcQES9K+gawKH3fuRHxYpo+CZgNbArcmR4AM7vJMDMzM+toLRdvEfEkMKGL+S8AH+lifgB/2822ZgGzupi/GNijpxlmZmZmnc53WDAzMzOrERdvZmZmZjXS148KMbOamDFjRq/XmT59ehv2xMzM+sI9b2ZmZmY14uLNzMzMrEY8bGo2SMwa2fshUA+ampkNPC7ezAaJoW+Ma2GtJ7Lvh5mZ9Y2HTc3MzMxqxMWbmZmZWY24eDMzMzOrERdvZmZmZjXi4s3MzMysRly8mZmZmdWIizczMzOzGnHxZmZmZlYjLt7MzMzMasTFm5mZmVmNuHgzMzMzqxEXb2ZmZmY14uLNzMzMrEZcvJmZmZnViIs3MzMzsxpx8WZmZmZWIy7ezMzMzGrExZuZmZlZjQzt7x0wM2vVjBkzer3O9OnT27AnZmbVcfFmNkicdvqc3q80Lf9+mJlZ37h4M7Paun7z63u9znTc82Zm9eZz3szMzMxqxMWbmZmZWY24eDMzMzOrEZ/zZtaPWrlaEnzFpJnZYOaeNzMzM7MacfFmSBOHGAAADOFJREFUZmZmViMu3szMzMxqxMWbmZmZWY34ggWzftTKh8yCP2i2wXeNMLPByMWbmVkP+D6qZjZQuHgzM+uB6eec08JKLt7MLD8Xb2ZmPTDuwnf3ep0n2rAfZmYu3irkD2StDw+RmZnZQOXijc78Q11lm6rK6sTfk5mZWW+5eLPamDWy94VYq6VbK1eBtnIFaEtXS4KvmOxgu1za+3We/mr+/TCzgcvFm/VZVYXO0DfG9Xodn3VkdePXuZltTK2LN0lTgEuBIcA1ETGzn3dpwKiqoDIz25hWehPBPYpm3alt8SZpCHAF8DFgFbBI0tyIeKR/92zwqeqDUqv8QFZ/+Ks168TXxLiLet/L98S03vfytdabCK30KPrcWBsMalu8Ae8HfhMRTwJIugU4BOh18bbtOYf2Pr2F93pLOS1mdeIfGrPBwP+k9E1Vow5VfnpAlQVpVQV9VTnQmQW9IqK/96Elkg4DpkTEF9PzLwD7RsRXSt8zFZianr4HeLyXMdsCf8ywu4M1y22qR1YntqnKLLepHlmd2KYqs9ym6rN2iYh3dLWgzj1v6mLeepVoRFwNXN1ygLQ4Iia1uv5gz3Kb6pHViW2qMsttqkdWJ7apyiy3aWBlbZJrQ/1gFbBz6flOwOp+2hczMzOzStS5eFsE7CZprKS3AUcCc/t5n8zMzMzaqrbDphGxVtJXgLsoPipkVkT8OnNMy0Ouzqo0p8ost8lZ/ZVTZZbb5Kz+yqkyq7Ztqu0FC2ZmZmaDUZ2HTc3MzMwGHRdvZmZmZjXi4s3MzMysRly8mVnHkbRdf+9DO0ga1d/7YGb9z8XbAOGD8sAjaYSkmZIek/RCejya5o2scD/uzLitrSSdL+kGSZ9rWnZlrpy0ve0lfU/SFZJGSTpH0jJJt0p6Z8acbZoeo4CFkraWtE2unJQ1pTQ9QtK1kpZKuknS6MxZMyVtm6YnSXoSeEDS05I+lDHnIUlnSWr1BqS9yZok6V5JN0raWdLdktZIWiTpLzLmbCHpXEm/Ttt/XtICScfmyihlDZV0oqT56bXwsKQ7JX1Z0rDced3sQ94rGaUhqU3fkLR/07KzMmdtJunvJZ0u6e2SjpU0V9IFkrbImdVF9oo2bfd9pelh6f01V9J5kjbLkeHiraSqA3NVB+W0/UoOzFUdlFNWVQfmW4GXgAMjYlREjAIOSvNuyxkkaa9uHnsDEzNGfZ/i7iS3A0dKul3S8LRsv4w5ALMp7jX8LHAv8BrwCeAXwD9nzPkj8GDpsRjYEXgoTed0Xmn6O8DvgE9RfO7kVZmzPhERjdvpXAgcERG7Ah9L2blsDYwE7pW0UNKpknbIuP2yK4ELgJ8AvwSuiogRwNfTslx+ADwJ/BUwA7gM+AJwkKTzNrRiC26geI+eAxxM8RqfAUwAbswV0sU/KeV/Vg7OlZNcBXwIeAG4TNLFpWWfzZw1GxgNjKV4XUwCLqI4Tn0vV4ikVyS9nB6vSHoFGNeYnysnmV2angnsSvGe3ZRcx76I8CM9gIdK09cA3wR2AU4F7siYs6w0fS+wT5reHVicuU0rKd4IzwALU1t2aMPPbiHwceAoij/Wh6X5HwHuz5w1BziW4q4apwH/BOwGXAeclzHn8VaWtZi1DrgnvR6aH69lzFnS9PxM4D+BUeXXf6asX5Wmn9nQfvQxZxowH9izNG9lzraUtls+RjT/LLO1KW3vMWBoml7QtGxZxpxymw6gKKKeS6+9qRW+Jn6VMefhpueL0tdNgMcyt2lDx4kVGXPWURSkK0uPxvPXM7dpaWl6KMVnlP0rMDzn7yltf0n6qvS6U+n50ow5lwPXA6NL81bmbEtpu+XX+RJgWO42ueete5Mi4qyIeDoiLgHGZNz2MEmND0jeNCIWAUTECoo3R04vRcS0iHgX8DWKIueh1Es2NWPOsIi4MyJuBiIifkgx8TPg7RlzAMZExOyIWBURFwOfjoj/Ao4j73+FT6fu/Ld6XSWNlnQGRYGa06PAiRFxUPODvDdOHi7prfd9RHyL4sD87xQFXE7l48v1TcuG5AqJiIuALwJnS7pY0pY03ec4o+0knSbpa8BWksr3WM59PL0CmCfpw8B8Sd+V9EFJMyj+IGQXEb+IiL+h6Ln8NvCXmSP+V9JkSYcDIekzAGnEYV3GnP+R9IG07U8BLwJExJt0fV/svnhJ0uHl95WkTSQdQdFLn8uTFKMAY0uPd0fEWOD3GXMA3taYiIi1ETEVeJjiH8y2DGVGUd3MS18bz7O9jyPiZOBS4GZJp6TfV7uOEyMkfVbSocDwiHgj7UO2Nrl4W19VB+bKD8rQ9gNzVQdlqO7AfARFQXOfpJckvQj8HNgG+OuMOVAMuXT3Gjs5Y86PgQ+XZ0TEdRSF/esZcwDmNM5ZiYi3zpORtCvweM6gVMgfTtFbdDeQ5bySLvwLsCXFH7DrgMbpD9uT+b0bEZdTDNOeCBxC0Yv9deC3wPEZo/7feT8RsS4i5kfEcRlzAL5M8Vo7nmJI8yBJ/03R23dKxpyTgIvTts9obFvSOyiOvzkdCRwGPCdpRTqP6jmKfySPzJjzXYoh7q5ckDEHYHH5NCKAiJhBcdrFmDZkNY4Tb72u06k+r+QMiogHgY+mp/eRv2Oh4T6K0yk+CSxodACk40SWf8Z9h4USSdObZl0ZEc+nH/gFEXF0xqwDKQ4wu1N0Sz8L3EFxm6+1GXNuiYicB5DuciZQHEDepBiaPQk4huIPzZci4pcZs95HMay9O7AcOD4iVqQD81ERcVnGrPdSDM8uiIg/leZPiYj5uXJKWTsCD7QzawM5H4+IbBdHbCSrbW2i+GdhXEQsr/Pvqcqsits0Htih3VkpZ0eqee/uS9Gj8gQwnuL80UciYl7mnPdTdOAskvRnwBSKYeCsOQMk63FKPXFtyDmA4hzmxW1q077Am237+eUYex0MD+C4Tspxm3q0rVMoDiB3AE8Bh5SW5T4/rJIsil68qtpUSVbFv6cqf36d+Jo4heJcvipeE23PSdubDiyguDjmfOBnwNkUpyKc2cace9qR06lZVf2eqsrKtrOd/qDp5Nq657hNPdrWMmCLND0mvRG/mp7nPmm3kiy3yVluU1vaNIRiqP5lYKs0f1PynnBfSU6nZnVamxonzRsgaWl3iyguZa5VTpVZndgmYEik4ZaIeCoNdf9Q0i7kP+m5qiy3yVn9lVNlVpVtWhsR64BXJT0RES+n3NckvVnDnE7N6qg2uXhb32iKk2ibrxASxWcS1S2nyqxObNNzkiZGxBKAiPiTpE8Cs4A9M+ZUmeU2Oau/cqrMqrJNr0vaLCJeBfZuzJQ0guIc4LrldGpWZ7UpZ1dh3R/AtcAHull2U91y3KY+5+wEbN/Nsv0zt6mSLLfJWW5T9jYN72b+tpQ+e7AuOZ2a1Wlt8tWmZmZmZjXiz3kzMzMzqxEXb2ZmZmY14uLNzMzMrEZcvJnZoCLp55Iel/SwpEWSJpaWPSXp9tLzwyTNblp/jqT7e5DznpS1RNKjkq5O8ydJynYXEDMbfFy8mdlg9PmImEBxT80Lm5ZNkvTnXa0kaSSwFzBS0tiNZFwGXBIREyNiPHA5QEQsjoic9/E0s0HGxZuZdSxJm0v6SeplWy7piKZvuZ/i/pdlFwH/2M0mDwV+DNzCxm86/k5gVeNJRCxL+3SgpH9L0/NSz9wSSWskHSNpiKQLU6/gUkkn9qy1ZjZYuHgzs042BVgdERMiYg+g+YbkUyjuf1l2K7CXpF272N5RwM3pcdRGsi8B7pF0p6RTU6/deiLi4IiYCJwAPJ325QRgTUTsA+wDfKkHvXxmNoi4eDOzTrYM+Kikb0s6ICLWpPk/kLQKOIM0nFmyjmIo9R/KMyWNBnYF/iMiVgBrJe3RXXBEfB8YD9wGHAgskDS8+fskbQvcAHwu7d9k4GhJS4AHgFHAbr1rtpl1MhdvZtaxUpG1N0URd76ks9OizwNjgZuAK7pY9Qbgg8C7SvOOALYGVkp6iuKG5xscOo2I1RExKyIOAdYC6xV7koZQDMGeGxHLG7OBk9O5chMjYmxE/LSHTTazQcDFm5l1LEk7AK9GxI0U57Lt1VgWEW8AZwH7SRpfXi8tuwT4u9Lso4ApETEmIsZQFIXdFm+Spkgalqa3p+hB+23Tt80ElkbELaV5dwEnldbdXdLmPW+1mXU6F29m1sn2BBamIcgzgW+WF0bEa8B3gGldrHstMBRA0hiKXrgFpXVXAi9L2reb7MnAckkPUxRkp0fEc03fMw2YXLpo4dPANcAjwEOSlgNXNfbDzAzwvU3NzMzM6sQ9b2ZmZmY14q54M7M+kHQmcHjT7Nsi4lv9sT9m1vk8bGpmZmZWIx42NTMzM6sRF29mZmZmNeLizczMzKxGXLyZmZmZ1YiLNzMzM7Ma+T9+i95Mzv7WAAAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 720x504 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "fileextension='.fastq.gz'\n",
    "directory = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later'\n",
    "outplotname = 'Ptiwi08RIP_53Later_Inserts_Stacked_Barplot_Colors.pdf'\n",
    "startseqlength = 15\n",
    "cleanreads=False\n",
    "blastdatabase = '/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabaselist=['/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2']  # '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1',\\\n",
    "\n",
    "colors = ['black', \\\n",
    "          'darkviolet', \\\n",
    "          'orange', \\\n",
    "          'forestgreen', \\\n",
    "          'red', \\\n",
    "          'dodgerblue', \\\n",
    "          'grey'] # this is length of align databases + 1 because we need a color for unmapped\n",
    "    \n",
    "# Vector, color=‘darkviolet’\n",
    "# MAC, color=‘forestgreen’\n",
    "# IES, color=‘red’\n",
    "# OES, color=‘dodgerblue’\n",
    "# Mitochondrial, color=‘orange’\n",
    "# Klebsiella, color=‘black’\n",
    "# Unmapped, color=‘grey’\n",
    "\n",
    "readsubtract.main(fileextension, directory, outplotname, startseqlength, cleanreads, blastdatabase, aligndatabaselist, colors)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
