{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "2021-11-09 14:10:35.158533\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import Tools\n",
    "from pygentoolbox import ReadSubtractionByGenomeMapping as readsubtract\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/media/sf_LinuxShare/PyPacks/PyGenToolbox/notebooks/Theresa\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.getcwd())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 54247\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1484\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 49144\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 5176\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 98\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 49046\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 48824\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 295\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 48824\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 63610\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 939\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 57083\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 96\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 56987\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 56750\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 287\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 56750\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 44791\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1531\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 35746\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 9263\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 35636\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 110\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 28156\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 18918\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 28032\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 133\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 27900\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 222\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 27526\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1008\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 42646\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 859\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 28593\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 14618\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 28496\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 18208\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21941\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 18097\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 111\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 18036\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 83\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 17633\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 774\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 45737\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 969\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26513\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 20508\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26423\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13953\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 21846\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 13776\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 181\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 13716\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 89\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 13300\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 745\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 52325\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 722\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 29502\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 23284\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 29409\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 93\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 15520\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 23043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 15315\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 210\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 15243\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 85\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 14528\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1091\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 128252\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 566\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 81814\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 47344\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 81694\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 120\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 42688\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 56915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 42345\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 349\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 42261\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 148\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 40925\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1824\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 378858\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 875\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 189392\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 193271\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 189221\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 171\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 33530\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 193913\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 32875\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 32710\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 353\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 29610\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 4335\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3361812\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 1288\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 1431194\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1943523\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 1430813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 381\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 152687\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 1306246\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 150602\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2112\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 149721\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 1753\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 138630\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 13343\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 184331\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 665\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 84813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 100062\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 84666\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 147\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 25585\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 85713\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 25049\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 543\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 24784\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 415\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 19639\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 5518\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 62181\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 626\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 41005\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 21230\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 40908\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 13228\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 58401\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 12211\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1027\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 10723\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 1834\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 8567\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2771\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 37487\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 387\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26846\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 10648\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26740\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 106\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 9227\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 30058\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 7882\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1357\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 7589\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 319\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 6583\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1376\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 20619\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 522\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 13788\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6834\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 13698\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8080\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8088\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6177\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1915\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 5885\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 400\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5275\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 752\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 19544\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 587\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14887\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4659\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 14779\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 108\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8016\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 17255\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6422\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1609\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 6178\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 410\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5772\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 505\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 12489\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 403\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 10962\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1527\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 10894\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 6680\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 11981\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5937\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 750\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 5813\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 179\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5627\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 252\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8192\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 492\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7149\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1043\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7081\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5375\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3620\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5251\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 124\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 5231\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 22\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5109\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 175\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6594\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 308\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5863\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 731\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5793\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 70\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4402\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2505\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4356\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 46\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 4349\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 7\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4282\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 8735\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 184\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 8191\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 544\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 8107\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 84\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5231\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8842\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 5218\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 13\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 5213\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 7\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5110\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 113\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 7713\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 104\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 7234\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 479\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 7198\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 36\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4470\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8819\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4458\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 12\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 4444\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 14\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4398\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 53\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5783\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5417\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 366\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5365\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 52\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3807\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2649\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3799\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 8\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 3796\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 3\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3768\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 29\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 4347\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 45\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 3983\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 364\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 3925\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 58\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2681\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2738\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 2679\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2640\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 44\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-53_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "wrote out table of counts aligned to each genome to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/ReadsMappedToEachReference.tsv\n",
      "\n",
      "Output Stacked Barplot to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later/Ptiwi08RIP_53Later_Inserts_Stacked_Barplot.pdf\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAm8AAAGtCAYAAABA/pOqAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzde5jN5f7/8edtHIYM5bR/MsmQ8xzWjOWUZOQ0Sk4RaotyLNpl76+QdqY2ZW92ByE73zKSjKhky5lKURgsZhgxw1RTdjnvmRhmdP/+mDXrO8OMORiZxetxXeuatd6f+/RZ17oub/f9uT8fY61FRERERLxDqWs9ABEREREpOCVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBElbyIiIiJepPS1HsDvpVq1arZOnTrXehgiIiIi+dqxY8cxa2313I7dMMlbnTp1iImJudbDEBEREcmXMea7vI5p2VRERETEiyh5ExEREfEi+SZvxhhfY8w2Y8xuY8xeY8wL7niUMeawMcblfjnccWOMmWGMSTDG7DHGhGVra5Ax5qD7NShbvJkxJtZdZ4YxxrjjVYwx69zl1xljbsmvDxEREZHrWUGueTsH3GOtTTXGlAG+Msasch8ba61delH5rkB996sl8CbQ0hhTBZgEOAEL7DDGLLfWnnSXGQ58A6wEIoBVwHhgg7V2qjFmvPvzuLz6KMoXICIicrH09HSSk5NJS0u71kOR65yvry/+/v6UKVOmwHXyTd5s5pPrU90fy7hfl3uafQ/gXXe9b4wxNxtjagLhwDpr7QkAY8w6IMIY8zlQyVr7tTv+LtCTzOSth7sewHzgczKTt1z7sNYeKeB5i4iI5Ck5ORk/Pz/q1KmDezFIpNhZazl+/DjJyckEBAQUuF6BrnkzxvgYY1zAL2QmYFvdh6a4ly1fNcaUc8dqAT9kq57sjl0unpxLHOAPWQmZ+2+NfPoQERG5YmlpaVStWlWJm1xVxhiqVq1a6BneAiVv1toL1loH4A+0MMYEAhOARkBzoAqZM2IAuf3SbRHil1OgOsaY4caYGGNMzNGjR/NpUkRE5P8ocZPfQ1F+Z4XabWqtPUXm0mWEtfaIzXQOmAe0cBdLBm7LVs0f+CmfuH8ucYCf3UuuuP/+kk8fF4/3LWut01rrrF491/vciYiI3DB8fHxwOBwEBgZy//33c+rUqXzrzJgxg8aNG/Pwww//DiOUgijIbtPqxpib3e/LAx2B/dmSKkPmNWpx7irLgUfcO0JbAafdS55rgM7GmFvcu0Y7A2vcx1KMMa3cbT0CfJKtraxdqYMuiufWh4iIiOShfPnyuFwu4uLiqFKlCrNmzcq3zuzZs1m5ciULFy4sUB8ZGRlXOkzJR0Fm3moCnxlj9gDbybzmbQWw0BgTC8QC1YDJ7vIrgUNAAjAXeALAvVHhb+42tgMvZm1eAB4H/tddJ5HMzQoAU4FOxpiDQCf35zz7EBERkYJp3bo1P/74o+fztGnTaN68OcHBwUyaNAmAkSNHcujQIbp3786rr77Kr7/+ymOPPUbz5s0JDQ3lk08y51SioqLo27cv999/P507d86zvaSkJBo3bsywYcNo2rQpnTt35uzZswAkJCTQsWNHQkJCCAsLIzExMc92bnjW2hvi1axZMysiIlIQ+/btu9ZDuCpuuukma621GRkZtk+fPnbVqlXWWmvXrFljhw0bZn/77Td74cIFe99999kvvvjCWmvt7bffbo8ePWqttXbChAl2wYIF1lprT548aevXr29TU1PtvHnzbK1atezx48cv297hw4etj4+P3bVrl7XW2r59+3raa9Gihf3oo4+stdaePXvW/vrrr5cd1/Ukt98bEGPzyGlumGebioiI3OjOnj2Lw+EgKSmJZs2a0alTJwDWrl3L2rVrCQ0NBSA1NZWDBw9y991356i/du1ali9fzvTp04HMXbnff/89AJ06daJKlSqXba927doEBATgcDgAaNasGUlJSaSkpPDjjz/Sq1cvIPPeZ4UZ141GyZuIiMgNIuuat9OnT9OtWzdmzZrFn/70J6y1TJgwgREjRly2vrWWDz/8kIYNG+aIb926lZtuuilHudzaS0pKoly5cp7PPj4+nD17lsyJptz7K8i4bjR6tqmIiMgNpnLlysyYMYPp06eTnp5Oly5deOedd0hNzbwn/48//sgvv/xySb0uXbrwxhtveJKtXbt25dp+QdvLUqlSJfz9/Vm2bBkA586d48yZM4Vu50ahmTcREZEbUGhoKCEhIURHRzNw4EDi4+Np3bo1ABUrVuS9996jRo0aOer89a9/5emnnyY4OBhrLXXq1GHFihWXtN25c+dc2/Px8clzPAsWLGDEiBE8//zzlClThiVLluTZzsXjutGYvKYqrzdOp9PGxMRc62GIiIgXiI+Pp3Hjxtd6GHKDyO33ZozZYa115lZeM28iUqwiIyN/13oiIjcaXfMmIiIi4kWUvImIiIh4ESVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBHdKkRERCQfdcZ/WqztJU29L98yFStW9DxZYOXKlTz11FNs2LCB2rVrF+tYLpaens5f//pXPvzwQ8qVK0eFChV44YUX6Nq161XrM/u55ubUqVO8//77PPHEE/m29fPPPzNmzBi++eYbbrnlFsqWLcszzzzjeW7q9UAzbyIiIiXYhg0bePLJJ1m9evVVT9wg8ykKR44cIS4ujri4OP7973+TkpJy1fu9nFOnTjF79ux8y1lr6dmzJ3fffTeHDh1ix44dREdHk5yc/DuM8vej5E1ERKSE+vLLLxk2bBiffvop9erVA2Dw4MGMHDmStm3b0qBBA8/jqaKioujduzcRERHUr1+fZ555xtPO2rVrad26NWFhYfTt2zfPWa4zZ84wd+5c3njjDc8D5P/whz/w4IMPArBo0SKCgoIIDAxk3LhxnnoVK1Zk3LhxNGvWjI4dO7Jt2zbCw8OpW7cuy5cv94yvR48eRERE0LBhQ1544YVcxzBt2jSaN29OcHAwkyZNAmD8+PEkJibicDgYO3ZsnuU2btxI2bJlGTlypKe922+/nSeffBKACxcuMHbsWE+9f/3rXwB8/vnnhIeH06dPHxo1asTDDz/seX7rhg0bCA0NJSgoiMcee4xz584BUKdOHZ599llat26N0+lk586ddOnShXr16jFnzpzLns+VUvImIiJSAp07d44ePXqwbNkyGjVqlONYUlISX3zxBZ9++ikjR44kLS0NAJfLxeLFi4mNjWXx4sX88MMPHDt2jMmTJ7N+/Xp27tyJ0+nklVdeybXPhIQEateuTaVKlS459tNPPzFu3Dg2btyIy+Vi+/btngfJ//rrr4SHh7Njxw78/Px47rnnWLduHR9//DHPP/+8p41t27axcOFCXC4XS5Ys4eLHVq5du5aDBw+ybds2XC4XO3bsYNOmTUydOpV69erhcrmYNm1anuX27t1LWFhYnt/p22+/TeXKldm+fTvbt29n7ty5HD58GIBdu3bx2muvsW/fPg4dOsTmzZtJS0tj8ODBnu80IyODN99809Pebbfdxtdff03btm0ZPHgwS5cu5ZtvvvGcc17jvFK65k1ERKQEKlOmDHfeeSdvv/02r7/+eo5jDz74IKVKlaJ+/frUrVuX/fv3A9ChQwcqV64MQJMmTfjuu+84deoU+/bto02bNgCcP3/e86D3wti+fTvh4eFUr14dgIcffphNmzbRs2dPypYtS0REBABBQUGUK1eOMmXKEBQURFJSkqeNTp06UbVqVQB69+7NV199hdP5f4/vXLt2LWvXriU0NBSA1NRUDh48eMlycV7lLjZq1Ci++uorypYty/bt21m7di179uxh6dKlAJw+fZqDBw9StmxZWrRogb+/PwAOh4OkpCT8/PwICAigQYMGAAwaNIhZs2bx9NNPA9C9e3fPOaempuLn54efnx++vr6cOnUqz3Hefffdhf7+s1PyJiIiUgKVKlWKDz74gI4dO/LSSy/x7LPPeo4ZY3KUzfqctdQJ4OPjQ0ZGBtZaOnXqxKJFi/Lt84477uD7778nJSUFPz+/HMeylhFzU6ZMGc8YSpUq5RlHqVKlyMjIyHfc2fuYMGECI0aMyBHPngBertyGDRv48MMPPZ9nzZrFsWPHPAmitZY33niDLl265Kj3+eef5/ndXU7288xeP+u88xrnldKyqYiISAlVoUIFVqxYwcKFC3n77bc98SVLlvDbb7+RmJjIoUOHaNiwYZ5ttGrVis2bN5OQkABkXtd24MCBPPsbMmQIf/rTnzh//jwAR44c4b333qNly5Z88cUXHDt2jAsXLrBo0SLatWtXqPNZt24dJ06c4OzZsyxbtswzG5ilS5cuvPPOO55r8n788Ud++eUX/Pz8cmyayKvcPffcQ1paWo6lzTNnzuSo9+abb5Keng7AgQMH+PXXX/Mcb6NGjUhKSvJ8dwsWLCjUOec1ziulmTcRKVZD0zpc6yGIFLuC3NrjaqlSpQqrV6/m7rvvplq1agA0bNiQdu3a8fPPPzNnzhx8fX3zrF+9enWioqIYMGCA52L7yZMne5YCLzZ58mSee+45mjRpgq+vLzfddBMvvvgiNWvW5OWXX6Z9+/ZYa7n33nvp0aNHoc7lrrvuYuDAgSQkJPDQQw/lWDIF6Ny5M/Hx8Z5l3YoVK/Lee+9Rr1492rRpQ2BgIF27dmXatGm5lqtRowbLli1jzJgx/OMf/6B69ercdNNN/P3vfwdg6NChJCUlERYWhrWW6tWre67by42vry/z5s2jb9++ZGRk0Lx58xybIfKT1/nUqFGjUN/bxUx+U4LXC6fTaS++MFJEil/y+C+LVM9/attiHolI0cXHx9O4ceNrPYxcDR48mG7dutGnT59rPZRCiYqKIiYmhpkzZ17roZQ4uf3ejDE7rLXO3Mpr5k1EilXKsuFFqzg1vngHIiJynVLyJiIi4kWioqKKpZ1evXp5bpOR5e9///slF/MXl8GDBzN48OCr0vaNRsmbiIjIDejjjz++1kOQItJuUxEREREvouRNRERExIsoeRMRERHxIkreRERERLyINiyIiIjkJ7JyMbd3+rKHw8PDmTBhQo6dn6+99hoHDhxg9uzZBe5m2bJlNGjQgCZNmhR5qFkiIyOZO3eu59mmERERTJ06lfDwcKZPn37JDXcLIrd7v13c3q5duwgLC2P16tX57oT18fEhKCiI9PR0SpcuzaBBg3j66acpVSrvuars/YWHh3PkyBHKly8PZD5D9UpvqHs1aOZNRESkhBkwYADR0dE5YtHR0QwYMKBQ7Sxbtox9+/YVqk72Z5FebMyYMbhcLlwuF1OnTi1Uu0W1aNEi7rrrrgI9m7V8+fK4XC727t3LunXrWLlyJS+88EKh+lu4cKHnHEti4gZK3kREREqcPn36sGLFCs/jrJKSkvjpp5+46667mDZtGs2bNyc4OJhJkyZ56rz77rsEBwcTEhLCwIED2bJlC8uXL2fs2LE4HA4SExNxuVy0atWK4OBgevXqxcmTJ4HM2adnn32Wdu3a8frrrxd53I8//jhOp5OmTZvmGNvKlStp1KgRd911F3/605/o1q1bgdqz1rJ06VKioqJYu3YtaWlpBR5LjRo1eOutt5g5cybWWi5cuMD//M//EBQURHBwMG+88Uahz6+k0LKpiIhICVO1alVatGjB6tWr6dGjB9HR0fTr149169Zx8OBBtm3bhrWW7t27s2nTJqpWrcqUKVPYvHkz1apV48SJE1SpUoXu3bvneJRWVtLSrl07nn/+eV544QVee+01AE6dOsUXX3xx2XG9+uqrvPfee0DuN/SdMmUKVapU4cKFC3To0IE9e/bQoEEDRowYwaZNmwgICLhk9nDx4sV89dVXns9ZD4EH2Lx5MwEBAdSrV4/w8HBWrlxJ7969C/w91q1bl99++41ffvmFjz76iMOHD7Nr1y5Kly7NiRMncq3z6KOP4uPjwwMPPMBzzz2HMabA/f1eNPMmIiJSAmVfOs1aMl27di1r164lNDSUsLAw9u/fz8GDB9m4cSN9+vTxPLi+SpUql7R3+vRpTp06Rbt27QAYNGgQmzZt8hzv169fvmPKvmya2/VnH3zwAWFhYYSGhrJ371727dvH/v37qVu3LgEBAZ7zyq5fv36eNl0uV45r5xYtWkT//v0B6N+/f4GWTi+W9Qz39evXM3LkSEqXzpy3yu07WrhwIbGxsXz55Zd8+eWXLFiwoND9/R6UvImIiJRAPXv2ZMOGDezcuZOzZ88SFhaGtZYJEyZ4Ep2EhASGDBmCtfaKZ4huuummK6p/+PBhpk+fzoYNG9izZw/33XcfaWlpnuSpsC5cuMCHH37Iiy++SJ06dXjyySdZtWoVKSkpBW7j0KFD+Pj4UKNGjQJ9R7Vq1QLAz8+Phx56iG3bthVp7FebkjcREZESqGLFioSHh/PYY495Zqu6dOnCO++8Q2pqKgA//vgjv/zyCx06dOCDDz7g+PHjAJ4lQT8/P0+yU7lyZW655Ra+/PJLABYsWOCZhSsO//3vf7npppuoXLkyP//8M6tWrQKgUaNGHDp0iKSkJCBzmbQg1q9fT0hICD/88ANJSUl89913PPDAAyxbtqxA9Y8ePcrIkSMZPXo0xhg6d+7MnDlzPBsyLl42zcjI4NixYwCkp6ezYsUKAgMDC9TX703XvImIiOQnn1t7XC0DBgygd+/enuXTzp07Ex8fT+vWrYHMBO+9996jadOmTJw4kXbt2uHj40NoaChRUVH079+fYcOGMWPGDJYuXcr8+fMZOXIkZ86coW7dusybN6/YxhoSEkJoaChNmzalbt26tGnTBsjcATp79mwiIiKoVq0aLVq0KFB7ixYtolevXjliDzzwAG+++SYDBw7Mtc7Zs2dxOByeW4UMHDiQP//5zwAMHTqUAwcOEBwcTJkyZRg2bBijR4/21D137hxdunQhPT2dCxcu0LFjR4YNG1aUr+KqM0WdzvQ2TqfTxsTEXOthiFz34hs1LlK9xvvji3kkIkUXHx9P48ZF+y3LpVJTU6lYsSLWWkaNGkX9+vUZM2bMtR5WiZHb780Ys8Nam+vN87RsKiIiIlfV3LlzcTgcNG3alNOnTzNixIhrPSSvpmVTERER8ZgyZQpLlizJEevbty8TJ04scptjxowptpm248eP06FDh0viGzZsoGrVqsXSR0mn5E1EREQ8Jk6ceEWJ2tVWtWpVXC7XtR7GNaVlUxEREREvouRNRERExIsoeRMRERHxIrrmTUREJB9B84OKtb3YQbHF2p7cWDTzJiIiUgL5+PjgcDgIDAykb9++nDlzhlOnTjF79uxiaX/ZsmXs27cvR+yVV16hUaNGBAUFERISwp///GfS09OLpT8pPvkmb8YYX2PMNmPMbmPMXmPMC+54gDFmqzHmoDFmsTGmrDtezv05wX28Tra2Jrjj3xpjumSLR7hjCcaY8dnihe5DRETkelC+fHlcLhdxcXGULVuWOXPmXNXkbc6cOaxdu5ZvvvmG2NhYtm/fTo0aNTh79myx9CfFpyAzb+eAe6y1IYADiDDGtAL+Drxqra0PnASGuMsPAU5aa+8AXnWXwxjTBOgPNAUigNnGGB9jjA8wC+gKNAEGuMtS2D5ERESuR23btiUhIYHx48eTmJiIw+Fg7NixuZb9/PPPadeuHQ8++CANGjRg/PjxLFy4kBYtWhAUFERiYiJbtmxh+fLljB07FofDQWJiIlOmTOHNN9/k5ptvBqBs2bKMHz+eSpUqAbB27Vpat25NWFgYffv29TxftU6dOkyaNImwsDCCgoLYv38/kPns0J49exIcHEyrVq3Ys2cPAJGRkQwaNIjOnTtTp04dPvroI5555hmCgoKIiIjwzPTt2LGDdu3a0axZM7p06cKRI0eu6nfsTfJN3mymVPfHMu6XBe4Blrrj84Ge7vc93J9xH+9gjDHueLS19py19jCQALRwvxKstYesteeBaKCHu05h+xAREbmuZGRksGrVKoKCgpg6dSr16tXD5XIxbdq0POvs3r2b119/ndjYWBYsWMCBAwfYtm0bQ4cO5Y033uDOO++ke/fuTJs2DZfLRY0aNUhNTSUgICDX9o4dO8bkyZNZv349O3fuxOl08sorr3iOV6tWjZ07d/L4448zffp0ACZNmkRoaCh79uzhpZde4pFHHvGUT0xM5NNPP+WTTz7hj3/8I+3btyc2Npby5cvz6aefkp6ezpNPPsnSpUvZsWMHjz32WIm+99zvrUAbFtyzYzuAO8icJUsETllrM9xFkoFa7ve1gB8ArLUZxpjTQFV3/JtszWav88NF8ZbuOoXt49hF4x4ODAeoXbt2QU5VRESkRMh6yDpkzrwNGTKEn376qUB1mzdvTs2aNQGoV68enTt3BiAoKIjPPvvskvLWWrLPgaxZs4Zx48Zx6tQp3n//fU6cOMG+ffs8D5s/f/48rVu39pTv3bs3AM2aNeOjjz4C4KuvvuLDDz8E4J577uH48eOcPn0agK5du1KmTBmCgoK4cOECERERnvElJSXx7bffEhcXR6dOnQC4cOGC53ykgMmbtfYC4DDG3Ax8DOT2tN6sJ9znNgNmLxPPbfbvcuUv10fOgLVvAW9B5oPpc6kjIiJSImVd81YU5cqV87wvVaqU53OpUqXIyMi4pHylSpW46aabOHz4MAEBAXTp0oUuXbrQrVs3zp8/j7WWTp06sWjRosv25+Pj42nf2kv/2c1KELOPp0yZMp541vistTRt2pSvv/66SOd/vSvUrUKstaeMMZ8DrYCbjTGl3TNj/kDWfweSgduAZGNMaaAycCJbPEv2OrnFjxWhDxERkWJXUm7t4efnR0pKylVpa8KECTz++ONER0dz8803Y60lLS0NgFatWjFq1CgSEhK44447OHPmDMnJyTRo0CDP9u+++24WLlzIX//6Vz7//HOqVavmuX4uPw0bNuTo0aN8/fXXtG7dmvT0dA4cOEDTpk2v7KSvEwXZbVrdPeOGMaY80BGIBz4D+riLDQI+cb9f7v6M+/hGm5l+Lwf6u3eKBgD1gW3AdqC+e2dpWTI3NSx31ylsHyIiItetqlWr0qZNGwIDA/PcsFBQ/fv3Z9q0aYSGhpKYmMjjjz9Ox44dadmyJcHBwbRp04bQ0FBCQ0OpXr06UVFRDBgwwLMBIWtjQl4iIyOJiYkhODiY8ePHM3/+/MuWz65s2bIsXbqUcePGERISgsPhYMuWLVd0vtcTk1/OY4wJJnNzgA+Zyd4H1toXjTF1ydxcUAXYBfzRWnvOGOMLLABCyZwN62+tPeRuayLwGJABPG2tXeWO3wu85u7jHWvtFHe80H3kxel02piYmEJ9OSJSePGNcruqIn+N98cX80hEii4+Pp7GjYv2WxYprNx+b8aYHdZaZ27l8102tdbuITNJujh+iMydohfH04C+ebQ1BZiSS3wlsLI4+hARERG5nunxWCIiIl4oNjaWgQMH5oiVK1eOrVu3XqMRye9FyZuIiIgXCgoKKvJuVPFuerapiIiIiBdR8iYiIiLiRZS8iYiIiHgRXfMmIiKSj6LeAicvujWOXAnNvImIiJRAPj4+OBwOAgMD6du3L2fOnOHUqVPMnj27wHUdDgfdu3f3xGfOnMkdd9yBMYZjx45dpgWIiorCGMOGDRs8sY8//hhjDEuXLi36ickVU/ImIiJSAmU92zQuLo6yZcsyZ86cAidvWXVdLhfLly/3xNu0acP69eu5/fbbCzSGoKCgHM8zjY6OJiQkpPAnI8VKyZuIiEgJ17ZtWxISEhg/fjyJiYk4HI4iPR4rNDSUOnXqFKrfbdu2kZ6eTmpqKgkJCTgcDs/xF198kebNmxMYGMjw4cM9D6NPSEigY8eOhISEEBYWRmJiYqHHKnlT8iYiIlKCZWRksGrVKoKCgpg6dSr16tXD5XIxbdq0POukpaXhdDpp1aoVy5YtK3Lfxhg6duzImjVr+OSTT3IswQKMHj2a7du3ExcXx9mzZ1mxYgUADz/8MKNGjWL37t1s2bKFmjVrFnkMciklbyIiIiXQ2bNncTgcOJ1OateuzZAhQwpc9/vvvycmJob333+fp59++opmvvr37090dDTR0dEMGDAgx7HPPvuMli1bEhQUxMaNG9m7dy8pKSn8+OOP9OrVCwBfX18qVKhQ5P7lUtptKiIiUgJlXbdWFLfeeisAdevWJTw8nF27dlGvXr0itdWiRQvi4uIoX748DRo08MTT0tJ44okniImJ4bbbbiMyMpK0tDTP0qlcPUreRERE8lFSbu3h5+dHSkrKZcucPHmSChUqUK5cOY4dO8bmzZt55plnrqjfl19+GV9f3xyxtLQ0AKpVq0ZqaipLly6lT58+VKpUCX9/f5YtW0bPnj05d+4cFy5c0OxbMdKyqYiIiJeoWrUqbdq0ITAwMM8NC/Hx8TidTkJCQmjfvj3jx4+nSZMmAMyYMQN/f3+Sk5MJDg5m6NChBeq3a9eutG/fPkfs5ptvZtiwYQQFBdGzZ0+aN2/uObZgwQJmzJhBcHAwd955J//5z3+KeMaSG3OjTG86nU4bExNzrYchct0r6s1MS8rMhghkJkCNGxfvjXlF8pLb780Ys8Na68ytvGbeRERERLyIrnkTERHxQrGxsQwcODBHrFy5cmzdurVQ7cybN4/XX389R6xNmzbMmjXriscoV4eSNxERES8UFBRU5N2o2T366KM8+uijxTAi+b1o2VRERETEiyh5ExEREfEiSt5EREREvIiueRMREcnHrJEbi7W9UXPuKdb25MaimTcREZESyMfHB4fDQWBgIH379uXMmTOcOnWK2bNnF7iuw+HI8TD5mTNncscdd2CM4dixY5dtIyoqCmMMGzZs8MQ+/vhjjDEsXbq0yOf16quv4uvry+nTpwtdNyoqitGjRwMQGRnJ9OnTARg8eDABAQGec77zzjsB+Pnnn+nWrRshISE0adKEe++9t8jjLkmUvImIiJRAWc82jYuLo2zZssyZM6fAyVtWXZfLxfLlyz3xNm3asH79em6//fYCjSEoKIhFixZ5PkdHRxMSElL4k8lm0U0llcQAACAASURBVKJFNG/enI8//viK2rnYtGnTPOe8ZcsWAJ5//nk6derE7t272bdvH1OnTi3WPq8VJW8iIiIlXNu2bUlISGD8+PEkJibicDjyfDzW5YSGhlKnTp1C9btt2zbS09NJTU0lISEBh8PhOf7iiy/SvHlzAgMDGT58uOeh9AkJCXTs2JGQkBDCwsJITEwEIDExkdTUVCZPnpwjKYyKiqJ3795ERERQv379HM9inTdvHg0aNKBdu3Zs3ry5UOd75MgR/P39PZ+Dg4MLVb+kUvImIiJSgmVkZLBq1SqCgoKYOnUq9erVw+VyMW3atDzrpKWl4XQ6adWqFcuWLSty38YYOnbsyJo1a/jkk09yLMECjB49mu3btxMXF8fZs2dZsWIFAA8//DCjRo1i9+7dbNmyhZo1awKZs24DBgygbdu2fPvtt/zyyy+etlwuF4sXLyY2NpbFixfzww8/cOTIESZNmsTmzZtZt24d+/bty3OsY8eO9SybPvzwwwCMGjWKIUOG0L59e6ZMmcJPP/1U5O+iJFHyJiIiUgKdPXsWh8OB0+mkdu3aDBkypMB1v//+e2JiYnj//fd5+umnPTNfRdG/f3+io6OJjo5mwIABOY599tlntGzZkqCgIDZu3MjevXtJSUnhxx9/pFevXgD4+vpSoUIFIHPZtX///pQqVYrevXuzZMkST1sdOnSgcuXK+Pr60qRJE7777ju2bt1KeHg41atXp2zZsvTr1y/PcWZfNl24cCEAXbp04dChQwwbNoz9+/cTGhrK0aNHi/xdlBTabSoiIlICZV23VhS33norAHXr1iU8PJxdu3ZRr169IrXVokUL4uLiKF++PA0aNPDE09LSeOKJJ4iJieG2224jMjKStLQ0z9Lpxfbs2cPBgwfp1KkTAOfPn6du3bqMGjUKyHy0VxYfHx8yMjKAzNm/K1GlShUeeughHnroIbp168amTZt44IEHrqjNa03Jm4iISD5Kyq09/Pz8SElJuWyZkydPUqFCBcqVK8exY8fYvHlzjmvIiuLll1/G19c3RywtLQ2AatWqkZqaytKlS+nTpw+VKlXC39+fZcuW0bNnT86dO8eFCxdYtGgRkZGRTJgwwdNGQEAA3333XZ79tmzZkqeeeorjx49TqVIllixZUqgNExs3bqRVq1ZUqFCBlJQUEhMTqV27diHPvuTRsqmIiIiXqFq1Km3atCEwMDDPDQvx8fE4nU5CQkJo374948ePp0mTJgDMmDEDf39/kpOTCQ4OZujQoQXqt2vXrrRv3z5H7Oabb2bYsGEEBQXRs2dPmjdv7jm2YMECZsyYQXBwMHfeeSf/+c9/iI6O9iylZunVqxfR0dF59luzZk0iIyNp3bo1HTt2JCwsLM+y2a95czgcnD9/nh07duB0OgkODqZ169YMHTo0xzi9lclrevN643Q6bUxMzLUehsh1L75R4yLVa7w/vphHIlJ08fHxNG5ctN+ySGHl9nszxuyw1jpzK6+ZNxEREREvomveREREvFBsbCwDBw7MEStXrhxbt24tVDvz5s3j9ddfzxFr06YNs2bNuuIxytWh5E1ERMQLBQUFFXk3anaPPvoojz76aDGMSH4vWjYVERER8SJK3kRERES8iJI3ERERES+ia95ERETy8c9+3Yq1vb8sXlGgcsnJyYwaNYp9+/bx22+/0a1bN6ZNm8aWLVvo0aMHAQEBnrLTp0+nY8eOTJkyhffffx8fHx9KlSrFv/71L1q2bJlnHxMnTuTdd9/l5MmTpKamXvG5ydWnmTcREZESyFpL79696dmzJwcPHuTAgQOkpqYyceJEANq2bet5lqfL5aJjx458/fXXrFixgp07d7Jnzx7Wr1/Pbbfddtl+7r//frZt2/Z7nJIUEyVvIiIiJdDGjRvx9fX17AT18fHh1Vdf5Z133uHMmTO51jly5AjVqlXzPCe0WrVq3HrrraxatYoHH3zQU+7zzz/n/vvvB6BVq1bUrFnzKp+NFCclbyIiIiXQ3r17adasWY5YpUqVqF27NgkJCXz55Zc5HgeVmJhI586d+eGHH2jQoAFPPPEEX3zxBQCdOnXim2++4ddffwVg8eLF9OvX73c/JykeSt5ERERKIGstxpg84xcvm9arV4+KFSuyY8cO3nrrLapXr06/fv2IioqidOnSRERE8O9//5uMjAw+/fRTevTocQ3OSoqDNiyIiIiUQE2bNuXDDz/MEfvvf//LDz/8QL169fKs5+PjQ3h4OOHh4QQFBTF//nwGDx5Mv379mDVrFlWqVKF58+b4+fld7VOQq0QzbyIiIiVQhw4dOHPmDO+++y4AFy5c4C9/+QuDBw+mQoUKudb59ttvOXjwoOezy+Xi9ttvByA8PJydO3cyd+5cLZl6uXxn3owxtwHvAv8P+A14y1r7ujEmEhgGHHUXfdZau9JdZwIwBLgA/Mlau8YdjwBeB3yA/7XWTnXHA4BooAqwExhorT1vjCnn7rsZcBzoZ61NulwfIiIixa2gt/YoTsYYPv74Y5544gn+9re/8dtvv3Hvvffy0ksv8fXXX3uuecvy3HPPERAQwJNPPsmpU6coXbo0d9xxB2+99RaQOSPXrVs3oqKimD9/vqfeM888w/vvv8+ZM2fw9/dn6NChREZG/t6nK4VgrLWXL2BMTaCmtXanMcYP2AH0BB4EUq210y8q3wRYBLQAbgXWAw3chw8AnYBkYDswwFq7zxjzAfCRtTbaGDMH2G2tfdMY8wQQbK0daYzpD/Sy1vbLqw9r7YW8zsPpdNqYmJhCfDUiUhTxjRoXqV7j/fHFPBKRoouPj6dx46L9lkUKK7ffmzFmh7XWmVv5fJdNrbVHrLU73e9TgHig1mWq9ACirbXnrLWHgQQyk6wWQIK19pC19jyZM209TObVmPcAS93155OZHGa1lfXfg6VAB3f5vPoQERERua4V6po3Y0wdIBTY6g6NNsbsMca8Y4y5xR2rBfyQrVqyO5ZXvCpwylqbcVE8R1vu46fd5fNq6+LxDjfGxBhjYo4ePXrxYRERERGvU+DkzRhTEfgQeNpa+1/gTaAe4ACOAP/MKppLdVuEeFHayhmw9i1rrdNa66xevXouVURERES8S4GSN2NMGTITt4XW2o8ArLU/W2svWGt/A+byf8uWyUD2Z3H4Az9dJn4MuNkYU/qieI623McrAycu05aIiIjIdS3f5M19jdnbQLy19pVs8ezP0ugFxLnfLwf6G2PKuXeR1ge2kblBob4xJsAYUxboDyy3mTsmPgP6uOsPAj7J1tYg9/s+wEZ3+bz6EBEREbmuFeQmvW2AgUCsMcbljj0LDDDGOMhcrkwCRgBYa/e6d4/uAzKAUVm7QI0xo4E1ZN4q5B1r7V53e+OAaGPMZGAXmcki7r8LjDEJZM649c+vDxEREZHrWb7Jm7X2K3K/xmzlZepMAabkEl+ZWz1r7SFy2S1qrU0D+hamDxERkeKWPP7LYm3Pf2rbYm1Pbix6woKIiEgJ5OPjg8PhIDAwkL59+3LmzBlOnTrF7NmzC1zX4XDQvXt3T3zmzJnccccdGGM4duzYZduIiorCGMOGDRs8sY8//hhjDEuXZt7da+jQoezbty/PNhYuXEhwcDDBwcHceeed7N69O9+xS/6UvImIiJRA5cuXx+VyERcXR9myZZkzZ06Bk7esui6Xi+XLl3vibdq0Yf369Z5HZuUnKCiIRYsWeT5HR0cTEhLi+fy///u/NGnSJM/6AQEBfPHFF+zZs4e//vWvDB8+vED9yuUpeRMRESnh2rZtS0JCAuPHjycxMRGHw8HYsWML3U5oaCh16tQpVL/btm0jPT2d1NRUEhIScjySKzw8nKynF61evZqwsDBCQkLo0KEDAHfeeSe33JJ5G9hWrVqRnJxc6DHLpQqyYUFERESukYyMDFatWkVERARdu3YlLi4Ol8t12TppaWk4nU5Kly7N+PHj6dmz52XL58UYQ8eOHVmzZg2nT5+me/fuHD58+JJyR48eZdiwYWzatImAgABOnDhxSZm3336brl27FmkckpOSNxERkRLo7Nmznlmutm3bMmTIEH76qWC3NP3++++59dZbOXToEPfccw9BQUHUq1evSOPo378/M2bM4PTp0/zzn//kpZdeuqTMN998w913301AQAAAVapUyXH8s88+4+233+arr74q0hgkJyVvIiIiJVDWdWtFceuttwJQt25dwsPD2bVrV5GTtxYtWhAXF0f58uVp0KBBrmWstWTeFvZSe/bsYejQoaxatYqqVasWaQySk5I3ERGRfJSUW3v4+fmRkpJy2TInT56kQoUKlCtXjmPHjrF582aeeeaZK+r35ZdfxtfXN8/jrVu3ZtSoURw+fNizbFqlShW+//57evfuzYIFC/JM/KTwtGFBRETES1StWpU2bdoQGBiY54aF+Ph4nE4nISEhtG/fnvHjx3t2hM6YMQN/f3+Sk5MJDg5m6NChBeq3a9eutG/fPs/j1atX56233qJ3796EhITQr18/AF588UWOHz/OE088gcPhwOl0FvKMJTcm82lT1z+n02mzdsSIyNUT36hxkeo13h9fzCMRKbr4+HgaNy7ab1mksHL7vRljdlhrc812NfMmIiIi4kV0zZuIiIgXio2NZeDAgTli5cqVY+vWrYVqZ968ebz++us5Ym3atGHWrFlXPEa5OpS8iYiIeKGgoKAi70bN7tFHH+XRRx8thhHJ70XLpiIiIiJeRMmbiIiIiBdR8iYiIiLiRXTNm4iISD4iIyNLdHtyY9HMm4iISAmUlJREYGBgjlhkZCTTp0+/RiMquqioKEaPHp3n8cjISGrVqoXD4aBJkyYsWrSo2PquU6cOx44dK7b2SgIlbyIiInLNjRkzBpfLxSeffMKIESNIT0+/1kMqsZS8iYiIeJnw8HDGjRtHixYtaNCgAV9++SWQOcPVs2dP7r//fgICApg5cyavvPIKoaGhtGrVihMnTgAwd+5cmjdvTkhICA888ABnzpwBYPDgwYwcOZK2bdvSoEEDVqxY4Wm3R48eRERE0LBhQ1544QXPWN577z1atGiBw+FgxIgRXLhwAci8f1yDBg1o164dmzdvLvC51a9fnwoVKnDy5EkAEhMTiYiIoFmzZrRt25b9+/cD8O9//5uWLVsSGhpKx44d+fnnnwE4fvw4nTt3JjQ0lBEjRpD1JKlff/2V++67j5CQEAIDA1m8eHGRv/9rTcmbiIiIF8rIyGDbtm289tprOZKpuLg43n//fbZt28bEiROpUKECu3btonXr1rz77rsA9O7dm+3bt7N7924aN27M22+/7amflJTEF198waeffsrIkSNJS0sDYNu2bSxcuBCXy8WSJUuIiYkhPj6exYsXs3nzZlwuFz4+PixcuJAjR44wadIkNm/ezLp169i3b1+Bz2vnzp3Ur1+fGjVqADB8+HDeeOMNduzYwfTp03niiScAuOuuu/jmm2/YtWsX/fv35x//+AcAL7zwAnfddRe7du2ie/fufP/99wCsXr2aW2+9ld27dxMXF0dERMQVfPvXljYsiIiIlEDGmMvGe/fuDUCzZs1ISkryHG/fvj1+fn74+flRuXJl7r//fiDzpr579uwBMhO85557jlOnTpGamkqXLl089R988EFKlSpF/fr1qVu3rmemq1OnTlStWtXT91dffUXp0qXZsWMHzZs3B+Ds2bPUqFGDrVu3Eh4eTvXq1QHo168fBw4cuOz5vvrqq8ydO5dDhw6xevVqAFJTU9myZQt9+/b1lDt37hwAycnJ9OvXjyNHjnD+/HkCAgIA2LRpEx999BEA9913H7fccovn/P/nf/6HcePG0a1bN9q2bXvZ8ZRkmnkTEREpgapWrepZOsxy4sQJqlWrBmQ+CgvAx8eHjIwMT5msOECpUqU8n0uVKuUpN3jwYGbOnElsbCyTJk3yzK7BpUlj1ufc4tZaBg0ahMvlwuVy8e2333p20uaVfOZlzJgxfPvttyxevJhHHnmEtLQ0fvvtN26++WZP+y6Xi/j4eACefPJJRo8eTWxsLP/6178uew4ADRo0YMeOHQQFBTFhwgRefPHFQo2vJNHMm4iISD6uxa09KlasSM2aNdmwYQMdOnTgxIkTrF69mqeeeop58+ZdUdspKSnUrFmT9PR0Fi5cSK1atTzHlixZwqBBgzh8+DCHDh2iYcOG7Nq1i3Xr1nHixAnKly/PsmXLeOedd6hQoQI9evRgzJgx1KhRgxMnTpCSkkLLli156qmnOH78OJUqVWLJkiWEhIQUaGy9e/dm/vz5zJ8/nxEjRhAQEMCSJUvo27cv1lr27NlDSEgIp0+f9ox7/vz5nvp33303Cxcu5LnnnmPVqlWeBPinn36iSpUq/PGPf6RixYpERUVd0Xd4LSl5ExERKaHeffddRo0axV/+8hcAJk2aRL169a643b/97W+0bNmS22+/naCgIFJSUjzHGjZsSLt27fj555+ZM2cOvr6+QOY1ZgMHDiQhIYGHHnoIp9MJwOTJk+ncuTO//fYbZcqUYdasWbRq1YrIyEhat25NzZo1CQsL82xkKIjnn3+ehx56iGHDhrFw4UIef/xxJk+eTHp6Ov379yckJITIyEj69u1LrVq1aNWqFYcPH/Z8RwMGDCAsLIx27dpRu3ZtAGJjYxk7diylSpWiTJkyvPnmm1f8PV4rJmsXxvXO6XTamJiYaz0MketefKPGRarXeH98MY9EpOji4+Np3Lhov2VvNnjwYLp160afPn1yxKOiooiJiWHmzJnXaGTXt9x+b8aYHdZaZ27ldc2biIiIiBfRsqmIiIgA5Hkd2ODBgxk8ePAVtz9lyhSWLFmSI9a3b18mTpx4xW3fSJS8iYiI5MJaW+gdk3J5EydOVKJ2kaJcvqZlUxERkYv4+vpy/PjxIv3DKlJQ1lqOHz/u2RRSUJp5ExERuYi/vz/JyckcPXr0Wg9FrnO+vr74+/sXqo6SNxERkYuUKVPGc8d+kZJGy6YiIiIiXkTJm4iIiIgXUfImIiIi4kWUvImIiIh4ESVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBElbyIiIiJeRMmbiIiIiBdR8iYiIiLiRZS8iYiIiHgRJW8iIiIiXiTf5M0Yc5sx5jNjTLwxZq8x5il3vIoxZp0x5qD77y3uuDHGzDDGJBhj9hhjwrK1Nchd/qAxZlC2eDNjTKy7zgxjjClqHyIiIiLXs4LMvGUAf7HWNgZaAaOMMU2A8cAGa219YIP7M0BXoL77NRx4EzITMWAS0BJoAUzKSsbcZYZnqxfhjheqDxEREZHrXb7Jm7X2iLV2p/t9ChAP1AJ6APPdxeYDPd3vewDv2kzfADcbY2oCXYB11toT1tqTwDogwn2skrX2a2utBd69qK3C9CEiIiJyXSvUNW/GmDpAKLAV+IO19ghkJnhADXexWsAP2aolu2OXiyfnEqcIfVw83uHGmBhjTMzRo0cLc6oiIiIiJVKBkzdjTEXgQ+Bpa+1/L1c0l5gtQvyywylIHWvtW9Zap7XWWb169XyaFBERESn5CpS8GWPKkJm4LbTWfuQO/5y1VOn++4s7ngzclq26P/BTPnH/XOJF6UNERETkulaQ3aYGeBuIt9a+ku3QciBrx+gg4JNs8UfcO0JbAafdS55rgM7GmFvcGxU6A2vcx1KMMa3cfT1yUVuF6UNERETkula6AGXaAAOBWGOMyx17FpgKfGCMGQJ8D/R1H1sJ3AskAGeARwGstSeMMX8DtrvLvWitPeF+/zgQBZQHVrlfFLYPERERkeudydzgef1zOp02JibmWg9D5LoX36hxkeo13h9fzCMREfFexpgd1lpnbscKMvMmIlJgjfvr8lMRkatJj8cSERER8SJK3kRERES8iJI3ERERES+i5E1ERETEiyh5ExEREfEiSt5EREREvIiSNxEREREvouRNRERExIsoeRMRERHxIkreRERERLyIkjcRERERL6LkTURERMSLKHkTERER8SJK3kRERES8iJI3ERERES+i5E1ERETEiyh5ExEREfEiSt5EREREvIiSNxEREREvouRNRERExIsoeRMRERHxIkreRERERLyIkjcRERERL6LkTURERMSLKHkTERER8SJK3kRERES8iJI3ERERES+i5E1ERETEiyh5ExEREfEiSt5EREREvIiSNxEREREvouRNRERExIsoeRMRERHxIkreRERERLyIkjcRERERL6LkTURERMSLKHkTERER8SJK3kRERES8iJI3ERERES+i5E1ERETEiyh5ExEREfEiSt5EREREvIiSNxEREREvkm/yZox5xxjzizEmLlss0hjzozHG5X7dm+3YBGNMgjHmW2NMl2zxCHcswRgzPls8wBiz1Rhz0Biz2BhT1h0v5/6c4D5eJ78+RERERK53BZl5iwIicom/aq11uF8rAYwxTYD+QFN3ndnGGB9jjA8wC+gKNAEGuMsC/N3dVn3gJDDEHR8CnLTW3gG86i6XZx+FO20RERER75Rv8mat3QScKGB7PYBoa+05a+1hIAFo4X4lWGsPWWvPA9FAD2OMAe4Blrrrzwd6Zmtrvvv9UqCDu3xefYiIiIhc967kmrfRxpg97mXVW9yxWsAP2coku2N5xasCp6y1GRfFc7TlPn7aXT6vtkRERESue0VN3t4E6gEO4AjwT3fc5FLWFiFelLYuYYwZboyJMcbEHD16NLciIiIiIl6lSMmbtfZna+0Fa+1vwFz+b9kyGbgtW1F/4KfLxI8BNxtjSl8Uz9GW+3hlMpdv82ort3G+Za11Wmud1atXL8qpioiIiJQoRUrejDE1s33sBWTtRF0O9HfvFA0A6gPbgO1AfffO0rJkbjhYbq21wGdAH3f9QcAn2doa5H7fB9joLp9XHyIiIiLXvdL5FTDGLALCgWrGmGRgEhBujHGQuVyZBIwAsNbuNcZ8AOwDMoBR1toL7nZGA2sAH+Ada+1edxfjgGhjzGRgF/C2O/42sMAYk0DmjFv//PoQERERud6ZzMms65/T6bQxMTHXehgi17/IykWsd7p4xyEi4sWMMTustc7cjukJCyIiIiJeRMmbiIiIiBdR8iYiIiLiRZS8iYiIiHgRJW8iIiIiXkTJm4iIiIgXUfImIiIi4kWUvImIiIh4ESVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBElbyIiIiJeRMmbiIiIiBdR8iYiIiLiRZS8iYiIiHgRJW8iIiIiXkTJm4iIiIgXUfImIiIi4kWUvImIiIh4ESVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBElbyIiIiJeRMmbiIiIiBdR8iYiIiLiRZS8iYiIiHgRJW8iIiIiXkTJm4iIiIgXUfImIiIi4kWUvImIiIh4ESVvIiIiIl5EyZuIiIiIF1HyJiIiIuJFlLyJiIiIeBElbyIiIiJeRMmbiIiIiBdR8iYiIiLiRZS8iYiIiHgRJW8iIiIiXkTJm4iIiIgXUfImIiIi4kWUvImIiIh4kXyTN2PMO8aYX4wxcdliVYwx64wxB91/b3HHjTFmhjEmwRizxxgTlq3OIHf5g8aYQdnizYwxse46M4wxpqh9iIiIiFzvCjLzFgVEXBQbD2yw1tYHNrg/A3QF6rtfw4E3ITMRAyYBLYEWwKSsZMxdZni2ehFF6UNERETkRpBv8mat3QScuCjcA5jvfj8f6Jkt/q7N9A1wszGmJtAFWGetPWGtPQmsAyLcxypZa7+21lrg3YvaKkwfIiIiIte9ol7z9gdr7REA998a7ngt4Ids5ZLdscvFk3OJF6UPERERketecW9YMLnEbBHiRenj0oLGDDfGxBhjYo4ePZpPsyIiIiIlX1GTt5+zlirdf39xx5OB27KV8wd+yifun0u8KH1cwlr7lrXWaa11Vq9evVAnKCIiIlISFTV5Ww5k7RgdBHySLf6Ie0doK+C0e8lzDdDZGHOLe6NCZ2CN+1iKMaaVe5fpIxe1VZg+RERERK57pfMrYIxZBIQD1YwxyWTuGp0K/7+9ew+zra7rOP7+CIgQyE1EFBJUSEqR5HgLUDAjoEwzCKwnBS/HSKU0S55H0oPXIxkWhoSZAl5TS6TiIo8CoYKcA3EHEQH1hBgIIQZpwLc/1m9sM825zVmzZ9Y+79fz7Gf2/q29fp/1m9mz5jvrymeSvBL4DnBIe/uZwEHAjcC9wBEAVXVnkncAy9r73l5VUydBHEl3RusmwFntwdpmSJIkrQ/SneQ5+RYtWlTLly+f78WQJt+SLWY53939LockDViSS6tq0UzTvMOCJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNiMWbJEnSgFi8SZIkDYjFmyRJ0oCsU/GW5JYkVyW5PMny1rZ1knOTfLN93aq1J8kJSW5McmWSp4/08/L2/m8meflI+56t/xvbvFlVhiRJ0qTrY8vbflW1R1Utaq+PBr5UVbsAX2qvAQ4EdmmPxcBJ0BViwNuAZwHPBN42Uoyd1N47Nd8Bq8mQJEmaaHOx2/RFwKnt+anAi0faT6vOxcCWSbYHfhU4t6rurKq7gHOBA9q0R1bVRVVVwGnT+popQ5IkaaKtpDwuWAAAEEhJREFUa/FWwBeTXJpkcWvbrqq+B9C+Prq1Pw747si8K1rbqtpXzNC+qoyHSLI4yfIky2+//fZZDlGSJGnh2HAd59+rqm5N8mjg3CTXr+K9maGtZtG+xqrqQ8CHABYtWrRW80qSJC1E67TlrapubV//A/g83TFr32+7PGlf/6O9fQWw48jsOwC3rqZ9hxnaWUWGJEnSRJt18ZbkZ5JsPvUc2B+4GjgDmDpj9OXAF9rzM4CXtbNOnw3c3XZ5ngPsn2SrdqLC/sA5bdo9SZ7dzjJ92bS+ZsqQJEmaaOuy23Q74PPt6h0bAp+sqrOTLAM+k+SVwHeAQ9r7zwQOAm4E7gWOAKiqO5O8A1jW3vf2qrqzPT8SOAXYBDirPQCWriRDkiRpos26eKuqm4CnzdD+A+CXZ2gv4LUr6esjwEdmaF8OPGVNMyRJkiadd1iQJEkaEIs3SZKkAVnXS4VIGoglS5aMZR5J0txyy5skSdKAWLxJkiQNiLtNpfXEq/7bE7QlaRJYvEnriXtOX7z6N0239Lr+F0SStE7cbSpJkjQgFm+SJEkDYvEmSZI0IBZvkiRJA2LxJkmSNCAWb5IkSQNi8SZJkjQgFm+SJEkDYvEmSZI0IBZvkiRJA2LxJkmSNCAWb5IkSQNi8SZJkjQgFm+SJEkDYvEmSZI0IBZvkiRJA2LxJkmSNCAWb5IkSQOy4XwvgCTN1pIlS8YyjyQtJBZv0npit8Nune9FkCT1wOJN0mAd+um/X/uZ3PImaeA85k2SJGlALN4kSZIGxOJNkiRpQDzmTZpHsz3z0TMmJWn95ZY3SZKkAbF4kyRJGhCLN0mSpAGxeJMkSRoQT1iQ5tGsLjILXmi28a4RktZHFm+StAa8j6qkhcLiTZLWwLa3PXe+F0GSAIs3SVojzz//tbOY67rel0OSLN7GyAuyDoe7yCRJC5XFG5P5h3qcYxpX1iT+nCRJWlsWbxqMza9bPrasWZ0FOotC0bMlNd2Koy9c63l2WLrPHCyJpIXK4k3rbFyFzkFXfGvtc6SBuef0xWs/01KPrZPWJ4Mu3pIcAPwVsAHw4apaOs+LtGCMq6CSpNWZzdZEcIuitDKDLd6SbACcCPwKsAJYluSMqrp2fpds/TOuXX/j3MXo7kxNN4mfieuevNtaz7Pb9Wu/lW9WWxNhVlsUPTZW64PBFm/AM4Ebq+omgCSfBl4ErHXxtoT3zyJ+ySzmmE3O7LIm8Q+NtD7wn5R1M669DpN4gheMr6AfVw5MZkGfqprvZZiVJAcDB1TVq9rr3wOeVVWvG3nPYmDqX76fA76xljGPAu7oYXHX1yzHNIysSRzTOLMc0zCyJnFM48xyTOPPenxVbTvThCFvecsMbQ+pRKvqQ8CHZh2QLK+qRbOdf33PckzDyJrEMY0zyzENI2sSxzTOLMe0sLIe1ldH82AFsOPI6x2AydsHIEmSNGLIxdsyYJckOyd5OHAYcMY8L5MkSdKcGuxu06q6P8nrgHPoLhXykaq6pueYWe9yNWusOePMckxmzVfOOLMck1nzlTPOrMGOabAnLEiSJK2PhrzbVJIkab1j8SZJkjQgFm+SJEkDYvEmaeIkefR8L8NcSLLNfC+DpPln8bZAuFJeeJJskWRpkuuT/KA9rmttW45xOc7qsa9HJnlPko8l+Z1p0z7YV07r7zFJTkpyYpJtkixJclWSzyTZvsecrac9tgEuSbJVkq37ymlZB4w83yLJ3yW5Msknk2zXc9bSJI9qzxcluQn4epJvJ3lejzmXJTkmyRP76nMVWYuSnJfk40l2THJukruTLEvyiz3mbJbk7Umuaf3fnuTiJIf3lTGStWGS1yQ5u30WrkhyVpLfT7JR33krWYZ+z2RMNmhjekeSvaZNO6bnrE2T/GmSP0nyiCSHJzkjyXFJNusza4bsG+ao391Hnm/Ufr/OSPLuJJv2kWHxNmJcK+ZxrZRb/2NZMY9rpdyyxrVi/gxwF7BvVW1TVdsA+7W2z/YZlOTpK3nsCezRY9RH6e5O8g/AYUn+IcnGbdqze8wBOIXuXsPfBc4D7gN+DbgQ+Jsec+4ALh15LAceB1zWnvfp3SPP/wL4HvBCuutOntxz1q9V1dTtdP4cOLSqngT8Ssvuy1bAlsB5SS5J8oYkj+2x/1EfBI4D/gX4GnByVW0BHN2m9eUTwE3ArwLHAicAvwfsl+Tdq5pxFj5G9zu6BDiI7jN+LPA04ON9hczwT8roPysH9ZXTnAw8D/gBcEKS40emvaTnrFOA7YCd6T4Xi4D30a2nTuorJMk9SX7YHvckuQd44lR7XznNKSPPlwJPovud3YS+1n1V5aM9gMtGnn8YeCfweOANwOk95lw18vw84Bnt+a7A8p7HdDPdL8J3gEvaWB47B9+7S4ADgZfS/bE+uLX/MnBRz1lfAA6nu6vGG4E/A3YBTgXe3WPON2YzbZZZDwBfbp+H6Y/7esy5fNrrtwBfBbYZ/fz3lPVvI8+/s6rlWMecNwFnA08dabu5z7GM9Du6jpj+vextTK2/64EN2/OLp027qsec0THtQ1dE3dY+e4vH+Jn4tx5zrpj2eln7+jDg+p7HtKr1xA095jxAV5DePPKYev2Tnsd05cjzDemuUfaPwMZ9/pxa/5e3r2mfu4y8vrLHnA8ApwHbjbTd3OdYRvod/ZxfDmzU95jc8rZyi6rqmKr6dlW9H9ipx743SjJ1geRNqmoZQFXdQPfL0ae7qupNVfWzwB/TFTmXta1ki3vM2aiqzqqqTwFVVZ+je/Il4BE95gDsVFWnVNWKqjoe+I2q+iZwBP3+V/jttjn/p1tdk2yX5M10BWqfrgNeU1X7TX/Q742TN07y09/7qnoX3Yr5X+kKuD6Nrl9OmzZtg75Cqup9wKuAtyY5PsnmTLvPcY8eneSNSf4YeGSS0Xss970+PRE4M8nzgbOT/GWS5yY5lu4PQu+q6sKq+gO6LZfvBZ7Tc8R/J9k/ySFAJXkxQNvj8ECPOf+VZO/W9wuBOwGq6kFmvi/2urgrySGjv1dJHpbkULqt9H25iW4vwM4jjydU1c7A93vMAXj41JOqur+qFgNX0P2DOSe7Mqurbs5sX6de9/Z7XFWvB/4K+FSSo9rPa67WE1skeUmS3wI2rqr/acvQ25gs3h5qXCvmsa+UYc5XzONaKcP4VsyH0hU0FyS5K8mdwPnA1sBv95gD3S6XlX3GXt9jzj8Bzx9tqKpT6Qr7n/SYA/CFqWNWquqnx8kkeRLwjT6DWiF/CN3WonOBXo4rmcHfApvT/QE7FZg6/OEx9Py7W1UfoNtN+xrgRXRbsY8G/h14RY9R/++4n6p6oKrOrqojeswB+H26z9or6HZp7pfkP+m29h3VY86RwPGt7zdP9Z1kW7r1b58OAw4GbktyQzuO6ja6fyQP6zHnL+l2cc/kuB5zAJaPHkYEUFXH0h12sdMcZE2tJ376uW6H+tzTZ1BVXQq8oL28gP43LEy5gO5wil8HLp7aANDWE738M+4dFkYkedu0pg9W1e3tG35cVb2sx6x96VYwu9Jtlv4ucDrdbb7u7zHn01XV5wpkZTlPo1uBPEi3a/ZI4OV0f2heXVVf6zFrd7rd2rsCVwOvqKob2or5pVV1Qo9ZT6bbPXtxVf1opP2Aqjq7r5yRrMcBX5/LrFXkHFhVvZ0csZqsORsT3T8LT6yqq4f8cxpn1pjHtBvw2LnOajmPYzy/u8+i26LyLWA3uuNHr62qM3vOeSbdBpxlSX4eOIBuN3CvOQsk6xuMbImbg5x96I5hXj5HY3oW8OCcff/62Pe6PjyAIyYpxzGtUV9H0a1ATgduAV40Mq3v48PGkkW3FW9cYxpL1ph/TuP8/k3iZ+IoumP5xvGZmPOc1t/bgIvpTo55D/Al4K10hyK8ZQ5zvjwXOZOaNa6f07iyelvYSX8w7eDaoec4pjXq6ypgs/Z8p/aL+Iftdd8H7Y4lyzGZ5ZjmZEwb0O2q/yHwyNa+Cf0ecD+WnEnNmrQxTR00LyDJlSubRHcq86Byxpk1iWMCNqi2u6Wqbmm7uj+X5PH0f9DzuLIck1nzlTPOrHGO6f6qegC4N8m3quqHLfe+JA8OMGdSsyZqTBZvD7Ud3UG0088QCt01iYaWM86sSRzTbUn2qKrLAarqR0l+HfgI8NQec8aZ5ZjMmq+ccWaNc0w/SbJpVd0L7DnVmGQLumOAh5YzqVmTNaY+NxUO/QH8HbD3SqZ9cmg5jmmdc3YAHrOSaXv1PKaxZDkmsxxT72PaeCXtj2Lk2oNDyZnUrEkbk2ebSpIkDYjXeZMkSRoQizdJkqQBsXiTJEkaEIs3SQtOkrckuSbJlUkub1crH7Qk5ydZNIf975vkl0Zen5Lk4LnKkzR/vFSIpAUlyXPo7gn49Kr6cZJHMXKjbK3UvsCP6P/SPJIWGLe8SVpotgfuqKofA1TVHVV1a5I9k1yQ5NIk5yTZHqC1X5HkoiR/nuTq1n54kr+e6jTJP7eLtZJk//b+y5J8Nu3G2EluSXJsa7+q3fOTJJsl+WhruzLJb62qnzWVZIO2zMtav69p7fu2LXWfS3J9kk8kSZt2UGv7SpIT2rh2orvp+xvalsp9WsRzk3wtyU1uhZMmh8WbpIXmi8COSW5I8sEkz0uyEfAB4OCq2pPuYqvvau//KHBUVT1nTTpvW/KOAV5QVU+nu3XSG0feckdrPwl4U2v7M+DuqnpqVe0OfHkN+lkTr2z9PgN4BvDqJDu3ab8I/BHw88ATgL2SPAI4GTiwqvYGtoXuLgLA3wDvr6o9qurC1sf2wN50WzKXruWySVqg3G0qaUGp7mr4ewL7APsBfw+8E3gKcG7bALUB8L12xfItq+qCNvvHgANXE/FsuoLoq62vhwMXjUz/x/b1UuAl7fkLgMNGlvGudsX+VfWzJvYHdh/ZKrYFsAvwE+CSqloBkORyunt0/gi4qapubu//FLB4Ff2fXlUPAtcm6fU2dZLmj8WbpAWnuvsCng+cn+Qq4LXANdO3riXZEljZlcbv56F7Fx4xNRtwblW9dCXz/bh9fYD/W0dmhpzV9bMmAry+qs55SGO3e/fHI01Ty7K29+Uc7aPve3pKmifuNpW0oCT5uSS7jDTtAVwHbNtOZiDJRkl+oar+E7g7yd7tvb87Mt8twB5JHpZkR+CZrf1iul2QT2p9bZpk19Us1heB140s41az7Ge6c4Aj225hkuya5GdW8f7rgSe0Y9wADh2Zdg+w+VrmSxogizdJC81mwKlJrk1yJd2uybcCBwPvTXIFcDkwdVmMI4ATk1wE3DfSz1eBm4GrgPcBlwFU1e3A4cCnWv8XA09ezTK9E9gqydUtf79Z9vMvSVa0x2eBDwPXApe1Ey1OZhV7RKrqPuAPgLOTfAX4PnB3m/xPwG9OO2FB0gTy3qaSJkbbIvXPVfWUeV6UOZNks3ZcYIATgW9W1fvne7kkjY9b3iRpWF7dTmC4hu4Eh5PneXkkjZlb3iSpR0k+D+w8rfnN009KkKTZsniTJEkaEHebSpIkDYjFmyRJ0oBYvEmSJA2IxZskSdKAWLxJkiQNyP8CLYcFIbIuZ5EAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 720x504 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "fileextension='.fastq.gz'\n",
    "directory = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts53Later'\n",
    "outplotname = 'Ptiwi08RIP_53Later_Inserts_Stacked_Barplot.pdf'\n",
    "startseqlength = 15\n",
    "cleanreads=False\n",
    "blastdatabase = '/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabaselist=['/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2']\n",
    "\n",
    "readsubtract.main(fileextension, directory, outplotname, startseqlength, cleanreads, blastdatabase, aligndatabaselist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
