{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2021-11-09 14:10:21.192659\n"
     ]
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "import datetime\n",
    "print(datetime.datetime.now())\n",
    "\n",
    "from pygentoolbox import Tools\n",
    "from pygentoolbox import ReadSubtractionByGenomeMapping as readsubtract\n",
    "#dir(pygentoolbox.Tools)\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/media/sf_LinuxShare/PyPacks/PyGenToolbox/notebooks/Theresa\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.getcwd())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  Created \n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  Created \n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 50905\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 554\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 47474\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 3510\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 47381\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 93\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 47381\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 47381\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 47215\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 237\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  Created \n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 47215\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_15bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 70887\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 569\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 66290\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4721\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 66200\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 90\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 66200\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 66200\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 66018\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 212\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 66018\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 0\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_16bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 39364\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 685\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 32939\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 6597\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 32844\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 95\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 25605\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 16726\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 25524\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 81\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 25435\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 125\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 25006\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1525\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_17bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 35574\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 489\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 26161\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 9758\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 26096\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 65\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 16427\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 19561\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 16307\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 123\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 16270\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 57\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 15951\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 659\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_18bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 31942\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 414\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 19463\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 13502\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 19390\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 73\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 8021\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 18436\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 7889\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 132\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 7853\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 49\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 7548\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 518\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_19bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 38696\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 339\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 23798\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 15226\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 23709\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 89\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 9369\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 22268\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 9215\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 158\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 9163\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 8451\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 936\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_20bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 103841\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 255\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 73825\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 30604\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 73730\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 95\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 29792\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 66087\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 29523\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 270\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 29446\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 117\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 27481\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2641\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_21bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 322662\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 330\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 194944\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 130408\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 194806\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 138\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 26972\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 205472\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 26348\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 625\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 26210\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 221\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 22468\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 5579\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_22bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 2967770\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 865\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 1711920\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1264627\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 1711688\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 232\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 119970\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 1618609\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 117500\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 2493\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 116921\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 1053\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 107443\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 10400\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_23bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 168066\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 389\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 100313\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 68193\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 100136\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 177\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 24138\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 97592\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 23653\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 487\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 23483\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 271\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 18227\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 5518\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_24bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 59944\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 350\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 44803\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 15166\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 44666\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 137\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 11882\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 57448\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 11173\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 720\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 9972\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 1465\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 7845\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 2603\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_25bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 42627\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 212\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 35225\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 7413\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 35126\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 7204\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 39245\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 6332\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 889\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 6208\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 148\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 5138\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 1317\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_26bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 19018\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 286\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14733\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4289\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 14636\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 97\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5682\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 12180\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4673\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 1027\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 4589\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 103\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4003\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 749\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_27bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 18390\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 287\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 14103\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 4295\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 13994\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 109\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 5350\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 17286\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 4565\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 796\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 4488\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 126\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 4138\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 492\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_28bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 10326\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 229\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 9200\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 1130\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 9132\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 4009\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 12570\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3644\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 367\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 3615\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 41\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3438\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 256\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_29bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6088\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 154\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5422\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 666\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5376\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 46\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3428\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3286\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3354\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 75\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 3344\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 17\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3224\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 179\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_30bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5401\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 119\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5009\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 392\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 4940\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 69\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3200\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 3147\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3165\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 35\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 3160\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 8\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3075\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 98\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_31bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 6786\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 104\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 6500\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 286\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 6432\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 68\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 3495\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8591\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 3479\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 16\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 3476\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 4\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 3393\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 99\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_32bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 5710\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 40\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 5454\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 256\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 5403\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 51\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2499\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 8835\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2480\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 19\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 2467\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 13\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2415\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 72\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_33bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3877\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 43\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 3694\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 183\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 3658\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 36\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 2215\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2321\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 2211\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 4\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 2209\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 2\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 2164\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 48\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_34bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "Start trimming of:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.fastq.gz\n",
      "\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp  already exists\n",
      "fastp -i /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.fastq.gz -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "b''\n",
      "Finished trimming, files output to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "\n",
      "Start unpacking trimmed.gz file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "gzip -f -d -k /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq.gz\n",
      "Finished unpacking:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "\n",
      "Starting fastqc\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc  already exists\n",
      "/media/sf_LinuxShare/Programs/FastQC/fastqc -o /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc -f fastq fastq /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Finished fastqc, output at directory:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/fastqc\n",
      "\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam = 3128\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.NotMatch4.sort.bam = 26\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam = 2985\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.NotMatch4.sort.bam = 143\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam = 2937\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.NotMatch4.sort.bam = 48\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam = 1707\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.NotMatch4.sort.bam = 2853\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam = 1700\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.NotMatch4.sort.bam = 7\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam = 1695\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.NotMatch4.sort.bam = 6\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Finished\n",
      "Starting Hisat2: aligning\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.fastq\n",
      "Aligning reads to Reference:\n",
      "/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2\n",
      "Directory  /media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2  already exists\n",
      "Finished with Hisat2\n",
      "\n",
      "Starting samtools converting sam to sorted indexed bam\n",
      "Finished with Samtools\n",
      "\n",
      "Start Filtering file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.sort.bam\n",
      "with flag: 4\n",
      "Finished\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam = 1674\n",
      "Finished counting reads in file\n",
      "Start counting reads in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam\n",
      "Finished\n",
      "Number of reads mapped in file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.NotMatch4.sort.bam = 25\n",
      "Finished counting reads in file\n",
      "Start converting bam to fastq\n",
      "Finished\n",
      "Converted file:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.bam\n",
      "to\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/fastp/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/hisat2/170519_SND405_A_L005_HTD-52_AdapterTrimmed_R1_35bp.trim.Kpne_CompleteGenome.Match4.sort.Vector_FlagHA_Dcl5.Match4.sort.Pt_mtGenome.Match4.sort.Pt_51_Mac.Match4.sort.Pt_51_MacAndIES.Match4.sort.OESv1.Match4.sort.Pt_51_Mic2.Match4.sort.fastq\n",
      "Finished\n",
      "wrote out table of counts aligned to each genome to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/ReadsMappedToEachReference.tsv\n",
      "\n",
      "Output Stacked Barplot to:\n",
      "/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts/Ptiwi08RIP_52Late_Inserts_Stacked_Barplot.pdf\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAm8AAAGtCAYAAABA/pOqAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjEsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8QZhcZAAAgAElEQVR4nOzde5zOZeL/8ddlHIYMcvr+lGTIee45uZ2TkdNohUSotZRj0ZbdFVZlaim7bAchW181koxQsoqcKkVhhhvDiBmmGtnKcU2MZnT9/ph77u8MM+ZgZD68n4/HPMx9fa7T537M49G76/ocjLUWEREREXGGUld7AiIiIiJScApvIiIiIg6i8CYiIiLiIApvIiIiIg6i8CYiIiLiIApvIiIiIg5S+mpP4LdSvXp1W7du3as9DREREZF8xcXFHbXW1sjt2HUT3urWrUtsbOzVnoaIiIhIvowx3+R1TNumIiIiIg6i8CYiIiLiIApvIiIiIg5y3VzzJiIiUlDp6emkpKSQlpZ2taci1zh/f39q165NmTJlCtxG4U1EROQCKSkpBAQEULduXYwxV3s6co2y1nLs2DFSUlIIDAwscDttm4qIiFwgLS2NatWqKbjJFWWMoVq1aoVe4VV4ExERyYWCm/wWivJ3pvAmIiJynfDz8yM0NJSgoCDuvvtuTp48mW+bmTNn0qRJEx544IHfYIZSEApvIiIi14ny5cvj8XiIj4+natWqzJ49O982c+bM4aOPPmLhwoUFGiMjI+Nypyn5UHgTERG5DrVp04bDhw/7Pk+fPp0WLVoQHBzM5MmTARg1ahQHDx6kZ8+evPjii/z888889NBDtGjRgrCwMD744AMAoqOj6devH3fffTddu3bNs7/k5GSaNGnC8OHDadasGV27duXs2bMAJCYm0rlzZ0JCQggPDycpKSnPfq53Cm8iIiLXmfPnz7N+/Xp69uwJwJo1azhw4ABbt27F4/EQFxfHxo0bmTt3LjfddBOffPIJY8eOZerUqdx5551s27aNTz75hHHjxvHzzz8D8OWXXzJ//nw2bNiQZ38ABw4cYPTo0ezZs4cqVaqwbNkyAB544AFGjx7Nzp072bx5M7Vq1bpkP9czPSpERETkOnH27FlCQ0NJTk6mefPmdOnSBcgMb2vWrCEsLAyA1NRUDhw4wB133JGj/Zo1a1ixYgUzZswAMu/K/fbbbwHo0qULVatWvWR/derUITAwkNDQUACaN29OcnIyp0+f5vDhw9xzzz1A5rPPCjOv643Cm4iIyHUi65q3U6dO0aNHD2bPns0f//hHrLVMnDiRkSNHXrK9tZZly5bRqFGjHOVbtmzhhhtuyFEvt/6Sk5MpV66c77Ofnx9nz57FWpvneAWZ1/Um321TY4y/MWarMWanMWaPMeYZb3mgMWaLMeaAMWaxMaast7yc93Oi93jdbH1N9JZ/bYzplq080luWaIyZkK280GOIiIjIpVWuXJmZM2cyY8YM0tPT6datG2+88QapqakAHD58mB9//PGidt26deOVV17xha0dO3bk2n9B+8tSqVIlateuzfLlywE4d+4cZ86cKXQ/14uCXPN2DrjTWhsChAKRxpjWwN+BF621DYATwFBv/aHACWvtbcCL3noYY5oCA4BmQCQwxxjjZ4zxA2YD3YGmwEBvXQo7hoiIiBRMWFgYISEhxMTE0LVrV+6//37atGmDy+Wib9++nD59+qI2Tz31FOnp6QQHBxMUFMRTTz2Va98F7S+7BQsWMHPmTIKDg2nbti3/+c9/itTP9cDktVSZa2VjKgBfAA8DHwL/z1qbYYxpA0RZa7sZYz72/v6lMaY08B+gBjABwFr7vLevj4Eob9dR1tpu3vKJ3rJpwE+FGcNe4mTcbreNjY0t8LmKiMj1KyEhgSZNmlztach1Ire/N2NMnLXWnVv9Al3z5l0diwNuI3OVLAk4aa3NephLCnCz9/ebge8AvKHrFFDNW/5Vtm6zt/nugvJW3jaFHePoBfMeAYwAqFOnTkFOVUQuU1RU1G/aTkTkelOgR4VYa89ba0OB2kBLILf/Hcla9crtPQ+2GMsvNUbOAmtfs9a6rbXuGjVq5NJERERExFkK9Zw3a+1J4FOgNVDFu2UJmaHue+/vKcAtAN7jlYHj2csvaJNX+dEijCEiIiJyTSvI3aY1jDFVvL+XBzoDCcAnQF9vtcHAB97fV3g/4z2+wXst2gpggPdO0UCgAbAV2AY08N5ZWpbMmxpWeNsUdgwRERGRa1pBrnmrBcz3XvdWCnjXWrvSGLMXiDHGTAF2APO89ecBC4wxiWSuhg0AsNbuMca8C+wFMoDR1trzAMaYMcDHgB/whrV2j7ev8YUZQ0RERORal294s9buAsJyKT9I5vVvF5anAf3y6GsqMDWX8o+Aj4pjDBEREZFrmd5tKiIiIuIgej2WiIhIPupO+LBY+0ue9rt861SsWNH3ZoGPPvqIxx57jPXr11/xR1+lp6fz1FNPsWzZMsqVK0eFChV45pln6N69+xUbM/u55ubkyZO88847PPLII/n29cMPPzB27Fi++uorbrzxRsqWLcsTTzzhe2/qtUArbyIiIiXY+vXrefTRR1m9evVv8szSp556iiNHjhAfH098fDz//ve/r/pbDU6ePMmcOXPyrWetpXfv3txxxx0cPHiQuLg4YmJiSElJ+Q1m+dtReBMRESmhPv/8c4YPH86HH35I/fr1ARgyZAijRo2iffv2NGzYkJUrVwIQHR1Nnz59iIyMpEGDBjzxxBO+ftasWUObNm0IDw+nX79+ea5ynTlzhtdff51XXnnF9wL5//mf/+G+++4DYNGiRbhcLoKCghg/fryvXcWKFRk/fjzNmzenc+fObN26lYiICOrVq8eKFSt88+vVqxeRkZE0atSIZ555Jtc5TJ8+nRYtWhAcHMzkyZMBmDBhAklJSYSGhjJu3Lg8623YsIGyZcsyatQoX3+33norjz76KADnz59n3Lhxvnb/+te/APj000+JiIigb9++NG7cmAceeMD3/tb169cTFhaGy+XioYce4ty5cwDUrVuXv/71r7Rp0wa328327dvp1q0b9evXZ+7cuZc8n8ul8CYiIlICnTt3jl69erF8+XIaN26c41hycjKfffYZH374IaNGjSItLQ0Aj8fD4sWL2b17N4sXL+a7777j6NGjTJkyhXXr1rF9+3bcbjcvvPBCrmMmJiZSp04dKlWqdNGx77//nvHjx7NhwwY8Hg/btm3zvUj+559/JiIigri4OAICAnjyySdZu3Yt77//Pk8//bSvj61bt7Jw4UI8Hg9LlizhwtdWrlmzhgMHDrB161Y8Hg9xcXFs3LiRadOmUb9+fTweD9OnT8+z3p49ewgPD8/zO503bx6VK1dm27ZtbNu2jddff51Dhw4BsGPHDl566SX27t3LwYMH2bRpE2lpaQwZMsT3nWZkZPDqq6/6+rvlllv48ssvad++PUOGDGHp0qV89dVXvnPOa56XS9e8iYiIlEBlypShbdu2zJs3j5dffjnHsfvuu49SpUrRoEED6tWrx759+wDo1KkTlStXBqBp06Z88803nDx5kr1799KuXTsAfvnlF9q0aVPo+Wzbto2IiAiy3lj0wAMPsHHjRnr37k3ZsmWJjIwEwOVyUa5cOcqUKYPL5SI5OdnXR5cuXahWrRoAffr04YsvvsDt/r/Xd65Zs4Y1a9YQFpb5kIvU1FQOHDhw0XZxXvUuNHr0aL744gvKli3Ltm3bWLNmDbt27WLp0qUAnDp1igMHDlC2bFlatmxJ7dq1AQgNDSU5OZmAgAACAwNp2LAhAIMHD2b27Nk8/vjjAPTs2dN3zqmpqQQEBBAQEIC/vz8nT57Mc5533HFHob//7BTeRERESqBSpUrx7rvv0rlzZ5577jn++te/+o4Zk/MtkVmfs7Y6Afz8/MjIyMBaS5cuXVi0aFG+Y9522218++23nD59moCAgBzHLvUs/DJlyvjmUKpUKd88SpUqRUZGhq9eXvPOPsbEiRMZOXJkjvLsAfBS9davX8+yZct8n2fPns3Ro0d9AdFayyuvvEK3bt1ytPv000/z/O4uJft5Zm+fdd55zfNyadtURIrVsLRORfoRkYtVqFCBlStXsnDhQubNm+crX7JkCb/++itJSUkcPHiQRo0a5dlH69at2bRpE4mJiUDmdW379+/Pc7yhQ4fyxz/+kV9++QWAI0eO8Pbbb9OqVSs+++wzjh49yvnz51m0aBEdOnQo1PmsXbuW48ePc/bsWZYvX+5bDczSrVs33njjDd81eYcPH+bHH38kICAgx00TedW78847SUtLy7G1eebMmRztXn31VdLT0wHYv38/P//8c57zbdy4McnJyb7vbsGCBYU657zmebm08iYixer08hFFazgtoXgnIlKMCvJojyulatWqrF69mjvuuIPq1asD0KhRIzp06MAPP/zA3Llz8ff3z7N9jRo1iI6OZuDAgb6L7adMmeLbCrzQlClTePLJJ2natCn+/v7ccMMNPPvss9SqVYvnn3+ejh07Yq3lrrvuolevXoU6l9tvv51BgwaRmJjI/fffn2PLFKBr164kJCT4tnUrVqzI22+/Tf369WnXrh1BQUF0796d6dOn51qvZs2aLF++nLFjx/KPf/yDGjVqcMMNN/D3v/8dgGHDhpGcnEx4eDjWWmrUqOG7bi83/v7+vPnmm/Tr14+MjAxatGiR42aI/OR1PjVr1izU93Yhc728EtTtdtsLL4wUkeKX0LhJkdo12afwJiVHQkICTZoU7W/5ShsyZAg9evSgb9+++VcuQaKjo4mNjWXWrFlXeyolTm5/b8aYOGutO7f62jYVERERcRBtm4qIiDhIdHR0sfRzzz33+B6TkeXvf//7RRfzF5chQ4YwZMiQK9L39UbhTURE5Dr0/vvvX+0pSBFp21RERETEQRTeRERERBxE4U1ERETEQRTeRERERBxENyyIiIjkJ6pyMfd36pKHIyIimDhxYo47P1966SX279/PnDlzCjzM8uXLadiwIU2bNi3yVLNERUXx+uuv+95tGhkZybRp04iIiGDGjBkXPXC3IHJ79tuF/e3YsYPw8HBWr16d752wfn5+uFwu0tPTKV26NIMHD+bxxx+nVKm816qyjxcREcGRI0coX748kPkO1ct9oO6VoJU3ERGREmbgwIHExMTkKIuJiWHgwIGF6mf58uXs3bu3UG2yv4v0QmPHjsXj8eDxeJg2bVqh+i2qRYsWcfvttxfo3azly5fH4/GwZ88e1q5dy0cffcQzzzxTqPEWLlzoO8eSGNxA4U1ERKTE6du3LytXrvS9zio5OZnvv/+e22+/nenTp9OiRQuCg4OZPHmyr81bb71FcHAwISEhDBo0iM2bN7NixQrGjRtHaGgoSUlJeDweWrduTXBwMPfccw8nTpwAMlef/vrXv9KhQwdefvnlIs/74Ycfxu1206xZsxxz++ijj2jcuDG33347f/zjH+nRo0eB+rPWsnTpUqKjo1mzZg1paWkFnkvNmjV57bXXmDVrFtZazp8/z1/+8hdcLhfBwcG88sorhT6/kkLbpiIiIiVMtWrVaNmyJatXr6ZXr17ExMTQv39/1q5dy4EDB9i6dSvWWnr27MnGjRupVq0aU6dOZdOmTVSvXp3jx49TtWpVevbsmeNVWlmhpUOHDjz99NM888wzvPTSSwCcPHmSzz777JLzevHFF3n77beB3B/oO3XqVKpWrcr58+fp1KkTu3btomHDhowcOZKNGzcSGBh40erh4sWL+eKLL3yfs14CD7Bp0yYCAwOpX78+ERERfPTRR/Tp06fA32O9evX49ddf+fHHH3nvvfc4dOgQO3bsoHTp0hw/fjzXNg8++CB+fn7ce++9PPnkkxhjCjzeb0UrbyIiIiVQ9q3TrC3TNWvWsGbNGsLCwggPD2ffvn0cOHCADRs20LdvX9+L66tWrXpRf6dOneLkyZN06NABgMGDB7Nx40bf8f79++c7p+zbprldf/buu+8SHh5OWFgYe/bsYe/evezbt4969eoRGBjoO6/s+vfv7+vT4/HkuHZu0aJFDBgwAIABAwYUaOv0QlnvcF+3bh2jRo2idOnMdavcvqOFCxeye/duPv/8cz7//HMWLFhQ6PF+CwpvIiIiJVDv3r1Zv34927dv5+zZs4SHh2OtZeLEib6gk5iYyNChQ7HWXvYK0Q033HBZ7Q8dOsSMGTNYv349u3bt4ne/+x1paWm+8FRY58+fZ9myZTz77LPUrVuXRx99lFWrVnH69OkC93Hw4EH8/PyoWbNmgb6jm2++GYCAgADuv/9+tm7dWqS5X2kKbyIiIiVQxYoViYiI4KGHHvKtVnXr1o033niD1NRUAA4fPsyPP/5Ip06dePfddzl27BiAb0swICDAF3YqV67MjTfeyOeffw7AggULfKtwxeG///0vN9xwA5UrV+aHH35g1apVADRu3JiDBw+SnJwMZG6TFsS6desICQnhu+++Izk5mW+++YZ7772X5cuXF6j9Tz/9xKhRoxgzZgzGGLp27crcuXN9N2RcuG2akZHB0aNHAUhPT2flypUEBQUVaKzfmq55ExERyU8+j/a4UgYOHEifPn1826ddu3YlISGBNm3aAJkB7+2336ZZs2ZMmjSJDh064OfnR1hYGNHR0QwYMIDhw4czc+ZMli5dyvz58xk1ahRnzpyhXr16vPnmm8U215CQEMLCwmjWrBn16tWjXbt2QOYdoHPmzCEyMpLq1avTsmXLAvW3aNEi7rnnnhxl9957L6+++iqDBg3Ktc3Zs2cJDQ31PSpk0KBB/OlPfwJg2LBh7N+/n+DgYMqUKcPw4cMZM2aMr+25c+fo1q0b6enpnD9/ns6dOzN8+PCifBVXnCnqcqbTuN1uGxsbe7WnIXLNS2jcpEjtmuxLKOaZiBRdQkICTZoU7W9ZLpaamkrFihWx1jJ69GgaNGjA2LFjr/a0Sozc/t6MMXHW2lwfnqdtUxEREbmiXn/9dUJDQ2nWrBmnTp1i5MiRV3tKjqZtUxEREfGZOnUqS5YsyVHWr18/Jk2aVOQ+x44dW2wrbceOHaNTp04Xla9fv55q1aoVyxglncKbiIiI+EyaNOmygtqVVq1aNTwez9WexlWlbVMRERERB1F4ExEREXEQhTcRERERB9E1byIiIvlwzXcVa3+7B+8u1v7k+qKVNxERkRLIz8+P0NBQgoKC6NevH2fOnOHkyZPMmTOnWPpfvnw5e/fuzVH2wgsv0LhxY1wuFyEhIfzpT38iPT29WMaT4qPwJiIiUgKVL18ej8dDfHw8ZcuWZe7cuVc0vM2dO5c1a9bw1VdfsXv3brZt20bNmjU5e/ZssYwnxUfhTUREpIRr3749iYmJTJgwgaSkJEJDQxk3blyudT/99FM6dOjAfffdR8OGDZkwYQILFy6kZcuWuFwukpKS2Lx5MytWrGDcuHGEhoaSlJTE1KlTefXVV6lSpQoAZcuWZcKECVSqVAmANWvW0KZNG8LDw+nXr5/v/ap169Zl8uTJhIeH43K52LdvH5D57tDevXsTHBxM69at2bVrFwBRUVEMHjyYrl27UrduXd577z2eeOIJXC4XkZGRvpW+uLg4OnToQPPmzenWrRtHjhy5ot+xkyi8iYiIlGAZGRmsWrUKl8vFtGnTqF+/Ph6Ph+nTp+fZZufOnbz88svs3r2bBQsWsH//frZu3cqwYcN45ZVXaNu2LT179mT69Ol4PB5q1qxJamoqgYGBufZ39OhRpkyZwrp169i+fTtut5sXXnjBd7x69eps376dhx9+mBkzZgAwefJkwsLC2LVrF8899xx/+MMffPWTkpL48MMP+eCDD/j9739Px44d2b17N+XLl+fDDz8kPT2dRx99lKVLlxIXF8dDDz1Uop8991vTDQsiIiIlUNZL1iFz5W3o0KF8//33BWrbokULatWqBUD9+vXp2rUrAC6Xi08++eSi+tZajDG+zx9//DHjx4/n5MmTvPPOOxw/fpy9e/f6Xjb/yy+/0KZNG1/9Pn36ANC8eXPee+89AL744guWLVsGwJ133smxY8c4deoUAN27d6dMmTK4XC7Onz9PZGSkb37Jycl8/fXXxMfH06VLFwDOnz/vOx9ReBMRESmRsq55K4py5cr5fi9VqpTvc6lSpcjIyLiofqVKlbjhhhs4dOgQgYGBdOvWjW7dutGjRw9++eUXrLV06dKFRYsWXXI8Pz8/X//W2ovqZQXE7PMpU6aMrzxrftZamjVrxpdfflmk87/WKbyJiIjko6Q82iMgIIDTp09fkb4mTpzIww8/TExMDFWqVMFaS1paGgCtW7dm9OjRJCYmctttt3HmzBlSUlJo2LBhnv3fcccdLFy4kKeeeopPP/2U6tWr+66fy0+jRo346aef+PLLL2nTpg3p6ens37+fZs2aXd5JXyN0zZuIiIhDVKtWjXbt2hEUFJTnDQsFNWDAAKZPn05YWBhJSUk8/PDDdO7cmVatWhEcHEy7du0ICwsjLCyMGjVqEB0dzcCBA303IGTdmJCXqKgoYmNjCQ4OZsKECcyfP7/AcytbtixLly5l/PjxhISEEBoayubNmy/rfK8lJrdlzWuR2+22sbGxV3saIte8hMZNitSuyb6EYp6JSNElJCTQpEnR/pZFCiu3vzdjTJy11p1bfa28iYiIiDiIrnkTERFxoN27dzNo0KAcZeXKlWPLli1XaUbyW1F4ExERcSCXy1Xku1HF2bRtKiIiIuIgCm8iIiIiDqLwJiIiIuIguuZNREQkH0V9BE5e9GgcuRxaeRMRESmB/Pz8CA0NJSgoiH79+nHmzBlOnjzJnDlzCtw2NDSUnj17+spnzZrFbbfdhjGGo0ePXrKP6OhojDGsX7/eV/b+++9jjGHp0qVFPzG5bApvIiIiJVDWu03j4+MpW7Ysc+fOLXB4y2rr8XhYsWKFr7xdu3asW7eOW2+9tUBzcLlcOd5nGhMTQ0hISOFPRopVvuHNGHOLMeYTY0yCMWaPMeYxb3mUMeawMcbj/bkrW5uJxphEY8zXxphu2cojvWWJxpgJ2coDjTFbjDEHjDGLjTFlveXlvJ8Tvcfr5jeGiIjItaZ9+/YkJiYyYcIEkpKSCA0NLdLrscLCwqhbt26hxt26dSvp6emkpqaSmJhIaGio7/izzz5LixYtCAoKYsSIEb6X0ScmJtK5c2dCQkIIDw8nKSmp0HOVvBVk5S0D+LO1tgnQGhhtjGnqPfaitTbU+/MRgPfYAKAZEAnMMcb4GWP8gNlAd6ApMDBbP3/39tUAOAEM9ZYPBU5Ya28DXvTWy3OMIn8LIiIiJVRGRgarVq3C5XIxbdo06tevj8fjYfr06Xm2SUtLw+1207p1a5YvX17ksY0xdO7cmY8//pgPPvggxxYswJgxY9i2bRvx8fGcPXuWlStXAvDAAw8wevRodu7cyebNm6lVq1aR5yAXyze8WWuPWGu3e38/DSQAN1+iSS8gxlp7zlp7CEgEWnp/Eq21B621vwAxQC9jjAHuBLI20OcDvbP1lfUm26VAJ2/9vMYQERG5Jpw9e5bQ0FDcbjd16tRh6NCh+Tfy+vbbb4mNjeWdd97h8ccfv6yVrwEDBhATE0NMTAwDBw7MceyTTz6hVatWuFwuNmzYwJ49ezh9+jSHDx/mnnvuAcDf358KFSoUeXy5WKHuNvVuW4YBW4B2wBhjzB+AWDJX506QGey+ytYshf8Le99dUN4KqAactNZm5FL/5qw21toMY8wpb/1LjSEiIuJ4WdetFcVNN90EQL169YiIiGDHjh3Ur1+/SH21bNmS+Ph4ypcvT8OGDX3laWlpPPLII8TGxnLLLbcQFRVFWlqab+tUrpwChzdjTEVgGfC4tfa/xphXgb8B1vvvP4GHAJNLc0vuq3z2EvW5xLFLtck+5xHACIA6derk0kRERCR/JeXRHgEBAZw+ffqSdU6cOEGFChUoV64cR48eZdOmTTzxxBOXNe7zzz+Pv79/jrK0tDQAqlevTmpqKkuXLqVv375UqlSJ2rVrs3z5cnr37s25c+c4f/68Vt+KUYHuNjXGlCEzuC201r4HYK39wVp73lr7K/A6/7dtmQLckq15beD7S5QfBaoYY0pfUJ6jL+/xysDxS/SVg7X2NWut21rrrlGjRkFOVUREpMSqVq0a7dq1IygoKM8bFhISEnC73YSEhNCxY0cmTJhA06aZl5jPnDmT2rVrk5KSQnBwMMOGDSvQuN27d6djx445yqpUqcLw4cNxuVz07t2bFi1a+I4tWLCAmTNnEhwcTNu2bfnPf/5TxDOW3Jj8lje915jNB45bax/PVl7LWnvE+/tYoJW1doAxphnwDplh7iZgPdCAzNWy/UAn4DCwDbjfWrvHGLMEWGatjTHGzAV2WWvnGGNGAy5r7ShjzACgj7X2vrzGsNaez+s83G63jY2NLfw3JCKFUtSHmZaUlQ0RyAxATZoU74N5RfKS29+bMSbOWuvOrX5Btk3bAYOA3caYrM33v5J5t2gomduVycBIAG8YexfYS+adqqOzQpUxZgzwMeAHvGGt3ePtbzwQY4yZAuwA5nnL5wELjDGJZK64DchvDBEREZFrWb7hzVr7BblfY/bRJdpMBabmUv5Rbu2stQfJ5W5Ra20a0K8wY4iIiFwPdu/ezaBBg3KUlStXji1bthSqnzfffJOXX345R1m7du2YPXv2Zc9Rrgy921RERMSBXC5Xke9Gze7BBx/kwQcfLIYZyW9Fr8cSERERcRCFNxEREREHUXgTERERcRBd8yYiIpKP2aM2FGt/o+feWaz9yfVFK28iIiIlkJ+fH6GhoQQFBdGvXz/OnDnDyZMnmTNnToHbhoaG5niZ/KxZs7jtttswxnD06NFL9hEdHY0xhvXr1/vK3n//fYwxLF269BItL+3FF1/E39+fU6dOFbptdHQ0Y8aMASAqKooZM2YAMGTIEAIDA33n3LZtWwB++OEHevToQUhICE2bNuWuu+4q8rxLEoU3ERGREijr3abx8fGULVuWuXPnFji8ZbX1eDysWLHCV96uXTvWrVvHrbfeWqA5uFwuFi1a5HzBONMAACAASURBVPscExNDSEhI4U8mm0WLFtGiRQvef//9y+rnQtOnT/ed8+bNmwF4+umn6dKlCzt37mTv3r1MmzatWMe8WhTeRERESrj27duTmJjIhAkTSEpKIjQ0NM/XY11KWFgYdevWLdS4W7duJT09ndTUVBITEwkNDfUdf/bZZ2nRogVBQUGMGDHC91L6xMREOnfuTEhICOHh4SQlJQGQlJREamoqU6ZMyREKo6Oj6dOnD5GRkTRo0CDHu1jffPNNGjZsSIcOHdi0aVOhzvfIkSPUrl3b9zk4OLhQ7UsqhTcREZESLCMjg1WrVuFyuZg2bRr169fH4/Ewffr0PNukpaXhdrtp3bo1y5cvL/LYxhg6d+7Mxx9/zAcffJBjCxZgzJgxbNu2jfj4eM6ePcvKlSsBeOCBBxg9ejQ7d+5k8+bN1KpVC8hcdRs4cCDt27fn66+/5scff/T15fF4WLx4Mbt372bx4sV89913HDlyhMmTJ7Np0ybWrl3L3r1785zruHHjfNumDzzwAACjR49m6NChdOzYkalTp/L99xe9Bt2RFN5ERERKoLNnzxIaGorb7aZOnToMHTq0wG2//fZbYmNjeeedd3j88cd9K19FMWDAAGJiYoiJiWHgwIE5jn3yySe0atUKl8vFhg0b2LNnD6dPn+bw4cPcc889APj7+1OhQgUgc9t1wIABlCpVij59+rBkyRJfX506daJy5cr4+/vTtGlTvvnmG7Zs2UJERAQ1atSgbNmy9O/fP895Zt82XbhwIQDdunXj4MGDDB8+nH379hEWFsZPP/1U5O+ipNDdpiIiIiVQ1nVrRXHTTTcBUK9ePSIiItixYwf169cvUl8tW7YkPj6e8uXL07BhQ195WloajzzyCLGxsdxyyy1ERUWRlpbm2zq90K5duzhw4ABdunQB4JdffqFevXqMHj0ayHy1VxY/Pz8yMjKAzNW/y1G1alXuv/9+7r//fnr06MHGjRu59957L6vPq03hTUREJB8l5dEeAQEBnD59+pJ1Tpw4QYUKFShXrhxHjx5l06ZNOa4hK4rnn38ef3//HGVpaWkAVK9endTUVJYuXUrfvn2pVKkStWvXZvny5fTu3Ztz585x/vx5Fi1aRFRUFBMnTvT1ERgYyDfffJPnuK1ateKxxx7j2LFjVKpUiSVLlhTqhokNGzbQunVrKlSowOnTp0lKSqJOnTqFPPuSR9umIiIiDlGtWjXatWtHUFBQnjcsJCQk4Ha7CQkJoWPHjkyYMIGmTZsCMHPmTGrXrk1KSgrBwcEMGzasQON2796djh075iirUqUKw4cPx+Vy0bt3b1q0aOE7tmDBAmbOnElwcDBt27blP//5DzExMb6t1Cz33HMPMTExeY5bq1YtoqKiaNOmDZ07dyY8PDzPutmveQsNDeWXX34hLi4Ot9tNcHAwbdq0YdiwYTnm6VQmr+XNa43b7baxsbFXexoi17yExk2K1K7JvoRinolI0SUkJNCkSdH+lkUKK7e/N2NMnLXWnVt9rbyJiIiIOIiueRMREXGg3bt3M2jQoBxl5cqVY8uWLYXq58033+Tll1/OUdauXTtmz5592XOUK0PhTURExIFcLleR70bN7sEHH+TBBx8shhnJb0XbpiIiIiIOovAmIiIi4iAKbyIiIiIOomveRERE8vHP/j2Ktb8/L15ZoHopKSmMHj2avXv38uuvv9KjRw+mT5/O5s2b6dWrF4GBgb66M2bMoHPnzkydOpV33nkHPz8/SpUqxb/+9S9atWqV5xiTJk3irbfe4sSJE6Smpl72ucmVp5U3ERGREshaS58+fejduzcHDhxg//79pKamMmnSJADat2/ve5enx+Ohc+fOfPnll6xcuZLt27eza9cu1q1bxy233HLJce6++262bt36W5ySFBOFNxERkRJow4YN+Pv7++4E9fPz48UXX+SNN97gzJkzubY5cuQI1atX970ntHr16tx0002sWrWK++67z1fv008/5e677wagdevW1KpV6wqfjRQnhTcREZESaM+ePTRv3jxHWaVKlahTpw6JiYl8/vnnOV4HlZSURNeuXfnuu+9o2LAhjzzyCJ999hkAXbp04auvvuLnn38GYPHixfTv3/83PycpHgpvIiIiJZC1FmNMnuUXbpvWr1+fihUrEhcXx2uvvUaNGjXo378/0dHRlC5dmsjISP7973+TkZHBhx9+SK9eva7CWUlx0A0LIiIiJVCzZs1YtmxZjrL//ve/fPfdd9SvXz/Pdn5+fkRERBAREYHL5WL+/PkMGTKE/v37M3v2bKpWrUqLFi0ICAi40qcgV4hW3kREREqgTp06cebMGd566y0Azp8/z5///GeGDBlChQoVcm3z9ddfc+DAAd9nj8fDrbfeCkBERATbt2/n9ddf15apw2nlTUREJB8FfbRHcTLG8P777/PII4/wt7/9jV9//ZW77rqL5557ji+//NJ3zVuWJ598ksDAQB599FFOnjxJ6dKlue2223jttdeAzBW5Hj16EB0dzfz5833tnnjiCd555x3OnDlD7dq1GTZsGFFRUb/16UohGGvt1Z7Db8LtdtvY2NirPQ2Ra15C4yZFatdkX0Ixz0Sk6BISEmjSpGh/yyKFldvfmzEmzlrrzq2+tk1FREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRB9Jw3ERGRfKRM+LxY+6s9rX2x9ifXF628iYiIlEB+fn6EhoYSFBREv379OHPmDCdPnmTOnDkFbhsaGkrPnj195bNmzeK2227DGMPRo0cv2Ud0dDTGGNavX+8re//99zHGsHTpUgCGDRvG3r178+xj4cKFBAcHExwcTNu2bdm5c2e+c5f8KbyJiIiUQOXLl8fj8RAfH0/ZsmWZO3dugcNbVluPx8OKFSt85e3atWPdunW+V2blx+VysWjRIt/nmJgYQkJCfJ//93//l6ZNm+bZPjAwkM8++4xdu3bx1FNPMWLEiAKNK5em8CYiIlLCtW/fnsTERCZMmEBSUhKhoaGMGzeu0P2EhYVRt27dQo27detW0tPTSU1NJTExMccruSIiIsh6e9Hq1asJDw8nJCSETp06AdC2bVtuvPFGAFq3bk1KSkqh5ywX0zVvIiIiJVhGRgarVq0iMjKS7t27Ex8fj8fjuWSbtLQ03G43pUuXZsKECfTu3btIYxtj6Ny5Mx9//DGnTp2iZ8+eHDp06KJ6P/30E8OHD2fjxo0EBgZy/Pjxi+rMmzeP7t27F2kekpPCm4iISAl09uxZ3ypX+/btGTp0KN9//32B2n777bfcdNNNHDx4kDvvvBOXy0X9+vWLNI8BAwYwc+ZMTp06xT//+U+ee+65i+p89dVX3HHHHQQGBgJQtWrVHMc/+eQT5s2bxxdffFGkOUhOCm8iIiIlUNZ1a0Vx0003AVCvXj0iIiLYsWNHkcNby5YtiY+Pp3z58jRs2DDXOtZajDG5Htu1axfDhg1j1apVVKtWrUhzkJwU3kRERPJRUh7tERAQwOnTpy9Z58SJE1SoUIFy5cpx9OhRNm3axBNPPHFZ4z7//PP4+/vnebxNmzaMHj2aQ4cO+bZNq1atyrfffkufPn1YsGBBnsFPCk83LIiIiDhEtWrVaNeuHUFBQXnesJCQkIDb7SYkJISOHTsyYcIE3x2hM2fOpHbt2qSkpBAcHMywYcMKNG737t3p2LFjnsdr1KjBa6+9Rp8+fQgJCaF///4APPvssxw7doxHHnmE0NBQ3G53Ic9YcmOstVd7Dr8Jt9tts+6IEZErJ6FxkyK1a7IvoZhnIlJ0CQkJNGlStL9lkcLK7e/NGBNnrc017WrlTURERMRBdM2biIiIA+3evZtBgwblKCtXrhxbtmwpVD9vvvkmL7/8co6ydu3aMXv27Mueo1wZCm8iIiIO5HK5inw3anYPPvggDz74YDHMSH4r2jYVERERcRCFNxEREREHUXgTERERcRBd8yYiIpKPqKioEt2fXF/yXXkzxtxijPnEGJNgjNljjHnMW17VGLPWGHPA+++N3nJjjJlpjEk0xuwyxoRn62uwt/4BY8zgbOXNjTG7vW1mGu87NooyhoiIyLUgOTmZoKCgHGVRUVHMmDHjKs2o6KKjoxkzZkyex6Oiorj55psJDQ2ladOmLFq0qNjGrlu3LkePHi22/kqCgmybZgB/ttY2AVoDo40xTYEJwHprbQNgvfczQHeggfdnBPAqZAYxYDLQCmgJTM4KY946I7K1i/SWF2oMERERcaaxY8fi8Xj44IMPGDlyJOnp6Vd7SiVWvuHNWnvEWrvd+/tpIAG4GegFzPdWmw/09v7eC3jLZvoKqGKMqQV0A9Zaa49ba08Aa4FI77FK1tovbebrHt66oK/CjCEiInLNi4iIYPz48bRs2ZKGDRvy+eefA5krXL179+buu+8mMDCQWbNm8cILLxAWFkbr1q05fvw4AK+//jotWrQgJCSEe++9lzNnzgAwZMgQRo0aRfv27WnYsCErV6709durVy8iIyNp1KgRzzzzjG8ub7/9Ni1btiQ0NJSRI0dy/vx5IPP5cQ0bNqRDhw5s2rSpwOfWoEEDKlSowIkTJwBISkoiMjKS5s2b0759e/bt2wfAv//9b1q1akVYWBidO3fmhx9+AODYsWN07dqVsLAwRo4cSdabpH7++Wd+97vfERISQlBQEIsXLy7y93+1FeqGBWNMXSAM2AL8j7X2CGQGPKCmt9rNwHfZmqV4yy5VnpJLOUUY48L5jjDGxBpjYn/66afCnKqIiEiJlpGRwdatW3nppZdyhKn4+Hjeeecdtm7dyqRJk6hQoQI7duygTZs2vPXWWwD06dOHbdu2sXPnTpo0acK8efN87ZOTk/nss8/48MMPGTVqFGlpaQBs3bqVhQsX4vF4WLJkCbGxsSQkJLB48WI2bdqEx+PBz8+PhQsXcuTIESZPnsymTZtYu3Yte/fuLfB5bd++nQYNGlCzZuZ/8keMGMErr7xCXFwcM2bM4JFHHgHg9ttv56uvvmLHjh0MGDCAf/zjHwA888wz3H777ezYsYOePXvy7bffArB69Wpuuukmdu7cSXx8PJGRkblPwAEKfMOCMaYisAx43Fr7X+9lablWzaXMFqH8ktMpSBtr7WvAa5D5btN8+hQRESkx8vrvbFZ5nz59AGjevDnJycm+4x07diQgIICAgAAqV67M3XffDWQ+1HfXrl1AZsB78sknOXnyJKmpqXTr1s3X/r777qNUqVI0aNCAevXq+Va6unTpQrVq1Xxjf/HFF5QuXZq4uDhatGgBwNmzZ6lZsyZbtmwhIiKCGjVqANC/f3/2799/yfN98cUXef311zl48CCrV68GIDU1lc2bN9OvXz9fvXPnzgGQkpJC//79OXLkCL/88guBgYEAbNy4kffeew+A3/3ud9x4442+8//LX/7C+PHj6dGjB+3bt7/kfEqyAq28GWPKkBncFlpr3/MW/5C1Ven990dveQpwS7bmtYHv8ymvnUt5UcYQERG5JlSrVs23dZjl+PHjVK9eHch8FRaAn58fGRkZvjpZ5QClSpXyfS5VqpSv3pAhQ5g1axa7d+9m8uTJvtU1uDg0Zn3Ordxay+DBg/F4PHg8Hr7++mvfnbSXWOTJ1dixY/n6669ZvHgxf/jDH0hLS+PXX3+lSpUqvv49Hg8JCQkAPProo4wZM4bdu3fzr3/965LnANCwYUPi4uJwuVxMnDiRZ599tlDzK0nyXXnz3vk5D0iw1r6Q7dAKYDAwzfvvB9nKxxhjYsi8OeGUtfaIMeZj4LlsNyl0BSZaa48bY04bY1qTuR37B+CVooxR+NMXERHJ39V4tEfFihWpVasW69evp1OnThw/fpzVq1fz2GOP8eabb15W36dPn6ZWrVqkp6ezcOFCbr75/648WrJkCYMHD+bQoUMcPHiQRo0asWPHDtauXcvx48cpX748y5cv54033qBChQr06tWLsWPHUrNmTY4fP87p06dp1aoVjz32GMeOHaNSpUosWbKEkJCQAs2tT58+zJ8/n/nz5zNy5EgCAwNZsmQJ/fr1w1rLrl27CAkJ4dSpU755z58/39f+jjvuYOHChTz55JOsWrXKF4C///57qlatyu9//3sqVqxIdHT0ZX2HV1NBtk3bAYOA3caYrJeo/ZXMQPWuMWYo8C2Qtab5EXAXkAicAR4E8Ia0vwHbvPWetdYe9/7+MBANlAdWeX8o7BgiIiLXkrfeeovRo0fz5z//GYDJkydTv379y+73b3/7G61ateLWW2/F5XJx+vRp37FGjRrRoUMHfvjhB+bOnYu/vz+QeY3ZoEGDSExM5P7778ftdgMwZcoUunbtyq+//kqZMmWYPXs2rVu3JioqijZt2lCrVi3Cw8N9NzIUxNNPP83999/P8OHDWbhwIQ8//DBTpkwhPT2dAQMGEBISQlRUFP369ePmm2+mdevWHDp0yPcdDRw4kPDwcDp06ECdOnUA2L17N+PGjaNUqVKUKVOGV1917oMqTNZdGNc6t9ttY2Njr/Y0RK55CY2bFKldk30JxTwTkaJLSEigSZOi/S072ZAhQ+jRowd9+/bNUR4dHU1sbCyzZs26SjO7tuX292aMibPWunOrr9djiYiIiDiIXo8lIiIiAHleBzZkyBCGDBly2f1PnTqVJUuW5Cjr168fkyZNuuy+rycKbyIiIrmw1hb6jkm5tEmTJimoXaAol69p21REROQC/v7+HDt2rEj/YRUpKGstx44d890UUlBaeRMREblA7dq1SUlJQW/nkSvN39+f2rVr518xG4U3ERGRC5QpU8b3xH6RkkbbpiIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOkm94M8a8YYz50RgTn60syhhz2Bjj8f7cle3YRGNMojHma2NMt2zlkd6yRGPMhGzlgcaYLcaYA8aYxcaYst7yct7Pid7jdfMbQ0RERORaV5CVt2ggMpfyF621od6fjwCMMU2BAUAzb5s5xhg/Y4wfMBvoDjQFBnrrAvzd21cD4AQw1Fs+FDhhrb0NeNFbL88xCnfaIiIiIs6Ub3iz1m4Ejhewv15AjLX2nLX2EJAItPT+JFprD1prfwFigF7GGAPcCSz1tp8P9M7W13zv70uBTt76eY0hIiIics27nGvexhhjdnm3VW/0lt0MfJetToq3LK/yasBJa23GBeU5+vIeP+Wtn1dfFzHGjDDGxBpjYn/66aeinaWIiIhICVLU8PYqUB8IBY4A//SWm1zq2iKUF6Wviwutfc1a67bWumvUqJFbFRERERFHKVJ4s9b+YK09b639FXid/9u2TAFuyVa1NvD9JcqPAlWMMaUvKM/Rl/d4ZTK3b/PqS0REROSaV6TwZoyple3jPUDWnagrgAHeO0UDgQbAVmAb0MB7Z2lZMm84WGGttcAnQF9v+8HAB9n6Guz9vS+wwVs/rzFERERErnml86tgjFkERADVjTEpwGQgwhgTSuZ2ZTIwEsBau8cY8y6wF8gARltrz3v7GQN8DPgBb1hr93iHGA/EGGOmADuAed7yecACY0wimStuA/IbQ0RERORaZzIXs659brfbxsbGXu1piFzzEho3KVK7JvsSinkmIiLOZYyJs9a6czumNyyIiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOIjCm4iIiIiDKLyJiIiIOEjpqz0BEbm2NBnw/dWegojINU0rbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOovAmIiIi4iAKbyIiIiIOkm94M8a8YYz50RgTn62sqjFmrTHmgPffG73lxhgz0xiTaIzZZYwJz9ZmsLf+AWPM4GzlzY0xu71tZhpjTFHHEBEREbnWFWTlLRqIvKBsArDeWtsAWO/9DNAdaOD9GQG8CplBDJgMtAJaApOzwpi3zohs7SKLMoaIiIjI9SDf8Gat3Qgcv6C4FzDf+/t8oHe28rdspq+AKsaYWkA3YK219ri19gSwFoj0Hqtkrf3SWmuBty7oqzBjiIiIiFzzinrN2/9Ya48AeP+t6S2/GfguW70Ub9mlylNyKS/KGCIiIiLXvOK+YcHkUmaLUF6UMS6uaMwIY0ysMSb2p59+yqdbERERkZKvqOHth6ytSu+/P3rLU4BbstWrDXyfT3ntXMqLMsZFrLWvWWvd1lp3jRo1CnWCIiIiIiVRUcPbCiDrjtHBwAfZyv/gvSO0NXDKu+X5MdDVGHOj90aFrsDH3mOnjTGtvXeZ/uGCvgozhoiIiMg1r3R+FYwxi4AIoLoxJoXMu0anAe8aY4YC3wL9vNU/Au4CEoEzwIMA1trjxpi/Adu89Z611mbdBPEwmXe0lgdWeX8o7BgiIiIi1wOTeZPntc/tdtvY2NirPQ2Ra19U5SK2O1W88xARcTBjTJy11p3bMb1hQURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBFN5EREREHEThTURERMRBLiu8GWOSjTG7jTEeY0yst6yqMWatMeaA998bveXGGDPTGJNojNlljAnP1s9gb/0DxpjB2cqbe/tP9LY1lxpDRERE5FpXHCtvHa21odZat/fzBGC9tbYBsN77GaA70MD7MwJ4FTKDGDAZaAW0BCZnC2OveutmtYvMZwwRERGRa9qV2DbtBcz3/j4f6J2t/C2b6SugijGmFtCN/9/evYfbNd95HH9/JHFrCCLiTtxKx63EbVBSmgltx4zGiJmnihLVYoaa0eehGq1qqoaWQWOUoEVdKtKWRJ66jCKVSHNBIq3EJeNSl5QYtxHf+WP9ji5nzjk5Ofnttc/a+byeZz9n77X2/n3W72Tvle/5/dZeC6ZExGsRsRiYAoxI69aOiIcjIoDr2rXVUYaZmZlZS1vR4i2AuyU9Kml0WjY4Il4ASD83SMs3AZ4rvXZRWtbV8kUdLO8qw8zMzKyl9V3B1+8bEc9L2gCYImleF89VB8uiB8u7LRWUowE233zz5XmpmZmZWa+0QiNvEfF8+vkn4HaKY9ZeSlOepJ9/Sk9fBGxWevmmwPPLWL5pB8vpIqP99l0ZEUMjYuigQYN62k0zMzOzXqPHxZukj0laq+0+MBx4DJgItH1j9EvAHen+RODo9K3TvYHX05TnZGC4pHXTFxWGA5PTuiWS9k7fMj26XVsdZZiZmZm1tBWZNh0M3J7O3tEXuCEiJkmaBtws6cvAs8AR6fl3AocCfwTeAo4FiIjXJH0HmJae9+2IeC3dPwkYD6wB3JVuAGM7yTAzMzNraT0u3iJiAbBLB8tfBQ7qYHkAX+ukrauBqztYPh3YsbsZZmZmZq3OV1gwMzMzqxEXb2ZmZmY14uLNzMzMrEZcvJmZmZnViIs3MzMzsxpx8WZmZmZWIy7ezMzMzGrExZuZmZlZjbh4MzMzM6sRF29mZmZmNeLizczMzKxGVuTC9GZWI2PGjKnkNWZm1lgeeTMzMzOrERdvZmZmZjXiaVOzlcRac6c3exPMzCwDF29mK4lDZz3V7E0wM7MMPG1qZmZmViMu3szMzMxqxMWbmZmZWY24eDMzMzOrERdvZmZmZjXi4s3MzMysRly8mZmZmdWIizczMzOzGnHxZmZmZlYjLt7MzMzMasTFm5mZmVmNuHgzMzMzqxEXb2ZmZmY14uLNzMzMrEZcvJmZmZnViIs3MzMzsxrp2+wNMDPrqTFj9a7rSwAADfdJREFUxlTyGjOz3sQjb2ZmZmY14pE3M6ut4985qNmbYGZWORdvZlZbSyaMXv4XjZ2bf0PMzCrkaVMzMzOzGnHxZmZmZlYjLt7MzMzMasTHvJmtJHYY9XyzN8HMzDJw8WbWRD0955jPVWZmtvLytKmZmZlZjbh4MzMzM6sRT5uaNdGRN/28Zy/0tKmZ2UrLI29mZmZmNeKRNzOzbujJl0T8xRIzawQXb2Zm3dCjKW4Xb2bWAC7eKuTTQpjl5XPXmdnKyMe8mZmZmdWIR95ozWNZWrFPVfLvz5pl7vY7LPdrdpg3twFbYma9lYs3W2FVFTpVFlTHv3NQj15nZmbWaC7ebIVVVehUecD4kgmjl/9FY5d/9MPHbJmZ2fKqdfEmaQTwI6APcFVEjG3yJvUarVjomFleVY1m92QqGDwdbNaZ2hZvkvoAlwGfARYB0yRNjIgnmrtl1ihVjlJ5RMzaa8X3RCue/sTHDNrKoLbFG7An8MeIWAAg6SbgMKDXFm+3DbmtR68bw5jlfo0LHbN68uepHqo89VOVx/tWVfy6yF4xiohmb0OPSBoJjIiI49PjLwJ7RcTJpeeMBtrm9D4OPLmcMesDr2TY3JU1y32qR1Yr9qnKLPepHlmt2Kcqs9yn6rO2iIhBHa2o88ibOlj2kUo0Iq4EruxxgDQ9Iob29PUre5b7VI+sVuxTlVnuUz2yWrFPVWa5T70rq84n6V0EbFZ6vCng+QYzMzNraXUu3qYB20oaImlVYBQwscnbZGZmZtZQtZ02jYj3JZ0MTKY4VcjVEfF45pgeT7k6q9KcKrPcJ2c1K6fKLPfJWc3KqTKrtn2q7RcWzMzMzFZGdZ42NTMzM1vpuHgzMzMzqxEXb2ZmZmY14uLNzFqOpA2avQ2NIGlgs7fBzJrPxVsv4Z1y7yNpgKSxkuZJejXd5qZl61S4HXdlbGttSd+TdL2kf2y37vJcOam9DSVdIekySQMljZE0R9LNkjbKmLNeu9tA4BFJ60paL1dOyhpRuj9A0k8kzZZ0g6TBmbPGSlo/3R8qaQHwO0nPSDogY84MSWdL2jpXm11kDZV0r6SfStpM0hRJr0uaJumTGXP6S/q2pMdT+y9LmirpmFwZpay+kk6UNCm9F2ZJukvSVyT1y53XyTbk/Saj1Cf16TuS9m237uzMWWtK+jdJ/yppdUnHSJoo6QJJ/XNmdZA9v0Ht7ly63y99viZKOl/SmjkyXLyVVLVjrmqnnNqvZMdc1U45ZVW1Y74ZWAwcGBEDI2IgMCwtuyVnkKTdOrntDuyaMeoaiquT3AaMknSbpNXSur0z5gCMp7jW8HPAvcDbwGeBB4AfZ8x5BXi0dJsObALMSPdzOr90/9+BF4DPU5x3clzmrM9GRNvldH4AHBkR2wCfSdm5rAusA9wr6RFJp0naOGP7ZZcDFwC/Bh4CxkXEAOAbaV0uPwMWAH8DnAtcAnwRGCbp/K5e2APXU3xGxwCHUrzHzwV2AX6aK6SDP1LKf6wcmisnGQccALwKXCLpotK6wzNnjQcGA0Mo3hdDgQsp9lNX5AqRtETSG+m2RNISYOu25blykvGl+2OBbSg+s2uQa98XEb6lGzCjdP8q4DxgC+A0YELGnDml+/cCe6T72wHTM/dpIcUH4VngkdSXjRvwu3sEOAQ4iuI/65Fp+UHAw5mz7gCOobiqxunAN4FtgWuB8zPmPNmTdT3MWgrck94P7W9vZ8yZ2e7xWcCDwMDy+z9T1u9L95/tajtWMOcMYBKwU2nZwpx9KbVb3ke0/11m61Nqbx7QN92f2m7dnIw55T7tT1FEvZjee6MrfE/8PmPOrHaPp6WfqwDzMvepq/3E/Iw5SykK0oWlW9vj9zL3aXbpfl+Kc5T9Algt579Tan9m+qn0vlPp8eyMOZcC1wGDS8sW5uxLqd3y+3wm0C93nzzy1rmhEXF2RDwTERcDW2Zsu5+kthMkrxER0wAiYj7FhyOnxRFxRkRsDnydosiZkUbJRmfM6RcRd0XEjUBExK0Ud34DrJ4xB2DLiBgfEYsi4iLgbyPiD8Cx5P2r8Jk0nP/hqKukwZLOpChQc5oLnBgRw9rfyHvh5NUkffi5j4jvUuyY/4uigMupvH+5rt26PrlCIuJC4HjgHEkXSVqLdtc5zmgDSadL+jqwtqTyNZZz708vA+6U9GlgkqQfSvqUpHMp/kPILiIeiIivUoxcfh/YJ3PEO5KGSzoCCEl/B5BmHJZmzPkfSfultj8PvAYQER/Q8XWxV8RiSUeUP1eSVpF0JMUofS4LKGYBhpRuW0XEEOCljDkAq7bdiYj3I2I0MIviD8yGTGVGUd3cmX62Pc72OY6IU4AfATdKOjX9ezVqPzFA0uGSvgCsFhH/m7YhW59cvH1UVTvmynfK0PAdc1U7Zahux3wkRUFzv6TFkl4D7gPWA/4hYw4UUy6dvcdOyZjzS+DT5QURcS1FYf9exhyAO9qOWYmID4+TkbQN8GTOoFTIH0ExWjQFyHJcSQf+E1iL4j+wa4G2wx82JPNnNyIupZimPRE4jGIU+xvAfwPHZYz6f8f9RMTSiJgUEcdmzAH4CsV77TiKKc1hkv5MMdp3asack4CLUttntrUtaRDF/jenUcBI4EVJ89NxVC9S/CE5KmPODymmuDtyQcYcgOnlw4gAIuJcisMutmxAVtt+4sP3dTrUZ0nOoIh4FDg4Pbyf/AMLbe6nOJzic8DUtgGAtJ/I8se4r7BQIulb7RZdHhEvp1/4BRFxdMasAyl2MNtRDEs/B0yguMzX+xlzboqInDuQznJ2odiBfEAxNXsS8CWK/2hOiIiHMmbtTDGtvR3wGHBcRMxPO+ajIuKSjFnbU0zPTo2IN0vLR0TEpFw5paxNgN81MquLnEMiItuXI5aR1bA+UfyxsHVEPFbnf6cqsyru0w7Axo3OSjmbUM1ndy+KEZWngB0ojh99IiLuzJyzJ8UAzjRJnwBGUEwDZ83pJVlPUhqJa0DO/hTHME9vUJ/2Aj5o2O8vx9zrynADjm2lHPepW22dSrEDmQA8DRxWWpf7+LBKsihG8arqUyVZFf87Vfn7a8X3xKkUx/JV8Z5oeE5q71vAVIovx3wP+A1wDsWhCGc1MOeeRuS0alZV/05VZWXb2Fa/0e7g2rrnuE/damsO0D/d3zJ9EP85Pc590G4lWe6Ts9ynhvSpD8VU/RvA2mn5GuQ94L6SnFbNarU+tR00b4Ck2Z2tovgqc61yqsxqxT4BfSJNt0TE02mq+1ZJW5D/oOeqstwnZzUrp8qsKvv0fkQsBd6S9FREvJFy35b0QQ1zWjWrpfrk4u2jBlMcRNv+G0KiOCdR3XKqzGrFPr0oadeImAkQEW9K+hxwNbBTxpwqs9wnZzUrp8qsKvv0nqQ1I+ItYPe2hZIGUBwDXLecVs1qrT7lHCqs+w34CbBfJ+tuqFuO+7TCOZsCG3aybt/Mfaoky31ylvuUvU+rdbJ8fUrnHqxLTqtmtVqf/G1TMzMzsxrxed7MzMzMasTFm5mZmVmNuHgzMzMzqxEXb2bW60g6S9LjkmZLmpnOVl5rku6TNLSB7R8o6a9Lj8dLGtmoPDNrHp8qxMx6FUn7UFwTcLeIeFfS+pQulG2dOhB4k/yn5jGzXsYjb2bW22wEvBIR7wJExCsR8byk3SXdL+lRSZMlbQSQls+S9LCkH0h6LC0/RtJ/tDUq6VfpZK1IGp6eP0PSLUoXxpb0tKRz0/I56ZqfSOov6Zq0bLakL3TVTndJ6pO2eVpq98S0/MA0UnerpHmSfiZJad2hadlvJV2S+rUlxUXfT0sjlfuniE9JekjSAo/CmbUOF29m1tvcDWwmab6kyyUdIKkfcCkwMiJ2pzjZ6nfT868BTo2IfbrTeBrJOxs4OCJ2o7h00umlp7ySll8BnJGWfRN4PSJ2ioidgXu60U53fDm1uwewB3CCpCFp3SeBfwE+AWwF7CtpdWAccEhE7AcMguIqAsCPgYsjYteIeCC1sRGwH8VI5tjl3DYz66U8bWpmvUoUZ8PfHdgfGAb8HDgP2BGYkgag+gAvpDOWrxMR96eXXw8csoyIvSkKogdTW6sCD5fW/yL9fBQ4PN0/GBhV2sbF6Yz9XbXTHcOBnUujYgOAbYH3gEciYhGApJkU1+h8E1gQEQvT828ERnfR/oSI+AB4QlLWy9SZWfO4eDOzXieK6wLeB9wnaQ7wNeDx9qNrktYBOjvT+Pt8dHZh9baXAVMi4qhOXvdu+rmUv+wj1UHOstrpDgGnRMTkjywspnffLS1q25blvS5nuY3c1/Q0sybxtKmZ9SqSPi5p29KiXYG5wKD0ZQYk9ZP0VxHxZ+B1Sful5/5T6XVPA7tKWkXSZsCeaflUiinIbVJba0rabhmbdTdwcmkb1+1hO+1NBk5K08JI2k7Sx7p4/jxgq3SMG8CRpXVLgLWWM9/MasjFm5n1Nv2BayU9IWk2xdTkOcBI4PuSZgEzgbbTYhwLXCbpYeDtUjsPAguBOcCFwAyAiHgZOAa4MbU/Fdh+Gdt0HrCupMdS/rAetvNrSYvS7RbgKuAJYEb6osU4upgRiYi3ga8CkyT9FngJeD2t/iXw9+2+sGBmLcjXNjWzlpFGpH4VETs2eVMaRlL/dFyggMuAP0TExc3eLjOrjkfezMzq5YT0BYbHKb7gMK7J22NmFfPIm5lZRpJuB4a0W3xm+y8lmJn1lIs3MzMzsxrxtKmZmZlZjbh4MzMzM6sRF29mZmZmNeLizczMzKxGXLyZmZmZ1cj/AVYfNMduha3rAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 720x504 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "fileextension='.fastq.gz'\n",
    "directory = '/media/sf_LinuxShare/Projects/Theresa/Hisat2/FlagHA_Pt08/Inserts'\n",
    "outplotname = 'Ptiwi08RIP_52Late_Inserts_Stacked_Barplot.pdf'\n",
    "startseqlength = 15\n",
    "cleanreads=False\n",
    "blastdatabase = '/media/sf_LinuxShare/Humans/Genome/Seqs/GRCh38_top_level.fa'\n",
    "aligndatabaselist=['/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Kpne_CompleteGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Vector_FlagHA_Dcl5',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_mtGenome',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mac',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_MacAndIES',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/OESv1',\\\n",
    "                   '/media/sf_LinuxShare/Ciliates/Genomes/Hisat2_Indexes/Pt_51_Mic2']\n",
    "\n",
    "readsubtract.main(fileextension, directory, outplotname, startseqlength, cleanreads, blastdatabase, aligndatabaselist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
